---
title: "Papers citing PR2"
author: vaulot
date: '2019-10-15'
categories: ["papers"]
tags: ["PR2"]
---

<script src="/rmarkdown-libs/header-attrs/header-attrs.js"></script>


<p><img src="2013_guillou_pr2_title.png" /></p>
<p>Guillou, L., Bachar, D., Audic, S., Bass, D., Berney, C., Bittner, L., Boutte, C., Burgaud, G., de Vargas, C., Decelle, J., del Campo, J., Dolan, J. R., Dunthorn, M., Edvardsen, B., Holzmann, M., Kooistra, W. H. C. F., Lara, E., Le Bescot, N., Logares, R., … Christen, R. (2013). The Protist Ribosomal Reference database (PR2): A catalog of unicellular eukaryote Small Sub-Unit rRNA sequences with curated taxonomy. Nucleic Acids Research, 41(D1), D597–D604. <a href="https://doi.org/10.1093/nar/gks1160" class="uri">https://doi.org/10.1093/nar/gks1160</a></p>
<p>References are available as a <a href="https://www.zotero.org/groups/2791531/pr2/library">Zotero group</a></p>
<p><img src="/papers/papers-citing-pr2_files/figure-html/unnamed-chunk-1-1.png" width="672" /></p>
<div id="section" class="section level2">
<h2>2022</h2>
<p><a name=bib-Anderson2022291></a><a href="#cite-Anderson2022291">Anderson, S. and L.
Thompson</a> (2022). “Optimizing an enclosed bead
beating extraction method for microbial and fish environmental DNA”.
In: <em>Environmental DNA</em> 4.2, pp. 291-303. DOI:
<a href="https://doi.org/10.1002%2Fedn3.251">10.1002/edn3.251</a>.</p>
<p><a name=bib-Bachy2022></a><a href="#cite-Bachy2022">Bachy, C., L. Sudek, C. Choi, et
al.</a> (2022). “Phytoplankton surveys in the arctic fram
strait demonstrate the tiny eukaryotic alga micromonas and other
picoprasinophytes contribute to deep sea export”. In: <em>Microorganisms</em>
10.5. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms10050961">10.3390/microorganisms10050961</a>.</p>
<p><a name=bib-Bai2022></a><a href="#cite-Bai2022">Bai, R., M. Zhou, Y. Guo, et
al.</a> (2022). “Long-term mowing reinforces connections
between soil microbial and plant communities in a temperate steppe”.
In: <em>Plant and Soil</em>. DOI:
<a href="https://doi.org/10.1007%2Fs11104-022-05374-9">10.1007/s11104-022-05374-9</a>.</p>
<p><a name=bib-Biard2022></a><a href="#cite-Biard2022">Biard, T.</a> (2022).
“Diversity and ecology of Radiolaria in modern oceans”. In:
<em>Environmental Microbiology</em>. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.16004">10.1111/1462-2920.16004</a>.</p>
<p><a name=bib-Blais2022></a><a href="#cite-Blais2022">Blais, M., A. Matveev, C. Lovejoy, et
al.</a> (2022). “Size-fractionated microbiome structure
in subarctic rivers and a coastal plume across DOC and salinity
gradients”. In: <em>Frontiers in Microbiology</em> 12. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2021.760282">10.3389/fmicb.2021.760282</a>.</p>
<p><a name=bib-Bock2022327></a><a href="#cite-Bock2022327">Bock, C., J. Olefeld, J. Vogt, et
al.</a> (2022). “Phylogenetic and functional diversity
of Chrysophyceae in inland waters”. In: <em>Organisms Diversity and
Evolution</em> 22.2, pp. 327-341. DOI:
<a href="https://doi.org/10.1007%2Fs13127-022-00554-y">10.1007/s13127-022-00554-y</a>.</p>
<p><a name=bib-Brasell2022></a><a href="#cite-Brasell2022">Brasell, K., X. Pochon, J. Howarth, et
al.</a> (2022). “Shifts in DNA yield and biological
community composition in stored sediment: Implications for paleogenomic
studies”. In: <em>Metabarcoding and Metagenomics</em> 6. DOI:
<a href="https://doi.org/10.3897%2Fmbmg.6.78128">10.3897/mbmg.6.78128</a>.</p>
<p><a name=bib-Briscoe20221303></a><a href="#cite-Briscoe20221303">Briscoe, A., S. Nichols, H.
Hartikainen, et al.</a> (2022). “High-throughput
sequencing of faeces provides evidence for dispersal of parasites and
pathogens by migratory waterbirds”. In: <em>Molecular Ecology Resources</em>
22.4, pp. 1303-1318. DOI:
<a href="https://doi.org/10.1111%2F1755-0998.13548">10.1111/1755-0998.13548</a>.</p>
<p><a name=bib-Cahoon2022></a><a href="#cite-Cahoon2022">Cahoon, A. and R.
VanGundy</a> (2022). “Alveolates (dinoflagellates,
ciliates and apicomplexans) and Rhizarians are the most common
microbial eukaryotes in temperate Appalachian karst caves”. In:
<em>Environmental Microbiology Reports</em>. DOI:
<a href="https://doi.org/10.1111%2F1758-2229.13060">10.1111/1758-2229.13060</a>.</p>
<p><a name=bib-Capo2022></a><a href="#cite-Capo2022">Capo, E., M. Monchamp, M. Coolen, et
al.</a> (2022). “Environmental paleomicrobiology: using
DNA preserved in aquatic sediments to its full potential”. In:
<em>Environmental Microbiology</em>. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.15913">10.1111/1462-2920.15913</a>.</p>
<p><a name=bib-Cho2022></a><a href="#cite-Cho2022">Cho, A., D. Tikhonenkov, E. Hehenberger, et
al.</a> (2022). “Monophyly of diverse Bigyromonadea and
their impact on phylogenomic relationships within stramenopiles”. In:
<em>Molecular Phylogenetics and Evolution</em> 171. DOI:
<a href="https://doi.org/10.1016%2Fj.ympev.2022.107468">10.1016/j.ympev.2022.107468</a>.</p>
<p><a name=bib-Couch2022></a><a href="#cite-Couch2022">Couch, C., J. Sanders, D. Sweitzer, et
al.</a> (2022). “The relationship between dietary trophic
level, parasites and the microbiome of Pacific walrus (Odobenus
rosmarus divergens)”. In: <em>Proceedings of the Royal Society B:
Biological Sciences</em> 289.1972. DOI:
<a href="https://doi.org/10.1098%2Frspb.2022.0079">10.1098/rspb.2022.0079</a>.</p>
<p><a name=bib-DeGruyter2022></a><a href="#cite-DeGruyter2022">De Gruyter, J., J. Weedon, E. Elst, et
al.</a> (2022). “Arbuscular mycorrhizal inoculation
and plant response strongly shape bacterial and eukaryotic soil
community trajectories”. In: <em>Soil Biology and Biochemistry</em> 165. DOI:
<a href="https://doi.org/10.1016%2Fj.soilbio.2021.108524">10.1016/j.soilbio.2021.108524</a>.</p>
<p><a name=bib-Djaoudi2022></a><a href="#cite-Djaoudi2022">Djaoudi, K., J. Onrubia, A. Boukra, et
al.</a> (2022). “Seawater copper content controls
biofilm bioaccumulation and microbial community on microplastics”. In:
<em>Science of the Total Environment</em> 814. DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2021.152278">10.1016/j.scitotenv.2021.152278</a>.</p>
<p><a name=bib-Duerschlag2022465></a><a href="#cite-Duerschlag2022465">Duerschlag, J., W. Mohr, T.
Ferdelman, et al.</a> (2022). “Niche partitioning
by photosynthetic plankton as a driver of CO2-fixation across the
oligotrophic South Pacific Subtropical Ocean”. In: <em>ISME Journal</em> 16.2,
pp. 465-476. DOI:
<a href="https://doi.org/10.1038%2Fs41396-021-01072-z">10.1038/s41396-021-01072-z</a>.</p>
<p><a name=bib-Dumack20221345></a><a href="#cite-Dumack20221345">Dumack, K., O. Ferlian, D. Morselli
Gysi, et al.</a> (2022). “Contrasting protist
communities (Cercozoa: Rhizaria) in pristine and earthworm-invaded
North American deciduous forests”. In: <em>Biological Invasions</em> 24.5, pp.
1345-1357. DOI:
<a href="https://doi.org/10.1007%2Fs10530-021-02726-x">10.1007/s10530-021-02726-x</a>.</p>
<p><a name=bib-Durkin2022></a><a href="#cite-Durkin2022">Durkin, C., I. Cetinic, M. Estapa, et
al.</a> (2022). “Tracing the path of carbon export in
the ocean though DNA sequencing of individual sinking particles”. In:
<em>ISME Journal</em>. DOI:
<a href="https://doi.org/10.1038%2Fs41396-022-01239-2">10.1038/s41396-022-01239-2</a>.</p>
<p><a name=bib-EshghiSahraei2022></a><a href="#cite-EshghiSahraei2022">Eshghi Sahraei, S., B. Furneaux, K.
Kluting, et al.</a> (2022). “Effects of
operational taxonomic unit inference methods on soil microeukaryote
community analysis using long-read metabarcoding”. In: <em>Ecology and
Evolution</em> 12.3. DOI:
<a href="https://doi.org/10.1002%2Fece3.8676">10.1002/ece3.8676</a>.</p>
<p><a name=bib-Fall2022886></a><a href="#cite-Fall2022886">Fall, F., H. Sanguin, D. Fall, et
al.</a> (2022). “Changes in intraspecific diversity of
the arbuscular mycorrhizal community involved in Plant
interactions between sporobolus robustus kunth and prosopis juliflora
(swartz) DC along an environmental gradient”. In: <em>Microbial Ecology</em>
83.4, pp. 886-898. DOI:
<a href="https://doi.org/10.1007%2Fs00248-021-01779-8">10.1007/s00248-021-01779-8</a>.</p>
<p><a name=bib-Ferreira2022></a><a href="#cite-Ferreira2022">Ferreira, M. and D.
Cleary</a> (2022). “Microeukaryotic communities of
golf-ball sponges inside and outside of marine lakes”. In: <em>Journal of
Sea Research</em> 180. DOI:
<a href="https://doi.org/10.1016%2Fj.seares.2021.102151">10.1016/j.seares.2021.102151</a>.</p>
<p><a name=bib-Fiore-Donno2022></a><a href="#cite-Fiore-Donno2022">Fiore-Donno, A., Z. Human, M.
Å tursovÃ¡, et al.</a> (2022). “Soil compartments
(bulk soil, litter, root and rhizosphere) as main drivers of soil
protistan communities distribution in forests with different nitrogen
deposition”. In: <em>Soil Biology and Biochemistry</em> 168. DOI:
<a href="https://doi.org/10.1016%2Fj.soilbio.2022.108628">10.1016/j.soilbio.2022.108628</a>.</p>
<p><a name=bib-Fournier2022></a><a href="#cite-Fournier2022">Fournier, I., C. Lovejoy, and W.
Vincent</a> (2022). “Road salt versus urban snow
effects on lake microbial communities”. In: <em>Microorganisms</em> 10.4. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms10040803">10.3390/microorganisms10040803</a>.</p>
<p><a name=bib-Freudenthal2022></a><a href="#cite-Freudenthal2022">Freudenthal, J., F. Ju, H. Bürgmann,
et al.</a> (2022). “Microeukaryotic gut parasites
in wastewater treatment plants: diversity, activity, and removal”. In:
<em>Microbiome</em> 10.1. DOI:
<a href="https://doi.org/10.1186%2Fs40168-022-01225-y">10.1186/s40168-022-01225-y</a>.</p>
<p><a name=bib-Fuchsman20221790></a><a href="#cite-Fuchsman20221790">Fuchsman, C., L. Cherubini, and M.
Hays</a> (2022). “An analysis of protists in
Pacific oxygen deficient zones: implications for Prochlorococcus and
N2-producing bacteria”. In: <em>Environmental Microbiology</em> 24.4, pp.
1790-1804. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.15893">10.1111/1462-2920.15893</a>.</p>
<p><a name=bib-Funaki2022></a><a href="#cite-Funaki2022">Funaki, H., C. Gaonkar, T. Kataoka, et
al.</a> (2022). “Horizontal and vertical distribution of
Gambierdiscus spp. (Dinophyceae) including novel phylotypes in Japan
identified by 18S rDNA metabarcoding”. In: <em>Harmful Algae</em> 111. DOI:
<a href="https://doi.org/10.1016%2Fj.hal.2021.102163">10.1016/j.hal.2021.102163</a>.</p>
<p><a name=bib-Gibson2022></a><a href="#cite-Gibson2022">Gibson, K., H. Song, and N.
Chen</a> (2022). “Metabarcoding analysis of microbiome
dynamics during a Phaeocystis globosa bloom in the Beibu Gulf, China”.
In: <em>Harmful Algae</em> 114. DOI:
<a href="https://doi.org/10.1016%2Fj.hal.2022.102217">10.1016/j.hal.2022.102217</a>.</p>
<p><a name=bib-Girard2022></a><a href="#cite-Girard2022">Girard, E., A. Langerak, J. Jompa, et
al.</a> (2022). “Mitochondrial cytochrome oxidase
subunit 1: A promising molecular marker for species identification in
foraminifera”. In: <em>Frontiers in Marine Science</em> 9. DOI:
<a href="https://doi.org/10.3389%2Ffmars.2022.809659">10.3389/fmars.2022.809659</a>.</p>
<p><a name=bib-Guo2022></a><a href="#cite-Guo2022">Guo, S., C. Tao, A. Jousset, et
al.</a> (2022). “Trophic interactions between predatory
protists and pathogen-suppressive bacteria impact plant health”. In:
<em>ISME Journal</em>. DOI:
<a href="https://doi.org/10.1038%2Fs41396-022-01244-5">10.1038/s41396-022-01244-5</a>.</p>
<p><a name=bib-Hu2022></a><a href="#cite-Hu2022">Hu, J., M. Gebremikael, B. Tytgat, et
al.</a> (2022). “Combined selective gamma irradiation and
pulverized soil inoculation for ecologically relevant soil microfauna
studies”. In: <em>Applied Soil Ecology</em> 169. DOI:
<a href="https://doi.org/10.1016%2Fj.apsoil.2021.104223">10.1016/j.apsoil.2021.104223</a>.</p>
<p><a name=bib-James2022></a><a href="#cite-James2022">James, C., A. Barton, L. Allen, et
al.</a> (2022). “Influence of nutrient supply on plankton
microbiome biodiversity and distribution in a coastal upwelling
region”. In: <em>Nature Communications</em> 13.1. DOI:
<a href="https://doi.org/10.1038%2Fs41467-022-30139-4">10.1038/s41467-022-30139-4</a>.</p>
<p><a name=bib-Jamieson2022></a><a href="#cite-Jamieson2022">Jamieson, T., H. Whiley, J. Gascooke, et
al.</a> (2022). “Can aggregate-associated organisms
influence the fouling in a SWRO desalination plant?” In:
<em>Microorganisms</em> 10.4. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms10040682">10.3390/microorganisms10040682</a>.</p>
<p><a name=bib-Keck20221820></a><a href="#cite-Keck20221820">Keck, F., R. Blackman, R. Bossart, et
al.</a> (2022). “Meta-analysis shows both congruence
and complementarity of DNA and eDNA metabarcoding to traditional
methods for biological community assessment”. In: <em>Molecular Ecology</em>
31.6, pp. 1820-1835. DOI:
<a href="https://doi.org/10.1111%2Fmec.16364">10.1111/mec.16364</a>.</p>
<p><a name=bib-Kitagami2022></a><a href="#cite-Kitagami2022">Kitagami, Y., K. Obase, and Y.
Matsuda</a> (2022). “High-throughput sequencing and
conventional morphotyping show different soil nematode assemblages but
similar community responses to altitudinal gradients on Mt. Ibuki,
Japan”. In: <em>Pedobiologia</em> 90. DOI:
<a href="https://doi.org/10.1016%2Fj.pedobi.2021.150788">10.1016/j.pedobi.2021.150788</a>.</p>
<p><a name=bib-LapeyraMartin2022></a><a href="#cite-LapeyraMartin2022">Lapeyra Martin, J., U. John, C.
Royer, et al.</a> (2022). “Fantastic beasts:
Unfolding mixoplankton temporal variability in the belgian coastal zone
through DNA-Metabarcoding”. In: <em>Frontiers in Marine Science</em> 9. DOI:
<a href="https://doi.org/10.3389%2Ffmars.2022.786787">10.3389/fmars.2022.786787</a>.</p>
<p><a name=bib-Lara2022></a><a href="#cite-Lara2022">Lara, E., D. Singer, and S.
Geisen</a> (2022). “Discrepancies between prokaryotes and
eukaryotes need to be considered in soil DNA-based studies”. In:
<em>Environmental Microbiology</em>. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.16019">10.1111/1462-2920.16019</a>.</p>
<p><a name=bib-Latz2022></a><a href="#cite-Latz2022">Latz, M., V. Grujcic, S. Brugel, et
al.</a> (2022). “Short- and long-read metabarcoding of the
eukaryotic rRNA operon: Evaluation of primers and comparison to shotgun
metagenomics sequencing”. In: <em>Molecular Ecology Resources</em>. DOI:
<a href="https://doi.org/10.1111%2F1755-0998.13623">10.1111/1755-0998.13623</a>.</p>
<p><a name=bib-Leray2022></a><a href="#cite-Leray2022">Leray, M., N. Knowlton, and R.
Machida</a> (2022). “MIDORI2: A collection of quality
controlled, preformatted, and regularly updated reference databases for
taxonomic assignment of eukaryotic mitochondrial sequences”. In:
<em>Environmental DNA</em>. DOI:
<a href="https://doi.org/10.1002%2Fedn3.303">10.1002/edn3.303</a>.</p>
<p><a name=bib-Li2022></a><a href="#cite-Li2022">Li, H., D. Zhu, A. Sun, et al.</a>
(2022). “Effects of soil protists on the antibiotic resistome under
long term fertilization”. In: <em>Environmental Pollution</em> 307. DOI:
<a href="https://doi.org/10.1016%2Fj.envpol.2022.119516">10.1016/j.envpol.2022.119516</a>.</p>
<p><a name=bib-Li20223></a><a href="#cite-Li20223">Li, Q., K. Edwards, C. Schvarcz, et
al.</a> (2022). “Broad phylogenetic and functional
diversity among mixotrophic consumers of Prochlorococcus”. In: <em>ISME
Journal</em>. DOI:
<a href="https://doi.org/10.1038%2Fs41396-022-01204-z">10.1038/s41396-022-01204-z</a>.</p>
<p><a name=bib-Li20222></a><a href="#cite-Li20222">Li, Y., R. Yang, L. Guo, et
al.</a> (2022). “The composition, biotic network, and
assembly of plastisphere protistan taxonomic and functional communities
in plastic-mulching croplands”. In: <em>Journal of Hazardous Materials</em>
430. DOI:
<a href="https://doi.org/10.1016%2Fj.jhazmat.2022.128390">10.1016/j.jhazmat.2022.128390</a>.</p>
<p><a name=bib-Li20221></a><a href="#cite-Li20221">Li, Z., A. Sun, X. Liu, et al.</a>
(2022). “Climate warming increases the proportions of specific
antibiotic resistance genes in natural soil ecosystems”. In: <em>Journal
of Hazardous Materials</em> 430. DOI:
<a href="https://doi.org/10.1016%2Fj.jhazmat.2022.128442">10.1016/j.jhazmat.2022.128442</a>.</p>
<p><a name=bib-Liu20221></a><a href="#cite-Liu20221">Liu, S., Z. Cui, Y. Zhao, et
al.</a> (2022). “Composition and spatial-temporal dynamics
of phytoplankton community shaped by environmental selection and
interactions in the Jiaozhou Bay”. In: <em>Water Research</em> 218. DOI:
<a href="https://doi.org/10.1016%2Fj.watres.2022.118488">10.1016/j.watres.2022.118488</a>.</p>
<p><a name=bib-Liu2022></a><a href="#cite-Liu2022">Liu, Y., D. Li, H. Gao, et al.</a>
(2022). “Regulation of soil micro-foodwebs to root secondary
metabolites in cultivated and wild licorice plants”. In: <em>Science of
the Total Environment</em> 828. DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2022.154302">10.1016/j.scitotenv.2022.154302</a>.</p>
<p><a name=bib-Metz20221731></a><a href="#cite-Metz20221731">Metz, S., P. Huber, V. Accattatis, et
al.</a> (2022). “Freshwater protists: unveiling the
unexplored in a large floodplain system”. In: <em>Environmental
Microbiology</em> 24.4, pp. 1731-1745. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.15838">10.1111/1462-2920.15838</a>.</p>
<p><a name=bib-Minardi2022664></a><a href="#cite-Minardi2022664">Minardi, D., D. Ryder, J. del Campo, et
al.</a> (2022). “Improved high throughput protocol
for targeting eukaryotic symbionts in metazoan and eDNA samples”. In:
<em>Molecular Ecology Resources</em> 22.2, pp. 664-678. DOI:
<a href="https://doi.org/10.1111%2F1755-0998.13509">10.1111/1755-0998.13509</a>.</p>
<p><a name=bib-Moreno-Mesonero2022></a><a href="#cite-Moreno-Mesonero2022">Moreno-Mesonero, L., I. Amorós, Y.
Moreno, et al.</a> (2022). “Simultaneous
detection of less frequent waterborne parasitic protozoa in reused
wastewater using amplicon sequencing and qPCR techniques”. In: <em>Journal
of Environmental Management</em> 314. DOI:
<a href="https://doi.org/10.1016%2Fj.jenvman.2022.115029">10.1016/j.jenvman.2022.115029</a>.</p>
<p><a name=bib-Moynihan2022233></a><a href="#cite-Moynihan2022233">Moynihan, M., N. Goodkin, K. Morgan,
et al.</a> (2022). “Coral-associated nitrogen
fixation rates and diazotrophic diversity on a nutrient-replete
equatorial reef”. In: <em>ISME Journal</em> 16.1, pp. 233-246. DOI:
<a href="https://doi.org/10.1038%2Fs41396-021-01054-1">10.1038/s41396-021-01054-1</a>.</p>
<p><a name=bib-Nguyen2022></a><a href="#cite-Nguyen2022">Nguyen, B., Q. Chen, J. He, et
al.</a> (2022). “Livestock manure spiked with the
antibiotic tylosin significantly altered soil protist functional
groups”. In: <em>Journal of Hazardous Materials</em> 427. DOI:
<a href="https://doi.org/10.1016%2Fj.jhazmat.2021.127867">10.1016/j.jhazmat.2021.127867</a>.</p>
<p><a name=bib-Okamoto20223210></a><a href="#cite-Okamoto20223210">Okamoto, N., P. Keeling, B. Leander,
et al.</a> (2022). “Microbial communities in sandy
beaches from the three domains of life differ by microhabitat and
intertidal location”. In: <em>Molecular Ecology</em> 31.11, pp. 3210-3227.
DOI: <a href="https://doi.org/10.1111%2Fmec.16453">10.1111/mec.16453</a>.</p>
<p><a name=bib-Oliva2022></a><a href="#cite-Oliva2022">Oliva, A., R. Garner, D. Walsh, et
al.</a> (2022). “The occurrence of potentially pathogenic
fungi and protists in Canadian lakes predicted using geomatics, in situ
and satellite-derived variables: Towards a tele-epidemiological
approach”. In: <em>Water Research</em> 209. DOI:
<a href="https://doi.org/10.1016%2Fj.watres.2021.117935">10.1016/j.watres.2021.117935</a>.</p>
<p><a name=bib-PierellaKarlusich2022></a><a href="#cite-PierellaKarlusich2022">Pierella Karlusich, J., E.
Pelletier, L. Zinger, et al.</a> (2022). “A
robust approach to estimate relative phytoplankton cell abundances from
metagenomes”. In: <em>Molecular Ecology Resources</em>. DOI:
<a href="https://doi.org/10.1111%2F1755-0998.13592">10.1111/1755-0998.13592</a>.</p>
<p><a name=bib-Potvin2022></a><a href="#cite-Potvin2022">Potvin, M., M. Rautio, and C.
Lovejoy</a> (2022). “Freshwater microbial eukaryotic
core communities, open-water and under-ice specialists in southern
victoria island lakes (ekaluktutiak, NU, canada)”. In: <em>Frontiers in
Microbiology</em> 12. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2021.786094">10.3389/fmicb.2021.786094</a>.</p>
<p><a name=bib-Qi2022></a><a href="#cite-Qi2022">Qi, Q., C. Hu, J. Lin, et al.</a>
(2022). “Contamination with multiple heavy metals decreases microbial
diversity and favors generalists as the keystones in microbial
occurrence networks”. In: <em>Environmental Pollution</em> 306. DOI:
<a href="https://doi.org/10.1016%2Fj.envpol.2022.119406">10.1016/j.envpol.2022.119406</a>.</p>
<p><a name=bib-Reid202289></a><a href="#cite-Reid202289">Reid, B., J. Servis, M. Timmers, et
al.</a> (2022). “18S rDNA amplicon sequence data (V1-V3)
of the palmyra atoll national wildlife refuge, central pacific”. In:
<em>Metabarcoding and Metagenomics</em> 6, pp. 89-99. DOI:
<a href="https://doi.org/10.3897%2Fmbmg.6.78762">10.3897/mbmg.6.78762</a>.</p>
<p><a name=bib-Resch2022></a><a href="#cite-Resch2022">Resch, M., M. Schütz, R. Ochoa-Hueso, et
al.</a> (2022). “Long-term recovery of above- and
below-ground interactions in restored grasslands after topsoil removal
and seed addition”. In: <em>Journal of Applied Ecology</em>. DOI:
<a href="https://doi.org/10.1111%2F1365-2664.14145">10.1111/1365-2664.14145</a>.</p>
<p><a name=bib-Rii2022S331></a><a href="#cite-Rii2022S331">Rii, Y., L. Peoples, D. Karl, et
al.</a> (2022). “Seasonality and episodic variation in
picoeukaryote diversity and structure reveal community resilience to
disturbances in the North Pacific Subtropical Gyre”. In: <em>Limnology and
Oceanography</em> 67.S1, pp. S331-S351. DOI:
<a href="https://doi.org/10.1002%2Flno.11916">10.1002/lno.11916</a>.</p>
<p><a name=bib-Romdhane2022></a><a href="#cite-Romdhane2022">Romdhane, S., A. Spor, S. Banerjee, et
al.</a> (2022). “Land-use intensification
differentially affects bacterial, fungal and protist communities and
decreases microbiome network complexity”. In: <em>Environmental
Microbiomes</em> 17.1. DOI:
<a href="https://doi.org/10.1186%2Fs40793-021-00396-9">10.1186/s40793-021-00396-9</a>.</p>
<p><a name=bib-Salmaso2022></a><a href="#cite-Salmaso2022">Salmaso, N., V. Vasselon, F. Rimet, et
al.</a> (2022). “DNA sequence and taxonomic gap
analyses to quantify the coverage of aquatic cyanobacteria and
eukaryotic microalgae in reference databases: Results of a survey in
the Alpine region”. In: <em>Science of the Total Environment</em> 834. DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2022.155175">10.1016/j.scitotenv.2022.155175</a>.</p>
<p><a name=bib-Selak20222282></a><a href="#cite-Selak20222282">Selak, L., H. Osterholz, I. Stankovic,
et al.</a> (2022). “Adaptations of microbial
communities and dissolved organics to seasonal pressures in a
mesotrophic coastal Mediterranean lake”. In: <em>Environmental
Microbiology</em> 24.5, pp. 2282-2298. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.15924">10.1111/1462-2920.15924</a>.</p>
<p><a name=bib-Stefanni2022></a><a href="#cite-Stefanni2022">Stefanni, S., L. Mirimin, D. Stankovic,
et al.</a> (2022). “Framing cutting-edge integrative
deep-sea biodiversity monitoring via environmental DNA and optoacoustic
augmented infrastructures”. In: <em>Frontiers in Marine Science</em> 8. DOI:
<a href="https://doi.org/10.3389%2Ffmars.2021.797140">10.3389/fmars.2021.797140</a>.</p>
<p><a name=bib-Sun20221></a><a href="#cite-Sun20221">Sun, F., X. Xia, M. Simon, et
al.</a> (2022). “Anticyclonic eddy driving significant
changes in prokaryotic and eukaryotic communities in the south china
sea”. In: <em>Frontiers in Marine Science</em> 9. DOI:
<a href="https://doi.org/10.3389%2Ffmars.2022.773548">10.3389/fmars.2022.773548</a>.</p>
<p><a name=bib-Sun20222></a><a href="#cite-Sun20222">Sun, P., Y. Liao, Y. Wang, et
al.</a> (2022). “Contrasting community composition and
co-occurrence relationships of the active pico-sized haptophytes in the
surface and subsurface chlorophyll maximum layers of the arctic ocean
in summer”. In: <em>Microorganisms</em> 10.2. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms10020248">10.3390/microorganisms10020248</a>.</p>
<p><a name=bib-Sun2022></a><a href="#cite-Sun2022">Sun, P., Y. Wang, X. Huang, et
al.</a> (2022). “Water masses and their associated
temperature and cross-domain biotic factors co-shape upwelling
microbial communities”. In: <em>Water Research</em> 215. DOI:
<a href="https://doi.org/10.1016%2Fj.watres.2022.118274">10.1016/j.watres.2022.118274</a>.</p>
<p><a name=bib-Suter2022></a><a href="#cite-Suter2022">Suter, E., M. Pachiadaki, G. Taylor, et
al.</a> (2022). “Eukaryotic parasites are integral to a
productive microbial food web in oxygen-depleted waters”. In:
<em>Frontiers in Microbiology</em> 12. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2021.764605">10.3389/fmicb.2021.764605</a>.</p>
<p><a name=bib-Vannoni2022></a><a href="#cite-Vannoni2022">Vannoni, M., V. Créach, D. Ryder, et
al.</a> (2022). “Resilience of a microphytobenthos
community from the Severn Estuary, UK, to chlorination: A mesocosm
approach”. In: <em>Marine Pollution Bulletin</em> 176. DOI:
<a href="https://doi.org/10.1016%2Fj.marpolbul.2022.113443">10.1016/j.marpolbul.2022.113443</a>.</p>
<p><a name=bib-Vasar20221120></a><a href="#cite-Vasar20221120">Vasar, M., J. Davison, S. Sepp, et
al.</a> (2022). “Global soil microbiomes: A new
frontline of biome-ecology research”. In: <em>Global Ecology and
Biogeography</em> 31.6, pp. 1120-1132. DOI:
<a href="https://doi.org/10.1111%2Fgeb.13487">10.1111/geb.13487</a>.</p>
<p><a name=bib-Vaulot2022168></a><a href="#cite-Vaulot2022168">Vaulot, D., S. Geisen, F. Mahé, et
al.</a> (2022). “pr2-primers: An 18S rRNA primer
database for protists”. In: <em>Molecular Ecology Resources</em> 22.1, pp.
168-179. DOI:
<a href="https://doi.org/10.1111%2F1755-0998.13465">10.1111/1755-0998.13465</a>.</p>
<p><a name=bib-Wang2022></a><a href="#cite-Wang2022">Wang, Y., X. Li, Y. Chi, et
al.</a> (2022). “Changes of the freshwater microbial
community structure and assembly processes during different sample
storage conditions”. In: <em>Microorganisms</em> 10.6. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms10061176">10.3390/microorganisms10061176</a>.</p>
<p><a name=bib-Wu20221></a><a href="#cite-Wu20221">Wu, B., L. Zhou, S. Liu, et
al.</a> (2022). “Biogeography of soil protistan consumer
and parasite is contrasting and linked to microbial nutrient
mineralization in forest soils at a wide-scale”. In: <em>Soil Biology and
Biochemistry</em> 165. DOI:
<a href="https://doi.org/10.1016%2Fj.soilbio.2021.108513">10.1016/j.soilbio.2021.108513</a>.</p>
<p><a name=bib-Wu2022></a><a href="#cite-Wu2022">Wu, C., Y. Chao, L. Shu, et al.</a>
(2022). “Interactions between soil protists and pollutants: An unsolved
puzzle”. In: <em>Journal of Hazardous Materials</em> 429. DOI:
<a href="https://doi.org/10.1016%2Fj.jhazmat.2022.128297">10.1016/j.jhazmat.2022.128297</a>.</p>
<p><a name=bib-Xiong2022></a><a href="#cite-Xiong2022">Xiong, W., M. Delgado-Baquerizo, Q. Shen, et
al.</a> (2022). “Pedogenesis shapes predator-prey
relationships within soil microbiomes”. In: <em>Science of the Total
Environment</em> 828. DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2022.154405">10.1016/j.scitotenv.2022.154405</a>.</p>
<p><a name=bib-Xu2022></a><a href="#cite-Xu2022">Xu, D., H. Kong, E. Yang, et al.</a>
(2022). “Spatial dynamics of active microeukaryotes along a latitudinal
gradient: Diversity, assembly process, and co-occurrence
relationships”. In: <em>Environmental Research</em> 212. DOI:
<a href="https://doi.org/10.1016%2Fj.envres.2022.113234">10.1016/j.envres.2022.113234</a>.</p>
<p><a name=bib-Yin2022></a><a href="#cite-Yin2022">Yin, Y., M. Li, C. Yang, et
al.</a> (2022). “Phagotrophic protists can change microbial
nitrogen conversion patterns during swine manure composting”. In:
<em>Biomass Conversion and Biorefinery</em>. DOI:
<a href="https://doi.org/10.1007%2Fs13399-022-02318-2">10.1007/s13399-022-02318-2</a>.</p>
<p><a name=bib-Yung2022></a><a href="#cite-Yung2022">Yung, C., E. Redondo, F. Sanchez, et
al.</a> (2022). “Diversity and evolution of
mamiellophyceae: Early-diverging phytoplanktonic green algae containing
many cosmopolitan species”. In: <em>Journal of Marine Science and
Engineering</em> 10.2. DOI:
<a href="https://doi.org/10.3390%2Fjmse10020240">10.3390/jmse10020240</a>.</p>
<p><a name=bib-Zafeiropoulos2022></a><a href="#cite-Zafeiropoulos2022">Zafeiropoulos, H., S. Paragkamian,
S. Ninidakis, et al.</a> (2022). “PREGO: A
literature and data-mining resource to associate microorganisms,
biological processes, and environment types”. In: <em>Microorganisms</em>
10.2. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms10020293">10.3390/microorganisms10020293</a>.</p>
<p><a name=bib-Zhao2022></a><a href="#cite-Zhao2022">Zhao, H., Z. Zhang, S. Nair, et
al.</a> (2022). “Vertically exported phytoplankton (¡ 20
µm) and their correlation network with bacterioplankton along a
deep-sea seamount”. In: <em>Frontiers in Marine Science</em> 9. DOI:
<a href="https://doi.org/10.3389%2Ffmars.2022.862494">10.3389/fmars.2022.862494</a>.</p>
<p><a name=bib-Zhao20221775></a><a href="#cite-Zhao20221775">Zhao, Z., H. Li, Y. Sun, et
al.</a> (2022). “How habitat heterogeneity shapes
bacterial and protistan communities in temperate coastal areas near
estuaries”. In: <em>Environmental Microbiology</em> 24.4, pp. 1775-1789. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.15892">10.1111/1462-2920.15892</a>.</p>
</div>
<div id="section-1" class="section level2">
<h2>2021</h2>
<p><a name=bib-AbdullahAl2021></a><a href="#cite-AbdullahAl2021">Abdullah Al, M., Y. Xue, P. Xiao, et
al.</a> (2021). “DNA metabarcoding reveals the
significant influence of anthropogenic effects on microeukaryotic
communities in urban waterbodies”. In: <em>Environmental Pollution</em> 285.
DOI:
<a href="https://doi.org/10.1016%2Fj.envpol.2021.117336">10.1016/j.envpol.2021.117336</a>.</p>
<p><a name=bib-Acinas2021></a><a href="#cite-Acinas2021">Acinas, S., P. Sánchez, G. Salazar, et
al.</a> (2021). “Deep ocean metagenomes provide insight
into the metabolic architecture of bathypelagic microbial communities”.
In: <em>Communications Biology</em> 4.1. DOI:
<a href="https://doi.org/10.1038%2Fs42003-021-02112-2">10.1038/s42003-021-02112-2</a>.</p>
<p><a name=bib-Armbrecht2021></a><a href="#cite-Armbrecht2021">Armbrecht, L., G. Hallegraeff, C. Bolch,
et al.</a> (2021). “Hybridisation capture allows DNA
damage analysis of ancient marine eukaryotes”. In: <em>Scientific Reports</em>
11.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-021-82578-6">10.1038/s41598-021-82578-6</a>.</p>
<p><a name=bib-Asiloglu2021></a><a href="#cite-Asiloglu2021">Asiloglu, R., K. Shiroishi, K. Suzuki, et
al.</a> (2021). “Soil properties have more significant
effects on the community composition of protists than the rhizosphere
effect of rice plants in alkaline paddy field soils”. In: <em>Soil Biology
and Biochemistry</em> 161. DOI:
<a href="https://doi.org/10.1016%2Fj.soilbio.2021.108397">10.1016/j.soilbio.2021.108397</a>.</p>
<p><a name=bib-Beng20211602></a><a href="#cite-Beng20211602">Beng, K., J. Wolinska, E. Funke, et
al.</a> (2021). “Temporal dynamics of freshwater
planktonic parasites inferred using a DNA metabarcoding time-series”.
In: <em>Parasitology</em> 148.13, pp. 1602-1611. DOI:
<a href="https://doi.org/10.1017%2FS0031182021001293">10.1017/S0031182021001293</a>.</p>
<p><a name=bib-Bonfantine2021></a><a href="#cite-Bonfantine2021">Bonfantine, K., S. Trevathan-Tackett,
T. Matthews, et al.</a> (2021). “Dumpster diving for
diatom plastid 16S rRNA genes”. In: <em>PeerJ</em> 9. DOI:
<a href="https://doi.org/10.7717%2Fpeerj.11576">10.7717/peerj.11576</a>.</p>
<p><a name=bib-Brandt2021></a><a href="#cite-Brandt2021">Brandt, M., F. Pradillon, B. Trouche, et
al.</a> (2021). “Evaluating sediment and water sampling
methods for the estimation of deep-sea biodiversity using environmental
DNA”. In: <em>Scientific Reports</em> 11.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-021-86396-8">10.1038/s41598-021-86396-8</a>.</p>
<p><a name=bib-Bruhn2021></a><a href="#cite-Bruhn2021">Bruhn, C., S. Wohlrab, B. Krock, et
al.</a> (2021). “Seasonal plankton succession is in
accordance with phycotoxin occurrence in Disko Bay, West Greenland”.
In: <em>Harmful Algae</em> 103. DOI:
<a href="https://doi.org/10.1016%2Fj.hal.2021.101978">10.1016/j.hal.2021.101978</a>.</p>
<p><a name=bib-Burki2021R1267></a><a href="#cite-Burki2021R1267">Burki, F., M. Sandin, and M.
Jamy</a> (2021). “Diversity and ecology of protists
revealed by metabarcoding”. In: <em>Current Biology</em> 31.19, pp.
R1267-R1280. DOI:
<a href="https://doi.org/10.1016%2Fj.cub.2021.07.066">10.1016/j.cub.2021.07.066</a>.</p>
<p><a name=bib-Canino2021199></a><a href="#cite-Canino2021199">Canino, A., A. Bouchez, C.
Laplace-Treyture, et al.</a> (2021). “Phytool, a
ShinyApp to homogenise taxonomy of freshwater microalgae from DNA
barcodes and microscopic observations”. In: <em>Metabarcoding and
Metagenomics</em> 5, pp. 199-205. DOI:
<a href="https://doi.org/10.3897%2Fmbmg.5.74096">10.3897/mbmg.5.74096</a>.</p>
<p><a name=bib-Capo2021></a><a href="#cite-Capo2021">Capo, E., C. Giguet-Covex, A. Rouillard, et
al.</a> (2021). “Lake sedimentary dna research on past
terrestrial and aquatic biodiversity: Overview and recommendations”.
In: <em>Quaternary</em> 4.1. DOI:
<a href="https://doi.org/10.3390%2Fquat4010006">10.3390/quat4010006</a>.</p>
<p><a name=bib-Casey20211229></a><a href="#cite-Casey20211229">Casey, J., E. Ransome, A. Collins, et
al.</a> (2021). “DNA metabarcoding marker choice
skews perception of marine eukaryotic biodiversity”. In: <em>Environmental
DNA</em> 3.6, pp. 1229-1246. DOI:
<a href="https://doi.org/10.1002%2Fedn3.245">10.1002/edn3.245</a>.</p>
<p><a name=bib-Castagna2021></a><a href="#cite-Castagna2021">Castagna, A., H. Dierssen, E. Organelli,
et al.</a> (2021). “Optical detection of harmful algal
blooms in the belgian coastal zone: A cautionary tale of chlorophyll
c3”. In: <em>Frontiers in Marine Science</em> 8. DOI:
<a href="https://doi.org/10.3389%2Ffmars.2021.770340">10.3389/fmars.2021.770340</a>.</p>
<p><a name=bib-Catlett2021></a><a href="#cite-Catlett2021">Catlett, D., K. Son, and C.
Liang</a> (2021). “EnsembleTax: An R package for
determinations of ensemble taxonomic assignments of
phylogenetically-informative marker gene sequences”. In: <em>PeerJ</em> 9.
DOI: <a href="https://doi.org/10.7717%2Fpeerj.11865">10.7717/peerj.11865</a>.</p>
<p><a name=bib-Cecchetto2021></a><a href="#cite-Cecchetto2021">Cecchetto, M., A. Di Cesare, E. Eckert,
et al.</a> (2021). “Antarctic coastal nanoplankton
dynamics revealed by metabarcoding of desalination plant filters:
Detection of short-term events and implications for routine
monitoring”. In: <em>Science of the Total Environment</em> 757. DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2020.143809">10.1016/j.scitotenv.2020.143809</a>.</p>
<p><a name=bib-Ceja-Navarro2021></a><a href="#cite-Ceja-Navarro2021">Ceja-Navarro, J., Y. Wang, D. Ning,
et al.</a> (2021). “Protist diversity and
community complexity in the rhizosphere of switchgrass are dynamic as
plants develop”. In: <em>Microbiome</em> 9.1. DOI:
<a href="https://doi.org/10.1186%2Fs40168-021-01042-9">10.1186/s40168-021-01042-9</a>.</p>
<p><a name=bib-Chang2021></a><a href="#cite-Chang2021">Chang, F., J. Yang, A. Liu, et
al.</a> (2021). “Community assembly processes as a
mechanistic explanation of the predator-prey diversity relationship in
marine microbes”. In: <em>Frontiers in Marine Science</em> 8. DOI:
<a href="https://doi.org/10.3389%2Ffmars.2021.651565">10.3389/fmars.2021.651565</a>.</p>
<p><a name=bib-Charvet2021></a><a href="#cite-Charvet2021">Charvet, S., E. Kim, A. Subramaniam, et
al.</a> (2021). “Small pigmented eukaryote assemblages
of the western tropical North Atlantic around the Amazon River plume
during spring discharge”. In: <em>Scientific Reports</em> 11.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-021-95676-2">10.1038/s41598-021-95676-2</a>.</p>
<p><a name=bib-Chen2021></a><a href="#cite-Chen2021">Chen, B., W. Xiong, J. Qi, et
al.</a> (2021). “Trophic interrelationships drive the
biogeography of protistan community in agricultural ecosystems”. In:
<em>Soil Biology and Biochemistry</em> 163. DOI:
<a href="https://doi.org/10.1016%2Fj.soilbio.2021.108445">10.1016/j.soilbio.2021.108445</a>.</p>
<p><a name=bib-Chen20211773></a><a href="#cite-Chen20211773">Chen, J., L. Dong, X. Ma, et
al.</a> (2021). “Microbial community monitoring in
baiyangdian lake based on eDNA technology [åŸºäºŽ eDNA
æŠ€æœ¯çš„ç™½æ´‹æ·€å¾®åž‹ç”Ÿç‰©ç¾¤è½ç›’æµ‹]”. In: <em>Journal of
Agro-Environment Science</em> 40.9, pp. 1773-1786. DOI:
<a href="https://doi.org/10.11654%2Fjaes.2021-0632">10.11654/jaes.2021-0632</a>.</p>
<p><a name=bib-Chen20211></a><a href="#cite-Chen20211">Chen, Y., Q. Xu, K. Gibson, et
al.</a> (2021). “Metabarcoding dissection of harmful
algal bloom species in the East China Sea off Southern Zhejiang
Province in late spring”. In: <em>Marine Pollution Bulletin</em> 169. DOI:
<a href="https://doi.org/10.1016%2Fj.marpolbul.2021.112586">10.1016/j.marpolbul.2021.112586</a>.</p>
<p><a name=bib-Choe2021></a><a href="#cite-Choe2021">Choe, Y., M. Kim, and Y. Lee</a>
(2021). “Distinct microbial communities in adjacent rock and soil
substrates on a high arctic polar desert”. In: <em>Frontiers in
Microbiology</em> 11. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2020.607396">10.3389/fmicb.2020.607396</a>.</p>
<p><a name=bib-Chrismas2021667></a><a href="#cite-Chrismas2021667">Chrismas, N., R. Allen, A.
Hollingsworth, et al.</a> (2021). “Complex
photobiont diversity in the marine lichen Lichina pygmaea”. In:
<em>Journal of the Marine Biological Association of the United Kingdom</em>
101.4, pp. 667-674. DOI:
<a href="https://doi.org/10.1017%2FS002531542100062X">10.1017/S002531542100062X</a>.</p>
<p><a name=bib-Christaki2021></a><a href="#cite-Christaki2021">Christaki, U., I. Skouroliakou, A.
Delegrange, et al.</a> (2021). “Microzooplankton
diversity and potential role in carbon cycling of contrasting Southern
Ocean productivity regimes”. In: <em>Journal of Marine Systems</em> 219. DOI:
<a href="https://doi.org/10.1016%2Fj.jmarsys.2021.103531">10.1016/j.jmarsys.2021.103531</a>.</p>
<p><a name=bib-Cohen2021173></a><a href="#cite-Cohen2021173">Cohen, N., M. McIlvin, D. Moran, et
al.</a> (2021). “Dinoflagellates alter their carbon
and nutrient metabolic strategies across environmental gradients in the
central Pacific Ocean”. In: <em>Nature Microbiology</em> 6.2, pp. 173-186.
DOI:
<a href="https://doi.org/10.1038%2Fs41564-020-00814-7">10.1038/s41564-020-00814-7</a>.</p>
<p><a name=bib-Cohen20215397></a><a href="#cite-Cohen20215397">Cohen, N., A. Noble, D. Moran, et
al.</a> (2021). “Hydrothermal trace metal release and
microbial metabolism in the northeastern Lau Basin of the South Pacific
Ocean”. In: <em>Biogeosciences (Online)</em> 18.19, pp. 5397-5422. DOI:
<a href="https://doi.org/10.5194%2Fbg-18-5397-2021">10.5194/bg-18-5397-2021</a>.</p>
<p><a name=bib-Cordier20212937></a><a href="#cite-Cordier20212937">Cordier, T., L. Alonso-SÃ¡ez, L.
ApothÃ©loz-Perret-Gentil, et al.</a> (2021).
“Ecosystems monitoring powered by environmental genomics: A review of
current strategies with an implementation roadmap”. In: <em>Molecular
Ecology</em> 30.13, pp. 2937-2958. DOI:
<a href="https://doi.org/10.1111%2Fmec.15472">10.1111/mec.15472</a>.</p>
<p><a name=bib-Cui2021></a><a href="#cite-Cui2021">Cui, Z., Q. Xu, K. Gibson, et
al.</a> (2021). “Metabarcoding analysis of harmful algal
bloom species in the Changjiang Estuary, China”. In: <em>Science of the
Total Environment</em> 782. DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2021.146823">10.1016/j.scitotenv.2021.146823</a>.</p>
<p><a name=bib-Dai2021></a><a href="#cite-Dai2021">Dai, Z., X. Lv, B. Ma, et al.</a>
(2021). “Concurrent and rapid recovery of bacteria and protist
communities in Canadian boreal forest ecosystems following wildfire”.
In: <em>Soil Biology and Biochemistry</em> 163. DOI:
<a href="https://doi.org/10.1016%2Fj.soilbio.2021.108452">10.1016/j.soilbio.2021.108452</a>.</p>
<p><a name=bib-David20211436></a><a href="#cite-David20211436">David, G., D. Moreira, G. Reboul, et
al.</a> (2021). “Environmental drivers of plankton
protist communities along latitudinal and vertical gradients in the
oldest and deepest freshwater lake”. In: <em>Environmental Microbiology</em>
23.3, pp. 1436-1451. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.15346">10.1111/1462-2920.15346</a>.</p>
<p><a name=bib-DeMesquita2021></a><a href="#cite-DeMesquita2021">De Mesquita, C., L. Nichols, M. Gebert,
et al.</a> (2021). “Structure of chimpanzee gut
microbiomes across tropical Africa”. In: <em>mSystems</em> 6.3. DOI:
<a href="https://doi.org/10.1128%2FmSystems.01269-20">10.1128/mSystems.01269-20</a>.</p>
<p><a name=bib-Degenhardt2021></a><a href="#cite-Degenhardt2021">Degenhardt, J., S. Khodami, F. Milke,
et al.</a> (2021). “The three domains of life within
the discharge area of a shallow subterranean estuary at a high energy
beach”. In: <em>Frontiers in Environmental Science</em> 9. DOI:
<a href="https://doi.org/10.3389%2Ffenvs.2021.642098">10.3389/fenvs.2021.642098</a>.</p>
<p><a name=bib-Deutschmann2021></a><a href="#cite-Deutschmann2021">Deutschmann, I., G. Lima-Mendez, A.
Krabberød, et al.</a> (2021). “Disentangling
environmental effects in microbial association networks”. In:
<em>Microbiome</em> 9.1. DOI:
<a href="https://doi.org/10.1186%2Fs40168-021-01141-7">10.1186/s40168-021-01141-7</a>.</p>
<p><a name=bib-DiCapua20213303></a><a href="#cite-DiCapua20213303">Di Capua, I., R. Piredda, M.
Mazzocchi, et al.</a> (2021). “Metazoan diversity
and seasonality through eDNA metabarcoding at a Mediterranean long-term
ecological research site”. In: <em>ICES Journal of Marine Science</em> 78.9,
pp. 3303-3316. DOI:
<a href="https://doi.org/10.1093%2Ficesjms%2Ffsab059">10.1093/icesjms/fsab059</a>.</p>
<p><a name=bib-Diner2021></a><a href="#cite-Diner2021">Diner, R., D. Kaul, A. Rabines, et
al.</a> (2021). “Pathogenic vibrio species are associated
with distinct environmental niches and planktonic taxa in southern
california (usa) aquatic microbiomes”. In: <em>mSystems</em> 6.4. DOI:
<a href="https://doi.org/10.1128%2FmSystems.00571-21">10.1128/mSystems.00571-21</a>.</p>
<p><a name=bib-Doliwa2021549></a><a href="#cite-Doliwa2021549">Doliwa, A., M. Dunthorn, E.
Rassoshanska, et al.</a> (2021). “Identifying
potential hosts of short-branch microsporidia”. In: <em>Microbial Ecology</em>
82.2, pp. 549-553. DOI:
<a href="https://doi.org/10.1007%2Fs00248-020-01657-9">10.1007/s00248-020-01657-9</a>.</p>
<p><a name=bib-Dubuffet20214344></a><a href="#cite-Dubuffet20214344">Dubuffet, A., M. Chauvet, A. Moné, et
al.</a> (2021). “A phylogenetic framework to
investigate the microsporidian communities through metabarcoding and
its application to lake ecosystems”. In: <em>Environmental Microbiology</em>
23.8, pp. 4344-4359. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.15618">10.1111/1462-2920.15618</a>.</p>
<p><a name=bib-Dumack202177></a><a href="#cite-Dumack202177">Dumack, K. and M.
Bonkowski</a> (2021). “Protists in the plant
microbiome: an untapped field of research”. In: <em>Methods in Molecular
Biology</em> 2232, pp. 77-84. DOI:
<a href="https://doi.org/10.1007%2F978-1-0716-1040-4_8">10.1007/978-1-0716-1040-4_8</a>.</p>
<p><a name=bib-Egge20214913></a><a href="#cite-Egge20214913">Egge, E., S. Elferink, D. Vaulot, et
al.</a> (2021). “An 18S V4 rRNA metabarcoding dataset
of protist diversity in the Atlantic inflow to the Arctic Ocean,
through the year and down to 1000m depth”. In: <em>Earth System Science
Data</em> 13.10, pp. 4913-4928. DOI:
<a href="https://doi.org/10.5194%2Fessd-13-4913-2021">10.5194/essd-13-4913-2021</a>.</p>
<p><a name=bib-Fermani2021></a><a href="#cite-Fermani2021">Fermani, P., S. Metz, V. Balagué, et
al.</a> (2021). “Microbial eukaryote assemblages and
potential novel diversity in four tropical East African Great Lakes”.
In: <em>FEMS Microbiology Ecology</em> 97.9. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiab114">10.1093/femsec/fiab114</a>.</p>
<p><a name=bib-Forster20211></a><a href="#cite-Forster20211">Forster, D., Z. Qu, G. Pitsch, et
al.</a> (2021). “Lake ecosystem robustness and
resilience inferred from a climate-stressed protistan plankton
network”. In: <em>Microorganisms</em> 9.3, pp. 1-20. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms9030549">10.3390/microorganisms9030549</a>.</p>
<p><a name=bib-Fournier2021></a><a href="#cite-Fournier2021">Fournier, I., C. Lovejoy, and W.
Vincent</a> (2021). “Changes in the community
structure of under-ice and open-water microbiomes in urban lakes
exposed to road salts”. In: <em>Frontiers in Microbiology</em> 12. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2021.660719">10.3389/fmicb.2021.660719</a>.</p>
<p><a name=bib-Francioli20211></a><a href="#cite-Francioli20211">Francioli, D., G. Lentendu, S. Lewin,
et al.</a> (2021). “DNA metabarcoding for the
characterization of terrestrial microbiota—pitfalls and solutions”. In:
<em>Microorganisms</em> 9.2, pp. 1-29. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms9020361">10.3390/microorganisms9020361</a>.</p>
<p><a name=bib-Freyria2021></a><a href="#cite-Freyria2021">Freyria, N., N. Joli, and C.
Lovejoy</a> (2021). “A decadal perspective on north
water microbial eukaryotes as Arctic Ocean sentinels”. In: <em>Scientific
Reports</em> 11.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-021-87906-4">10.1038/s41598-021-87906-4</a>.</p>
<p><a name=bib-Ganser2021></a><a href="#cite-Ganser2021">Ganser, M., D. Forster, W. Liu, et
al.</a> (2021). “Genetic diversity in marine planktonic
ciliates (alveolata, ciliophora) suggests distinct geographical
patterns data from chinese and european coastal waters”.
In: <em>Frontiers in Marine Science</em> 8. DOI:
<a href="https://doi.org/10.3389%2Ffmars.2021.643822">10.3389/fmars.2021.643822</a>.</p>
<p><a name=bib-Gauthier20213020></a><a href="#cite-Gauthier20213020">Gauthier, J., D. Walsh, D. Selbie, et
al.</a> (2021). “Evaluating the congruence between
DNA-based and morphological taxonomic approaches in water and sediment
trap samples: Analyses of a 36-month time series from a temperate
monomictic lake”. In: <em>Limnology and Oceanography</em> 66.8, pp. 3020-3039.
DOI: <a href="https://doi.org/10.1002%2Flno.11856">10.1002/lno.11856</a>.</p>
<p><a name=bib-Genitsaris2021></a><a href="#cite-Genitsaris2021">Genitsaris, S., N. Stefanidou, Y.
Beeri-Shlevin, et al.</a> (2021). “Air-dispersed
aquatic microorganisms show establishment and growth preferences in
different freshwater colonisation habitats”. In: <em>FEMS Microbiology
Ecology</em> 97.9. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiab122">10.1093/femsec/fiab122</a>.</p>
<p><a name=bib-Glasner2021></a><a href="#cite-Glasner2021">Glasner, B., C. Henr'iquez-Castillo, F.
Alfaro, et al.</a> (2021). “Decoupling of biotic and
abiotic patterns in a coastal area affected by chronic metal
micronutrients disturbances”. In: <em>Marine Pollution Bulletin</em> 166. DOI:
<a href="https://doi.org/10.1016%2Fj.marpolbul.2020.111608">10.1016/j.marpolbul.2020.111608</a>.</p>
<p><a name=bib-Gong2021></a><a href="#cite-Gong2021">Gong, X., X. Chen, S. Geisen, et
al.</a> (2021). “Agricultural habitats are dominated by
rapidly evolving nematodes revealed through phylogenetic comparative
methods”. In: <em>Soil Biology and Biochemistry</em> 155. DOI:
<a href="https://doi.org/10.1016%2Fj.soilbio.2021.108183">10.1016/j.soilbio.2021.108183</a>.</p>
<p><a name=bib-Gottschling2021></a><a href="#cite-Gottschling2021">Gottschling, M., L. Czech, F. Mahé, et
al.</a> (2021). “The windblown: Possible
explanations for dinophyte DNA in forest soils”. In: <em>Journal of
Eukaryotic Microbiology</em> 68.1. DOI:
<a href="https://doi.org/10.1111%2Fjeu.12833">10.1111/jeu.12833</a>.</p>
<p><a name=bib-Greco2021113></a><a href="#cite-Greco2021113">Greco, M., R. Morard, and M.
Kucera</a> (2021). “Single-cell metabarcoding reveals
biotic interactions of the Arctic calcifier Neogloboquadrina pachyderma
with the eukaryotic pelagic community”. In: <em>Journal of Plankton
Research</em> 43.2, pp. 113-125. DOI:
<a href="https://doi.org/10.1093%2Fplankt%2Ffbab015">10.1093/plankt/fbab015</a>.</p>
<p><a name=bib-Guo2021></a><a href="#cite-Guo2021">Guo, S., W. Xiong, X. Hang, et
al.</a> (2021). “Protists as main indicators and
determinants of plant performance”. In: <em>Microbiome</em> 9.1. DOI:
<a href="https://doi.org/10.1186%2Fs40168-021-01025-w">10.1186/s40168-021-01025-w</a>.</p>
<p><a name=bib-Hamard20213424></a><a href="#cite-Hamard20213424">Hamard, S., R. Céréghino, M. Barret, et
al.</a> (2021). “Contribution of microbial
photosynthesis to peatland carbon uptake along a latitudinal gradient”.
In: <em>Journal of Ecology</em> 109.9, pp. 3424-3441. DOI:
<a href="https://doi.org/10.1111%2F1365-2745.13732">10.1111/1365-2745.13732</a>.</p>
<p><a name=bib-Hamilton2021></a><a href="#cite-Hamilton2021">Hamilton, M., M. Mascioni, E.
Hehenberger, et al.</a> (2021). “Spatiotemporal
variations in antarctic protistan communities highlight phytoplankton
diversity and seasonal dominance by a novel cryptophyte lineage”. In:
<em>mBio</em> 12.6. DOI:
<a href="https://doi.org/10.1128%2FmBio.02973-21">10.1128/mBio.02973-21</a>.</p>
<p><a name=bib-Herrera2021></a><a href="#cite-Herrera2021">Herrera, G., L. Vega, M. Patarroyo, et
al.</a> (2021). “Gut microbiota composition in
health-care facility-and community-onset diarrheic patients with
Clostridioides difficile infection”. In: <em>Scientific Reports</em> 11.1.
DOI:
<a href="https://doi.org/10.1038%2Fs41598-021-90380-7">10.1038/s41598-021-90380-7</a>.</p>
<p><a name=bib-Hii2021></a><a href="#cite-Hii2021">Hii, K., M. Mohd-Din, Z. Luo, et
al.</a> (2021). “Diverse harmful microalgal community
assemblages in the Johor Strait and the environmental effects on its
community dynamics”. In: <em>Harmful Algae</em> 107. DOI:
<a href="https://doi.org/10.1016%2Fj.hal.2021.102077">10.1016/j.hal.2021.102077</a>.</p>
<p><a name=bib-Hilton2021></a><a href="#cite-Hilton2021">Hilton, S., E. Picot, S. Schreiter, et
al.</a> (2021). “Identification of microbial signatures
linked to oilseed rape yield decline at the landscape scale”. In:
<em>Microbiome</em> 9.1. DOI:
<a href="https://doi.org/10.1186%2Fs40168-020-00972-0">10.1186/s40168-020-00972-0</a>.</p>
<p><a name=bib-Hohlfeld2021></a><a href="#cite-Hohlfeld2021">Hohlfeld, M., A. Schoenle, and H.
Arndt</a> (2021). “Horizontal and vertical small-scale
patterns of protist communities at the Atlantic deep-sea floor”. In:
<em>Deep-Sea Research Part I: Oceanographic Research Papers</em> 171. DOI:
<a href="https://doi.org/10.1016%2Fj.dsr.2021.103515">10.1016/j.dsr.2021.103515</a>.</p>
<p><a name=bib-Hu2021></a><a href="#cite-Hu2021">Hu, S., E. Herrera, A. Smith, et
al.</a> (2021). “Protistan grazing impacts microbial
communities and carbon cycling at deep-sea hydrothermal vents”. In:
<em>Proceedings of the National Academy of Sciences of the United States
of America</em> 118.29. DOI:
<a href="https://doi.org/10.1073%2Fpnas.2102674118">10.1073/pnas.2102674118</a>.</p>
<p><a name=bib-Huang20211></a><a href="#cite-Huang20211">Huang, H., Q. Xu, K. Gibson, et
al.</a> (2021). “Molecular characterization of harmful
algal blooms in the Bohai Sea using metabarcoding analysis”. In:
<em>Harmful Algae</em> 106. DOI:
<a href="https://doi.org/10.1016%2Fj.hal.2021.102066">10.1016/j.hal.2021.102066</a>.</p>
<p><a name=bib-Huang2021></a><a href="#cite-Huang2021">Huang, X., J. Wang, K. Dumack, et
al.</a> (2021). “Protists modulate fungal community
assembly in paddy soils across climatic zones at the continental
scale”. In: <em>Soil Biology and Biochemistry</em> 160. DOI:
<a href="https://doi.org/10.1016%2Fj.soilbio.2021.108358">10.1016/j.soilbio.2021.108358</a>.</p>
<p><a name=bib-Iniesto202151></a><a href="#cite-Iniesto202151">Iniesto, M., D. Moreira, G. Reboul, et
al.</a> (2021). “Core microbial communities of
lacustrine microbialites sampled along an alkalinity gradient”. In:
<em>Environmental Microbiology</em> 23.1, pp. 51-68. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.15252">10.1111/1462-2920.15252</a>.</p>
<p><a name=bib-Irion20212509></a><a href="#cite-Irion20212509">Irion, S., U. Christaki, H. Berthelot,
et al.</a> (2021). “Small phytoplankton contribute
greatly to CO2-fixation after the diatom bloom in the Southern Ocean”.
In: <em>ISME Journal</em> 15.9, pp. 2509-2522. DOI:
<a href="https://doi.org/10.1038%2Fs41396-021-00915-z">10.1038/s41396-021-00915-z</a>.</p>
<p><a name=bib-Izaguirre2021941></a><a href="#cite-Izaguirre2021941">Izaguirre, I., F. Unrein, M.
Schiaffino, et al.</a> (2021). “Phylogenetic
diversity and dominant ecological traits of freshwater Antarctic
Chrysophyceae”. In: <em>Polar Biology</em> 44.5, pp. 941-957. DOI:
<a href="https://doi.org/10.1007%2Fs00300-021-02850-3">10.1007/s00300-021-02850-3</a>.</p>
<p><a name=bib-Jamieson2021></a><a href="#cite-Jamieson2021">Jamieson, T., S. Balzano, C. Le Lan, et
al.</a> (2021). “Survival of the fittest: Prokaryotic
communities within a SWRO desalination plant”. In: <em>Desalination</em> 514.
DOI:
<a href="https://doi.org/10.1016%2Fj.desal.2021.115152">10.1016/j.desal.2021.115152</a>.</p>
<p><a name=bib-Jauss2021></a><a href="#cite-Jauss2021">Jauss, R., A. Nowack, S. Walden, et
al.</a> (2021). “To the canopy and beyond: Air dispersal
as a mechanism of ubiquitous protistan pathogen assembly in tree
canopies”. In: <em>European Journal of Protistology</em> 80. DOI:
<a href="https://doi.org/10.1016%2Fj.ejop.2021.125805">10.1016/j.ejop.2021.125805</a>.</p>
<p><a name=bib-Joglar20211559></a><a href="#cite-Joglar20211559">Joglar, V., X. Ãlvarez-Salgado, A.
Gago-Martinez, et al.</a> (2021). “Cobalamin and
microbial plankton dynamics along a coastal to offshore transect in the
Eastern North Atlantic Ocean”. In: <em>Environmental Microbiology</em> 23.3,
pp. 1559-1583. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.15367">10.1111/1462-2920.15367</a>.</p>
<p><a name=bib-Joglar2021></a><a href="#cite-Joglar2021">Joglar, V., B. Pontiller, S.
Mart'inez-Garc'ia, et al.</a> (2021). “Microbial
plankton community structure and function responses to vitamin B12and
B1Amendments in an upwelling system”. In: <em>Applied and Environmental
Microbiology</em> 87.22. DOI:
<a href="https://doi.org/10.1128%2FAEM.01525-21">10.1128/AEM.01525-21</a>.</p>
<p><a name=bib-Kawachi20212395></a><a href="#cite-Kawachi20212395">Kawachi, M., T. Nakayama, M. Kayama,
et al.</a> (2021). “Rappemonads are haptophyte
phytoplankton”. In: <em>Current Biology</em> 31.11, pp. 2395-2403.e4. DOI:
<a href="https://doi.org/10.1016%2Fj.cub.2021.03.012">10.1016/j.cub.2021.03.012</a>.</p>
<p><a name=bib-Kawamoto2021></a><a href="#cite-Kawamoto2021">Kawamoto, Y., H. Kato, Y. Nagata, et
al.</a> (2021). “Microbial communities developing
within bulk sediments under fish carcasses on a tidal flat”. In: <em>PLoS
ONE</em> 16.2 February. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pone.0247220">10.1371/journal.pone.0247220</a>.</p>
<p><a name=bib-Kobari2021></a><a href="#cite-Kobari2021">Kobari, T., Y. Tokumo, I. Sato, et
al.</a> (2021). “Metabarcoding analysis of trophic
sources and linkages in the plankton community of the Kuroshio and
neighboring waters”. In: <em>Scientific Reports</em> 11.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-021-02083-8">10.1038/s41598-021-02083-8</a>.</p>
<p><a name=bib-Lambert20212592></a><a href="#cite-Lambert20212592">Lambert, S., J. Lozano, F. Bouget, et
al.</a> (2021). “Seasonal marine microorganisms
change neighbours under contrasting environmental conditions”. In:
<em>Environmental Microbiology</em> 23.5, pp. 2592-2604. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.15482">10.1111/1462-2920.15482</a>.</p>
<p><a name=bib-Latorre2021></a><a href="#cite-Latorre2021">Latorre, F., I. Deutschmann, A. Labarre,
et al.</a> (2021). “Niche adaptation promoted the
evolutionary diversification of tiny ocean predators”. In: <em>Proceedings
of the National Academy of Sciences of the United States of America</em>
118.25. DOI:
<a href="https://doi.org/10.1073%2Fpnas.2020955118">10.1073/pnas.2020955118</a>.</p>
<p><a name=bib-Lattaud2021></a><a href="#cite-Lattaud2021">Lattaud, J., S. Balzano, M. van der Meer,
et al.</a> (2021). “Sources and seasonality of
long-chain diols in a temperate lake (Lake Geneva)”. In: <em>Organic
Geochemistry</em> 156. DOI:
<a href="https://doi.org/10.1016%2Fj.orggeochem.2021.104223">10.1016/j.orggeochem.2021.104223</a>.</p>
<p><a name=bib-Lee2021></a><a href="#cite-Lee2021">Lee, S., D. Lee, H. Kang, et
al.</a> (2021). “Analysis of in vitro digestion using human
gut microbiota in adult and elderly individuals”. In: <em>Food Chemistry</em>
362. DOI:
<a href="https://doi.org/10.1016%2Fj.foodchem.2021.130228">10.1016/j.foodchem.2021.130228</a>.</p>
<p><a name=bib-Lejzerowicz2021></a><a href="#cite-Lejzerowicz2021">Lejzerowicz, F., A. Gooday, I.
Barrenechea Angeles, et al.</a> (2021). “Eukaryotic
biodiversity and spatial patterns in the clarion-clipperton zone and
other abyssal regions: Insights from sediment DNA and RNA
metabarcoding”. In: <em>Frontiers in Marine Science</em> 8. DOI:
<a href="https://doi.org/10.3389%2Ffmars.2021.671033">10.3389/fmars.2021.671033</a>.</p>
<p><a name=bib-Lentendu20211532></a><a href="#cite-Lentendu20211532">Lentendu, G. and M.
Dunthorn</a> (2021). “Phylogenetic relatedness
drives protist assembly in marine and terrestrial environments”. In:
<em>Global Ecology and Biogeography</em> 30.7, pp. 1532-1544. DOI:
<a href="https://doi.org/10.1111%2Fgeb.13317">10.1111/geb.13317</a>.</p>
<p><a name=bib-Li20213801></a><a href="#cite-Li20213801">Li, F., A. Sun, X. Jiao, et
al.</a> (2021). “Specific protistan consumers and
parasites are responsive to inorganic fertilization in rhizosphere and
bulk soils”. In: <em>Journal of Soils and Sediments</em> 21.12, pp. 3801-3812.
DOI:
<a href="https://doi.org/10.1007%2Fs11368-021-03052-4">10.1007/s11368-021-03052-4</a>.</p>
<p><a name=bib-Li202116434></a><a href="#cite-Li202116434">Li, F., Y. Zhang, F. Altermatt, et
al.</a> (2021). “Consideration of multitrophic
biodiversity and ecosystem functions improves indices on river
ecological status”. In: <em>Environmental Science and Technology</em> 55.24,
pp. 16434-16444. DOI:
<a href="https://doi.org/10.1021%2Facs.est.1c05899">10.1021/acs.est.1c05899</a>.</p>
<p><a name=bib-Li20211></a><a href="#cite-Li20211">Li, R., C. Hu, J. Wang, et al.</a>
(2021). “Biogeographical distribution and community assembly of active
protistan assemblages along an estuary to a basin transect of the
northern south China sea”. In: <em>Microorganisms</em> 9.2, pp. 1-20. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms9020351">10.3390/microorganisms9020351</a>.</p>
<p><a name=bib-Liu20211></a><a href="#cite-Liu20211">Liu, S., M. Zhang, Y. Zhao, et
al.</a> (2021). “Biodiversity and spatial-temporal
dynamics of margalefidinium species in jiaozhou bay, China”. In:
<em>International Journal of Environmental Research and Public Health</em>
18.21. DOI:
<a href="https://doi.org/10.3390%2Fijerph182111637">10.3390/ijerph182111637</a>.</p>
<p><a name=bib-Liu2021></a><a href="#cite-Liu2021">Liu, X., N. Xie, M. Bai, et
al.</a> (2021). “Composition change and decreased diversity
of microbial eukaryotes in the coastal upwelling waters of South China
Sea”. In: <em>Science of the Total Environment</em> 795. DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2021.148892">10.1016/j.scitotenv.2021.148892</a>.</p>
<p><a name=bib-Ma2021></a><a href="#cite-Ma2021">Ma, Y., Y. Pan, Q. Liu, et al.</a>
(2021). “Co-occurrence patterns and assembly processes of
microeukaryotic communities in a semi-enclosed aquaculture bay”. In:
<em>Continental Shelf Research</em> 228. DOI:
<a href="https://doi.org/10.1016%2Fj.csr.2021.104550">10.1016/j.csr.2021.104550</a>.</p>
<p><a name=bib-Majaneva2021></a><a href="#cite-Majaneva2021">Majaneva, M., J. Rintala, and J.
Blomster</a> (2021). “Taxonomically and functionally
distinct ciliophora assemblages inhabiting baltic sea ice”. In:
<em>Microbial Ecology</em>. DOI:
<a href="https://doi.org/10.1007%2Fs00248-021-01915-4">10.1007/s00248-021-01915-4</a>.</p>
<p><a name=bib-Martini2021965></a><a href="#cite-Martini2021965">Martini, S., F. Larras, A. Boyé, et
al.</a> (2021). “Functional trait-based approaches
as a common framework for aquatic ecologists”. In: <em>Limnology and
Oceanography</em> 66.3, pp. 965-994. DOI:
<a href="https://doi.org/10.1002%2Flno.11655">10.1002/lno.11655</a>.</p>
<p><a name=bib-McLennan20211></a><a href="#cite-McLennan20211">McLennan, K., R. Ruvindy, M. Ostrowski,
et al.</a> (2021). “Assessing the use of molecular
barcoding and QPCR for investigating the ecology of prorocentrum
minimum (Dinophyceae), a harmful algal species”. In: <em>Microorganisms</em>
9.3, pp. 1-20. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms9030510">10.3390/microorganisms9030510</a>.</p>
<p><a name=bib-McNichol2021></a><a href="#cite-McNichol2021">McNichol, J., P. Berube, S. Biller, et
al.</a> (2021). “Evaluating and improving small
subunit rrna pcr primer coverage for bacteria, archaea, and eukaryotes
using metagenomes from global ocean surveys”. In: <em>mSystems</em> 6.3. DOI:
<a href="https://doi.org/10.1128%2FmSystems.00565-21">10.1128/mSystems.00565-21</a>.</p>
<p><a name=bib-Meng20211></a><a href="#cite-Meng20211">Meng, L., H. Endo, R. Blanc-Mathieu, et
al.</a> (2021). “Quantitative assessment of
nucleocytoplasmic large DNA virus and host interactions predicted by
co-occurrence analyses”. In: <em>mSphere</em> 6.2, pp. 1-18. DOI:
<a href="https://doi.org/10.1128%2FmSphere.01298-20">10.1128/mSphere.01298-20</a>.</p>
<p><a name=bib-Meredith202183></a><a href="#cite-Meredith202183">Meredith, C., J. Hoffman, A. Trebitz,
et al.</a> (2021). “Evaluating the performance of
DNA metabarcoding for assessment of zooplankton communities in Western
Lake Superior using multiple markers”. In: <em>Metabarcoding and
Metagenomics</em> 5, pp. 83-97. DOI:
<a href="https://doi.org/10.3897%2FMBMG.5.64735">10.3897/MBMG.5.64735</a>.</p>
<p><a name=bib-Michealsamy2021109></a><a href="#cite-Michealsamy2021109">Michealsamy, A., L. Thangamani, G.
Manivel, et al.</a> (2021). “Current research
and applications of meta-omics stratagems in bioremediation: A
bird’s-eye view”. In: <em>Journal of Applied Biotechnology Reports</em> 8.2,
pp. 109-115. DOI:
<a href="https://doi.org/10.30491%2Fjabr.2020.237662.1248">10.30491/jabr.2020.237662.1248</a>.</p>
<p><a name=bib-Mikryukov2021188></a><a href="#cite-Mikryukov2021188">Mikryukov, V., O. Dulya, G.
Likhodeevskii, et al.</a> (2021). “Analysis of
ecological networks in multicomponent communities of microorganisms:
Possibilities, limitations, and potential errors”. In: <em>Russian Journal
of Ecology</em> 52.3, pp. 188-200. DOI:
<a href="https://doi.org/10.1134%2FS1067413621030085">10.1134/S1067413621030085</a>.</p>
<p><a name=bib-Mo2021></a><a href="#cite-Mo2021">Mo, Y., F. Peng, X. Gao, et al.</a>
(2021). “Low shifts in salinity determined assembly processes and
network stability of microeukaryotic plankton communities in a
subtropical urban reservoir”. In: <em>Microbiome</em> 9.1. DOI:
<a href="https://doi.org/10.1186%2Fs40168-021-01079-w">10.1186/s40168-021-01079-w</a>.</p>
<p><a name=bib-Mthethwa2021></a><a href="#cite-Mthethwa2021">Mthethwa, N., I. Amoah, P. Reddy, et
al.</a> (2021). “A review on application of
next-generation sequencing methods for profiling of protozoan parasites
in water: Current methodologies, challenges, and perspectives”. In:
<em>Journal of Microbiological Methods</em> 187. DOI:
<a href="https://doi.org/10.1016%2Fj.mimet.2021.106269">10.1016/j.mimet.2021.106269</a>.</p>
<p><a name=bib-Neupane2021></a><a href="#cite-Neupane2021">Neupane, S., C. Saski, and D.
Nayduch</a> (2021). “House fly larval grazing alters
dairy cattle manure microbial communities”. In: <em>BMC Microbiology</em>
21.1. DOI:
<a href="https://doi.org/10.1186%2Fs12866-021-02418-5">10.1186/s12866-021-02418-5</a>.</p>
<p><a name=bib-Nguyen2021></a><a href="#cite-Nguyen2021">Nguyen, B., Q. Chen, Z. Yan, et
al.</a> (2021). “Distinct factors drive the diversity
and composition of protistan consumers and phototrophs in natural soil
ecosystems”. In: <em>Soil Biology and Biochemistry</em> 160. DOI:
<a href="https://doi.org/10.1016%2Fj.soilbio.2021.108317">10.1016/j.soilbio.2021.108317</a>.</p>
<p><a name=bib-Novotny2021></a><a href="#cite-Novotny2021">Novotny, A., S. Zamora-Terol, and M.
Winder</a> (2021). “DNA metabarcoding reveals trophic
niche diversity of micro and mesozooplankton species”. In: <em>Proceedings
of the Royal Society B: Biological Sciences</em> 288.1953. DOI:
<a href="https://doi.org/10.1098%2Frspb.2021.0908">10.1098/rspb.2021.0908</a>.</p>
<p><a name=bib-Nowinski2021524></a><a href="#cite-Nowinski2021524">Nowinski, B. and M.
Moran</a> (2021). “Niche dimensions of a marine
bacterium are identified using invasion studies in coastal seawater”.
In: <em>Nature Microbiology</em> 6.4, pp. 524-532. DOI:
<a href="https://doi.org/10.1038%2Fs41564-020-00851-2">10.1038/s41564-020-00851-2</a>.</p>
<p><a name=bib-Obertegger2021640></a><a href="#cite-Obertegger2021640">Obertegger, U., M. Pindo, and G.
Flaim</a> (2021). “Do inferences about freshwater
phytoplankton communities change when based on microscopy or
high-throughput sequencing data?” In: <em>Freshwater Biology</em> 66.4, pp.
640-655. DOI: <a href="https://doi.org/10.1111%2Ffwb.13667">10.1111/fwb.13667</a>.</p>
<p><a name=bib-Ollison2021></a><a href="#cite-Ollison2021">Ollison, G., S. Hu, L. Mesrop, et
al.</a> (2021). “Come rain or shine: Depth not season
shapes the active protistan community at station ALOHA in the North
Pacific Subtropical Gyre”. In: <em>Deep-Sea Research Part I: Oceanographic
Research Papers</em> 170. DOI:
<a href="https://doi.org/10.1016%2Fj.dsr.2021.103494">10.1016/j.dsr.2021.103494</a>.</p>
<p><a name=bib-Pappalardo20213397></a><a href="#cite-Pappalardo20213397">Pappalardo, P., A. Collins, K.
Pagenkopp Lohan, et al.</a> (2021). “The role of
taxonomic expertise in interpretation of metabarcoding studies”. In:
<em>ICES Journal of Marine Science</em> 78.9, pp. 3397-3410. DOI:
<a href="https://doi.org/10.1093%2Ficesjms%2Ffsab082">10.1093/icesjms/fsab082</a>.</p>
<p><a name=bib-Pauli2021></a><a href="#cite-Pauli2021">Pauli, N., K. Metfies, E. Pakhomov, et
al.</a> (2021). “Selective feeding in Southern Ocean key
grazers—diet composition of krill and salps”. In: <em>Communications
Biology</em> 4.1. DOI:
<a href="https://doi.org/10.1038%2Fs42003-021-02581-5">10.1038/s42003-021-02581-5</a>.</p>
<p><a name=bib-Pearman2021173></a><a href="#cite-Pearman2021173">Pearman, J., U. von Ammon, O. Laroche,
et al.</a> (2021). “Metabarcoding as a tool to
enhance marine surveillance of nonindigenous species in tropical
harbors: A case study in Tahiti”. In: <em>Environmental DNA</em> 3.1, pp.
173-189. DOI: <a href="https://doi.org/10.1002%2Fedn3.154">10.1002/edn3.154</a>.</p>
<p><a name=bib-Picot2021></a><a href="#cite-Picot2021">Picot, E., C. Hale, S. Hilton, et
al.</a> (2021). “Contrasting responses of rhizosphere
bacterial, fungal, protist, and nematode communities to nitrogen
fertilization and crop genotype in field grown oilseed rape (brassica
napus)”. In: <em>Frontiers in Sustainable Food Systems</em> 5. DOI:
<a href="https://doi.org/10.3389%2Ffsufs.2021.613269">10.3389/fsufs.2021.613269</a>.</p>
<p><a name=bib-Ramond2021></a><a href="#cite-Ramond2021">Ramond, P., R. Siano, S. Schmitt, et
al.</a> (2021). “Phytoplankton taxonomic and functional
diversity patterns across a coastal tidal front”. In: <em>Scientific
Reports</em> 11.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-021-82071-0">10.1038/s41598-021-82071-0</a>.</p>
<p><a name=bib-Ricci2021275></a><a href="#cite-Ricci2021275">Ricci, F., A. Fordyce, W. Leggat, et
al.</a> (2021). “Multiple techniques point to oxygenic
phototrophs dominating the Isopora palifera skeletal microbiome”. In:
<em>Coral reefs (Online)</em> 40.2, pp. 275-282. DOI:
<a href="https://doi.org/10.1007%2Fs00338-021-02068-z">10.1007/s00338-021-02068-z</a>.</p>
<p><a name=bib-Rimet202117></a><a href="#cite-Rimet202117">Rimet, F., E. Aylagas, A. Borja, et
al.</a> (2021). “Metadata standards and practical
guidelines for specimen and DNA curation when building barcode
reference libraries for aquatic life”. In: <em>Metabarcoding and
Metagenomics</em> 5, pp. 17-33. DOI:
<a href="https://doi.org/10.3897%2Fmbmg.5.58056">10.3897/mbmg.5.58056</a>.</p>
<p><a name=bib-Roshan20211></a><a href="#cite-Roshan20211">Roshan, S., K. Dumack, M. Bonkowski, et
al.</a> (2021). “Taxonomic and functional diversity of
heterotrophic protists (Cercozoa and endomyxa) from biological soil
crusts”. In: <em>Microorganisms</em> 9.2, pp. 1-14. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms9020205">10.3390/microorganisms9020205</a>.</p>
<p><a name=bib-Sakamoto2021></a><a href="#cite-Sakamoto2021">Sakamoto, T. and J.
Ortega</a> (2021). “Taxallnomy: an extension of NCBI
Taxonomy that produces a hierarchically complete taxonomic tree”. In:
<em>BMC Bioinformatics</em> 22.1. DOI:
<a href="https://doi.org/10.1186%2Fs12859-021-04304-3">10.1186/s12859-021-04304-3</a>.</p>
<p><a name=bib-Schiwitza2021></a><a href="#cite-Schiwitza2021">Schiwitza, S. and F.
Nitsche</a> (2021). “A needle in the haystack
mapping sequences to morphology exemplified by the loricate
choanoflagellate enibas thessalia sp. nov. (acanthoecida,
acanthoecidae)”. In: <em>Protist</em> 172.1. DOI:
<a href="https://doi.org/10.1016%2Fj.protis.2020.125782">10.1016/j.protis.2020.125782</a>.</p>
<p><a name=bib-Schoenle2021></a><a href="#cite-Schoenle2021">Schoenle, A., M. Hohlfeld, K. Hermanns,
et al.</a> (2021). “High and specific diversity of
protists in the deep-sea basins dominated by diplonemids,
kinetoplastids, ciliates and foraminiferans”. In: <em>Communications
Biology</em> 4.1. DOI:
<a href="https://doi.org/10.1038%2Fs42003-021-02012-5">10.1038/s42003-021-02012-5</a>.</p>
<p><a name=bib-Shemi20211357></a><a href="#cite-Shemi20211357">Shemi, A., U. Alcolombri, D. Schatz, et
al.</a> (2021). “Dimethyl sulfide mediates microbial
predatorprey interactions between zooplankton and algae in
the ocean”. In: <em>Nature Microbiology</em> 6.11, pp. 1357-1366. DOI:
<a href="https://doi.org/10.1038%2Fs41564-021-00971-3">10.1038/s41564-021-00971-3</a>.</p>
<p><a name=bib-Shi20211718></a><a href="#cite-Shi20211718">Shi, J., Y. Lei, H. Li, et
al.</a> (2021). “NGS-metabarcoding revealing novel
foraminiferal diversity in the Western Pacific Magellan Seamount
sediments”. In: <em>Journal of Oceanology and Limnology</em> 39.5, pp.
1718-1729. DOI:
<a href="https://doi.org/10.1007%2Fs00343-021-0427-0">10.1007/s00343-021-0427-0</a>.</p>
<p><a name=bib-Shore20211></a><a href="#cite-Shore20211">Shore, A., R. Day, J. Stewart, et
al.</a> (2021). “Dichotomy between regulation of coral
bacterial communities and calcification physiology under ocean
acidification conditions”. In: <em>Applied and Environmental Microbiology</em>
87.6, pp. 1-18. DOI:
<a href="https://doi.org/10.1128%2FAEM.02189-20">10.1128/AEM.02189-20</a>.</p>
<p><a name=bib-Siano20212682></a><a href="#cite-Siano20212682">Siano, R., M. Lassudrie, P. Cuzin, et
al.</a> (2021). “Sediment archives reveal
irreversible shifts in plankton communities after World War II and
agricultural pollution”. In: <em>Current Biology</em> 31.12, pp. 2682-2689.e7.
DOI:
<a href="https://doi.org/10.1016%2Fj.cub.2021.03.079">10.1016/j.cub.2021.03.079</a>.</p>
<p><a name=bib-Singer2021></a><a href="#cite-Singer2021">Singer, D., C. Seppey, G. Lentendu, et
al.</a> (2021). “Protist taxonomic and functional
diversity in soil, freshwater and marine ecosystems”. In: <em>Environment
International</em> 146. DOI:
<a href="https://doi.org/10.1016%2Fj.envint.2020.106262">10.1016/j.envint.2020.106262</a>.</p>
<p><a name=bib-Snyder20213356></a><a href="#cite-Snyder20213356">Snyder, R., J. Moss, L. Santoferrara,
et al.</a> (2021). “Ciliate microzooplankton from
the northeastern gulf of mexico”. In: <em>ICES Journal of Marine Science</em>
78.9, pp. 3356-3371. DOI:
<a href="https://doi.org/10.1093%2Ficesjms%2Ffsab002">10.1093/icesjms/fsab002</a>.</p>
<p><a name=bib-Soler2021></a><a href="#cite-Soler2021">Soler, P., L. Moreno-Mesonero, A. Zornoza,
et al.</a> (2021). “Characterization of eukaryotic
microbiome and associated bacteria communities in a drinking water
treatment plant”. In: <em>Science of the Total Environment</em> 797. DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2021.149070">10.1016/j.scitotenv.2021.149070</a>.</p>
<p><a name=bib-Steven20211></a><a href="#cite-Steven20211">Steven, B., J. LaReau, S. Taerum, et
al.</a> (2021). “What’s under the christmas tree? A
soil sulfur amendment lowers soil pH and alters fir tree rhizosphere
bacterial and eukaryotic communities, their interactions, and
functional traits”. In: <em>Microbiology Spectrum</em> 9.1, pp. 1-14. DOI:
<a href="https://doi.org/10.1128%2FSpectrum.00166-21">10.1128/Spectrum.00166-21</a>.</p>
<p><a name=bib-Sun20212169></a><a href="#cite-Sun20212169">Sun, A., X. Jiao, Q. Chen, et
al.</a> (2021). “Fertilization alters protistan
consumers and parasites in crop-associated microbiomes”. In:
<em>Environmental Microbiology</em> 23.4, pp. 2169-2183. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.15385">10.1111/1462-2920.15385</a>.</p>
<p><a name=bib-Sun2021></a><a href="#cite-Sun2021">Sun, P., X. Huang, Y. Wang, et
al.</a> (2021). “Protistan-bacterial microbiota exhibit
stronger species sorting and greater network connectivity offshore than
nearshore across a coast-to-basin continuum”. In: <em>mSystems</em> 6.5. DOI:
<a href="https://doi.org/10.1128%2FmSystems.00100-21">10.1128/mSystems.00100-21</a>.</p>
<p><a name=bib-Sun20211></a><a href="#cite-Sun20211">Sun, P., S. Zhang, Y. Wang, et
al.</a> (2021). “Biogeographic role of the kuroshio
current intrusion in the microzooplankton community in the boundary
zone of the northern south china sea”. In: <em>Microorganisms</em> 9.5. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms9051104">10.3390/microorganisms9051104</a>.</p>
<p><a name=bib-Thioye2021361></a><a href="#cite-Thioye2021361">Thioye, B., H. Sanguin, A. Kane, et
al.</a> (2021). “Mycorrhizal inoculation increases
fruit production without disturbance of native arbuscular mycorrhizal
community in jujube tree orchards (Senegal)”. In: <em>Symbiosis
(Philadelphia, Pa.)</em> 83.3, pp. 361-372. DOI:
<a href="https://doi.org/10.1007%2Fs13199-021-00757-5">10.1007/s13199-021-00757-5</a>.</p>
<p><a name=bib-Thompson20211467></a><a href="#cite-Thompson20211467">Thompson, A.</a>
(2021). “Phagotrophic protists (protozoa) in Antarctic terrestrial
ecosystems: diversity, distribution, ecology, and best research
practices”. In: <em>Polar Biology</em> 44.8, pp. 1467-1484. DOI:
<a href="https://doi.org/10.1007%2Fs00300-021-02896-3">10.1007/s00300-021-02896-3</a>.</p>
<p><a name=bib-Thompson2021></a><a href="#cite-Thompson2021">Thompson, A., A. Roth-monzón, Z.
Aanderud, et al.</a> (2021). “Phagotrophic protists
and their associates: Evidence for preferential grazing in an
abiotically driven soil ecosystem”. In: <em>Microorganisms</em> 9.8. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms9081555">10.3390/microorganisms9081555</a>.</p>
<p><a name=bib-Trefault2021></a><a href="#cite-Trefault2021">Trefault, N., R. De la Iglesia, M.
Moreno-Pino, et al.</a> (2021). “Annual phytoplankton
dynamics in coastal waters from fildes bay, western antarctic
peninsula”. In: <em>Scientific Reports</em> 11.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-020-80568-8">10.1038/s41598-020-80568-8</a>.</p>
<p><a name=bib-Vega2021></a><a href="#cite-Vega2021">Vega, L., G. Herrera, M. Munoz, et
al.</a> (2021). “Gut microbiota profiles in diarrheic
patients with co-occurrence of Clostridioides difficile and
Blastocystis”. In: <em>PLoS ONE</em> 16.3 March. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pone.0248185">10.1371/journal.pone.0248185</a>.</p>
<p><a name=bib-Vega2021847></a><a href="#cite-Vega2021847">Vega, L., J. Jaimes, D. Morales, et
al.</a> (2021). “Microbial communities’
characterization in urban recreational surface waters using next
generation sequencing”. In: <em>Microbial Ecology</em> 81.4, pp. 847-863. DOI:
<a href="https://doi.org/10.1007%2Fs00248-020-01649-9">10.1007/s00248-020-01649-9</a>.</p>
<p><a name=bib-Vernette20211347></a><a href="#cite-Vernette20211347">Vernette, C., N. Henry, J. Lecubin,
et al.</a> (2021). “The Ocean barcode atlas: A web
service to explore the biodiversity and biogeography of marine
organisms”. In: <em>Molecular Ecology Resources</em> 21.4, pp. 1347-1358. DOI:
<a href="https://doi.org/10.1111%2F1755-0998.13322">10.1111/1755-0998.13322</a>.</p>
<p><a name=bib-Vilne2021></a><a href="#cite-Vilne2021">Vilne, B., L. Granti-Ievi, J.
, et al.</a> (2021). “Investigating bacterial
and free-living protozoa diversity in biofilms of hot water pipes of
apartment buildings in the city of riga (latvia)”. In: <em>Frontiers in
Water</em> 3. DOI:
<a href="https://doi.org/10.3389%2Ffrwa.2021.799840">10.3389/frwa.2021.799840</a>.</p>
<p><a name=bib-Walden2021></a><a href="#cite-Walden2021">Walden, S., R. Jauss, K. Feng, et
al.</a> (2021). “On the phenology of protists: Recurrent
patterns reveal seasonal variation of protistan (Rhizaria: Cercozoa and
Endomyxa) communities in tree canopies”. In: <em>FEMS Microbiology
Ecology</em> 97.7. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiab081">10.1093/femsec/fiab081</a>.</p>
<p><a name=bib-Walker2021195></a><a href="#cite-Walker2021195">Walker, L., T. Hoppe, and M.
Silliker</a> (2021). <em>Molecular techniques and
current research approaches</em> , pp. 195-229. DOI:
<a href="https://doi.org/10.1016%2FB978-0-12-824281-0.00006-3">10.1016/B978-0-12-824281-0.00006-3</a>.</p>
<p><a name=bib-Wang20211></a><a href="#cite-Wang20211">Wang, C., A. Masoudi, M. Wang, et
al.</a> (2021). “Land-use types shape soil microbial
compositions under rapid urbanization in the Xiong’an New Area, China”.
In: <em>Science of the Total Environment</em> 777. DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2021.145976">10.1016/j.scitotenv.2021.145976</a>.</p>
<p><a name=bib-Wang202114732></a><a href="#cite-Wang202114732">Wang, C., W. Zhu, P. Strong, et
al.</a> (2021). “Disentangling the effects of
physicochemical, genetic, and microbial properties on phase-driven
resistome dynamics during multiple manure composting processes”. In:
<em>Environmental Science and Technology</em> 55.21, pp. 14732-14745. DOI:
<a href="https://doi.org/10.1021%2Facs.est.1c03933">10.1021/acs.est.1c03933</a>.</p>
<p><a name=bib-Wang2021></a><a href="#cite-Wang2021">Wang, K., Y. Huang, M. Majaneva, et
al.</a> (2021). “Group 2i Isochrysidales produce
characteristic alkenones reflecting sea ice distribution”. In: <em>Nature
Communications</em> 12.1. DOI:
<a href="https://doi.org/10.1038%2Fs41467-020-20187-z">10.1038/s41467-020-20187-z</a>.</p>
<p><a name=bib-Wilms2021></a><a href="#cite-Wilms2021">Wilms, S.</a> (2021). “A
beginner’s guide on integrating *Omics approaches to study marine
microbial communities: Details and discussions from sample collection
to bioinformatics analysis”. In: <em>Frontiers in Marine Science</em> 8. DOI:
<a href="https://doi.org/10.3389%2Ffmars.2021.692538">10.3389/fmars.2021.692538</a>.</p>
<p><a name=bib-Xiong2021></a><a href="#cite-Xiong2021">Xiong, W., A. Jousset, R. Li, et
al.</a> (2021). “A global overview of the trophic
structure within microbiomes across ecosystems”. In: <em>Environment
International</em> 151. DOI:
<a href="https://doi.org/10.1016%2Fj.envint.2021.106438">10.1016/j.envint.2021.106438</a>.</p>
<p><a name=bib-Xu2021></a><a href="#cite-Xu2021">Xu, Q., C. Wang, K. Xu, et al.</a>
(2021). “Metabarcoding analysis of harmful algal bloom species in the
western pacific seamount regions”. In: <em>International Journal of
Environmental Research and Public Health</em> 18.21. DOI:
<a href="https://doi.org/10.3390%2Fijerph182111470">10.3390/ijerph182111470</a>.</p>
<p><a name=bib-Yang2021></a><a href="#cite-Yang2021">Yang, J., J. Lv, Q. Liu, et
al.</a> (2021). “Seasonal and spatial patterns of
eukaryotic phytoplankton communities in an urban river based on marker
gene”. In: <em>Scientific Reports</em> 11.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-021-02183-5">10.1038/s41598-021-02183-5</a>.</p>
<p><a name=bib-Yap2021></a><a href="#cite-Yap2021">Yap, W., A. Switzer, C. Gouramanis, et
al.</a> (2021). “Environmental DNA signatures distinguish
between tsunami and storm deposition in overwash sand”. In:
<em>Communications Earth and Environment</em> 2.1. DOI:
<a href="https://doi.org/10.1038%2Fs43247-021-00199-3">10.1038/s43247-021-00199-3</a>.</p>
<p><a name=bib-Yeh20213240></a><a href="#cite-Yeh20213240">Yeh, Y., J. McNichol, D. Needham, et
al.</a> (2021). “Comprehensive single-PCR 16S and 18S
rRNA community analysis validated with mock communities, and estimation
of sequencing bias against 18S”. In: <em>Environmental Microbiology</em> 23.6,
pp. 3240-3250. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.15553">10.1111/1462-2920.15553</a>.</p>
<p><a name=bib-Zamora-Terol2021125></a><a href="#cite-Zamora-Terol2021125">Zamora-Terol, S., A. Novotny, and
M. Winder</a> (2021). “Molecular evidence of
host-parasite interactions between zooplankton and Syndiniales”. In:
<em>Aquatic Ecology</em> 55.1, pp. 125-134. DOI:
<a href="https://doi.org/10.1007%2Fs10452-020-09816-3">10.1007/s10452-020-09816-3</a>.</p>
<p><a name=bib-Zhang20211></a><a href="#cite-Zhang20211">Zhang, J., P. Xing, M. Niu, et
al.</a> (2021). “Taxonomic compositions and
co-occurrence relationships of protists in bulk soil and rhizosphere of
soybean fields in different regions of china”. In: <em>Frontiers in
Microbiology</em> 12. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2021.738129">10.3389/fmicb.2021.738129</a>.</p>
<p><a name=bib-Zhang2021796></a><a href="#cite-Zhang2021796">Zhang, L., S. Xu, Z. Zhao, et
al.</a> (2021). “Precision of eDNA metabarcoding
technology for biodiversity monitoring of eukaryotic phytoplankton in
lakes
[çŽ¯å¢ƒDNAå®æ¡å½¢ç ç›’æµ‹æ¹–æ³ŠçœŸæ ¸æµ®æ¸¸æ¤ç‰©çš„ç²¾å‡†æ€§]”. In:
<em>Huanjing Kexue/Environmental Science</em> 42.2, pp. 796-807. DOI:
<a href="https://doi.org/10.13227%2Fj.hjkx.202007236">10.13227/j.hjkx.202007236</a>.</p>
<p><a name=bib-Zhang20212></a><a href="#cite-Zhang20212">Zhang, S., X. Xia, Y. Ke, et
al.</a> (2021). “Population dynamics and interactions of
Noctiluca scintillans and Mesodinium rubrum during their successive
blooms in a subtropical coastal water”. In: <em>Science of the Total
Environment</em> 755. DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2020.142349">10.1016/j.scitotenv.2020.142349</a>.</p>
<p><a name=bib-Zhang2021></a><a href="#cite-Zhang2021">Zhang, Y., N. Huang, M. Wang, et
al.</a> (2021). “Microbial eukaryotes associated with
sediments in deep-sea cold seeps”. In: <em>Frontiers in Microbiology</em> 12.
DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2021.782004">10.3389/fmicb.2021.782004</a>.</p>
<p><a name=bib-Zhao20211></a><a href="#cite-Zhao20211">Zhao, F., S. Filker, C. Wang, et
al.</a> (2021). “Bathymetric gradient shapes the
community composition rather than the species richness of deep-sea
benthic ciliates”. In: <em>Science of the Total Environment</em> 755. DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2020.142623">10.1016/j.scitotenv.2020.142623</a>.</p>
<p><a name=bib-Zhao2021></a><a href="#cite-Zhao2021">Zhao, F., Y. Wang, S. Zheng, et
al.</a> (2021). “Patterns and drivers of microeukaryotic
distribution along the North Equatorial Current from the Central
Pacific Ocean to the South China Sea”. In: <em>Marine Pollution Bulletin</em>
165. DOI:
<a href="https://doi.org/10.1016%2Fj.marpolbul.2021.112091">10.1016/j.marpolbul.2021.112091</a>.</p>
<p><a name=bib-Zheng2021></a><a href="#cite-Zheng2021">Zheng, X., K. Zhang, T. Yang, et
al.</a> (2021). “Sediment resuspension drives protist
metacommunity structure and assembly in grass carp (Ctenopharyngodon
idella) aquaculture ponds”. In: <em>Science of the Total Environment</em> 764.
DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2020.142840">10.1016/j.scitotenv.2020.142840</a>.</p>
<p><a name=bib-Zhou20216331></a><a href="#cite-Zhou20216331">Zhou, Y., B. Sun, B. Xie, et
al.</a> (2021). “Warming reshaped the microbial
hierarchical interactions”. In: <em>Global Change Biology</em> 27.24, pp.
6331-6347. DOI:
<a href="https://doi.org/10.1111%2Fgcb.15891">10.1111/gcb.15891</a>.</p>
<p><a name=bib-Zou2021></a><a href="#cite-Zou2021">Zou, K., R. Wang, S. Xu, et
al.</a> (2021). “Changes in protist communities in
drainages across the Pearl River Delta under anthropogenic influence”.
In: <em>Water Research</em> 200. DOI:
<a href="https://doi.org/10.1016%2Fj.watres.2021.117294">10.1016/j.watres.2021.117294</a>.</p>
</div>
<div id="section-2" class="section level2">
<h2>2020</h2>
<p><a name=bib-AbdalaAsbun2020></a><a href="#cite-AbdalaAsbun2020">Abdala Asbun, A., M. Besseling, S.
Balzano, et al.</a> (2020). “Cascabel: A scalable
and versatile amplicon sequence data analysis pipeline delivering
reproducible and documented results”. In: <em>Frontiers in Genetics</em> 11.
DOI:
<a href="https://doi.org/10.3389%2Ffgene.2020.489357">10.3389/fgene.2020.489357</a>.</p>
<p><a name=bib-Allen202019></a><a href="#cite-Allen202019">Allen, R., T. Summerfield, K. Currie, et
al.</a> (2020). “Distinct processes structure
bacterioplankton and protist communities across an oceanic front”. In:
<em>Aquatic Microbial Ecology</em> 85, pp. 19-34. DOI:
<a href="https://doi.org/10.3354%2FAME01938">10.3354/AME01938</a>.</p>
<p><a name=bib-Alric2020468></a><a href="#cite-Alric2020468">Alric, B., C. ter Braak, Y. Desdevises,
et al.</a> (2020). “Investigating microbial
associations from sequencing survey data with co-correspondence
analysis”. In: <em>Molecular Ecology Resources</em> 20.2, pp. 468-480. DOI:
<a href="https://doi.org/10.1111%2F1755-0998.13126">10.1111/1755-0998.13126</a>.</p>
<p><a name=bib-vonAmmon2020></a><a href="#cite-vonAmmon2020">Ammon, U. von, A. Jeffs, A. Zaiko, et
al.</a> (2020). “A portable cruising speed net:
Expanding global collection of sea surface plankton data”. In:
<em>Frontiers in Marine Science</em> 7. DOI:
<a href="https://doi.org/10.3389%2Ffmars.2020.615458">10.3389/fmars.2020.615458</a>.</p>
<p><a name=bib-Anderson2020></a><a href="#cite-Anderson2020">Anderson, S. and E.
Harvey</a> (2020). “Temporal variability and
ecological interactions of parasitic marine syndiniales in coastal
protist communities”. In: <em>mSphere</em> 5.3. DOI:
<a href="https://doi.org/10.1128%2FmSphere.00209-20">10.1128/mSphere.00209-20</a>.</p>
<p><a name=bib-Annenkova2020></a><a href="#cite-Annenkova2020">Annenkova, N., C. Giner, and R.
Logares</a> (2020). “Tracing the origin of planktonic
protists in an ancient lake”. In: <em>Microorganisms</em> 8.4. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms8040543">10.3390/microorganisms8040543</a>.</p>
<p><a name=bib-Armbrecht2020116></a><a href="#cite-Armbrecht2020116">Armbrecht,
L.</a> (2020). “The potential of sedimentary
ancient DNA to reconstruct past ocean ecosystems”. In: <em>Oceanography</em>
33.2, pp. 116-123. DOI:
<a href="https://doi.org/10.5670%2Foceanog.2020.211">10.5670/oceanog.2020.211</a>.</p>
<p><a name=bib-Arsenieff20201966></a><a href="#cite-Arsenieff20201966">Arsenieff, L., F. Le Gall, F.
Rigaut-Jalabert, et al.</a> (2020). “Diversity
and dynamics of relevant nanoplanktonic diatoms in the Western English
Channel”. In: <em>ISME Journal</em> 14.8, pp. 1966-1981. DOI:
<a href="https://doi.org/10.1038%2Fs41396-020-0659-6">10.1038/s41396-020-0659-6</a>.</p>
<p><a name=bib-Atienza2020></a><a href="#cite-Atienza2020">Atienza, S., M. Guardiola, K. Præbel, et
al.</a> (2020). “DNA metabarcoding of deep-sea sediment
communities using COI: Community assessment, spatio-temporal patterns
and comparison with 18S rDNA”. In: <em>Diversity</em> 12.4. DOI:
<a href="https://doi.org/10.3390%2FD12040123">10.3390/D12040123</a>.</p>
<p><a name=bib-BarrenecheaAngeles2020></a><a href="#cite-BarrenecheaAngeles2020">Barrenechea Angeles, I., F.
Lejzerowicz, T. Cordier, et al.</a> (2020).
“Planktonic foraminifera eDNA signature deposited on the seafloor
remains preserved after burial in marine sediments”. In: <em>Scientific
Reports</em> 10.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-020-77179-8">10.1038/s41598-020-77179-8</a>.</p>
<p><a name=bib-Bastida2020752></a><a href="#cite-Bastida2020752">Bastida, F., D. Eldridge, S. Abades, et
al.</a> (2020). “Climatic vulnerabilities and
ecological preferences of soil invertebrates across biomes”. In:
<em>Molecular Ecology</em> 29.4, pp. 752-761. DOI:
<a href="https://doi.org/10.1111%2Fmec.15299">10.1111/mec.15299</a>.</p>
<p><a name=bib-Beam2020></a><a href="#cite-Beam2020">Beam, J., S. George, N. Record, et
al.</a> (2020). “Mud, microbes, and macrofauna: Seasonal
dynamics of the iron biogeochemical cycle in an intertidal mudflat”.
In: <em>Frontiers in Marine Science</em> 7. DOI:
<a href="https://doi.org/10.3389%2Ffmars.2020.562617">10.3389/fmars.2020.562617</a>.</p>
<p><a name=bib-Bird2020></a><a href="#cite-Bird2020">Bird, C., M. Schweizer, A. Roberts, et
al.</a> (2020). “The genetic diversity, morphology,
biogeography, and taxonomic designations of Ammonia (Foraminifera) in
the Northeast Atlantic”. In: <em>Marine Micropaleontology</em> 155. DOI:
<a href="https://doi.org/10.1016%2Fj.marmicro.2019.02.001">10.1016/j.marmicro.2019.02.001</a>.</p>
<p><a name=bib-Botnen2020></a><a href="#cite-Botnen2020">Botnen, S., E. Thoen, P. Eidesen, et
al.</a> (2020). “Community composition of arctic
root-associated fungi mirrors host plant phylogeny”. In: <em>FEMS
Microbiology Ecology</em> 96.11. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiaa185">10.1093/femsec/fiaa185</a>.</p>
<p><a name=bib-Busseni20201915></a><a href="#cite-Busseni20201915">Busseni, G., L. Caputi, R. Piredda, et
al.</a> (2020). “Large scale patterns of marine
diatom richness: Drivers and trends in a changing ocean”. In: <em>Global
Ecology and Biogeography</em> 29.11, pp. 1915-1928. DOI:
<a href="https://doi.org/10.1111%2Fgeb.13161">10.1111/geb.13161</a>.</p>
<p><a name=bib-Cai2020></a><a href="#cite-Cai2020">Cai, R., E. Kayal, C. Alves-de-Souza, et
al.</a> (2020). “Cryptic species in the parasitic
Amoebophrya species complex revealed by a polyphasic approach”. In:
<em>Scientific Reports</em> 10.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-020-59524-z">10.1038/s41598-020-59524-z</a>.</p>
<p><a name=bib-Canals20202824></a><a href="#cite-Canals20202824">Canals, O., A. Obiol, I. Muhovic, et
al.</a> (2020). “Ciliate diversity and distribution
across horizontal and vertical scales in the open ocean”. In:
<em>Molecular Ecology</em> 29.15, pp. 2824-2839. DOI:
<a href="https://doi.org/10.1111%2Fmec.15528">10.1111/mec.15528</a>.</p>
<p><a name=bib-Carley20208878></a><a href="#cite-Carley20208878">Carley, L., R. Panchagavi, X. Song, et
al.</a> (2020). “Long-term effects of copper
nanopesticides on soil and sediment community diversity in two outdoor
mesocosm experiments”. In: <em>Environmental Science and Technology</em>
54.14, pp. 8878-8889. DOI:
<a href="https://doi.org/10.1021%2Facs.est.0c00510">10.1021/acs.est.0c00510</a>.</p>
<p><a name=bib-Catlett202020></a><a href="#cite-Catlett202020">Catlett, D., P. Matson, C. Carlson, et
al.</a> (2020). “Evaluation of accuracy and precision
in an amplicon sequencing workflow for marine protist communities”. In:
<em>Limnology and Oceanography: Methods</em> 18.1, pp. 20-40. DOI:
<a href="https://doi.org/10.1002%2Flom3.10343">10.1002/lom3.10343</a>.</p>
<p><a name=bib-Clark2020></a><a href="#cite-Clark2020">Clark, D., C. Pilditch, J. Pearman, et
al.</a> (2020). “Environmental DNA metabarcoding reveals
estuarine benthic community response to nutrient enrichment
Evidence from an in-situ experiment”. In: <em>Environmental Pollution</em>
267. DOI:
<a href="https://doi.org/10.1016%2Fj.envpol.2020.115472">10.1016/j.envpol.2020.115472</a>.</p>
<p><a name=bib-Clarke2020104538></a><a href="#cite-Clarke2020104538">Clarke, L. and B.
Deagle</a> (2020). “Eukaryote plankton assemblages
in the southern Kerguelen Axis region: Ecological drivers differ
between size fractions”. In: <em>Deep-Sea Research Part II: Topical
Studies in Oceanography</em> 174, p. 104538. DOI:
<a href="https://doi.org/10.1016%2Fj.dsr2.2018.12.003">10.1016/j.dsr2.2018.12.003</a>.</p>
<p><a name=bib-Clerissi2020></a><a href="#cite-Clerissi2020">Clerissi, C., L. Guillou, J. Escoubas, et
al.</a> (2020). “Unveiling protist diversity
associated with the Pacific oyster Crassostrea gigas using blocking and
excluding primers”. In: <em>BMC Microbiology</em> 20.1. DOI:
<a href="https://doi.org/10.1186%2Fs12866-020-01860-1">10.1186/s12866-020-01860-1</a>.</p>
<p><a name=bib-Cruaud20201></a><a href="#cite-Cruaud20201">Cruaud, P., A. Vigneron, C. Dorea, et
al.</a> (2020). “Rapid changes in microbial community
structures along a meandering river”. In: <em>Microorganisms</em> 8.11, pp.
1-21. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms8111631">10.3390/microorganisms8111631</a>.</p>
<p><a name=bib-DeLaIglesia2020514></a><a href="#cite-DeLaIglesia2020514">De La Iglesia, R., I.
Echenique-Subiabre, S. Rodr'iguez-Marconi, et
al.</a> (2020). “Distinct oxygen environments
shape picoeukaryote assemblages thriving oxygen minimum zone waters off
central Chile”. In: <em>Journal of Plankton Research</em> 42.5, pp. 514-529.
DOI:
<a href="https://doi.org/10.1093%2Fplankt%2Ffbaa036">10.1093/plankt/fbaa036</a>.</p>
<p><a name=bib-Delgado-Baquerizo2020210></a><a href="#cite-Delgado-Baquerizo2020210">Delgado-Baquerizo, M., P.
Reich, C. Trivedi, et al.</a> (2020).
“Multiple elements of soil biodiversity drive ecosystem functions
across biomes”. In: <em>Nature Ecology and Evolution</em> 4.2, pp. 210-220.
DOI:
<a href="https://doi.org/10.1038%2Fs41559-019-1084-y">10.1038/s41559-019-1084-y</a>.</p>
<p><a name=bib-Deng202015911></a><a href="#cite-Deng202015911">Deng, L., D. Bölsterli, E. Kristensen,
et al.</a> (2020). “Macrofaunal control of microbial
community structure in continental margin sediments”. In: <em>Proceedings
of the National Academy of Sciences of the United States of America</em>
117.27, pp. 15911-15922. DOI:
<a href="https://doi.org/10.1073%2Fpnas.1917494117">10.1073/pnas.1917494117</a>.</p>
<p><a name=bib-vanderReis2020107></a><a href="#cite-vanderReis2020107">der Reis, A. van, A. Jeffs, and S.
Lavery</a> (2020). “From feeding habits to food
webs: Exploring the diet of an opportunistic benthic generalist”. In:
<em>Marine Ecology Progress Series</em>, pp. 107-121. DOI:
<a href="https://doi.org/10.3354%2Fmeps13511">10.3354/meps13511</a>.</p>
<p><a name=bib-Djemiel2020></a><a href="#cite-Djemiel2020">Djemiel, C., D. Plassard, S. Terrat, et
al.</a> (2020). “µgreen-db: A reference database for
the 23S rRNA gene of eukaryotic plastids and cyanobacteria”. In:
<em>Scientific Reports</em> 10.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-020-62555-1">10.1038/s41598-020-62555-1</a>.</p>
<p><a name=bib-Dudek20205></a><a href="#cite-Dudek20205">Dudek, K., B. Cruz, B. Polidoro, et
al.</a> (2020). “Microbial colonization of microplastics
in the Caribbean Sea”. In: <em>Limnology And Oceanography Letters</em> 5.1,
pp. 5-17. DOI:
<a href="https://doi.org/10.1002%2Flol2.10141">10.1002/lol2.10141</a>.</p>
<p><a name=bib-Duret2020S49></a><a href="#cite-Duret2020S49">Duret, M., R. Lampitt, and P.
Lam</a> (2020). “Eukaryotic influence on the oceanic
biological carbon pump in the Scotia Sea as revealed by 18S rRNA gene
sequencing of suspended and sinking particles”. In: <em>Limnology and
Oceanography</em> 65.S1, pp. S49-S70. DOI:
<a href="https://doi.org/10.1002%2Flno.11319">10.1002/lno.11319</a>.</p>
<p><a name=bib-Fawley20202699></a><a href="#cite-Fawley20202699">Fawley, M. and K.
Fawley</a> (2020). “Identification of eukaryotic
microalgal strains”. In: <em>Journal of Applied Phycology</em> 32.5, pp.
2699-2709. DOI:
<a href="https://doi.org/10.1007%2Fs10811-020-02190-5">10.1007/s10811-020-02190-5</a>.</p>
<p><a name=bib-Finkel2020></a><a href="#cite-Finkel2020">Finkel, Z., Y. Liang, D. Nanjappa, et
al.</a> (2020). “A ribosomal sequence-based oil
sensitivity index for phytoplankton groups”. In: <em>Marine Pollution
Bulletin</em> 151. DOI:
<a href="https://doi.org/10.1016%2Fj.marpolbul.2019.110798">10.1016/j.marpolbul.2019.110798</a>.</p>
<p><a name=bib-Fiore-Donno2020></a><a href="#cite-Fiore-Donno2020">Fiore-Donno, A., T. Richter-Heitmann,
and M. Bonkowski</a> (2020). “Contrasting responses
of protistan plant parasites and phagotrophs to ecosystems, land
management and soil properties”. In: <em>Frontiers in Microbiology</em> 11.
DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2020.01823">10.3389/fmicb.2020.01823</a>.</p>
<p><a name=bib-Flegontova20204014></a><a href="#cite-Flegontova20204014">Flegontova, O., P. Flegontov, P.
Londoño, et al.</a> (2020). “Environmental
determinants of the distribution of planktonic diplonemids and
kinetoplastids in the oceans”. In: <em>Environmental Microbiology</em> 22.9,
pp. 4014-4031. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.15190">10.1111/1462-2920.15190</a>.</p>
<p><a name=bib-Fu2020961></a><a href="#cite-Fu2020961">Fu, Y., P. Zheng, X. Zhang, et
al.</a> (2020). “Protist interactions and seasonal
dynamics in the coast of yantai, northern yellow sea of china as
revealed by metabarcoding”. In: <em>Journal of Ocean University of China</em>
19.4, pp. 961-974. DOI:
<a href="https://doi.org/10.1007%2Fs11802-020-4461-x">10.1007/s11802-020-4461-x</a>.</p>
<p><a name=bib-Gad2020></a><a href="#cite-Gad2020">Gad, M., L. Hou, J. Li, et al.</a>
(2020). “Distinct mechanisms underlying the assembly of microeukaryotic
generalists and specialists in an anthropogenically impacted river”.
In: <em>Science of the Total Environment</em> 748. DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2020.141434">10.1016/j.scitotenv.2020.141434</a>.</p>
<p><a name=bib-Gaonkar20201917></a><a href="#cite-Gaonkar20201917">Gaonkar, C., R. Piredda, D. Sarno, et
al.</a> (2020). “Species detection and delineation
in the marine planktonic diatoms Chaetoceros and Bacteriastrum through
metabarcoding: making biological sense of haplotype diversity”. In:
<em>Environmental Microbiology</em> 22.5, pp. 1917-1929. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.14984">10.1111/1462-2920.14984</a>.</p>
<p><a name=bib-Genitsaris2020></a><a href="#cite-Genitsaris2020">Genitsaris, S., N. Stefanidou, M.
Moustaka-Gouni, et al.</a> (2020). “Variability and
community composition of marine unicellular eukaryote assemblages in a
eutrophic mediterranean urban coastal area with marked plankton blooms
and red tides”. In: <em>Diversity</em> 12.3. DOI:
<a href="https://doi.org/10.3390%2Fd12030114">10.3390/d12030114</a>.</p>
<p><a name=bib-Giner2020437></a><a href="#cite-Giner2020437">Giner, C., M. Pernice, V. Balagué, et
al.</a> (2020). “Marked changes in diversity and
relative activity of picoeukaryotes with depth in the world ocean”. In:
<em>ISME Journal</em> 14.2, pp. 437-449. DOI:
<a href="https://doi.org/10.1038%2Fs41396-019-0506-9">10.1038/s41396-019-0506-9</a>.</p>
<p><a name=bib-Gu2020></a><a href="#cite-Gu2020">Gu, R., P. Sun, Y. Wang, et al.</a>
(2020). “Genetic diversity, community assembly, and shaping factors of
benthic microbial eukaryotes in dongshan bay, southeast china”. In:
<em>Frontiers in Microbiology</em> 11. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2020.592489">10.3389/fmicb.2020.592489</a>.</p>
<p><a name=bib-Guo2020></a><a href="#cite-Guo2020">Guo, X., L. Wu, and L. Huang</a>
(2020). “Spatiotemporal patterns in diversity and assembly process of
marine protist communities of the changjiang (yangtze river) plume and
its adjacent waters”. In: <em>Frontiers in Microbiology</em> 11. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2020.579290">10.3389/fmicb.2020.579290</a>.</p>
<p><a name=bib-Harakeh2020></a><a href="#cite-Harakeh2020">Harakeh, S., E. Angelakis, T. Karamitros,
et al.</a> (2020). “Impact of smoking cessation, coffee
and bread consumption on the intestinal microbial composition among
Saudis: A cross-sectional study”. In: <em>PLoS ONE</em> 15.4. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pone.0230895">10.1371/journal.pone.0230895</a>.</p>
<p><a name=bib-Harkes2020></a><a href="#cite-Harkes2020">Harkes, P., J. van Steenbrugge, S. van den
Elsen, et al.</a> (2020). “Shifts in the active
rhizobiome paralleling low meloidogyne chitwoodi densities in fields
under prolonged organic soil management”. In: <em>Frontiers in Plant
Science</em> 10. DOI:
<a href="https://doi.org/10.3389%2Ffpls.2019.01697">10.3389/fpls.2019.01697</a>.</p>
<p><a name=bib-Helfer2020141></a><a href="#cite-Helfer2020141">Helfer, V. and C.
Hassenrück</a> (2020). <em>Microbial communities in
mangrove sediments</em> , pp. 141-175. DOI:
<a href="https://doi.org/10.1016%2FB978-0-12-816437-2.00003-3">10.1016/B978-0-12-816437-2.00003-3</a>.</p>
<p><a name=bib-Hirose2020></a><a href="#cite-Hirose2020">Hirose, Y., T. Shiozaki, I. Hamano, et
al.</a> (2020). “A specific combination of dual index
adaptors decreases the sensitivity of amplicon sequencing with the
Illumina platform”. In: <em>DNA Research</em> 27.4. DOI:
<a href="https://doi.org/10.1093%2Fdnares%2Fdsaa017">10.1093/dnares/dsaa017</a>.</p>
<p><a name=bib-Hirose20201></a><a href="#cite-Hirose20201">Hirose, Y., T. Shiozaki, M. Otani, et
al.</a> (2020). “Investigating algal communities in
lacustrine and hydro-terrestrial environments of East Antarctica using
deep amplicon sequencing”. In: <em>Microorganisms</em> 8.4. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms8040497">10.3390/microorganisms8040497</a>.</p>
<p><a name=bib-Hupfauf2020></a><a href="#cite-Hupfauf2020">Hupfauf, S., M. Etemadi, M. Juárez, et
al.</a> (2020). “CoMA an intuitive and
user-friendly pipeline for amplicon-sequencing data analysis”. In:
<em>PLoS ONE</em> 15.12 December. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pone.0243241">10.1371/journal.pone.0243241</a>.</p>
<p><a name=bib-Irion20202835></a><a href="#cite-Irion20202835">Irion, S., L. Jardillier, I.
Sassenhagen, et al.</a> (2020). “Marked
spatiotemporal variations in small phytoplankton structure in
contrasted waters of the Southern Ocean (Kerguelen area)”. In:
<em>Limnology and Oceanography</em> 65.11, pp. 2835-2852. DOI:
<a href="https://doi.org/10.1002%2Flno.11555">10.1002/lno.11555</a>.</p>
<p><a name=bib-Jauss2020></a><a href="#cite-Jauss2020">Jauss, R., S. Walden, A. Fiore-Donno, et
al.</a> (2020). “From forest soil to the canopy:
Increased habitat diversity does not increase species richness of
cercozoa and oomycota in tree canopies”. In: <em>Frontiers in
Microbiology</em> 11. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2020.592189">10.3389/fmicb.2020.592189</a>.</p>
<p><a name=bib-Jiang2020></a><a href="#cite-Jiang2020">Jiang, Y., L. Luan, K. Hu, et
al.</a> (2020). “Trophic interactions as determinants of
the arbuscular mycorrhizal fungal community with cascading
plant-promoting consequences”. In: <em>Microbiome</em> 8.1. DOI:
<a href="https://doi.org/10.1186%2Fs40168-020-00918-6">10.1186/s40168-020-00918-6</a>.</p>
<p><a name=bib-Jin2020293></a><a href="#cite-Jin2020293">Jin, S., K. Kim, M. Kim, et
al.</a> (2020). “An assessment of the taxonomic
reliability of dna barcode sequences in publicly available databases”.
In: <em>Algae (Korean Phycological Society)</em> 35.3, pp. 293-301. DOI:
<a href="https://doi.org/10.4490%2Falgae.2020.35.9.4">10.4490/algae.2020.35.9.4</a>.</p>
<p><a name=bib-Joglar20202807></a><a href="#cite-Joglar20202807">Joglar, V., A. Prieto, E. Barber-Lluch,
et al.</a> (2020). “Spatial and temporal variability
in the response of phytoplankton and prokaryotes to B-vitamin
amendments in an upwelling system”. In: <em>Biogeosciences (Online)</em>
17.10, pp. 2807-2823. DOI:
<a href="https://doi.org/10.5194%2Fbg-17-2807-2020">10.5194/bg-17-2807-2020</a>.</p>
<p><a name=bib-Kase2020305></a><a href="#cite-Kase2020305">Kase, L., A. Kraberg, K. Metfies, et
al.</a> (2020). “Rapid succession drives spring
community dynamics of small protists at Helgoland Roads, North Sea”.
In: <em>Journal of Plankton Research</em> 42.3, pp. 305-319. DOI:
<a href="https://doi.org/10.1093%2Fplankt%2Ffbaa017">10.1093/plankt/fbaa017</a>.</p>
<p><a name=bib-Keck2020></a><a href="#cite-Keck2020">Keck, F., L. Millet, D. Debroas, et
al.</a> (2020). “Assessing the response of
micro-eukaryotic diversity to the Great Acceleration using lake
sedimentary DNA”. In: <em>Nature Communications</em> 11.1. DOI:
<a href="https://doi.org/10.1038%2Fs41467-020-17682-8">10.1038/s41467-020-17682-8</a>.</p>
<p><a name=bib-Klimek2020></a><a href="#cite-Klimek2020">Klimek, B., M. Jazwa, M. Golebiewski, et
al.</a> (2020). “No apparent effect of invasive alien
goldenrod on soil microbial communities or soil fauna feeding
activity”. In: <em>Acta Oecologica</em> 109. DOI:
<a href="https://doi.org/10.1016%2Fj.actao.2020.103669">10.1016/j.actao.2020.103669</a>.</p>
<p><a name=bib-Kling2020413></a><a href="#cite-Kling2020413">Kling, J., M. Lee, F. Fu, et
al.</a> (2020). “Transient exposure to novel high
temperatures reshapes coastal phytoplankton communities”. In: <em>ISME
Journal</em> 14.2, pp. 413-424. DOI:
<a href="https://doi.org/10.1038%2Fs41396-019-0525-6">10.1038/s41396-019-0525-6</a>.</p>
<p><a name=bib-Kolisko2020></a><a href="#cite-Kolisko2020">Kolisko, M., O. Flegontova, A. Karnkowska,
et al.</a> (2020). “EukRef-excavates: Seven curated SSU
ribosomal RNA gene databases”. In: <em>Database-the Magazine of Electronic
Database Reviews</em> 2020. DOI:
<a href="https://doi.org/10.1093%2Fdatabase%2Fbaaa080">10.1093/database/baaa080</a>.</p>
<p><a name=bib-Labarre2020S149></a><a href="#cite-Labarre2020S149">Labarre, A., A. Obiol, S. Wilken, et
al.</a> (2020). “Expression of genes involved in
phagocytosis in uncultured heterotrophic flagellates”. In: <em>Limnology
and Oceanography</em> 65.S1, pp. S149-S160. DOI:
<a href="https://doi.org/10.1002%2Flno.11379">10.1002/lno.11379</a>.</p>
<p><a name=bib-Leconte2020></a><a href="#cite-Leconte2020">Leconte, J., L. Benites, T. Vannier, et
al.</a> (2020). “Genome resolved biogeography of
mamiellales”. In: <em>Genes</em> 11.1. DOI:
<a href="https://doi.org/10.3390%2Fgenes11010066">10.3390/genes11010066</a>.</p>
<p><a name=bib-Li20206867></a><a href="#cite-Li20206867">Li, F., F. Altermatt, J. Yang, et
al.</a> (2020). “Human activities’ fingerprint on
multitrophic biodiversity and ecosystem functions across a major river
catchment in China”. In: <em>Global Change Biology</em> 26.12, pp. 6867-6879.
DOI: <a href="https://doi.org/10.1111%2Fgcb.15357">10.1111/gcb.15357</a>.</p>
<p><a name=bib-Li2020></a><a href="#cite-Li2020">Li, Q., Y. Lei, R. Morard, et
al.</a> (2020). “Diversity hotspot and unique community
structure of foraminifera in the world’s deepest marine blue hole
Sansha Yongle Blue Hole”. In: <em>Scientific Reports</em> 10.1.
DOI:
<a href="https://doi.org/10.1038%2Fs41598-020-67221-0">10.1038/s41598-020-67221-0</a>.</p>
<p><a name=bib-Logares2020></a><a href="#cite-Logares2020">Logares, R., I. Deutschmann, P. Junger, et
al.</a> (2020). “Disentangling the mechanisms shaping
the surface ocean microbiota”. In: <em>Microbiome</em> 8.1. DOI:
<a href="https://doi.org/10.1186%2Fs40168-020-00827-8">10.1186/s40168-020-00827-8</a>.</p>
<p><a name=bib-Lohan2020145></a><a href="#cite-Lohan2020145">Lohan, K., R. DiMaria, D. Martin, et
al.</a> (2020). “Diversity and microhabitat
associations of labyrinthula spp. In the indian river lagoon system”.
In: <em>Diseases of Aquatic Organisms</em> 137.2, pp. 145-157. DOI:
<a href="https://doi.org/10.3354%2Fdao03431">10.3354/dao03431</a>.</p>
<p><a name=bib-Luan2020></a><a href="#cite-Luan2020">Luan, L., Y. Jiang, M. Cheng, et
al.</a> (2020). “Organism body size structures the soil
microbial and nematode community assembly at a continental and global
scale”. In: <em>Nature Communications</em> 11.1. DOI:
<a href="https://doi.org/10.1038%2Fs41467-020-20271-4">10.1038/s41467-020-20271-4</a>.</p>
<p><a name=bib-MarsBrisbin20202953></a><a href="#cite-MarsBrisbin20202953">Mars Brisbin, M., O. Brunner, M.
Grossmann, et al.</a> (2020). “Paired
high-throughput, in situ imaging and high-throughput sequencing
illuminate acantharian abundance and vertical distribution”. In:
<em>Limnology and Oceanography</em> 65.12, pp. 2953-2965. DOI:
<a href="https://doi.org/10.1002%2Flno.11567">10.1002/lno.11567</a>.</p>
<p><a name=bib-MarsBrisbin2020746></a><a href="#cite-MarsBrisbin2020746">Mars Brisbin, M., A. Conover, and
S. Mitarai</a> (2020). “Influence of regional
oceanography and hydrothermal activity on protist diversity and
community structure in the okinawa trough”. In: <em>Microbial Ecology</em>
80.4, pp. 746-761. DOI:
<a href="https://doi.org/10.1007%2Fs00248-020-01583-w">10.1007/s00248-020-01583-w</a>.</p>
<p><a name=bib-Metegnier2020></a><a href="#cite-Metegnier2020">Metegnier, G., S. Paulino, P. Ramond, et
al.</a> (2020). “Species specific gene expression
dynamics during harmful algal blooms”. In: <em>Scientific Reports</em> 10.1.
DOI:
<a href="https://doi.org/10.1038%2Fs41598-020-63326-8">10.1038/s41598-020-63326-8</a>.</p>
<p><a name=bib-Michaud2020308></a><a href="#cite-Michaud2020308">Michaud, C., V. Hervé, S. Dupont, et
al.</a> (2020). “Efficient but occasionally
imperfect vertical transmission of gut mutualistic protists in a
wood-feeding termite”. In: <em>Molecular Ecology</em> 29.2, pp. 308-324. DOI:
<a href="https://doi.org/10.1111%2Fmec.15322">10.1111/mec.15322</a>.</p>
<p><a name=bib-Minerovic2020></a><a href="#cite-Minerovic2020">Minerovic, A., M. Potapova, C. Sales, et
al.</a> (2020). “18S-V9 DNA metabarcoding detects the
effect of water-quality impairment on stream biofilm eukaryotic
assemblages”. In: <em>Ecological Indicators</em> 113. DOI:
<a href="https://doi.org/10.1016%2Fj.ecolind.2020.106225">10.1016/j.ecolind.2020.106225</a>.</p>
<p><a name=bib-Moccia2020340></a><a href="#cite-Moccia2020340">Moccia, K., S. Papoulis, A. Willems, et
al.</a> (2020). “Using the Microbiome Amplification
Preference Tool (MAPT) to reveal medicago sativa-associated eukaryotic
microbes”. In: <em>Phytobiomes Journal</em> 4.4, pp. 340-350. DOI:
<a href="https://doi.org/10.1094%2FPBIOMES-02-20-0022-R">10.1094/PBIOMES-02-20-0022-R</a>.</p>
<p><a name=bib-Mukherjee20204658></a><a href="#cite-Mukherjee20204658">Mukherjee, I., M. Salcher, A.
Andrei, et al.</a> (2020). “A freshwater
radiation of diplonemids”. In: <em>Environmental Microbiology</em> 22.11, pp.
4658-4668. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.15209">10.1111/1462-2920.15209</a>.</p>
<p><a name=bib-Nguyen20209556></a><a href="#cite-Nguyen20209556">Nguyen, B., Q. Chen, J. He, et
al.</a> (2020). “Oxytetracycline and ciprofloxacin
exposure altered the composition of protistan consumers in an
agricultural soil”. In: <em>Environmental Science and Technology</em> 54.15,
pp. 9556-9563. DOI:
<a href="https://doi.org/10.1021%2Facs.est.0c02531">10.1021/acs.est.0c02531</a>.</p>
<p><a name=bib-Obiol2020718></a><a href="#cite-Obiol2020718">Obiol, A., C. Giner, P. Sánchez, et
al.</a> (2020). “A metagenomic assessment of microbial
eukaryotic diversity in the global ocean”. In: <em>Molecular Ecology
Resources</em> 20.3, pp. 718-731. DOI:
<a href="https://doi.org/10.1111%2F1755-0998.13147">10.1111/1755-0998.13147</a>.</p>
<p><a name=bib-Oliverio2020></a><a href="#cite-Oliverio2020">Oliverio, A., S. Geisen, M.
Delgado-Baquerizo, et al.</a> (2020). “The
global-scale distributions of soil protists and their contributions to
belowground systems”. In: <em>Science Advances</em> 6.4. DOI:
<a href="https://doi.org/10.1126%2Fsciadv.aax8787">10.1126/sciadv.aax8787</a>.</p>
<p><a name=bib-Pan20201103></a><a href="#cite-Pan20201103">Pan, Y., J. Yang, G. McManus, et
al.</a> (2020). “Insights into protist diversity and
biogeography in intertidal sediments sampled across a range of spatial
scales”. In: <em>Limnology and Oceanography</em> 65.5, pp. 1103-1115. DOI:
<a href="https://doi.org/10.1002%2Flno.11375">10.1002/lno.11375</a>.</p>
<p><a name=bib-Pearman2020></a><a href="#cite-Pearman2020">Pearman, J., L. Biessy, G. Thomson-Laing,
et al.</a> (2020). “Local factors drive bacterial and
microeukaryotic community composition in lake surface sediment
collected across an altitudinal gradient”. In: <em>FEMS Microbiology
Ecology</em> 96.6. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiaa070">10.1093/femsec/fiaa070</a>.</p>
<p><a name=bib-Pereira2020519></a><a href="#cite-Pereira2020519">Pereira, T., A. De Santiago, T.
Schuelke, et al.</a> (2020). “The impact of
intragenomic rRNA variation on metabarcoding-derived diversity
estimates: A case study from marine nematodes”. In: <em>Environmental DNA</em>
2.4, pp. 519-534. DOI:
<a href="https://doi.org/10.1002%2Fedn3.77">10.1002/edn3.77</a>.</p>
<p><a name=bib-Pinseel2020></a><a href="#cite-Pinseel2020">Pinseel, E., S. Janssens, E. Verleyen, et
al.</a> (2020). “Global radiation in a rare biosphere
soil diatom”. In: <em>Nature Communications</em> 11.1. DOI:
<a href="https://doi.org/10.1038%2Fs41467-020-16181-0">10.1038/s41467-020-16181-0</a>.</p>
<p><a name=bib-Piwosz2020></a><a href="#cite-Piwosz2020">Piwosz, K., T. Shabarova, J. Pernthaler, et
al.</a> (2020). “Bacterial and eukaryotic small-subunit
amplicon data do not provide a quantitative picture of microbial
communities, but they are reliable in the context of ecological
interpretations”. In: <em>mSphere</em> 5.2. DOI:
<a href="https://doi.org/10.1128%2FMSPHERE.00052-20">10.1128/MSPHERE.00052-20</a>.</p>
<p><a name=bib-Porter2020></a><a href="#cite-Porter2020">Porter, T. and M.
Hajibabaei</a> (2020). “Putting COI metabarcoding in
context: The utility of exact sequence variants (ESVs) in biodiversity
analysis”. In: <em>Frontiers in Ecology and Evolution</em> 8. DOI:
<a href="https://doi.org/10.3389%2Ffevo.2020.00248">10.3389/fevo.2020.00248</a>.</p>
<p><a name=bib-Preston2020></a><a href="#cite-Preston2020">Preston, C., C. Durkin, and K.
Yamahara</a> (2020). “DNA metabarcoding reveals
organisms contributing to particulate matter flux to abyssal depths in
the North East Pacific ocean”. In: <em>Deep-Sea Research Part II: Topical
Studies in Oceanography</em> 173. DOI:
<a href="https://doi.org/10.1016%2Fj.dsr2.2019.104708">10.1016/j.dsr2.2019.104708</a>.</p>
<p><a name=bib-Qingxia2020825></a><a href="#cite-Qingxia2020825">Qingxia, L., L. Yanli, and L.
Tiegang</a> (2020). “A preliminary study on the dna
molecular diversity and community composition of deep-sea benthic
foraminifera from the northwest pacific ocean”. In: <em>Quaternary
Sciences</em> 40.3, pp. 825-836. DOI:
<a href="https://doi.org/10.11928%2Fj.issn.1001-7410.2020.03.19">10.11928/j.issn.1001-7410.2020.03.19</a>.</p>
<p><a name=bib-Reczuga2020></a><a href="#cite-Reczuga2020">Reczuga, M., C. Seppey, M. Mulot, et
al.</a> (2020). “Assessing the responses of Sphagnum
micro-eukaryotes to climate changes using high throughput sequencing”.
In: <em>PeerJ</em> 8. DOI:
<a href="https://doi.org/10.7717%2Fpeerj.9821">10.7717/peerj.9821</a>.</p>
<p><a name=bib-Ribeiro2020606></a><a href="#cite-Ribeiro2020606">Ribeiro, C., A. dos Santos, I. Probert,
et al.</a> (2020). “Taxonomic reassignment of
Pseudohaptolina birgeri comb. nov. (Haptophyta)”. In: <em>Phycologia</em>
59.6, pp. 606-615. DOI:
<a href="https://doi.org/10.1080%2F00318884.2020.1830255">10.1080/00318884.2020.1830255</a>.</p>
<p><a name=bib-Rossmann2020></a><a href="#cite-Rossmann2020">Rossmann, M., J. PÃ©rez-Jaramillo, V.
Kavamura, et al.</a> (2020). “Multitrophic
interactions in the rhizosphere microbiome of wheat: From bacteria and
fungi to protists”. In: <em>FEMS Microbiology Ecology</em> 96.4. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiaa032">10.1093/femsec/fiaa032</a>.</p>
<p><a name=bib-Sachs2020275></a><a href="#cite-Sachs2020275">Sachs, M., R. Schluckebier, K. Poll, et
al.</a> (2020). “Evidence of batrachochytrium
dendrobatidis and other amphibian parasites in the green toad (Bufotes
viridis), syntopic amphibians and environment in the Cologne Bay,
Germany”. In: <em>Salamandra</em> 56.3, pp. 275-284.</p>
<p><a name=bib-Salmaso2020></a><a href="#cite-Salmaso2020">Salmaso, N., A. Boscaini, and M.
Pindo</a> (2020). “Unraveling the diversity of
eukaryotic microplankton in a large and deep perialpine lake using a
high throughput sequencing approach”. In: <em>Frontiers in Microbiology</em>
11. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2020.00789">10.3389/fmicb.2020.00789</a>.</p>
<p><a name=bib-Santoferrara2020612></a><a href="#cite-Santoferrara2020612">Santoferrara, L., F. Burki, S.
Filker, et al.</a> (2020). “Perspectives from
ten years of protist studies by high-throughput metabarcoding”. In:
<em>Journal of Eukaryotic Microbiology</em> 67.5, pp. 612-622. DOI:
<a href="https://doi.org/10.1111%2Fjeu.12813">10.1111/jeu.12813</a>.</p>
<p><a name=bib-Santos2020></a><a href="#cite-Santos2020">Santos, S., A. Schöler, T. Nielsen, et
al.</a> (2020). “Land use as a driver for protist
community structure in soils under agricultural use across Europe”. In:
<em>Science of the Total Environment</em> 717. DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2020.137228">10.1016/j.scitotenv.2020.137228</a>.</p>
<p><a name=bib-Sassenhagen2020></a><a href="#cite-Sassenhagen2020">Sassenhagen, I., S. Irion, L.
Jardillier, et al.</a> (2020). “Protist
interactions and community structure during early autumn in the
kerguelen region (southern ocean)”. In: <em>Protist</em> 171.1. DOI:
<a href="https://doi.org/10.1016%2Fj.protis.2019.125709">10.1016/j.protis.2019.125709</a>.</p>
<p><a name=bib-Schoenle2020></a><a href="#cite-Schoenle2020">Schoenle, A., M. Hohlfeld, M. Rosse, et
al.</a> (2020). “Global comparison of bicosoecid
Cafeteria-like flagellates from the deep ocean and surface waters, with
reorganization of the family Cafeteriaceae”. In: <em>European Journal of
Protistology</em> 73. DOI:
<a href="https://doi.org/10.1016%2Fj.ejop.2019.125665">10.1016/j.ejop.2019.125665</a>.</p>
<p><a name=bib-Seppey2020866></a><a href="#cite-Seppey2020866">Seppey, C., O. Broennimann, A. Buri, et
al.</a> (2020). “Soil protist diversity in the Swiss
western Alps is better predicted by topo-climatic than by edaphic
variables”. In: <em>Journal of Biogeography</em> 47.4, pp. 866-878. DOI:
<a href="https://doi.org/10.1111%2Fjbi.13755">10.1111/jbi.13755</a>.</p>
<p><a name=bib-Shi2020></a><a href="#cite-Shi2020">Shi, J., Y. Lei, Q. Li, et al.</a>
(2020). “Molecular diversity and spatial distribution of benthic
foraminifera of the seamounts and adjacent abyssal plains in the
tropical Western Pacific Ocean”. In: <em>Marine Micropaleontology</em> 156.
DOI:
<a href="https://doi.org/10.1016%2Fj.marmicro.2020.101850">10.1016/j.marmicro.2020.101850</a>.</p>
<p><a name=bib-Simonin2020></a><a href="#cite-Simonin2020">Simonin, M., C. Dasilva, V. Terzi, et
al.</a> (2020). “Influence of plant genotype and soil
on the wheat rhizosphere microbiome: Evidences for a core microbiome
across eight African and European soils”. In: <em>FEMS Microbiology
Ecology</em> 96.6. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiaa067">10.1093/femsec/fiaa067</a>.</p>
<p><a name=bib-Sow2020></a><a href="#cite-Sow2020">Sow, S., T. Trull, and L.
Bodrossy</a> (2020). “Oceanographic fronts shape
phaeocystis assemblages: A high-resolution 18S rRNA gene survey from
the ice-edge to the equator of the south pacific”. In: <em>Frontiers in
Microbiology</em> 11. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2020.01847">10.3389/fmicb.2020.01847</a>.</p>
<p><a name=bib-Sprong2020></a><a href="#cite-Sprong2020">Sprong, P., V. Fofonova, K. Wiltshire, et
al.</a> (2020). “Spatial dynamics of eukaryotic
microbial communities in the German Bight”. In: <em>Journal of Sea
Research</em> 163. DOI:
<a href="https://doi.org/10.1016%2Fj.seares.2020.101914">10.1016/j.seares.2020.101914</a>.</p>
<p><a name=bib-Suleiman2020></a><a href="#cite-Suleiman2020">Suleiman, A., K. Lourenço, C. Clark, et
al.</a> (2020). “From toilet to agriculture:
Fertilization with microalgal biomass from wastewater impacts the soil
and rhizosphere active microbiomes, greenhouse gas emissions and plant
growth”. In: <em>Resources, Conservation and Recycling</em> 161. DOI:
<a href="https://doi.org/10.1016%2Fj.resconrec.2020.104924">10.1016/j.resconrec.2020.104924</a>.</p>
<p><a name=bib-Sun2020></a><a href="#cite-Sun2020">Sun, P., Y. Wang, E. Laws, et
al.</a> (2020). “Water massdriven spatial
effects and environmental heterogeneity shape microeukaryote
biogeography in a subtropical, hydrographically complex ocean system -
A case study of ciliates”. In: <em>Science of the Total Environment</em> 706.
DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2019.135753">10.1016/j.scitotenv.2019.135753</a>.</p>
<p><a name=bib-Sunagawa2020428></a><a href="#cite-Sunagawa2020428">Sunagawa, S., S. Acinas, P. Bork, et
al.</a> (2020). “Tara Oceans: towards global ocean
ecosystems biology”. In: <em>Nature Reviews Microbiology</em> 18.8, pp.
428-445. DOI:
<a href="https://doi.org/10.1038%2Fs41579-020-0364-5">10.1038/s41579-020-0364-5</a>.</p>
<p><a name=bib-Taerum2020291></a><a href="#cite-Taerum2020291">Taerum, S., B. Steven, D. Gage, et
al.</a> (2020). “Validation of a PNA clamping method
for reducing host DNA amplification and increasing eukaryotic diversity
in rhizosphere microbiome studies”. In: <em>Phytobiomes Journal</em> 4.4, pp.
291-302. DOI:
<a href="https://doi.org/10.1094%2FPBIOMES-05-20-0040-TA">10.1094/PBIOMES-05-20-0040-TA</a>.</p>
<p><a name=bib-Thompson20204620></a><a href="#cite-Thompson20204620">Thompson, A., S. Geisen, and B.
Adams</a> (2020). “Shotgun metagenomics reveal a
diverse assemblage of protists in a model Antarctic soil ecosystem”.
In: <em>Environmental Microbiology</em> 22.11, pp. 4620-4632. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.15198">10.1111/1462-2920.15198</a>.</p>
<p><a name=bib-Tian2020></a><a href="#cite-Tian2020">Tian, H., J. Feng, L. Zhang, et
al.</a> (2020). “Ecological drivers of methanotrophic
communities in paddy soils around mercury mining areas”. In: <em>Science
of the Total Environment</em> 721. DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2020.137760">10.1016/j.scitotenv.2020.137760</a>.</p>
<p><a name=bib-Tkacz2020></a><a href="#cite-Tkacz2020">Tkacz, A., E. Bestion, Z. Bo, et
al.</a> (2020). “Influence of plant fraction, soil, and
plant species on microbiota: A multikingdom comparison”. In: <em>mBio</em>
11.1. DOI:
<a href="https://doi.org/10.1128%2FmBio.02785-19">10.1128/mBio.02785-19</a>.</p>
<p><a name=bib-Wang20201></a><a href="#cite-Wang20201">Wang, W., K. Ren, H. Chen, et
al.</a> (2020). “Seven-year dynamics of testate amoeba
communities driven more by stochastic than deterministic processes in
two subtropical reservoirs”. In: <em>Water Research</em> 185. DOI:
<a href="https://doi.org/10.1016%2Fj.watres.2020.116232">10.1016/j.watres.2020.116232</a>.</p>
<p><a name=bib-Wang20204314></a><a href="#cite-Wang20204314">Wang, X., J. Long, J. Li, et
al.</a> (2020). “Diversity of soil eukaryotic microbes
in different succession stages in maolan karst forest of guizhou
[è´µå·žèŒ‚å…°å–€æ–¯ç‰¹æ£®æž—ä¸åŒæ¼”æ›¿ä¸‹åœŸå£¤çœŸæ ¸å¾®ç”Ÿç‰©å¤šæ ·æ€§]”.
In: <em>Huanjing Kexue/Environmental Science</em> 41.9, pp. 4314-4321. DOI:
<a href="https://doi.org/10.13227%2Fj.hjkx.202001010">10.13227/j.hjkx.202001010</a>.</p>
<p><a name=bib-Wang20203></a><a href="#cite-Wang20203">Wang, Y., X. Hu, Y. Sun, et
al.</a> (2020). “Influence of the cold bottom water on
taxonomic and functional composition and complexity of microbial
communities in the southern Yellow Sea during the summer”. In: <em>Science
of the Total Environment</em>. DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2020.143496">10.1016/j.scitotenv.2020.143496</a>.</p>
<p><a name=bib-Wang2020></a><a href="#cite-Wang2020">Wang, Y., G. Li, F. Shi, et
al.</a> (2020). “Taxonomic diversity of
pico-/nanoeukaryotes is related to dissolved oxygen and productivity,
but functional composition is shaped by limiting nutrients in eutrophic
coastal oceans”. In: <em>Frontiers in Microbiology</em> 11. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2020.601037">10.3389/fmicb.2020.601037</a>.</p>
<p><a name=bib-Wang20202></a><a href="#cite-Wang20202">Wang, Z., X. Gong, Y. Zheng, et
al.</a> (2020). “Soil protist communities in burrowing
and casting hotspots of different earthworm species”. In: <em>Soil Biology
and Biochemistry</em> 144. DOI:
<a href="https://doi.org/10.1016%2Fj.soilbio.2020.107774">10.1016/j.soilbio.2020.107774</a>.</p>
<p><a name=bib-Wideman2020154></a><a href="#cite-Wideman2020154">Wideman, J., A. Monier, R.
Rodr'iguez-Mart'inez, et al.</a> (2020).
“Unexpected mitochondrial genome diversity revealed by targeted
single-cell genomics of heterotrophic flagellated protists”. In:
<em>Nature Microbiology</em> 5.1, pp. 154-165. DOI:
<a href="https://doi.org/10.1038%2Fs41564-019-0605-4">10.1038/s41564-019-0605-4</a>.</p>
<p><a name=bib-Xiong20201></a><a href="#cite-Xiong20201">Xiong, W., X. Huang, Y. Chen, et
al.</a> (2020). “Zooplankton biodiversity monitoring in
polluted freshwater ecosystems: A technical review”. In: <em>Environmental
Science and Ecotechnology</em> 1. DOI:
<a href="https://doi.org/10.1016%2Fj.ese.2019.100008">10.1016/j.ese.2019.100008</a>.</p>
<p><a name=bib-Xiong2020></a><a href="#cite-Xiong2020">Xiong, W., Y. Song, K. Yang, et
al.</a> (2020). “Rhizosphere protists are key
determinants of plant health”. In: <em>Microbiome</em> 8.1. DOI:
<a href="https://doi.org/10.1186%2Fs40168-020-00799-9">10.1186/s40168-020-00799-9</a>.</p>
<p><a name=bib-Xu2020></a><a href="#cite-Xu2020">Xu, D., H. Kong, E. Yang, et al.</a>
(2020). “Contrasting community composition of active microbial
eukaryotes in melt ponds and sea water of the arctic ocean revealed by
high throughput sequencing”. In: <em>Frontiers in Microbiology</em> 11. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2020.01170">10.3389/fmicb.2020.01170</a>.</p>
<p><a name=bib-Xu20201></a><a href="#cite-Xu20201">Xu, Z., Y. Li, Y. Lu, et al.</a>
(2020). “Impacts of the Zhe-Min Coastal Current on the biogeographic
pattern of microbial eukaryotic communities”. In: <em>Progress in
Oceanography</em> 183. DOI:
<a href="https://doi.org/10.1016%2Fj.pocean.2020.102309">10.1016/j.pocean.2020.102309</a>.</p>
<p><a name=bib-Yeh2020></a><a href="#cite-Yeh2020">Yeh, H., J. Questel, K. Maas, et
al.</a> (2020). “Metabarcoding analysis of regional
variation in gut contents of the copepod Calanus finmarchicus in the
North Atlantic Ocean”. In: <em>Deep-Sea Research Part II: Topical Studies
in Oceanography</em> 180. DOI:
<a href="https://doi.org/10.1016%2Fj.dsr2.2020.104738">10.1016/j.dsr2.2020.104738</a>.</p>
<p><a name=bib-Zaiko20208443></a><a href="#cite-Zaiko20208443">Zaiko, A., S. Wood, X. Pochon, et
al.</a> (2020). “Elucidating biodiversity shifts in
ballast water tanks during a cross-latitudinal transfer: Complementary
insights from molecular analyses”. In: <em>Environmental Science and
Technology</em> 54.13, pp. 8443-8454. DOI:
<a href="https://doi.org/10.1021%2Facs.est.0c01931">10.1021/acs.est.0c01931</a>.</p>
<p><a name=bib-Zamora-Terol20203380></a><a href="#cite-Zamora-Terol20203380">Zamora-Terol, S., A. Novotny, and
M. Winder</a> (2020). “Reconstructing marine
plankton food web interactions using DNA metabarcoding”. In: <em>Molecular
Ecology</em> 29.17, pp. 3380-3395. DOI:
<a href="https://doi.org/10.1111%2Fmec.15555">10.1111/mec.15555</a>.</p>
<p><a name=bib-Zhao20202></a><a href="#cite-Zhao20202">Zhao, F., S. Filker, K. Xu, et
al.</a> (2020). “Microeukaryote communities exhibit
phyla-specific distance-decay patterns and an intimate link between
seawater and sediment habitats in the Western Pacific Ocean”. In:
<em>Deep-Sea Research Part I: Oceanographic Research Papers</em> 160. DOI:
<a href="https://doi.org/10.1016%2Fj.dsr.2020.103279">10.1016/j.dsr.2020.103279</a>.</p>
<p><a name=bib-Zhao2020></a><a href="#cite-Zhao2020">Zhao, F., C. Wang, K. Xu, et
al.</a> (2020). “Diversity and connectivity of
microeukaryote communities across multiple habitats from intertidal
zone to deep-sea floor in the Western Pacific Ocean”. In: <em>Deep-Sea
Research Part I: Oceanographic Research Papers</em> 165. DOI:
<a href="https://doi.org/10.1016%2Fj.dsr.2020.103395">10.1016/j.dsr.2020.103395</a>.</p>
<p><a name=bib-Zhao20201></a><a href="#cite-Zhao20201">Zhao, Z., J. He, Z. Quan, et
al.</a> (2020). “Fertilization changes soil microbiome
functioning, especially phagotrophic protists”. In: <em>Soil Biology and
Biochemistry</em> 148. DOI:
<a href="https://doi.org/10.1016%2Fj.soilbio.2020.107863">10.1016/j.soilbio.2020.107863</a>.</p>
</div>
<div id="section-3" class="section level2">
<h2>2019</h2>
<p><a name=bib-Ahmed201937></a><a href="#cite-Ahmed201937">Ahmed, M., M. Back, T. Prior, et
al.</a> (2019). “Metabarcoding of soil nematodes: The
importance of taxonomic coverage and availability of reference
sequences in choosing suitable marker(s)”. In: <em>Metabarcoding and
Metagenomics</em> 3, pp. 37-99. DOI:
<a href="https://doi.org/10.3897%2Fmbmg.3.36408">10.3897/mbmg.3.36408</a>.</p>
<p><a name=bib-Angelakis201914></a><a href="#cite-Angelakis201914">Angelakis, E., D. Bachar, M. Yasir, et
al.</a> (2019). “Treponema species enrich the gut
microbiota of traditional rural populations but are absent from urban
individuals”. In: <em>New Microbes and New Infections</em> 27, pp. 14-21. DOI:
<a href="https://doi.org/10.1016%2Fj.nmni.2018.10.009">10.1016/j.nmni.2018.10.009</a>.</p>
<p><a name=bib-Angelakis201940></a><a href="#cite-Angelakis201940">Angelakis, E., D. Bachar, M. Yasir, et
al.</a> (2019). “Comparison of the gut microbiota
of obese individuals from different geographic origins”. In: <em>New
Microbes and New Infections</em> 27, pp. 40-47. DOI:
<a href="https://doi.org/10.1016%2Fj.nmni.2018.11.005">10.1016/j.nmni.2018.11.005</a>.</p>
<p><a name=bib-Angelova2019></a><a href="#cite-Angelova2019">Angelova, A., G. Ellis, H. Wijesekera, et
al.</a> (2019). “Microbial composition and variability
of natural marine planktonic and biofouling communities from the bay of
bengal”. In: <em>Frontiers in Microbiology</em> 10. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2019.02738">10.3389/fmicb.2019.02738</a>.</p>
<p><a name=bib-ArmeliMinicante2019></a><a href="#cite-ArmeliMinicante2019">Armeli Minicante, S., R. Piredda,
G. Quero, et al.</a> (2019). “Habitat
heterogeneity and connectivity: Effects on the planktonic protist
community structure at two adjacent coastal sites (the lagoon and the
gulf of venice, northern adriatic sea, italy) revealed by
metabarcoding”. In: <em>Frontiers in Microbiology</em> 10. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2019.02736">10.3389/fmicb.2019.02736</a>.</p>
<p><a name=bib-Belilla20191552></a><a href="#cite-Belilla20191552">Belilla, J., D. Moreira, L.
Jardillier, et al.</a> (2019). “Hyperdiverse
archaea near life limits at the polyextreme geothermal Dallol area”.
In: <em>Nature Ecology and Evolution</em> 3.11, pp. 1552-1561. DOI:
<a href="https://doi.org/10.1038%2Fs41559-019-1005-0">10.1038/s41559-019-1005-0</a>.</p>
<p><a name=bib-Boeuf201911824></a><a href="#cite-Boeuf201911824">Boeuf, D., B. Edwards, J. Eppley, et
al.</a> (2019). “Biological composition and
microbial dynamics of sinking particulate organic matter at abyssal
depths in the oligotrophic open ocean”. In: <em>Proceedings of the
National Academy of Sciences of the United States of America</em> 116.24,
pp. 11824-11832. DOI:
<a href="https://doi.org/10.1073%2Fpnas.1903080116">10.1073/pnas.1903080116</a>.</p>
<p><a name=bib-Boscaro2019281></a><a href="#cite-Boscaro2019281">Boscaro, V., M. Syberg-Olsen, N. Irwin,
et al.</a> (2019). “What can environmental sequences
tell us about the distribution of low-rank taxa? The case of euplotes
(ciliophora, spirotrichea), including a description of euplotes enigma
sp. nov.” In: <em>Journal of Eukaryotic Microbiology</em> 66.2, pp. 281-293.
DOI: <a href="https://doi.org/10.1111%2Fjeu.12669">10.1111/jeu.12669</a>.</p>
<p><a name=bib-Buccheri2019></a><a href="#cite-Buccheri2019">Buccheri, M., E. Salvo, M. Coci, et
al.</a> (2019). “Investigating microbial indicators of
anthropogenic marine pollution by 16S and 18S High-Throughput
Sequencing (HTS) library analysis”. In: <em>FEMS Microbiology Letters</em>
366.14. DOI:
<a href="https://doi.org/10.1093%2Ffemsle%2Ffnz179">10.1093/femsle/fnz179</a>.</p>
<p><a name=bib-Caron2019197></a><a href="#cite-Caron2019197">Caron, D. and S. Hu</a>
(2019). “Are we overestimating protistan diversity in nature?” In:
<em>Trends in Microbiology</em> 27.3, pp. 197-205. DOI:
<a href="https://doi.org/10.1016%2Fj.tim.2018.10.009">10.1016/j.tim.2018.10.009</a>.</p>
<p><a name=bib-Chambouvet20194093></a><a href="#cite-Chambouvet20194093">Chambouvet, A., A. Monier, F.
Maguire, et al.</a> (2019). “Intracellular
infection of diverse diatoms by an evolutionary distinct relative of
the fungi”. In: <em>Current Biology</em> 29.23, pp. 4093-4101.e4. DOI:
<a href="https://doi.org/10.1016%2Fj.cub.2019.09.074">10.1016/j.cub.2019.09.074</a>.</p>
<p><a name=bib-Cleary2019861></a><a href="#cite-Cleary2019861">Cleary, D.</a> (2019).
“A comparison of microeukaryote communities inhabiting sponges and
seawater in a Taiwanese coral reef system”. In: <em>Annals of
Microbiology</em> 69.8, pp. 861-866. DOI:
<a href="https://doi.org/10.1007%2Fs13213-019-01476-5">10.1007/s13213-019-01476-5</a>.</p>
<p><a name=bib-Cleary201923></a><a href="#cite-Cleary201923">Cleary, D. and N.
Gomes</a> (2019). “Baseline information on prokaryotic
and microeukaryotic plankton communities inside and outside of
Indonesian marine lakes”. In: <em>Journal of Sea Research</em> 148-149, pp.
23-32. DOI:
<a href="https://doi.org/10.1016%2Fj.seares.2019.04.001">10.1016/j.seares.2019.04.001</a>.</p>
<p><a name=bib-Cordier2019387></a><a href="#cite-Cordier2019387">Cordier, T., A. Lanzén, L.
ApothÃ©loz-Perret-Gentil, et al.</a> (2019).
“Embracing environmental genomics and machine learning for routine
biomonitoring”. In: <em>Trends in Microbiology</em> 27.5, pp. 387-397. DOI:
<a href="https://doi.org/10.1016%2Fj.tim.2018.10.012">10.1016/j.tim.2018.10.012</a>.</p>
<p><a name=bib-Couton2019657></a><a href="#cite-Couton2019657">Couton, M., T. Comtet, S. Le Cam, et
al.</a> (2019). “Metabarcoding on planktonic larval
stages: An efficient approach for detecting and investigating life
cycle dynamics of benthic aliens”. In: <em>Management of Biological
Invasions</em> 10.4, pp. 657-689. DOI:
<a href="https://doi.org/10.3391%2Fmbi.2019.10.4.06">10.3391/mbi.2019.10.4.06</a>.</p>
<p><a name=bib-Cruaud2019></a><a href="#cite-Cruaud2019">Cruaud, P., A. Vigneron, M. Fradette, et
al.</a> (2019). “Annual protist community dynamics in a
freshwater ecosystem undergoing contrasted climatic conditions: The
saint-charles river (canada)”. In: <em>Frontiers in Microbiology</em> 10. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2019.02359">10.3389/fmicb.2019.02359</a>.</p>
<p><a name=bib-Czech20191151></a><a href="#cite-Czech20191151">Czech, L., P. Barbera, and A.
Stamatakis</a> (2019). “Methods for automatic
reference trees and multilevel phylogenetic placement”. In:
<em>Bioinformatics (Oxford, England)</em> 35.7, pp. 1151-1158. DOI:
<a href="https://doi.org/10.1093%2Fbioinformatics%2Fbty767">10.1093/bioinformatics/bty767</a>.</p>
<p><a name=bib-DeGruyter2019></a><a href="#cite-DeGruyter2019">De Gruyter, J., J. Weedon, S. Bazot, et
al.</a> (2019). “Patterns of local, intercontinental
and interseasonal variation of soil bacterial and eukaryotic microbial
communities”. In: <em>FEMS Microbiology Ecology</em> 96.3. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiaa018">10.1093/femsec/fiaa018</a>.</p>
<p><a name=bib-DeSchepper20192566></a><a href="#cite-DeSchepper20192566">De Schepper, S., J. Ray, K. Skaar,
et al.</a> (2019). “The potential of sedimentary
ancient DNA for reconstructing past sea ice evolution”. In: <em>ISME
Journal</em> 13.10, pp. 2566-2577. DOI:
<a href="https://doi.org/10.1038%2Fs41396-019-0457-1">10.1038/s41396-019-0457-1</a>.</p>
<p><a name=bib-Degrune2019></a><a href="#cite-Degrune2019">Degrune, F., K. Dumack, A. Fiore-Donno, et
al.</a> (2019). “Distinct communities of Cercozoa at
different soil depths in a temperate agricultural field”. In: <em>FEMS
Microbiology Ecology</em> 95.4. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiz041">10.1093/femsec/fiz041</a>.</p>
<p><a name=bib-DelCampo2019></a><a href="#cite-DelCampo2019">Del Campo, J., T. Heger, R.
Rodr'iguez-Mart'inez, et al.</a> (2019). “Assessing
the diversity and distribution of apicomplexans in host and free-living
environments using high-throughput amplicon data and a phylogenetically
informed reference framework”. In: <em>Frontiers in Microbiology</em> 10.OCT.
DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2019.02373">10.3389/fmicb.2019.02373</a>.</p>
<p><a name=bib-Demory2019132></a><a href="#cite-Demory2019132">Demory, D., A. Baudoux, A. Monier, et
al.</a> (2019). “Picoeukaryotes of the Micromonas
genus: sentinels of a warming ocean”. In: <em>ISME Journal</em> 13.1, pp.
132-146. DOI:
<a href="https://doi.org/10.1038%2Fs41396-018-0248-0">10.1038/s41396-018-0248-0</a>.</p>
<p><a name=bib-Duarte201959></a><a href="#cite-Duarte201959">Duarte, L., F. Coelho, D. Cleary, et
al.</a> (2019). “Bacterial and microeukaryotic
plankton communities in a semi-intensive aquaculture system of sea bass
(Dicentrarchus labrax): A seasonal survey”. In: <em>Aquaculture
(Amsterdam, Netherlands)</em> 503, pp. 59-69. DOI:
<a href="https://doi.org/10.1016%2Fj.aquaculture.2018.12.066">10.1016/j.aquaculture.2018.12.066</a>.</p>
<p><a name=bib-Dufresne2019></a><a href="#cite-Dufresne2019">Dufresne, Y., F. Lejzerowicz, L.
Perret-Gentil, et al.</a> (2019). “SLIM: A flexible
web application for the reproducible processing of environmental DNA
metabarcoding data”. In: <em>BMC Bioinformatics</em> 20.1. DOI:
<a href="https://doi.org/10.1186%2Fs12859-019-2663-2">10.1186/s12859-019-2663-2</a>.</p>
<p><a name=bib-Faure20191072></a><a href="#cite-Faure20191072">Faure, E., F. Not, A. Benoiston, et
al.</a> (2019). “Mixotrophic protists display
contrasted biogeographies in the global ocean”. In: <em>ISME Journal</em>
13.4, pp. 1072-1083. DOI:
<a href="https://doi.org/10.1038%2Fs41396-018-0340-5">10.1038/s41396-018-0340-5</a>.</p>
<p><a name=bib-Fiore-Donno2019></a><a href="#cite-Fiore-Donno2019">Fiore-Donno, A., T. Richter-Heitmann,
F. Degrune, et al.</a> (2019). “Functional traits
and spatio-temporal structure of a major group of soil protists
(rhizaria: Cercozoa) in a temperate grassland”. In: <em>Frontiers in
Microbiology</em> 10.JUN. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2019.01332">10.3389/fmicb.2019.01332</a>.</p>
<p><a name=bib-Forster2019294></a><a href="#cite-Forster2019294">Forster, D., S. Filker, R. Kochems, et
al.</a> (2019). “A comparison of different ciliate
metabarcode genes as bioindicators for environmental impact assessments
of salmon aquaculture”. In: <em>Journal of Eukaryotic Microbiology</em> 66.2,
pp. 294-308. DOI:
<a href="https://doi.org/10.1111%2Fjeu.12670">10.1111/jeu.12670</a>.</p>
<p><a name=bib-Gao2019></a><a href="#cite-Gao2019">Gao, Y., Y. Chen, W. Xiong, et
al.</a> (2019). “Distribution patterns of dinoflagellate
communities along the songhua river”. In: <em>PeerJ</em> 2019.4. DOI:
<a href="https://doi.org/10.7717%2Fpeerj.6733">10.7717/peerj.6733</a>.</p>
<p><a name=bib-Geisen2019></a><a href="#cite-Geisen2019">Geisen, S., M. Briones, H. Gan, et
al.</a> (2019). “A methodological framework to embrace
soil biodiversity”. In: <em>Soil Biology and Biochemistry</em> 136. DOI:
<a href="https://doi.org/10.1016%2Fj.soilbio.2019.107536">10.1016/j.soilbio.2019.107536</a>.</p>
<p><a name=bib-Giner2019923></a><a href="#cite-Giner2019923">Giner, C., V. Balagué, A. Krabberød, et
al.</a> (2019). “Quantifying long-term recurrence in
planktonic microbial eukaryotes”. In: <em>Molecular Ecology</em> 28.5, pp.
923-935. DOI: <a href="https://doi.org/10.1111%2Fmec.14929">10.1111/mec.14929</a>.</p>
<p><a name=bib-Gutierrez-Rodriguez2019964></a><a href="#cite-Gutierrez-Rodriguez2019964">Gutierrez-Rodriguez, A., M.
Stukel, A. Lopes dos Santos, et al.</a>
(2019). “High contribution of Rhizaria (Radiolaria) to vertical export
in the California Current Ecosystem revealed by DNA metabarcoding”. In:
<em>ISME Journal</em> 13.4, pp. 964-976. DOI:
<a href="https://doi.org/10.1038%2Fs41396-018-0322-7">10.1038/s41396-018-0322-7</a>.</p>
<p><a name=bib-Haas20193927></a><a href="#cite-Haas20193927">Haas, S., D. Desai, J. LaRoche, et
al.</a> (2019). “Geomicrobiology of the carbon,
nitrogen and sulphur cycles in Powell Lake: a permanently stratified
water column containing ancient seawater”. In: <em>Environmental
Microbiology</em> 21.10, pp. 3927-3952. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.14743">10.1111/1462-2920.14743</a>.</p>
<p><a name=bib-Harkes2019></a><a href="#cite-Harkes2019">Harkes, P., A. Suleiman, S. van den Elsen,
et al.</a> (2019). “Conventional and organic soil
management as divergent drivers of resident and active fractions of
major soil food web constituents”. In: <em>Scientific Reports</em> 9.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-019-49854-y">10.1038/s41598-019-49854-y</a>.</p>
<p><a name=bib-He20191138></a><a href="#cite-He20191138">He, X., T. Sutherland, J. Pawlowski, et
al.</a> (2019). “Responses of foraminifera communities
to aquaculture-derived organic enrichment as revealed by environmental
DNA metabarcoding”. In: <em>Molecular Ecology</em> 28.5, pp. 1138-1153. DOI:
<a href="https://doi.org/10.1111%2Fmec.15007">10.1111/mec.15007</a>.</p>
<p><a name=bib-Isabwe2019></a><a href="#cite-Isabwe2019">Isabwe, A., K. Ren, Y. Wang, et
al.</a> (2019). “Community assembly mechanisms
underlying the core and random bacterioplankton and microeukaryotes in
a river-reservoir system”. In: <em>Water (Switzerland)</em> 11.6. DOI:
<a href="https://doi.org/10.3390%2Fw11061127">10.3390/w11061127</a>.</p>
<p><a name=bib-Kataoka2019></a><a href="#cite-Kataoka2019">Kataoka, T. and R.
Kondo</a> (2019a). “Data on taxonomic annotation and
diversity of 18S rRNA gene amplicon libraries derived from high
throughput sequencing”. In: <em>Data in Brief</em> 25. DOI:
<a href="https://doi.org/10.1016%2Fj.dib.2019.104213">10.1016/j.dib.2019.104213</a>.</p>
<p><a name=bib-Kataoka201934></a><a href="#cite-Kataoka201934">Kataoka, T. and R.
Kondo</a> (2019b). “Protistan community composition
in anoxic sediments from three salinity-disparate Japanese lakes”. In:
<em>Estuarine, Coastal and Shelf Science</em> 224, pp. 34-42. DOI:
<a href="https://doi.org/10.1016%2Fj.ecss.2019.04.046">10.1016/j.ecss.2019.04.046</a>.</p>
<p><a name=bib-Kong2019></a><a href="#cite-Kong2019">Kong, J., Y. Wang, A. Warren, et
al.</a> (2019). “Diversity distribution and assembly
mechanisms of planktonic and benthic microeukaryote communities in
intertidal zones of southeast fujian, china”. In: <em>Frontiers in
Microbiology</em> 10. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2019.02640">10.3389/fmicb.2019.02640</a>.</p>
<p><a name=bib-Kulk2019314></a><a href="#cite-Kulk2019314">Kulk, G., A. Buist, W. van de Poll, et
al.</a> (2019). “Size scaling of photophysiology and
growth in four freshly isolated diatom species from Ryder Bay, western
Antarctic peninsula”. In: <em>Journal of Phycology</em> 55.2, pp. 314-328.
DOI: <a href="https://doi.org/10.1111%2Fjpy.12813">10.1111/jpy.12813</a>.</p>
<p><a name=bib-Lambert2019388></a><a href="#cite-Lambert2019388">Lambert, S., M. Tragin, J. Lozano, et
al.</a> (2019). “Rhythmicity of coastal marine
picoeukaryotes, bacteria and archaea despite irregular environmental
perturbations”. In: <em>ISME Journal</em> 13.2, pp. 388-401. DOI:
<a href="https://doi.org/10.1038%2Fs41396-018-0281-z">10.1038/s41396-018-0281-z</a>.</p>
<p><a name=bib-Lentendu2019592></a><a href="#cite-Lentendu2019592">Lentendu, G., P. Buosi, A. Cabral, et
al.</a> (2019). “Protist biodiversity and
biogeography in lakes from four brazilian River
systems”. In: <em>Journal of Eukaryotic Microbiology</em> 66.4, pp. 592-599.
DOI: <a href="https://doi.org/10.1111%2Fjeu.12703">10.1111/jeu.12703</a>.</p>
<p><a name=bib-Leray201922651></a><a href="#cite-Leray201922651">Leray, M., N. Knowlton, S. Ho, et
al.</a> (2019). “GenBank is a reliable resource for
21st century biodiversity research”. In: <em>Proceedings of the National
Academy of Sciences of the United States of America</em> 116.45, pp.
22651-22656. DOI:
<a href="https://doi.org/10.1073%2Fpnas.1911714116">10.1073/pnas.1911714116</a>.</p>
<p><a name=bib-Li2019></a><a href="#cite-Li2019">Li, F., X. Zhang, Y. Xie, et al.</a>
(2019). “Sedimentary DNA reveals over 150 years of ecosystem change by
human activities in Lake Chao, China”. In: <em>Environment International</em>
133. DOI:
<a href="https://doi.org/10.1016%2Fj.envint.2019.105214">10.1016/j.envint.2019.105214</a>.</p>
<p><a name=bib-Liu20195285></a><a href="#cite-Liu20195285">Liu, F., X. Zhang, S. Tang, et
al.</a> (2019). “Effects of acid mine drainage on
eukaryotic community in river sediments
[é…¸æ€§çŸ¿å±±åºŸæ°´å¯¹æ²‰ç§¯ç‰©çœŸæ ¸å¾®ç”Ÿç‰©ç¾¤è½çš„å½±å“]”. In:
<em>Zhongguo Huanjing Kexue/China Environmental Science</em> 39.12, pp.
5285-5292.</p>
<p><a name=bib-Liu20192196></a><a href="#cite-Liu20192196">Liu, L., H. Chen, M. Liu, et
al.</a> (2019). “Response of the eukaryotic plankton
community to the cyanobacterial biomass cycle over 6 years in two
subtropical reservoirs”. In: <em>ISME Journal</em> 13.9, pp. 2196-2208. DOI:
<a href="https://doi.org/10.1038%2Fs41396-019-0417-9">10.1038/s41396-019-0417-9</a>.</p>
<p><a name=bib-Liu2019></a><a href="#cite-Liu2019">Liu, M., Y. Xue, and J. Yang</a>
(2019). “Rare plankton subcommunities are far more affected by DNA
extraction kits than abundant plankton”. In: <em>Frontiers in
Microbiology</em> 10.MAR. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2019.00454">10.3389/fmicb.2019.00454</a>.</p>
<p><a name=bib-Lupatini2019480></a><a href="#cite-Lupatini2019480">Lupatini, M., G. Korthals, L. Roesch,
et al.</a> (2019). “Long-term farming systems
modulate multi-trophic responses”. In: <em>Science of the Total
Environment</em> 646, pp. 480-490. DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2018.07.323">10.1016/j.scitotenv.2018.07.323</a>.</p>
<p><a name=bib-Lutz2019115></a><a href="#cite-Lutz2019115">Lutz, S., L. Procházková, L. Benning, et
al.</a> (2019). “Evaluating amplicon high
throughput sequencing data of microalgae living in melting snow:
Improvements and limitations”. In: <em>Fottea</em> 19.2, pp. 115-131. DOI:
<a href="https://doi.org/10.5507%2Ffot.2019.003">10.5507/fot.2019.003</a>.</p>
<p><a name=bib-Macingo2019821></a><a href="#cite-Macingo2019821">Macingo, S., K. Kormas, A. Oikonomou,
et al.</a> (2019). “Taxa-area and distance-decay
relationships of unicellular eukaryotes along an elevation gradient of
mountainous freshwater ecosystems”. In: <em>Journal of Plankton Research</em>
41.6, pp. 821-834. DOI:
<a href="https://doi.org/10.1093%2Fplankt%2Ffbz066">10.1093/plankt/fbz066</a>.</p>
<p><a name=bib-Marquardt201965></a><a href="#cite-Marquardt201965">Marquardt, M., R. Skogseth, I.
Wiedmann, et al.</a> (2019). “Vertical export of
marine pelagic protists in an ice-free high-Arctic fjord
(Adventfjorden, West Spitsbergen) throughout 2011-2012”. In: <em>Aquatic
Microbial Ecology</em> 83.1, pp. 65-82. DOI:
<a href="https://doi.org/10.3354%2Fame01904">10.3354/ame01904</a>.</p>
<p><a name=bib-Metz2019></a><a href="#cite-Metz2019">Metz, S., A. Dos Santos, M. Berman, et
al.</a> (2019). “Diversity of photosynthetic
picoeukaryotes in eutrophic shallow lakes as assessed by combining flow
cytometry cell-sorting and high throughput sequencing”. In: <em>FEMS
Microbiology Ecology</em> 95.5. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiz038">10.1093/femsec/fiz038</a>.</p>
<p><a name=bib-Metz20193885></a><a href="#cite-Metz20193885">Metz, S., D. Singer, I. Domaizon, et
al.</a> (2019). “Global distribution of
Trebouxiophyceae diversity explored by high-throughput sequencing and
phylogenetic approaches”. In: <em>Environmental Microbiology</em> 21.10, pp.
3885-3895. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.14738">10.1111/1462-2920.14738</a>.</p>
<p><a name=bib-Moore2019></a><a href="#cite-Moore2019">Moore, L., T. Huang, M. Ostrowski, et
al.</a> (2019). “Unicellular cyanobacteria are important
components of phytoplankton communities in Australia’s northern oceanic
ecoregions”. In: <em>Frontiers in Microbiology</em> 10.JAN. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2018.03356">10.3389/fmicb.2018.03356</a>.</p>
<p><a name=bib-Mukherjee2019></a><a href="#cite-Mukherjee2019">Mukherjee, I., Y. Hodoki, Y. Okazaki, et
al.</a> (2019). “Widespread dominance of
kinetoplastids and unexpected presence of diplonemids in deep
freshwater lakes”. In: <em>Frontiers in Microbiology</em> 10. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2019.02375">10.3389/fmicb.2019.02375</a>.</p>
<p><a name=bib-Murdock20193796></a><a href="#cite-Murdock20193796">Murdock, S. and S.
Juniper</a> (2019). “Hydrothermal vent protistan
distribution along the Mariana arc suggests vent endemics may be rare
and novel”. In: <em>Environmental Microbiology</em> 21.10, pp. 3796-3815. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.14729">10.1111/1462-2920.14729</a>.</p>
<p><a name=bib-Nagler20191></a><a href="#cite-Nagler20191">Nagler, M., K. Kozjek, M. Etemadi, et
al.</a> (2019). “Simple yet effective: Microbial and
biotechnological benefits of rumen liquid addition to
lignocellulose-degrading biogas plants”. In: <em>Journal of Biotechnology</em>
300, pp. 1-10. DOI:
<a href="https://doi.org/10.1016%2Fj.jbiotec.2019.05.004">10.1016/j.jbiotec.2019.05.004</a>.</p>
<p><a name=bib-Panagiota-Chronopoulou2019></a><a href="#cite-Panagiota-Chronopoulou2019">Panagiota-Chronopoulou, M.,
I. Salonen, C. Bird, et al.</a> (2019).
“Metabarcoding insights into the trophic behavior and identity of
intertidal benthic foraminifera”. In: <em>Frontiers in Microbiology</em>
10.MAY. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2019.01169">10.3389/fmicb.2019.01169</a>.</p>
<p><a name=bib-Pasulka2019637></a><a href="#cite-Pasulka2019637">Pasulka, A., S. Hu, P. Countway, et
al.</a> (2019). “SSU-rRNA gene sequencing survey of
benthic microbial eukaryotes from guaymas basin hydrothermal vent”. In:
<em>Journal of Eukaryotic Microbiology</em> 66.4, pp. 637-653. DOI:
<a href="https://doi.org/10.1111%2Fjeu.12711">10.1111/jeu.12711</a>.</p>
<p><a name=bib-Piwosz20192172></a><a href="#cite-Piwosz20192172">Piwosz, K.</a>
(2019). “Weekly dynamics of abundance and size structure of specific
nanophytoplankton lineages in coastal waters (Baltic Sea)”. In:
<em>Limnology and Oceanography</em> 64.5, pp. 2172-2186. DOI:
<a href="https://doi.org/10.1002%2Flno.11177">10.1002/lno.11177</a>.</p>
<p><a name=bib-Polinski2019></a><a href="#cite-Polinski2019">Polinski, J., J. Bucci, M. Gasser, et
al.</a> (2019). “Metabarcoding assessment of
prokaryotic and eukaryotic taxa in sediments from Stellwagen Bank
National Marine Sanctuary”. In: <em>Scientific Reports</em> 9.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-019-51341-3">10.1038/s41598-019-51341-3</a>.</p>
<p><a name=bib-Ramond2019730></a><a href="#cite-Ramond2019730">Ramond, P., M. Sourisseau, N. Simon, et
al.</a> (2019). “Coupling between taxonomic and
functional diversity in protistan coastal communities”. In:
<em>Environmental Microbiology</em> 21.2, pp. 730-749. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.14537">10.1111/1462-2920.14537</a>.</p>
<p><a name=bib-Reboul2019464></a><a href="#cite-Reboul2019464">Reboul, G., D. Moreira, P. Bertolino, et
al.</a> (2019). “Microbial eukaryotes in the suboxic
chemosynthetic ecosystem of Movile Cave, Romania”. In: <em>Environmental
Microbiology Reports</em> 11.3, pp. 464-473. DOI:
<a href="https://doi.org/10.1111%2F1758-2229.12756">10.1111/1758-2229.12756</a>.</p>
<p><a name=bib-Rey201911849></a><a href="#cite-Rey201911849">Rey, A., K. Carney, L. Quinones, et
al.</a> (2019). “Environmental DNA metabarcoding: A
promising tool for ballast water monitoring”. In: <em>Environmental
Science and Technology</em> 53.20, pp. 11849-11859. DOI:
<a href="https://doi.org/10.1021%2Facs.est.9b01855">10.1021/acs.est.9b01855</a>.</p>
<p><a name=bib-Ribeiro2019></a><a href="#cite-Ribeiro2019">Ribeiro, H., A. Martins, M. Gonçalves, et
al.</a> (2019). “Development of an autonomous
biosampler to capture in situ aquatic microbiomes”. In: <em>PLoS ONE</em>
14.5. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pone.0216882">10.1371/journal.pone.0216882</a>.</p>
<p><a name=bib-Rimet2019></a><a href="#cite-Rimet2019">Rimet, F., E. Gusev, M. Kahlert, et
al.</a> (2019). “Diat.barcode, an open-access curated
barcode library for diatoms”. In: <em>Scientific Reports</em> 9.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-019-51500-6">10.1038/s41598-019-51500-6</a>.</p>
<p><a name=bib-Rudolph2019></a><a href="#cite-Rudolph2019">Rudolph, K., C. Fichtel, D. Schneider, et
al.</a> (2019). “One size fits all? Relationships among
group size, health, and ecology indicate a lack of an optimal group
size in a wild lemur population”. In: <em>Behavioral Ecology and
Sociobiology</em> 73.9. DOI:
<a href="https://doi.org/10.1007%2Fs00265-019-2746-0">10.1007/s00265-019-2746-0</a>.</p>
<p><a name=bib-Santoferrara2019571></a><a href="#cite-Santoferrara2019571">Santoferrara,
L.</a> (2019). “Current practice in plankton
metabarcoding: Optimization and error management”. In: <em>Journal of
Plankton Research</em> 41.5, pp. 571-582. DOI:
<a href="https://doi.org/10.1093%2Fplankt%2Ffbz041">10.1093/plankt/fbz041</a>.</p>
<p><a name=bib-Shiozaki2019></a><a href="#cite-Shiozaki2019">Shiozaki, T., Y. Hirose, K. Hamasaki, et
al.</a> (2019). “Eukaryotic phytoplankton contributing
to a seasonal bloom and carbon export revealed by tracking sequence
variants in the western north pacific”. In: <em>Frontiers in Microbiology</em>
10. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2019.02722">10.3389/fmicb.2019.02722</a>.</p>
<p><a name=bib-Singer2019714></a><a href="#cite-Singer2019714">Singer, D., S. Metz, F. Unrein, et
al.</a> (2019). “Contrasted micro-eukaryotic
diversity associated with sphagnum mosses in tropical, subtropical and
temperate climatic zones”. In: <em>Microbial Ecology</em> 78.3, pp. 714-724.
DOI:
<a href="https://doi.org/10.1007%2Fs00248-019-01325-7">10.1007/s00248-019-01325-7</a>.</p>
<p><a name=bib-deSousa2019388></a><a href="#cite-deSousa2019388">Sousa, A. de, M. Tomasino, P. Duarte,
et al.</a> (2019). “Diversity and composition of
pelagic prokaryotic and protist communities in a thin arctic sea-ice
regime”. In: <em>Microbial Ecology</em> 78.2, pp. 388-408. DOI:
<a href="https://doi.org/10.1007%2Fs00248-018-01314-2">10.1007/s00248-018-01314-2</a>.</p>
<p><a name=bib-Suleiman2019></a><a href="#cite-Suleiman2019">Suleiman, A., P. Harkes, S. van den
Elsen, et al.</a> (2019). “Organic amendment
strengthens interkingdom associations in the soil and rhizosphere of
barley (Hordeum vulgare)”. In: <em>Science of the Total Environment</em> 695.
DOI:
<a href="https://doi.org/10.1016%2Fj.scitotenv.2019.133885">10.1016/j.scitotenv.2019.133885</a>.</p>
<p><a name=bib-Sun2019></a><a href="#cite-Sun2019">Sun, P., L. Huang, D. Xu, et
al.</a> (2019). “Integrated space-time dataset reveals high
diversity and distinct community structure of ciliates in mesopelagic
waters of the northern south china sea”. In: <em>Frontiers in
Microbiology</em> 10. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2019.02178">10.3389/fmicb.2019.02178</a>.</p>
<p><a name=bib-Swalethorp20191></a><a href="#cite-Swalethorp20191">Swalethorp, R., J. Dinasquet, R.
Logares, et al.</a> (2019). “Microzooplankton
distribution in the Amundsen Sea Polynya (Antarctica) during an
extensive Phaeocystis antarctica bloom”. In: <em>Progress in Oceanography</em>
170, pp. 1-10. DOI:
<a href="https://doi.org/10.1016%2Fj.pocean.2018.10.008">10.1016/j.pocean.2018.10.008</a>.</p>
<p><a name=bib-Tito20191180></a><a href="#cite-Tito20191180">Tito, R., S. Chaffron, C. Caenepeel, et
al.</a> (2019). “Population-level analysis of
Blastocystis subtype prevalence and variation in the human gut
microbiota”. In: <em>Gut</em> 68.7, pp. 1180-1189. DOI:
<a href="https://doi.org/10.1136%2Fgutjnl-2018-316106">10.1136/gutjnl-2018-316106</a>.</p>
<p><a name=bib-Tragin2019></a><a href="#cite-Tragin2019">Tragin, M. and D. Vaulot</a>
(2019). “Novel diversity within marine Mamiellophyceae (Chlorophyta)
unveiled by metabarcoding”. In: <em>Scientific Reports</em> 9.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-019-41680-6">10.1038/s41598-019-41680-6</a>.</p>
<p><a name=bib-Tsao2019464></a><a href="#cite-Tsao2019464">Tsao, H., U. Scheikl, C. Herbold, et
al.</a> (2019). “The cooling tower water microbiota:
Seasonal dynamics and co-occurrence of bacterial and protist
phylotypes”. In: <em>Water Research</em> 159, pp. 464-479. DOI:
<a href="https://doi.org/10.1016%2Fj.watres.2019.04.028">10.1016/j.watres.2019.04.028</a>.</p>
<p><a name=bib-Voss2019></a><a href="#cite-Voss2019">Voss, C., A. Fiore-Donno, M. Guerreiro, et
al.</a> (2019). “Metatranscriptomics reveals unsuspected
protistan diversity in leaf litter across temperate beech forests, with
Amoebozoa the dominating lineage”. In: <em>FEMS Microbiology Ecology</em>
95.10. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiz142">10.1093/femsec/fiz142</a>.</p>
<p><a name=bib-Whalen2019></a><a href="#cite-Whalen2019">Whalen, K., J. Becker, A. Schrecengost, et
al.</a> (2019). “Bacterial alkylquinolone signaling
contributes to structuring microbial communities in the ocean”. In:
<em>Microbiome</em> 7.1. DOI:
<a href="https://doi.org/10.1186%2Fs40168-019-0711-9">10.1186/s40168-019-0711-9</a>.</p>
<p><a name=bib-Wilschut2019></a><a href="#cite-Wilschut2019">Wilschut, R., W. van der Putten, P.
Garbeva, et al.</a> (2019). “Root traits and
belowground herbivores relate to plantsoil feedback
variation among congeners”. In: <em>Nature Communications</em> 10.1. DOI:
<a href="https://doi.org/10.1038%2Fs41467-019-09615-x">10.1038/s41467-019-09615-x</a>.</p>
<p><a name=bib-Wilschut20192714></a><a href="#cite-Wilschut20192714">Wilschut, R., S. Geisen, H. Martens,
et al.</a> (2019). “Latitudinal variation in soil
nematode communities under climate warming-related range-expanding and
native plants”. In: <em>Global Change Biology</em> 25.8, pp. 2714-2726. DOI:
<a href="https://doi.org/10.1111%2Fgcb.14657">10.1111/gcb.14657</a>.</p>
<p><a name=bib-Wood20191407></a><a href="#cite-Wood20191407">Wood, S., X. Pochon, O. Laroche, et
al.</a> (2019). “A comparison of droplet digital
polymerase chain reaction (PCR), quantitative PCR and metabarcoding for
species-specific detection in environmental DNA”. In: <em>Molecular
Ecology Resources</em> 19.6, pp. 1407-1419. DOI:
<a href="https://doi.org/10.1111%2F1755-0998.13055">10.1111/1755-0998.13055</a>.</p>
<p><a name=bib-Wu201931></a><a href="#cite-Wu201931">Wu, W., C. Guo, P. Pitta, et
al.</a> (2019). “Response of active picoeukaryotes to the
deposition of Saharan dust and European aerosols in the eastern
Mediterranean Sea”. In: <em>Aquatic Microbial Ecology</em> 82.1, pp. 31-42.
DOI: <a href="https://doi.org/10.3354%2Fame01881">10.3354/ame01881</a>.</p>
<p><a name=bib-Wu2019></a><a href="#cite-Wu2019">Wu, W. and B. Huang</a> (2019).
“Protist diversity and community assembly in surface sediments of the
South China Sea”. In: <em>MicrobiologyOpen</em> 8.10. DOI:
<a href="https://doi.org/10.1002%2Fmbo3.891">10.1002/mbo3.891</a>.</p>
<p><a name=bib-Yu2019474></a><a href="#cite-Yu2019474">Yu, Z. and N. Chen</a> (2019).
“Emerging trends in red tide and major research progresses
[å›½å†…å¤–èµ¤æ½®çš„å’å±•è¶‹åŠ¿ä¸Žç ”ç©¶çƒ­ç‚¹]”. In: <em>Oceanologia et
Limnologia Sinica</em> 50.3, pp. 474-486. DOI:
<a href="https://doi.org/10.11693%2Fhyhz20190200041">10.11693/hyhz20190200041</a>.</p>
<p><a name=bib-Zhao2019></a><a href="#cite-Zhao2019">Zhao, Z., J. He, S. Geisen, et
al.</a> (2019). “Protist communities are more sensitive to
nitrogen fertilization than other microorganisms in diverse
agricultural soils”. In: <em>Microbiome</em> 7.1. DOI:
<a href="https://doi.org/10.1186%2Fs40168-019-0647-0">10.1186/s40168-019-0647-0</a>.</p>
</div>
<div id="section-4" class="section level2">
<h2>2018</h2>
<p><a name=bib-Ali2018429></a><a href="#cite-Ali2018429">Ali, S., M. Isa, S. Safari, et
al.</a> (2018). “Eukaryotic biodiversity in mixed peat
ecosystems in Sarawak, Malaysia”. In: <em>Journal of Oil Palm Research</em>
30.3, pp. 429-450. DOI:
<a href="https://doi.org/10.21894%2Fjopr.2018.0023">10.21894/jopr.2018.0023</a>.</p>
<p><a name=bib-vonAmmon201857></a><a href="#cite-vonAmmon201857">Ammon, U. von, S. Wood, O. Laroche, et
al.</a> (2018). “The impact of artificial surfaces
on marine bacterial and eukaryotic biofouling assemblages: A
high-throughput sequencing analysis”. In: <em>Marine Environmental
Research</em> 133, pp. 57-66. DOI:
<a href="https://doi.org/10.1016%2Fj.marenvres.2017.12.003">10.1016/j.marenvres.2017.12.003</a>.</p>
<p><a name=bib-Ammon2018></a><a href="#cite-Ammon2018">Ammon, U., S. Wood, O. Laroche, et
al.</a> (2018). “Combining morpho-taxonomy and
metabarcoding enhances the detection of non-indigenous marine pests in
biofouling communities”. In: <em>Scientific Reports</em> 8.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-018-34541-1">10.1038/s41598-018-34541-1</a>.</p>
<p><a name=bib-deAraujo2018></a><a href="#cite-deAraujo2018">Araujo, A. de, L. Mendes, L. Lemos, et
al.</a> (2018). “Protist species richness and soil
microbiome complexity increase towards climax vegetation in the
Brazilian Cerrado”. In: <em>Communications Biology</em> 1.1. DOI:
<a href="https://doi.org/10.1038%2Fs42003-018-0129-0">10.1038/s42003-018-0129-0</a>.</p>
<p><a name=bib-Arroyo2018></a><a href="#cite-Arroyo2018">Arroyo, A., D. LÃ³pez-EscardÃ³, E. Kim, et
al.</a> (2018). “Novel diversity of deeply branching
holomycota and unicellular holozoans revealed by metabarcoding in
Middle Paraná River, Argentina”. In: <em>Frontiers in Ecology and
Evolution</em> 6.JUL. DOI:
<a href="https://doi.org/10.3389%2Ffevo.2018.00099">10.3389/fevo.2018.00099</a>.</p>
<p><a name=bib-Bahram2018233></a><a href="#cite-Bahram2018233">Bahram, M., F. Hildebrand, S. Forslund,
et al.</a> (2018). “Structure and function of the
global topsoil microbiome”. In: <em>Nature</em> 560.7717, pp. 233-237. DOI:
<a href="https://doi.org/10.1038%2Fs41586-018-0386-6">10.1038/s41586-018-0386-6</a>.</p>
<p><a name=bib-Balzano20185951></a><a href="#cite-Balzano20185951">Balzano, S., J. Lattaud, L.
Villanueva, et al.</a> (2018). “A quest for the
biological sources of long chain alkyl diols in the western tropical
North Atlantic Ocean”. In: <em>Biogeosciences (Online)</em> 15.19, pp.
5951-5968. DOI:
<a href="https://doi.org/10.5194%2Fbg-15-5951-2018">10.5194/bg-15-5951-2018</a>.</p>
<p><a name=bib-Bass2018773></a><a href="#cite-Bass2018773">Bass, D., L. Czech, B. Williams, et
al.</a> (2018). “Clarifying the relationships between
microsporidia and cryptomycota”. In: <em>Journal of Eukaryotic
Microbiology</em> 65.6, pp. 773-782. DOI:
<a href="https://doi.org/10.1111%2Fjeu.12519">10.1111/jeu.12519</a>.</p>
<p><a name=bib-Bensaoud2018></a><a href="#cite-Bensaoud2018">Bensaoud, C., J. Nishiyama M.Y., C. Ben
Hamda, et al.</a> (2018). “De novo assembly and
annotation of Hyalomma dromedarii tick (Acari: Ixodidae)
sialotranscriptome with regard to gender differences in gene
expression”. In: <em>Parasites and Vectors</em> 11.1. DOI:
<a href="https://doi.org/10.1186%2Fs13071-018-2874-9">10.1186/s13071-018-2874-9</a>.</p>
<p><a name=bib-Berdjeb20181907></a><a href="#cite-Berdjeb20181907">Berdjeb, L., A. Parada, D. Needham, et
al.</a> (2018). “Short-term dynamics and
interactions of marine protist communities during the spring-summer
transition”. In: <em>ISME Journal</em> 12.8, pp. 1907-1917. DOI:
<a href="https://doi.org/10.1038%2Fs41396-018-0097-x">10.1038/s41396-018-0097-x</a>.</p>
<p><a name=bib-Boast20181546></a><a href="#cite-Boast20181546">Boast, A., L. Weyrich, J. Wood, et
al.</a> (2018). “Coprolites reveal ecological
interactions lost with the extinction of New Zealand birds”. In:
<em>Proceedings of the National Academy of Sciences of the United States
of America</em> 115.7, pp. 1546-1551. DOI:
<a href="https://doi.org/10.1073%2Fpnas.1712337115">10.1073/pnas.1712337115</a>.</p>
<p><a name=bib-Boaventura2018321></a><a href="#cite-Boaventura2018321">Boaventura, C., F. Coelho, P.
Martins, et al.</a> (2018). “Micro-eukaryotic
plankton diversity in an intensive aquaculture system for production of
Scophthalmus maximus and Solea senegalensis”. In: <em>Aquaculture
(Amsterdam, Netherlands)</em> 490, pp. 321-328. DOI:
<a href="https://doi.org/10.1016%2Fj.aquaculture.2018.02.033">10.1016/j.aquaculture.2018.02.033</a>.</p>
<p><a name=bib-Bock2018></a><a href="#cite-Bock2018">Bock, C., M. Salcher, M. Jensen, et
al.</a> (2018). “Synchrony of eukaryotic and prokaryotic
planktonic communities in three seasonally sampled Austrian lakes”. In:
<em>Frontiers in Microbiology</em> 9.JUN. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2018.01290">10.3389/fmicb.2018.01290</a>.</p>
<p><a name=bib-Boscaro20182218></a><a href="#cite-Boscaro20182218">Boscaro, V., L. Santoferrara, Q.
Zhang, et al.</a> (2018). “EukRef-Ciliophora: a
manually curated, phylogeny-based database of small subunit rRNA gene
sequences of ciliates”. In: <em>Environmental Microbiology</em> 20.6, pp.
2218-2230. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.14264">10.1111/1462-2920.14264</a>.</p>
<p><a name=bib-Briand2018657></a><a href="#cite-Briand2018657">Briand, J., X. Pochon, S. Wood, et
al.</a> (2018). “Metabarcoding and metabolomics offer
complementarity in deciphering marine eukaryotic biofouling community
shifts”. In: <em>Biofouling</em> 34.6, pp. 657-672. DOI:
<a href="https://doi.org/10.1080%2F08927014.2018.1480757">10.1080/08927014.2018.1480757</a>.</p>
<p><a name=bib-Brown2018></a><a href="#cite-Brown2018">Brown, M., J. Van De Kamp, M. Ostrowski, et
al.</a> (2018). “Data Descriptor: Systematic, continental
scale temporal monitoring of marine pelagic microbiota by the
Australian Marine Microbial Biodiversity Initiative”. In: <em>Scientific
Data</em> 5. DOI:
<a href="https://doi.org/10.1038%2Fsdata.2018.130">10.1038/sdata.2018.130</a>.</p>
<p><a name=bib-BruceCahoon2018></a><a href="#cite-BruceCahoon2018">Bruce Cahoon, A., A. Huffman, M.
Krager, et al.</a> (2018). “A meta-barcoding census
of freshwater planktonic protists in appalachia - natural tunnel state
park, virginia, USA”. In: <em>Metabarcoding and Metagenomics</em> 2. DOI:
<a href="https://doi.org/10.3897%2Fmbmg.2.26939">10.3897/mbmg.2.26939</a>.</p>
<p><a name=bib-Cai201831543></a><a href="#cite-Cai201831543">Cai, J., C. Bai, X. Tang, et
al.</a> (2018). “Characterization of bacterial and
microbial eukaryotic communities associated with an ephemeral hypoxia
event in Taihu Lake, a shallow eutrophic Chinese lake”. In:
<em>Environmental Science and Pollution Research</em> 25.31, pp. 31543-31557.
DOI:
<a href="https://doi.org/10.1007%2Fs11356-018-2987-x">10.1007/s11356-018-2987-x</a>.</p>
<p><a name=bib-delCampo2018></a><a href="#cite-delCampo2018">Campo, J. del, M. Kolisko, V. Boscaro, et
al.</a> (2018). “EukRef: Phylogenetic curation of
ribosomal RNA to enhance understanding of eukaryotic diversity and
distribution”. In: <em>PLoS Biology</em> 16.9. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pbio.2005849">10.1371/journal.pbio.2005849</a>.</p>
<p><a name=bib-Celussi2018210></a><a href="#cite-Celussi2018210">Celussi, M., G. Quero, L. Zoccarato, et
al.</a> (2018). “Planktonic prokaryote and protist
communities in a submarine canyon system in the Ligurian Sea (NW
Mediterranean)”. In: <em>Progress in Oceanography</em> 168, pp. 210-221. DOI:
<a href="https://doi.org/10.1016%2Fj.pocean.2018.10.002">10.1016/j.pocean.2018.10.002</a>.</p>
<p><a name=bib-Clerissi2018></a><a href="#cite-Clerissi2018">Clerissi, C., S. Brunet, J. Vidal-Dupiol,
et al.</a> (2018). “Protists within corals: The hidden
diversity”. In: <em>Frontiers in Microbiology</em> 9.AUG. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2018.02043">10.3389/fmicb.2018.02043</a>.</p>
<p><a name=bib-Decelle20183625></a><a href="#cite-Decelle20183625">Decelle, J., Q. Carradec, X. Pochon,
et al.</a> (2018). “Worldwide occurrence and
activity of the reef-building coral symbiont symbiodinium in the open
ocean”. In: <em>Current Biology</em> 28.22, pp. 3625-3633.e3. DOI:
<a href="https://doi.org/10.1016%2Fj.cub.2018.09.024">10.1016/j.cub.2018.09.024</a>.</p>
<p><a name=bib-vanderReis2018></a><a href="#cite-vanderReis2018">der Reis, A. van, O. Laroche, A. Jeffs,
et al.</a> (2018). “Preliminary analysis of New
Zealand scampi (Metanephrops challengeri) diet using metabarcoding”.
In: <em>PeerJ</em> 2018.9. DOI:
<a href="https://doi.org/10.7717%2Fpeerj.5641">10.7717/peerj.5641</a>.</p>
<p><a name=bib-Douglas2018131></a><a href="#cite-Douglas2018131">Douglas, G., A. Comeau, and M.
Langille</a> (2018). “Processing a 16S rRNA
sequencing dataset with the microbiome helper workflow”. In: <em>Methods
in Molecular Biology</em> 1849, pp. 131-141. DOI:
<a href="https://doi.org/10.1007%2F978-1-4939-8728-3_9">10.1007/978-1-4939-8728-3_9</a>.</p>
<p><a name=bib-Duarte2018></a><a href="#cite-Duarte2018">Duarte, S., L. Nunes, P. Borges, et
al.</a> (2018). “A bridge too far? An integrative
framework linking classical protist taxonomy and metabarcoding in lower
termites”. In: <em>Frontiers in Microbiology</em> 9.NOV. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2018.02620">10.3389/fmicb.2018.02620</a>.</p>
<p><a name=bib-Enberg201819></a><a href="#cite-Enberg201819">Enberg, S., M. Majaneva, R. Autio, et
al.</a> (2018). “Phases of microalgal succession in
sea ice and the water column in the baltic sea from autumn to spring”.
In: <em>Marine Ecology Progress Series</em> 599, pp. 19-34. DOI:
<a href="https://doi.org/10.3354%2Fmeps12645">10.3354/meps12645</a>.</p>
<p><a name=bib-Engesmo2018105></a><a href="#cite-Engesmo2018105">Engesmo, A., D. Strand, S.
Gran-StadniczeÃ±ko, et al.</a> (2018). “Development
of a qPCR assay to detect and quantify ichthyotoxic flagellates along
the Norwegian coast, and the first Norwegian record of Fibrocapsa
japonica (Raphidophyceae)”. In: <em>Harmful Algae</em> 75, pp. 105-117. DOI:
<a href="https://doi.org/10.1016%2Fj.hal.2018.04.007">10.1016/j.hal.2018.04.007</a>.</p>
<p><a name=bib-Fiore-Donno2018229></a><a href="#cite-Fiore-Donno2018229">Fiore-Donno, A., C. Rixen, M.
Rippin, et al.</a> (2018). “New barcoded primers
for efficient retrieval of cercozoan sequences in high-throughput
environmental diversity surveys, with emphasis on worldwide biological
soil crusts”. In: <em>Molecular Ecology Resources</em> 18.2, pp. 229-239. DOI:
<a href="https://doi.org/10.1111%2F1755-0998.12729">10.1111/1755-0998.12729</a>.</p>
<p><a name=bib-Flegontova2018878></a><a href="#cite-Flegontova2018878">Flegontova, O., P. Flegontov, S.
Malviya, et al.</a> (2018). “Neobodonids are
dominant kinetoplastids in the global ocean”. In: <em>Environmental
Microbiology</em> 20.2, pp. 878-889. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.14034">10.1111/1462-2920.14034</a>.</p>
<p><a name=bib-Gaonkar2018></a><a href="#cite-Gaonkar2018">Gaonkar, C., R. Piredda, C. Minucci, et
al.</a> (2018). “Annotated 18S and 28S rDNA reference
sequences of taxa in the planktonic diatom family Chaetocerotaceae”.
In: <em>PLoS ONE</em> 13.12. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pone.0208929">10.1371/journal.pone.0208929</a>.</p>
<p><a name=bib-Garvetto2018></a><a href="#cite-Garvetto2018">Garvetto, A., E. Nézan, Y. Badis, et
al.</a> (2018). “Novel widespread marine oomycetes
parasitising diatoms, including the toxic genus pseudo-nitzschia:
Genetic, morphological, and ecological characterisation”. In:
<em>Frontiers in Microbiology</em> 9. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2018.02918">10.3389/fmicb.2018.02918</a>.</p>
<p><a name=bib-Gebert20181></a><a href="#cite-Gebert20181">Gebert, M., M. Delgado-Baquerizo, A.
Oliverio, et al.</a> (2018). “Ecological analyses of
mycobacteria in showerhead biofilms and their relevance to human
health”. In: <em>mBio</em> 9.5, pp. 1-15. DOI:
<a href="https://doi.org/10.1128%2Fmbio.01614-18">10.1128/mbio.01614-18</a>.</p>
<p><a name=bib-Geisen2018293></a><a href="#cite-Geisen2018293">Geisen, S., E. Mitchell, S. Adl, et
al.</a> (2018). “Soil protists: A fertile frontier in
soil biology research”. In: <em>FEMS Microbiology Reviews</em> 42.3, pp.
293-323. DOI:
<a href="https://doi.org/10.1093%2Ffemsre%2Ffuy006">10.1093/femsre/fuy006</a>.</p>
<p><a name=bib-Geisen20181366></a><a href="#cite-Geisen20181366">Geisen, S., L. Snoek, F. ten Hooven, et
al.</a> (2018). “Integrating quantitative
morphological and qualitative molecular methods to analyse soil
nematode community responses to plant range expansion”. In: <em>Methods in
Ecology and Evolution</em> 9.6, pp. 1366-1378. DOI:
<a href="https://doi.org/10.1111%2F2041-210X.12999">10.1111/2041-210X.12999</a>.</p>
<p><a name=bib-Guo20188></a><a href="#cite-Guo20188">Guo, S., W. Xiong, H. Xu, et
al.</a> (2018). “Continuous application of different
fertilizers induces distinct bulk and rhizosphere soil protist
communities”. In: <em>European Journal of Soil Biology</em> 88, pp. 8-14. DOI:
<a href="https://doi.org/10.1016%2Fj.ejsobi.2018.05.007">10.1016/j.ejsobi.2018.05.007</a>.</p>
<p><a name=bib-Hu2018></a><a href="#cite-Hu2018">Hu, S., P. Connell, L. Mesrop, et
al.</a> (2018). “A hard day’s night: Diel shifts in
microbial eukaryotic activity in the north pacific subtropical gyre”.
In: <em>Frontiers in Marine Science</em> 5.OCT. DOI:
<a href="https://doi.org/10.3389%2Ffmars.2018.00351">10.3389/fmars.2018.00351</a>.</p>
<p><a name=bib-Hunsucker2018162></a><a href="#cite-Hunsucker2018162">Hunsucker, K., G. Vora, J. Hunsucker,
et al.</a> (2018). “Biofilm community structure
and the associated drag penalties of a groomed fouling release ship
hull coating”. In: <em>Biofouling</em> 34.2, pp. 162-172. DOI:
<a href="https://doi.org/10.1080%2F08927014.2017.1417395">10.1080/08927014.2017.1417395</a>.</p>
<p><a name=bib-Jing2018></a><a href="#cite-Jing2018">Jing, H., Y. Zhang, Y. Li, et
al.</a> (2018). “Spatial variability of picoeukaryotic
communities in the mariana trench”. In: <em>Scientific Reports</em> 8.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-018-33790-4">10.1038/s41598-018-33790-4</a>.</p>
<p><a name=bib-Kamp2018></a><a href="#cite-Kamp2018">Kamp, A., C. Petro, H. Røy, et
al.</a> (2018). “Intracellular nitrate in sediments of an
oxygen-deficient marine basin is linked to pelagic diatoms”. In: <em>FEMS
Microbiology Ecology</em> 94.8. DOI:
<a href="https://doi.org/10.1093%2FFEMSEC%2FFIY122">10.1093/FEMSEC/FIY122</a>.</p>
<p><a name=bib-Kisand2018628></a><a href="#cite-Kisand2018628">Kisand, V., L. Talas, A. Kisand, et
al.</a> (2018). “From microbial eukaryotes to
metazoan vertebrates: Wide spectrum paleo-diversity in sedimentary
ancient DNA over the last 14,500 years”. In: <em>Geobiology</em> 16.6, pp.
628-639. DOI: <a href="https://doi.org/10.1111%2Fgbi.12307">10.1111/gbi.12307</a>.</p>
<p><a name=bib-Kuwata2018></a><a href="#cite-Kuwata2018">Kuwata, A., K. Yamada, M. Ichinomiya, et
al.</a> (2018). “Bolidophyceae, a sister picoplanktonic
group of diatoms - A review”. In: <em>Frontiers in Marine Science</em> 5.OCT.
DOI:
<a href="https://doi.org/10.3389%2Ffmars.2018.00370">10.3389/fmars.2018.00370</a>.</p>
<p><a name=bib-Laroche201897></a><a href="#cite-Laroche201897">Laroche, O., S. Wood, L. Tremblay, et
al.</a> (2018). “A cross-taxa study using
environmental DNA/RNA metabarcoding to measure biological impacts of
offshore oil and gas drilling and production operations”. In: <em>Marine
Pollution Bulletin</em> 127, pp. 97-107. DOI:
<a href="https://doi.org/10.1016%2Fj.marpolbul.2017.11.042">10.1016/j.marpolbul.2017.11.042</a>.</p>
<p><a name=bib-Leblanc2018></a><a href="#cite-Leblanc2018">Leblanc, K., B. Quéguiner, F. Diaz, et
al.</a> (2018). “Nanoplanktonic diatoms are globally
overlooked but play a role in spring blooms and carbon export”. In:
<em>Nature Communications</em> 9.1. DOI:
<a href="https://doi.org/10.1038%2Fs41467-018-03376-9">10.1038/s41467-018-03376-9</a>.</p>
<p><a name=bib-Leff20181794></a><a href="#cite-Leff20181794">Leff, J., R. Bardgett, A. Wilkinson, et
al.</a> (2018). “Predicting the structure of soil
communities from plant community taxonomy, phylogeny, and traits”. In:
<em>ISME Journal</em> 12.7, pp. 1794-1805. DOI:
<a href="https://doi.org/10.1038%2Fs41396-018-0089-x">10.1038/s41396-018-0089-x</a>.</p>
<p><a name=bib-Lentendu20182846></a><a href="#cite-Lentendu20182846">Lentendu, G., F. Mahé, D. Bass, et
al.</a> (2018). “Consistent patterns of high alpha
and low beta diversity in tropical parasitic and free-living protists”.
In: <em>Molecular Ecology</em> 27.13, pp. 2846-2857. DOI:
<a href="https://doi.org/10.1111%2Fmec.14731">10.1111/mec.14731</a>.</p>
<p><a name=bib-Leray20183753></a><a href="#cite-Leray20183753">Leray, M., S. Ho, I. Lin, et
al.</a> (2018). “MIDORI server: a webserver for
taxonomic assignment of unknown metazoan mitochondrial-encoded
sequences using a curated database”. In: <em>Bioinformatics (Oxford,
England)</em> 34.21, pp. 3753-3754. DOI:
<a href="https://doi.org/10.1093%2Fbioinformatics%2Fbty454">10.1093/bioinformatics/bty454</a>.</p>
<p><a name=bib-Lewitus20181715></a><a href="#cite-Lewitus20181715">Lewitus, E., L. Bittner, S. Malviya,
et al.</a> (2018). “Clade-specific diversification
dynamics of marine diatoms since the Jurassic”. In: <em>Nature Ecology and
Evolution</em> 2.11, pp. 1715-1723. DOI:
<a href="https://doi.org/10.1038%2Fs41559-018-0691-3">10.1038/s41559-018-0691-3</a>.</p>
<p><a name=bib-Li201811708></a><a href="#cite-Li201811708">Li, F., Y. Peng, W. Fang, et
al.</a> (2018). “Application of environmental DNA
metabarcoding for predicting anthropogenic pollution in rivers”. In:
<em>Environmental Science and Technology</em> 52.20, pp. 11708-11719. DOI:
<a href="https://doi.org/10.1021%2Facs.est.8b03869">10.1021/acs.est.8b03869</a>.</p>
<p><a name=bib-Li201872></a><a href="#cite-Li201872">Li, R., N. Jiao, A. Warren, et
al.</a> (2018). “Changes in community structure of active
protistan assemblages from the lower Pearl River to coastal Waters of
the South China Sea”. In: <em>European Journal of Protistology</em> 63, pp.
72-82. DOI:
<a href="https://doi.org/10.1016%2Fj.ejop.2018.01.004">10.1016/j.ejop.2018.01.004</a>.</p>
<p><a name=bib-Morard2018></a><a href="#cite-Morard2018">Morard, R., M. Garet-Delmas, F. Mahé, et
al.</a> (2018). “Surface ocean metabarcoding confirms
limited diversity in planktonic foraminifera but reveals unknown
hyper-abundant lineages”. In: <em>Scientific Reports</em> 8.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-018-20833-z">10.1038/s41598-018-20833-z</a>.</p>
<p><a name=bib-Mordret2018974></a><a href="#cite-Mordret2018974">Mordret, S., R. Piredda, D. Vaulot, et
al.</a> (2018). “dinoref: A curated dinoflagellate
(Dinophyceae) reference database for the 18S rRNA gene”. In: <em>Molecular
Ecology Resources</em> 18.5, pp. 974-987. DOI:
<a href="https://doi.org/10.1111%2F1755-0998.12781">10.1111/1755-0998.12781</a>.</p>
<p><a name=bib-Moreno-Pino20185></a><a href="#cite-Moreno-Pino20185">Moreno-Pino, M., B. Krock, R. De la
Iglesia, et al.</a> (2018). “Next Generation
Sequencing and mass spectrometry reveal high taxonomic diversity and
complex phytoplankton-phycotoxins patterns in Southeastern Pacific
fjords”. In: <em>Toxicon : official journal of the International Society
on Toxinology</em> 151, pp. 5-14. DOI:
<a href="https://doi.org/10.1016%2Fj.toxicon.2018.06.078">10.1016/j.toxicon.2018.06.078</a>.</p>
<p><a name=bib-Mucko201814></a><a href="#cite-Mucko201814">Mucko, M., S. Bosak, R. Casotti, et
al.</a> (2018). “Winter picoplankton diversity in an
oligotrophic marginal sea”. In: <em>Marine Genomics</em> 42, pp. 14-24. DOI:
<a href="https://doi.org/10.1016%2Fj.margen.2018.09.002">10.1016/j.margen.2018.09.002</a>.</p>
<p><a name=bib-Oliverio20181918></a><a href="#cite-Oliverio20181918">Oliverio, A., J. Power, A. Washburne,
et al.</a> (2018). “The ecology and diversity of
microbial eukaryotes in geothermal springs”. In: <em>ISME Journal</em> 12.8,
pp. 1918-1928. DOI:
<a href="https://doi.org/10.1038%2Fs41396-018-0104-2">10.1038/s41396-018-0104-2</a>.</p>
<p><a name=bib-Pearman201831017></a><a href="#cite-Pearman201831017">Pearman, J., F. Afandi, P. Hong, et
al.</a> (2018). “Plankton community assessment in
anthropogenic-impacted oligotrophic coastal regions”. In:
<em>Environmental Science and Pollution Research</em> 25.31, pp. 31017-31030.
DOI:
<a href="https://doi.org/10.1007%2Fs11356-018-3072-1">10.1007/s11356-018-3072-1</a>.</p>
<p><a name=bib-Pearman2018></a><a href="#cite-Pearman2018">Pearman, J., M. Leray, R. Villalobos, et
al.</a> (2018). “Cross-shelf investigation of coral
reef cryptic benthic organisms reveals diversity patterns of the hidden
majority”. In: <em>Scientific Reports</em> 8.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-018-26332-5">10.1038/s41598-018-26332-5</a>.</p>
<p><a name=bib-Piwosz2018242></a><a href="#cite-Piwosz2018242">Piwosz, K., J. Calkiewicz, M.
Golebiewski, et al.</a> (2018). “Diversity and
community composition of pico- and nanoplanktonic protists in the
Vistula River estuary (Gulf of Gdansk, Baltic Sea)”. In: <em>Estuarine,
Coastal and Shelf Science</em> 207, pp. 242-249. DOI:
<a href="https://doi.org/10.1016%2Fj.ecss.2018.04.013">10.1016/j.ecss.2018.04.013</a>.</p>
<p><a name=bib-Rachik2018></a><a href="#cite-Rachik2018">Rachik, S., U. Christaki, L. Li, et
al.</a> (2018). “Diversity and potential activity
patterns of planktonic eukaryotic microbes in a mesoeutrophic coastal
area (eastern English Channel)”. In: <em>PLoS ONE</em> 13.5. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pone.0196987">10.1371/journal.pone.0196987</a>.</p>
<p><a name=bib-Ren2018177></a><a href="#cite-Ren2018177">Ren, K., Y. Xue, R. Rønn, et
al.</a> (2018). “Dynamics and determinants of amoeba
community, occurrence and abundance in subtropical reservoirs and
rivers”. In: <em>Water Research</em> 146, pp. 177-186. DOI:
<a href="https://doi.org/10.1016%2Fj.watres.2018.09.011">10.1016/j.watres.2018.09.011</a>.</p>
<p><a name=bib-Ser-Giacomi20181243></a><a href="#cite-Ser-Giacomi20181243">Ser-Giacomi, E., L. Zinger, S.
Malviya, et al.</a> (2018). “Ubiquitous
abundance distribution of non-dominant plankton across the global
ocean”. In: <em>Nature Ecology and Evolution</em> 2.8, pp. 1243-1249. DOI:
<a href="https://doi.org/10.1038%2Fs41559-018-0587-2">10.1038/s41559-018-0587-2</a>.</p>
<p><a name=bib-Stefanidou2018></a><a href="#cite-Stefanidou2018">Stefanidou, N., S. Genitsaris, J.
Lopez-Bautista, et al.</a> (2018). “Unicellular
eukaryotic community response to temperature and salinity variation in
mesocosm experiments”. In: <em>Frontiers in Microbiology</em> 9.OCT. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2018.02444">10.3389/fmicb.2018.02444</a>.</p>
<p><a name=bib-Stern2018519></a><a href="#cite-Stern2018519">Stern, R., A. Kraberg, E. Bresnan, et
al.</a> (2018). “Molecular analyses of protists in
long-term observation programmes - Current status and future
perspectives”. In: <em>Journal of Plankton Research</em> 40.5, pp. 519-536.
DOI:
<a href="https://doi.org/10.1093%2Fplankt%2Ffby035">10.1093/plankt/fby035</a>.</p>
<p><a name=bib-Stoeck2018153></a><a href="#cite-Stoeck2018153">Stoeck, T., R. Kochems, D. Forster, et
al.</a> (2018). “Metabarcoding of benthic ciliate
communities shows high potential for environmental monitoring in salmon
aquaculture”. In: <em>Ecological Indicators</em> 85, pp. 153-164. DOI:
<a href="https://doi.org/10.1016%2Fj.ecolind.2017.10.041">10.1016/j.ecolind.2017.10.041</a>.</p>
<p><a name=bib-Stoeck2018322></a><a href="#cite-Stoeck2018322">Stoeck, T., H. Pan, V. Dully, et
al.</a> (2018). “Towards an eDNA metabarcode-based
performance indicator for full-scale municipal wastewater treatment
plants”. In: <em>Water Research</em> 144, pp. 322-331. DOI:
<a href="https://doi.org/10.1016%2Fj.watres.2018.07.051">10.1016/j.watres.2018.07.051</a>.</p>
<p><a name=bib-Szelecz2018></a><a href="#cite-Szelecz2018">Szelecz, I., S. Lösch, C. Seppey, et
al.</a> (2018). “Comparative analysis of bones, mites,
soil chemistry, nematodes and soil micro-eukaryotes from a suspected
homicide to estimate the post-mortem interval”. In: <em>Scientific
Reports</em> 8.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-017-18179-z">10.1038/s41598-017-18179-z</a>.</p>
<p><a name=bib-Taberlet20181></a><a href="#cite-Taberlet20181">Taberlet, P., A. Bonin, L. Zinger, et
al.</a> (2018). <em>Environmental DNA: For biodiversity
research and monitoring</em> , pp. 1-253. DOI:
<a href="https://doi.org/10.1093%2Foso%2F9780198767220.001.0001">10.1093/oso/9780198767220.001.0001</a>.</p>
<p><a name=bib-Tan2018389></a><a href="#cite-Tan2018389">Tan, S. and H. Liu</a>
(2018). “Unravel the hidden protistan diversity: application of
blocking primers to suppress PCR amplification of metazoan DNA”. In:
<em>Applied Microbiology and Biotechnology</em> 102.1, pp. 389-401. DOI:
<a href="https://doi.org/10.1007%2Fs00253-017-8565-1">10.1007/s00253-017-8565-1</a>.</p>
<p><a name=bib-Tragin2018></a><a href="#cite-Tragin2018">Tragin, M. and D. Vaulot</a>
(2018). “Green microalgae in marine coastal waters: The ocean sampling
day (OSD) dataset”. In: <em>Scientific Reports</em> 8.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-018-32338-w">10.1038/s41598-018-32338-w</a>.</p>
<p><a name=bib-Tragin2018506></a><a href="#cite-Tragin2018506">Tragin, M., A. Zingone, and D.
Vaulot</a> (2018). “Comparison of coastal
phytoplankton composition estimated from the V4 and V9 regions of the
18S rRNA gene with a focus on photosynthetic groups and especially
Chlorophyta”. In: <em>Environmental Microbiology</em> 20.2, pp. 506-520. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.13952">10.1111/1462-2920.13952</a>.</p>
<p><a name=bib-Vader2018890></a><a href="#cite-Vader2018890">Vader, A., I. Laughinghouse H.D., C.
Griffiths, et al.</a> (2018). “Proton-pumping
rhodopsins are abundantly expressed by microbial eukaryotes in a
high-Arctic fjord”. In: <em>Environmental Microbiology</em> 20.2, pp. 890-902.
DOI:
<a href="https://doi.org/10.1111%2F1462-2920.14035">10.1111/1462-2920.14035</a>.</p>
<p><a name=bib-Varkey2018></a><a href="#cite-Varkey2018">Varkey, D., S. Mazard, T. Jeffries, et
al.</a> (2018). “Stormwater influences phytoplankton
assemblages within the diverse, but impacted Sydney Harbour estuary”.
In: <em>PLoS ONE</em> 13.12. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pone.0209857">10.1371/journal.pone.0209857</a>.</p>
<p><a name=bib-Venter2018432></a><a href="#cite-Venter2018432">Venter, P., F. Nitsche, and H.
Arndt</a> (2018). “The hidden diversity of
flagellated protists in soil”. In: <em>Protist</em> 169.3, pp. 432-449. DOI:
<a href="https://doi.org/10.1016%2Fj.protis.2018.04.007">10.1016/j.protis.2018.04.007</a>.</p>
<p><a name=bib-Venter2018521></a><a href="#cite-Venter2018521">Venter, P., F. Nitsche, A. Scherwass, et
al.</a> (2018). “Discrepancies between molecular and
morphological databases of soil ciliates studied for temperate
grasslands of central europe”. In: <em>Protist</em> 169.4, pp. 521-538. DOI:
<a href="https://doi.org/10.1016%2Fj.protis.2018.04.001">10.1016/j.protis.2018.04.001</a>.</p>
<p><a name=bib-Wang2018290></a><a href="#cite-Wang2018290">Wang, Y., T. Shi, G. Huang, et
al.</a> (2018). “Molecular detection of eukaryotic
diets and gut mycobiomes in two marine sediment-dwelling worms,
sipunculus nudus and urechis unicinctus”. In: <em>Microbes and
Environments</em> 33.3, pp. 290-300. DOI:
<a href="https://doi.org/10.1264%2Fjsme2.ME18065">10.1264/jsme2.ME18065</a>.</p>
<p><a name=bib-Wangensteen201854></a><a href="#cite-Wangensteen201854">Wangensteen, O., E. Cebrian, and C.
Palacín</a> (2018). “Under the canopy:
Community-wide effects of invasive algae in Marine Protected Areas
revealed by metabarcoding”. In: <em>Marine Pollution Bulletin</em> 127, pp.
54-66. DOI:
<a href="https://doi.org/10.1016%2Fj.marpolbul.2017.11.033">10.1016/j.marpolbul.2017.11.033</a>.</p>
<p><a name=bib-Ward2018574></a><a href="#cite-Ward2018574">Ward, G., S. Neuhauser, R. Groben, et
al.</a> (2018). “Environmental sequencing fills the gap
between parasitic haplosporidians and free-living giant amoebae”. In:
<em>Journal of Eukaryotic Microbiology</em> 65.5, pp. 574-586. DOI:
<a href="https://doi.org/10.1111%2Fjeu.12501">10.1111/jeu.12501</a>.</p>
<p><a name=bib-Wu20184627></a><a href="#cite-Wu20184627">Wu, W. and H. Liu</a>
(2018). “Disentangling protist communities identified from DNA and RNA
surveys in the Pearl RiverChina Sea Continuum during
the wet and dry seasons”. In: <em>Molecular Ecology</em> 27.22, pp. 4627-4640.
DOI: <a href="https://doi.org/10.1111%2Fmec.14867">10.1111/mec.14867</a>.</p>
<p><a name=bib-Wu2018485></a><a href="#cite-Wu2018485">Wu, W., H. Lu, A. Sastri, et
al.</a> (2018). “Contrasting the relative importance of
species sorting and dispersal limitation in shaping marine bacterial
versus protist communities”. In: <em>ISME Journal</em> 12.2, pp. 485-494. DOI:
<a href="https://doi.org/10.1038%2Fismej.2017.183">10.1038/ismej.2017.183</a>.</p>
<p><a name=bib-Wylezich201883></a><a href="#cite-Wylezich201883">Wylezich, C., D. Herlemann, and K.
Jürgens</a> (2018). “Improved 18S rDNA amplification
protocol for assessing protist diversity in oxygen-deficient marine
systems”. In: <em>Aquatic Microbial Ecology</em> 81.1, pp. 83-94. DOI:
<a href="https://doi.org/10.3354%2Fame01864">10.3354/ame01864</a>.</p>
<p><a name=bib-Xie2018189></a><a href="#cite-Xie2018189">Xie, Y., T. Floehr, X. Zhang, et
al.</a> (2018). “In situ microbiota distinguished
primary anthropogenic stressor in freshwater sediments”. In:
<em>Environmental Pollution</em> 239, pp. 189-197. DOI:
<a href="https://doi.org/10.1016%2Fj.envpol.2018.03.099">10.1016/j.envpol.2018.03.099</a>.</p>
<p><a name=bib-Xie2018739></a><a href="#cite-Xie2018739">Xie, Y., X. Zhang, J. Yang, et
al.</a> (2018). “eDNA-based bioassessment of coastal
sediments impacted by an oil spill”. In: <em>Environmental Pollution</em> 238,
pp. 739-748. DOI:
<a href="https://doi.org/10.1016%2Fj.envpol.2018.02.081">10.1016/j.envpol.2018.02.081</a>.</p>
<p><a name=bib-Zaiko2018></a><a href="#cite-Zaiko2018">Zaiko, A., X. Pochon, E. Garcia-Vazquez, et
al.</a> (2018). “Advantages and limitations of
environmental DNA/RNA tools for marine biosecurity: Management and
surveillance of non-indigenous species”. In: <em>Frontiers in Marine
Science</em> 5.SEP. DOI:
<a href="https://doi.org/10.3389%2Ffmars.2018.00322">10.3389/fmars.2018.00322</a>.</p>
<p><a name=bib-Zhao2018></a><a href="#cite-Zhao2018">Zhao, H., X. Li, Z. Zhang, et
al.</a> (2018). “Drivers and assemblies of soil eukaryotic
microbes among different soil habitat types in a semi-arid mountain in
China”. In: <em>PeerJ</em> 2018.12. DOI:
<a href="https://doi.org/10.7717%2Fpeerj.6042">10.7717/peerj.6042</a>.</p>
<p><a name=bib-Zhu2018751></a><a href="#cite-Zhu2018751">Zhu, P., Y. Wang, T. Shi, et
al.</a> (2018). “Genetic diversity of benthic microbial
eukaryotes in response to spatial heterogeneity of sediment
geochemistry in a mangrove ecosystem”. In: <em>Estuaries and Coasts</em> 41.3,
pp. 751-764. DOI:
<a href="https://doi.org/10.1007%2Fs12237-017-0317-z">10.1007/s12237-017-0317-z</a>.</p>
<p><a name=bib-Zouari201837></a><a href="#cite-Zouari201837">Zouari, A., M. Hassen, V. Balagué, et
al.</a> (2018). “Picoeukaryotic diversity in the Gulf
of Gabès: Variability patterns and relationships to nutrients and water
masses”. In: <em>Aquatic Microbial Ecology</em> 81.1, pp. 37-53. DOI:
<a href="https://doi.org/10.3354%2Fame01857">10.3354/ame01857</a>.</p>
</div>
<div id="section-5" class="section level2">
<h2>2017</h2>
<p><a name=bib-Alves-De-Souza2017698></a><a href="#cite-Alves-De-Souza2017698">Alves-De-Souza, C., T.
Benevides, J. Santos, et al.</a> (2017).
“Does environmental heterogeneity explain temporal <span class="math inline">\(\beta\)</span> diversity of
small eukaryotic phytoplankton? Example from a tropical eutrophic
coastal lagoon”. In: <em>Journal of Plankton Research</em> 39.4, pp. 698-714.
DOI:
<a href="https://doi.org/10.1093%2Fplankt%2Ffbx026">10.1093/plankt/fbx026</a>.</p>
<p><a name=bib-Berney2017407></a><a href="#cite-Berney2017407">Berney, C., A. Ciuprina, S. Bender, et
al.</a> (2017). “UniEuk: Time to speak a common
language in protistology!” In: <em>Journal of Eukaryotic Microbiology</em>
64.3, pp. 407-411. DOI:
<a href="https://doi.org/10.1111%2Fjeu.12414">10.1111/jeu.12414</a>.</p>
<p><a name=bib-Biard20171331></a><a href="#cite-Biard20171331">Biard, T., E. Bigeard, S. Audic, et
al.</a> (2017). “Biogeography and diversity of
Collodaria (Radiolaria) in the global ocean”. In: <em>ISME Journal</em> 11.6,
pp. 1331-1344. DOI:
<a href="https://doi.org/10.1038%2Fismej.2017.12">10.1038/ismej.2017.12</a>.</p>
<p><a name=bib-Christaki2017></a><a href="#cite-Christaki2017">Christaki, U., S. Genitsaris, S. Monchy,
et al.</a> (2017). “Parasitic eukaryotes in a
meso-eutrophic coastal system with marked Phaeocystis globosa blooms”.
In: <em>Frontiers in Marine Science</em> 4.DEC. DOI:
<a href="https://doi.org/10.3389%2Ffmars.2017.00416">10.3389/fmars.2017.00416</a>.</p>
<p><a name=bib-Deiner20175872></a><a href="#cite-Deiner20175872">Deiner, K., H. Bik, E. Mächler, et
al.</a> (2017). “Environmental DNA metabarcoding:
Transforming how we survey animal and plant communities”. In:
<em>Molecular Ecology</em> 26.21, pp. 5872-5895. DOI:
<a href="https://doi.org/10.1111%2Fmec.14350">10.1111/mec.14350</a>.</p>
<p><a name=bib-Domaizon2017></a><a href="#cite-Domaizon2017">Domaizon, I., A. Winegardner, E. Capo, et
al.</a> (2017). “DNA-based methods in paleolimnology:
new opportunities for investigating long-term dynamics of lacustrine
biodiversity”. In: <em>Journal of Paleolimnology</em> 58.1. DOI:
<a href="https://doi.org/10.1007%2Fs10933-017-9958-y">10.1007/s10933-017-9958-y</a>.</p>
<p><a name=bib-DosSantos2017512></a><a href="#cite-DosSantos2017512">Dos Santos, A., P. Gourvil, M.
Tragin, et al.</a> (2017). “Diversity and oceanic
distribution of prasinophytes clade VII, the dominant group of green
algae in oceanic waters”. In: <em>ISME Journal</em> 11.2, pp. 512-528. DOI:
<a href="https://doi.org/10.1038%2Fismej.2016.120">10.1038/ismej.2016.120</a>.</p>
<p><a name=bib-Dunthorn2017668></a><a href="#cite-Dunthorn2017668">Dunthorn, M., H. Kauserud, D. Bass, et
al.</a> (2017). “Yeasts dominate soil fungal
communities in three lowland Neotropical rainforests”. In:
<em>Environmental Microbiology Reports</em> 9.5, pp. 668-675. DOI:
<a href="https://doi.org/10.1111%2F1758-2229.12575">10.1111/1758-2229.12575</a>.</p>
<p><a name=bib-Filker20173186></a><a href="#cite-Filker20173186">Filker, S., D. Forster, L. Weinisch, et
al.</a> (2017). “Transition boundaries for protistan
species turnover in hypersaline waters of different biogeographic
regions”. In: <em>Environmental Microbiology</em> 19.8, pp. 3186-3200. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.13805">10.1111/1462-2920.13805</a>.</p>
<p><a name=bib-Fletcher20172851></a><a href="#cite-Fletcher20172851">Fletcher, L., A. Zaiko, J. Atalah, et
al.</a> (2017). “Bilge water as a vector for the
spread of marine pests: a morphological, metabarcoding and experimental
assessment”. In: <em>Biological Invasions</em> 19.10, pp. 2851-2867. DOI:
<a href="https://doi.org/10.1007%2Fs10530-017-1489-y">10.1007/s10530-017-1489-y</a>.</p>
<p><a name=bib-Foster2017></a><a href="#cite-Foster2017">Foster, Z., T. Sharpton, and N.
Grünwald</a> (2017). “Metacoder: An R package for
visualization and manipulation of community taxonomic diversity data”.
In: <em>PLoS Computational Biology</em> 13.2. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pcbi.1005404">10.1371/journal.pcbi.1005404</a>.</p>
<p><a name=bib-Frank201726></a><a href="#cite-Frank201726">Frank, N., T. Helge Abuldhauge, and J.
Daniel</a> (2017). “Bridging the gap between
morphological species and molecular barcodes Exemplified by
loricate choanoflagellates”. In: <em>European Journal of Protistology</em> 57,
pp. 26-37. DOI:
<a href="https://doi.org/10.1016%2Fj.ejop.2016.10.006">10.1016/j.ejop.2016.10.006</a>.</p>
<p><a name=bib-Gleason2017399></a><a href="#cite-Gleason2017399">Gleason, F., B. Scholz, T. Jephcott, et
al.</a> (2017). <em>Key ecological roles for zoosporic
true fungi in aquatic habitats</em> , pp. 399-416. DOI:
<a href="https://doi.org/10.1128%2F9781555819583.ch18">10.1128/9781555819583.ch18</a>.</p>
<p><a name=bib-Groendahl2017></a><a href="#cite-Groendahl2017">Groendahl, S., M. Kahlert, and P.
Fink</a> (2017). “The best of both worlds: A combined
approach for analyzing microalgal diversity via metabarcoding and
morphology-based methods”. In: <em>PLoS ONE</em> 12.2. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pone.0172808">10.1371/journal.pone.0172808</a>.</p>
<p><a name=bib-Heinz20171907></a><a href="#cite-Heinz20171907">Heinz, K., P. Zanoni, R. Oliveira, et
al.</a> (2017). “Recycled paper sludge microbial
community as a potential source of cellulase and xylanase enzymes”. In:
<em>Waste and Biomass Valorization</em> 8.6, pp. 1907-1917. DOI:
<a href="https://doi.org/10.1007%2Fs12649-016-9792-x">10.1007/s12649-016-9792-x</a>.</p>
<p><a name=bib-Hugerth2017></a><a href="#cite-Hugerth2017">Hugerth, L. and A.
Andersson</a> (2017). “Analysing microbial community
composition through amplicon sequencing: From sampling to hypothesis
testing”. In: <em>Frontiers in Microbiology</em> 8.SEP. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2017.01561">10.3389/fmicb.2017.01561</a>.</p>
<p><a name=bib-Jacob2017></a><a href="#cite-Jacob2017">Jacob, F., T. Melachio, G. Njitchouang, et
al.</a> (2017). “Intestinal bacterial communities of
trypanosome-infected and uninfected Glossina palpalis palpalis from
three human African trypanomiasis foci in Cameroon”. In: <em>Frontiers in
Microbiology</em> 8.AUG. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2017.01464">10.3389/fmicb.2017.01464</a>.</p>
<p><a name=bib-Kataoka2017></a><a href="#cite-Kataoka2017">Kataoka, T., H. Yamaguchi, M. Sato, et
al.</a> (2017). “Seasonal and geographical distribution
of near-surface small photosynthetic eukaryotes in the western north
pacific determined by pyrosequencing of 18S rDNA”. In: <em>FEMS
Microbiology Ecology</em> 93.2. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiw229">10.1093/femsec/fiw229</a>.</p>
<p><a name=bib-Kaysen201779></a><a href="#cite-Kaysen201779">Kaysen, A., A. Heintz-Buschart, E.
Muller, et al.</a> (2017). “Integrated meta-omic
analyses of the gastrointestinal tract microbiome in patients
undergoing allogeneic hematopoietic stem cell transplantation”. In:
<em>Translational Research</em> 186, pp. 79-94.e1. DOI:
<a href="https://doi.org/10.1016%2Fj.trsl.2017.06.008">10.1016/j.trsl.2017.06.008</a>.</p>
<p><a name=bib-Khomich2017></a><a href="#cite-Khomich2017">Khomich, M., H. Kauserud, R. Logares, et
al.</a> (2017). “Planktonic protistan communities in
lakes along a large-scale environmental gradient”. In: <em>FEMS
Microbiology Ecology</em> 93.4. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiw231">10.1093/femsec/fiw231</a>.</p>
<p><a name=bib-Laroche2017></a><a href="#cite-Laroche2017">Laroche, O., S. Wood, L. Tremblay, et
al.</a> (2017). “Metabarcoding monitoring analysis: The
pros and cons of using co-extracted environmental DNA and RNA data to
assess offshore oil production impacts on benthic communities”. In:
<em>PeerJ</em> 2017.5. DOI:
<a href="https://doi.org/10.7717%2Fpeerj.3347">10.7717/peerj.3347</a>.</p>
<p><a name=bib-Lee20171041></a><a href="#cite-Lee20171041">Lee, K., L. Ab Samad, P. Lwin, et
al.</a> (2017). “On the rocks: Microbiological quality
and microbial diversity of packaged ice in southern California”. In:
<em>Journal of Food Protection</em> 80.6, pp. 1041-1049. DOI:
<a href="https://doi.org/10.4315%2F0362-028X.JFP-16-295">10.4315/0362-028X.JFP-16-295</a>.</p>
<p><a name=bib-Leray2017></a><a href="#cite-Leray2017">Leray, M. and N. Knowlton</a>
(2017). “Random sampling causes the low reproducibility of rare
eukaryotic OTUs in Illumina COI metabarcoding”. In: <em>PeerJ</em> 2017.3.
DOI: <a href="https://doi.org/10.7717%2Fpeerj.3006">10.7717/peerj.3006</a>.</p>
<p><a name=bib-Machida2017></a><a href="#cite-Machida2017">Machida, R., M. Leray, S. Ho, et
al.</a> (2017). “Data Descriptor: Metazoan
mitochondrial gene sequence reference datasets for taxonomic assignment
of environmental samples”. In: <em>Scientific Data</em> 4. DOI:
<a href="https://doi.org/10.1038%2Fsdata.2017.27">10.1038/sdata.2017.27</a>.</p>
<p><a name=bib-Marie2017271></a><a href="#cite-Marie2017271">Marie, D., F. Le Gall, R. Edern, et
al.</a> (2017). “Improvement of phytoplankton culture
isolation using single cell sorting by flow cytometry”. In: <em>Journal of
Phycology</em> 53.2, pp. 271-282. DOI:
<a href="https://doi.org/10.1111%2Fjpy.12495">10.1111/jpy.12495</a>.</p>
<p><a name=bib-Meshram2017></a><a href="#cite-Meshram2017">Meshram, A., A. Vader, S. Kristiansen, et
al.</a> (2017). “Microbial eukaryotes in an arctic
under-ice spring bloom north of Svalbard”. In: <em>Frontiers in
Microbiology</em> 8.JUN. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2017.01099">10.3389/fmicb.2017.01099</a>.</p>
<p><a name=bib-Morard20172741></a><a href="#cite-Morard20172741">Morard, R., F. Lejzerowicz, K. Darling,
et al.</a> (2017). “Planktonic foraminifera-derived
environmental DNA extracted from abyssal sediments preserves patterns
of plankton macroecology”. In: <em>Biogeosciences (Online)</em> 14.11, pp.
2741-2754. DOI:
<a href="https://doi.org/10.5194%2Fbg-14-2741-2017">10.5194/bg-14-2741-2017</a>.</p>
<p><a name=bib-Mukherjee2017123></a><a href="#cite-Mukherjee2017123">Mukherjee, I., Y. Hodoki, and S.
Nakano</a> (2017). “Seasonal dynamics of
heterotrophic and plastidic protists in the water column of Lake Biwa,
Japan”. In: <em>Aquatic Microbial Ecology</em> 80.2, pp. 123-137. DOI:
<a href="https://doi.org/10.3354%2Fame01843">10.3354/ame01843</a>.</p>
<p><a name=bib-PagenkoppLohan2017680></a><a href="#cite-PagenkoppLohan2017680">Pagenkopp Lohan, K., R.
Fleischer, K. Carney, et al.</a> (2017).
“Molecular characterisation of protistan species and communities in
ships’ ballast water across three U.S. coasts”. In: <em>Diversity and
Distributions</em> 23.6, pp. 680-691. DOI:
<a href="https://doi.org/10.1111%2Fddi.12550">10.1111/ddi.12550</a>.</p>
<p><a name=bib-PagenkoppLohan2017183></a><a href="#cite-PagenkoppLohan2017183">Pagenkopp Lohan, K., R.
Fleischer, M. Torchin, et al.</a> (2017).
“Protistan biogeography: A snapshot across a major shipping corridor
spanning two oceans”. In: <em>Protist</em> 168.2, pp. 183-196. DOI:
<a href="https://doi.org/10.1016%2Fj.protis.2016.12.003">10.1016/j.protis.2016.12.003</a>.</p>
<p><a name=bib-Pan201788></a><a href="#cite-Pan201788">Pan, J., J. del Campo, and P.
Keeling</a> (2017). “Reference tree and environmental
sequence diversity of labyrinthulomycetes”. In: <em>Journal of Eukaryotic
Microbiology</em> 64.1, pp. 88-96. DOI:
<a href="https://doi.org/10.1111%2Fjeu.12342">10.1111/jeu.12342</a>.</p>
<p><a name=bib-Pearman2017></a><a href="#cite-Pearman2017">Pearman, J., J. Ellis, X. Irigoien, et
al.</a> (2017). “Microbial planktonic communities in
the Red Sea: high levels of spatial and temporal variability shaped by
nutrient availability and turbulence”. In: <em>Scientific Reports</em> 7.1.
DOI:
<a href="https://doi.org/10.1038%2Fs41598-017-06928-z">10.1038/s41598-017-06928-z</a>.</p>
<p><a name=bib-Penna201749></a><a href="#cite-Penna201749">Penna, A., S. Casabianca, A. Guerra, et
al.</a> (2017). “Analysis of phytoplankton assemblage
structure in the Mediterranean Sea based on high-throughput sequencing
of partial 18S rRNA sequences”. In: <em>Marine Genomics</em> 36, pp. 49-55.
DOI:
<a href="https://doi.org/10.1016%2Fj.margen.2017.06.001">10.1016/j.margen.2017.06.001</a>.</p>
<p><a name=bib-Piredda201731></a><a href="#cite-Piredda201731">Piredda, R., D. Sarno, C. Lange, et
al.</a> (2017). “Diatom resting stages in surface
sediments: A pilot study comparing next generation sequencing and
serial dilution cultures”. In: <em>Cryptogamie, Algologie</em> 38.1, pp.
31-46. DOI:
<a href="https://doi.org/10.7872%2Fcrya%2Fv38.iss1.2017.31">10.7872/crya/v38.iss1.2017.31</a>.</p>
<p><a name=bib-Piredda2017></a><a href="#cite-Piredda2017">Piredda, R., M. Tomasino, A. D’Erchia, et
al.</a> (2017). “Diversity and temporal patterns of
planktonic protist assemblages at a Mediterranean Long Term Ecological
Research site”. In: <em>FEMS Microbiology Ecology</em> 93.1. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiw200">10.1093/femsec/fiw200</a>.</p>
<p><a name=bib-Pochon2017></a><a href="#cite-Pochon2017">Pochon, X., A. Zaiko, L. Fletcher, et
al.</a> (2017). “Wanted dead or alive? Using
metabarcoding of environmental DNA and RNA to distinguish living
assemblages for biosecurity applications”. In: <em>PLoS ONE</em> 12.11. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pone.0187636">10.1371/journal.pone.0187636</a>.</p>
<p><a name=bib-Reich201757></a><a href="#cite-Reich201757">Reich, M. and A. Labes</a>
(2017). “How to boost marine fungal research: A first step towards a
multidisciplinary approach by combining molecular fungal ecology and
natural products chemistry”. In: <em>Marine Genomics</em> 36, pp. 57-75. DOI:
<a href="https://doi.org/10.1016%2Fj.margen.2017.09.007">10.1016/j.margen.2017.09.007</a>.</p>
<p><a name=bib-Salvitti2017335></a><a href="#cite-Salvitti2017335">Salvitti, L., S. Wood, R. Fairweather,
et al.</a> (2017). “In situ accumulation of
tetrodotoxin in non-toxic Pleurobranchaea maculata (Opisthobranchia)”.
In: <em>Aquatic Sciences</em> 79.2, pp. 335-344. DOI:
<a href="https://doi.org/10.1007%2Fs00027-016-0500-5">10.1007/s00027-016-0500-5</a>.</p>
<p><a name=bib-Santos2017283></a><a href="#cite-Santos2017283">Santos, S., I. Nunes, T. Nielsen, et
al.</a> (2017). “Soil DNA extraction procedure
influences protist 18S rRNA gene community profiling outcome”. In:
<em>Protist</em> 168.3, pp. 283-293. DOI:
<a href="https://doi.org/10.1016%2Fj.protis.2017.03.002">10.1016/j.protis.2017.03.002</a>.</p>
<p><a name=bib-Schnittler2017323></a><a href="#cite-Schnittler2017323">Schnittler, M., O. Shchepin, N.
Dagamac, et al.</a> (2017). “Barcoding
myxomycetes with molecular markers: Challenges and opportunities”. In:
<em>Nova Hedwigia</em> 104.1-3, pp. 323-341. DOI:
<a href="https://doi.org/10.1127%2Fnova_hedwigia%2F2017%2F0397">10.1127/nova_hedwigia/2017/0397</a>.</p>
<p><a name=bib-Seppey201768></a><a href="#cite-Seppey201768">Seppey, C., D. Singer, K. Dumack, et
al.</a> (2017). “Distribution patterns of soil
microbial eukaryotes suggests widespread algivory by phagotrophic
protists as an alternative pathway for nutrient cycling”. In: <em>Soil
Biology and Biochemistry</em> 112, pp. 68-76. DOI:
<a href="https://doi.org/10.1016%2Fj.soilbio.2017.05.002">10.1016/j.soilbio.2017.05.002</a>.</p>
<p><a name=bib-Simon2017612></a><a href="#cite-Simon2017612">Simon, N., E. Foulon, D. Grulois, et
al.</a> (2017). “Revision of the genus micromonas
manton et parke (chlorophyta, mamiellophyceae), of the type species M.
pusilla (butcher) manton &amp; parke and of the species M. commoda van
baren, bachy and worden and description of two new species based on the
genetic and phenotypic characterization of cultured isolates”. In:
<em>Protist</em> 168.5, pp. 612-635. DOI:
<a href="https://doi.org/10.1016%2Fj.protis.2017.09.002">10.1016/j.protis.2017.09.002</a>.</p>
<p><a name=bib-Smith2017555></a><a href="#cite-Smith2017555">Smith, K., G. Kohli, S. Murray, et
al.</a> (2017). “Assessment of the metabarcoding
approach for community analysis of benthic-epiphytic dinoflagellates
using mock communities”. In: <em>New Zealand Journal of Marine and
Freshwater Research</em> 51.4, pp. 555-576. DOI:
<a href="https://doi.org/10.1080%2F00288330.2017.1298632">10.1080/00288330.2017.1298632</a>.</p>
<p><a name=bib-Sun2017></a><a href="#cite-Sun2017">Sun, P., L. Huang, D. Xu, et
al.</a> (2017). “Marked seasonality and high spatial
variation in estuarine ciliates are driven by exchanges between the
‘abundant’ and ‘intermediate’ biospheres”. In: <em>Scientific Reports</em>
7.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-017-10308-y">10.1038/s41598-017-10308-y</a>.</p>
<p><a name=bib-Tirichine201746></a><a href="#cite-Tirichine201746">Tirichine, L., A. Rastogi, and C.
Bowler</a> (2017). “Recent progress in diatom
genomics and epigenomics”. In: <em>Current Opinion in Plant Biology</em> 36,
pp. 46-55. DOI:
<a href="https://doi.org/10.1016%2Fj.pbi.2017.02.001">10.1016/j.pbi.2017.02.001</a>.</p>
<p><a name=bib-Torruella2017113></a><a href="#cite-Torruella2017113">Torruella, G., D. Moreira, and P.
LÃ³pez-Garc'ia</a> (2017). “Phylogenetic and
ecological diversity of apusomonads, a lineage of deep-branching
eukaryotes”. In: <em>Environmental Microbiology Reports</em> 9.2, pp. 113-119.
DOI:
<a href="https://doi.org/10.1111%2F1758-2229.12507">10.1111/1758-2229.12507</a>.</p>
<p><a name=bib-Venter2017546></a><a href="#cite-Venter2017546">Venter, P., F. Nitsche, A. Domonell, et
al.</a> (2017). “The protistan microbiome of
grassland soil: Diversity in the mesoscale”. In: <em>Protist</em> 168.5, pp.
546-564. DOI:
<a href="https://doi.org/10.1016%2Fj.protis.2017.03.005">10.1016/j.protis.2017.03.005</a>.</p>
<p><a name=bib-Wampach2017></a><a href="#cite-Wampach2017">Wampach, L., A. Heintz-Buschart, A. Hogan,
et al.</a> (2017). “Colonization and succession within
the human gut microbiome by archaea, bacteria, and microeukaryotes
during the first year of life”. In: <em>Frontiers in Microbiology</em> 8.MAY.
DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2017.00738">10.3389/fmicb.2017.00738</a>.</p>
<p><a name=bib-Wang201755></a><a href="#cite-Wang201755">Wang, Y., X. Guo, P. Zheng, et
al.</a> (2017). “Distinct seasonality of
chytrid-dominated benthic fungal communities in the neritic oceans
(Bohai Sea and North Yellow Sea)”. In: <em>Fungal Ecology</em> 30, pp. 55-66.
DOI:
<a href="https://doi.org/10.1016%2Fj.funeco.2017.08.008">10.1016/j.funeco.2017.08.008</a>.</p>
<p><a name=bib-Weber2017153></a><a href="#cite-Weber2017153">Weber, F., A. Mylnikov, K. Jürgens, et
al.</a> (2017). “Culturing heterotrophic protists from
the baltic sea: Mostly the ``Usual suspects’’ but a few novelties as
well”. In: <em>Journal of Eukaryotic Microbiology</em> 64.2, pp. 153-163. DOI:
<a href="https://doi.org/10.1111%2Fjeu.12347">10.1111/jeu.12347</a>.</p>
<p><a name=bib-Wu2017287></a><a href="#cite-Wu2017287">Wu, W., R. Logares, B. Huang, et
al.</a> (2017). “Abundant and rare picoeukaryotic
sub-communities present contrasting patterns in the epipelagic waters
of marginal seas in the northwestern Pacific Ocean”. In: <em>Environmental
Microbiology</em> 19.1, pp. 287-300. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.13606">10.1111/1462-2920.13606</a>.</p>
<p><a name=bib-Xie2017166></a><a href="#cite-Xie2017166">Xie, Y., S. Hong, S. Kim, et
al.</a> (2017). “Ecogenomic responses of benthic
communities under multiple stressors along the marine and adjacent
riverine areas of northern Bohai Sea, China”. In: <em>Chemosphere</em> 172,
pp. 166-174. DOI:
<a href="https://doi.org/10.1016%2Fj.chemosphere.2016.12.121">10.1016/j.chemosphere.2016.12.121</a>.</p>
<p><a name=bib-Xie2017201></a><a href="#cite-Xie2017201">Xie, Y., J. Wang, J. Yang, et
al.</a> (2017). “Environmental DNA metabarcoding reveals
primary chemical contaminants in freshwater sediments from different
land-use types”. In: <em>Chemosphere</em> 172, pp. 201-209. DOI:
<a href="https://doi.org/10.1016%2Fj.chemosphere.2016.12.117">10.1016/j.chemosphere.2016.12.117</a>.</p>
<p><a name=bib-Xu2017370></a><a href="#cite-Xu2017370">Xu, D., N. Jiao, R. Ren, et
al.</a> (2017). “Distribution and diversity of microbial
eukaryotes in bathypelagic waters of the south china sea”. In: <em>Journal
of Eukaryotic Microbiology</em> 64.3, pp. 370-382. DOI:
<a href="https://doi.org/10.1111%2Fjeu.12372">10.1111/jeu.12372</a>.</p>
</div>
<div id="section-6" class="section level2">
<h2>2016</h2>
<p><a name=bib-Aguilar2016732></a><a href="#cite-Aguilar2016732">Aguilar, M., E. Richardson, B. Tan, et
al.</a> (2016). “Next-generation sequencing
assessment of eukaryotic diversity in oil sands tailings ponds
sediments and surface water”. In: <em>Journal of Eukaryotic Microbiology</em>
63.6, pp. 732-743. DOI:
<a href="https://doi.org/10.1111%2Fjeu.12320">10.1111/jeu.12320</a>.</p>
<p><a name=bib-Amato2016225></a><a href="#cite-Amato2016225">Amato, K., J. Metcalf, S. Song, et
al.</a> (2016). “Using the gut microbiota as a novel
tool for examining colobine primate GI health”. In: <em>Global Ecology and
Conservation</em> 7, pp. 225-237. DOI:
<a href="https://doi.org/10.1016%2Fj.gecco.2016.06.004">10.1016/j.gecco.2016.06.004</a>.</p>
<p><a name=bib-Angelakis2016></a><a href="#cite-Angelakis2016">Angelakis, E., M. Yasir, D. Bachar, et
al.</a> (2016). “Gut microbiome and dietary patterns
in different Saudi populations and monkeys”. In: <em>Scientific Reports</em>
6. DOI: <a href="https://doi.org/10.1038%2Fsrep32191">10.1038/srep32191</a>.</p>
<p><a name=bib-Bass2016195></a><a href="#cite-Bass2016195">Bass, D. and T. Bell</a>
(2016). <em>Protist systematics, ecology and next generation sequencing</em> ,
pp. 195-216. DOI:
<a href="https://doi.org/10.1017%2FCBO9781139236355.010">10.1017/CBO9781139236355.010</a>.</p>
<p><a name=bib-Bass20161604></a><a href="#cite-Bass20161604">Bass, D., J. Silberman, M. Brown, et
al.</a> (2016). “Coprophilic amoebae and flagellates,
including Guttulinopsis, Rosculus and Helkesimastix, characterise a
divergent and diverse rhizarian radiation and contribute to a large
diversity of faecal-associated protists”. In: <em>Environmental
Microbiology</em> 18.5, pp. 1604-1619. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.13235">10.1111/1462-2920.13235</a>.</p>
<p><a name=bib-Boyer2016176></a><a href="#cite-Boyer2016176">Boyer, F., C. Mercier, A. Bonin, et
al.</a> (2016). “obitools: A unix-inspired software
package for DNA metabarcoding”. In: <em>Molecular Ecology Resources</em> 16.1,
pp. 176-182. DOI:
<a href="https://doi.org/10.1111%2F1755-0998.12428">10.1111/1755-0998.12428</a>.</p>
<p><a name=bib-delCampo20164></a><a href="#cite-delCampo20164">Campo, J. del, L. Guillou, E.
Hehenberger, et al.</a> (2016). “Ecological and
evolutionary significance of novel protist lineages”. In: <em>European
Journal of Protistology</em> 55, pp. 4-11. DOI:
<a href="https://doi.org/10.1016%2Fj.ejop.2016.02.002">10.1016/j.ejop.2016.02.002</a>.</p>
<p><a name=bib-Cleary2016401></a><a href="#cite-Cleary2016401">Cleary, A. and E.
Durbin</a> (2016). “Unexpected prevalence of parasite
18S rDNA sequences in winter among Antarctic marine protists”. In:
<em>Journal of Plankton Research</em> 38.3, pp. 401-417. DOI:
<a href="https://doi.org/10.1093%2Fplankt%2Ffbw005">10.1093/plankt/fbw005</a>.</p>
<p><a name=bib-Coelho2016></a><a href="#cite-Coelho2016">Coelho, F., A. Louvado, P. Domingues, et
al.</a> (2016). “Integrated analysis of bacterial and
microeukaryotic communities from differentially active mud volcanoes in
the Gulf of Cadiz”. In: <em>Scientific Reports</em> 6. DOI:
<a href="https://doi.org/10.1038%2Fsrep35272">10.1038/srep35272</a>.</p>
<p><a name=bib-Cooper201638></a><a href="#cite-Cooper201638">Cooper, M., D. Phalen, S. Donahoe, et
al.</a> (2016). “The utility of diversity profiling
using Illumina 18S rRNA gene amplicon deep sequencing to detect and
discriminate Toxoplasma gondii among the cyst-forming coccidia”. In:
<em>Veterinary Parasitology</em> 216, pp. 38-45. DOI:
<a href="https://doi.org/10.1016%2Fj.vetpar.2015.12.011">10.1016/j.vetpar.2015.12.011</a>.</p>
<p><a name=bib-Cornejo-Castillo2016></a><a href="#cite-Cornejo-Castillo2016">Cornejo-Castillo, F., A. Cabello,
G. Salazar, et al.</a> (2016). “Cyanobacterial
symbionts diverged in the late Cretaceous towards lineage-specific
nitrogen fixation factories in single-celled phytoplankton”. In:
<em>Nature Communications</em> 7. DOI:
<a href="https://doi.org/10.1038%2Fncomms11071">10.1038/ncomms11071</a>.</p>
<p><a name=bib-Dubourg2016></a><a href="#cite-Dubourg2016">Dubourg, G., J. Lagier, S. Hüe, et
al.</a> (2016). “Gut microbiota associated with HIV
infection is significantly enriched in bacteria tolerant to oxygen”.
In: <em>BMJ Open Gastroenterology</em> 3.1. DOI:
<a href="https://doi.org/10.1136%2Fbmjgast-2016-000080">10.1136/bmjgast-2016-000080</a>.</p>
<p><a name=bib-Ferrera2016></a><a href="#cite-Ferrera2016">Ferrera, I., C. Giner, A. Reñé, et
al.</a> (2016). “Evaluation of alternative
high-throughput sequencing methodologies for the monitoring of marine
picoplanktonic biodiversity based on rRNA gene amplicons”. In:
<em>Frontiers in Marine Science</em> 3.AUG. DOI:
<a href="https://doi.org/10.3389%2Ffmars.2016.00147">10.3389/fmars.2016.00147</a>.</p>
<p><a name=bib-Forster2016></a><a href="#cite-Forster2016">Forster, D., M. Dunthorn, F. Mahé, et
al.</a> (2016). “Benthic protists: The under-charted
majority”. In: <em>FEMS Microbiology Ecology</em> 92.8. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiw120">10.1093/femsec/fiw120</a>.</p>
<p><a name=bib-Forster20161></a><a href="#cite-Forster20161">Forster, D., M. Dunthorn, T. Stoeck, et
al.</a> (2016). “Comparison of three clustering
approaches for detecting novel environmental microbial diversity”. In:
<em>PeerJ</em> 2016.2. DOI:
<a href="https://doi.org/10.7717%2Fpeerj.1692">10.7717/peerj.1692</a>.</p>
<p><a name=bib-Genitsaris201661></a><a href="#cite-Genitsaris201661">Genitsaris, S., S. Monchy, E. Breton,
et al.</a> (2016). “Small-scale variability of
protistan planktonic communities relative to environmental pressures
and biotic interactions at two adjacent coastal stations”. In: <em>Marine
Ecology Progress Series</em> 548, pp. 61-75. DOI:
<a href="https://doi.org/10.3354%2Fmeps11647">10.3354/meps11647</a>.</p>
<p><a name=bib-Genitsaris201630></a><a href="#cite-Genitsaris201630">Genitsaris, S., S. Monchy, J.
Denonfoux, et al.</a> (2016). “Marine microbial
community structure assessed from combined metagenomic analysis and
ribosomal amplicon deep-sequencing”. In: <em>Marine Biology Research</em>
12.1, pp. 30-42. DOI:
<a href="https://doi.org/10.1080%2F17451000.2015.1084425">10.1080/17451000.2015.1084425</a>.</p>
<p><a name=bib-Gimmler2016></a><a href="#cite-Gimmler2016">Gimmler, A., R. Korn, C. De Vargas, et
al.</a> (2016). “The Tara Oceans voyage reveals global
diversity and distribution patterns of marine planktonic ciliates”. In:
<em>Scientific Reports</em> 6. DOI:
<a href="https://doi.org/10.1038%2Fsrep33555">10.1038/srep33555</a>.</p>
<p><a name=bib-Giner20164757></a><a href="#cite-Giner20164757">Giner, C., I. Forn, S. Romac, et
al.</a> (2016). “Environmental sequencing provides
reasonable estimates of the relative abundance of specific
picoeukaryotes”. In: <em>Applied and Environmental Microbiology</em> 82.15,
pp. 4757-4766. DOI:
<a href="https://doi.org/10.1128%2FAEM.00560-16">10.1128/AEM.00560-16</a>.</p>
<p><a name=bib-deGroot2016125></a><a href="#cite-deGroot2016125">Groot, G. de, I. Laros, and S.
Geisen</a> (2016). “Molecular identification of soil
eukaryotes and focused approaches targeting protist and faunal groups
using high-throughput metabarcoding”. In: <em>Methods in Molecular
Biology</em> 1399, pp. 125-140. DOI:
<a href="https://doi.org/10.1007%2F978-1-4939-3369-3_7">10.1007/978-1-4939-3369-3_7</a>.</p>
<p><a name=bib-Guardiola2016></a><a href="#cite-Guardiola2016">Guardiola, M., O. Wangensteen, P.
Taberlet, et al.</a> (2016). “Spatio-temporal
monitoring of deep-sea communities using metabarcoding of sediment DNA
and RNA”. In: <em>PeerJ</em> 2016.12. DOI:
<a href="https://doi.org/10.7717%2Fpeerj.2807">10.7717/peerj.2807</a>.</p>
<p><a name=bib-Hu2016></a><a href="#cite-Hu2016">Hu, Y., B. Karlson, S. Charvet, et
al.</a> (2016). “Diversity of pico- to mesoplankton along
the 2000 km salinity gradient of the baltic sea”. In: <em>Frontiers in
Microbiology</em> 7.MAY. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2016.00679">10.3389/fmicb.2016.00679</a>.</p>
<p><a name=bib-Ichinomiya20162419></a><a href="#cite-Ichinomiya20162419">Ichinomiya, M., A. Dos Santos, P.
Gourvil, et al.</a> (2016). “Diversity and
oceanic distribution of the Parmales (Bolidophyceae), a picoplanktonic
group closely related to diatoms”. In: <em>ISME Journal</em> 10.10, pp.
2419-2434. DOI:
<a href="https://doi.org/10.1038%2Fismej.2016.38">10.1038/ismej.2016.38</a>.</p>
<p><a name=bib-Kozlov20165022></a><a href="#cite-Kozlov20165022">Kozlov, A., J. Zhang, P. Yilmaz, et
al.</a> (2016). “Phylogeny-aware identification and
correction of taxonomically mislabeled sequences”. In: <em>Nucleic Acids
Research</em> 44.11, pp. 5022-5033. DOI:
<a href="https://doi.org/10.1093%2Fnar%2Fgkw396">10.1093/nar/gkw396</a>.</p>
<p><a name=bib-LeBescot2016609></a><a href="#cite-LeBescot2016609">Le Bescot, N., F. Mahé, S. Audic, et
al.</a> (2016). “Global patterns of pelagic
dinoflagellate diversity across protist size classes unveiled by
metabarcoding”. In: <em>Environmental Microbiology</em> 18.2, pp. 609-626.
DOI:
<a href="https://doi.org/10.1111%2F1462-2920.13039">10.1111/1462-2920.13039</a>.</p>
<p><a name=bib-Leray2016></a><a href="#cite-Leray2016">Leray, M. and N. Knowlton</a>
(2016). “Censusing marine eukaryotic diversity in the twenty-first
century”. In: <em>Philosophical Transactions of the Royal Society B:
Biological Sciences</em> 371.1702. DOI:
<a href="https://doi.org/10.1098%2Frstb.2015.0331">10.1098/rstb.2015.0331</a>.</p>
<p><a name=bib-Malviya2016E1516></a><a href="#cite-Malviya2016E1516">Malviya, S., E. Scalco, S. Audic, et
al.</a> (2016). “Insights into global diatom
distribution and diversity in the world’s ocean”. In: <em>Proceedings of
the National Academy of Sciences of the United States of America</em>
113.11, pp. E1516-E1525. DOI:
<a href="https://doi.org/10.1073%2Fpnas.1509523113">10.1073/pnas.1509523113</a>.</p>
<p><a name=bib-Marquardt20161868></a><a href="#cite-Marquardt20161868">Marquardt, M., A. Vader, E. Stübner,
et al.</a> (2016). “Strong seasonality of marine
microbial eukaryotes in a high-Arctic fjord (Isfjorden, in West
Spitsbergen, Norway)”. In: <em>Applied and Environmental Microbiology</em>
82.6, pp. 1868-1880. DOI:
<a href="https://doi.org/10.1128%2FAEM.03208-15">10.1128/AEM.03208-15</a>.</p>
<p><a name=bib-More2016651></a><a href="#cite-More2016651">More, R. and H.
Purohit</a> (2016). “The identification of
discriminating patterns from 16S rRNA gene to generate signature for
bacillus genus”. In: <em>Journal of Computational Biology</em> 23.8, pp.
651-661. DOI:
<a href="https://doi.org/10.1089%2Fcmb.2016.0002">10.1089/cmb.2016.0002</a>.</p>
<p><a name=bib-Moustaka-Gouni2016389></a><a href="#cite-Moustaka-Gouni2016389">Moustaka-Gouni, M., K. Kormas,
M. Scotti, et al.</a> (2016). “Warming and
acidification effects on planktonic heterotrophic pico- and
nanoflagellates in a mesocosm experiment”. In: <em>Protist</em> 167.4, pp.
389-410. DOI:
<a href="https://doi.org/10.1016%2Fj.protis.2016.06.004">10.1016/j.protis.2016.06.004</a>.</p>
<p><a name=bib-Nitsche201658></a><a href="#cite-Nitsche201658">Nitsche, F.</a>
(2016). “A phylogenetic and morphological re-investigation of
Diaphanoeca spiralifurca, Didymoeca elongata and Polyoeca dichotoma
(Acanthoecida/Choanomonadida) from the Caribbean Sea”. In: <em>European
Journal of Protistology</em> 52, pp. 58-64. DOI:
<a href="https://doi.org/10.1016%2Fj.ejop.2015.11.001">10.1016/j.ejop.2015.11.001</a>.</p>
<p><a name=bib-Obbels20161></a><a href="#cite-Obbels20161">Obbels, D., E. Verleyen, M. Mano, et
al.</a> (2016). “Bacterial and eukaryotic biodiversity
patterns in terrestrial and aquatic habitats in the sør rondane
mountains, dronning maud land, east antarctica”. In: <em>FEMS Microbiology
Ecology</em> 92.6, pp. 1-13. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiw041">10.1093/femsec/fiw041</a>.</p>
<p><a name=bib-PagenkoppLohan2016530></a><a href="#cite-PagenkoppLohan2016530">Pagenkopp Lohan, K., R.
Fleischer, K. Carney, et al.</a> (2016).
“Amplicon-based pyrosequencing reveals high diversity of protistan
parasites in ships’ ballast water: Implications for biogeography and
infectious diseases”. In: <em>Microbial Ecology</em> 71.3, pp. 530-542. DOI:
<a href="https://doi.org/10.1007%2Fs00248-015-0684-6">10.1007/s00248-015-0684-6</a>.</p>
<p><a name=bib-Pawlowski201612></a><a href="#cite-Pawlowski201612">Pawlowski, J., F. Lejzerowicz, L.
Apotheloz-Perret-Gentil, et al.</a> (2016).
“Protist metabarcoding and environmental biomonitoring: Time for
change”. In: <em>European Journal of Protistology</em> 55, pp. 12-25. DOI:
<a href="https://doi.org/10.1016%2Fj.ejop.2016.02.003">10.1016/j.ejop.2016.02.003</a>.</p>
<p><a name=bib-Pearman201620></a><a href="#cite-Pearman201620">Pearman, J., H. Anlauf, X. Irigoien, et
al.</a> (2016). “Please mind the gap - Visual census
and cryptic biodiversity assessment at central Red Sea coral reefs”.
In: <em>Marine Environmental Research</em> 118, pp. 20-30. DOI:
<a href="https://doi.org/10.1016%2Fj.marenvres.2016.04.011">10.1016/j.marenvres.2016.04.011</a>.</p>
<p><a name=bib-Pearman2016></a><a href="#cite-Pearman2016">Pearman, J., S. Kürten, Y. Sarma, et
al.</a> (2016). “Biodiversity patterns of plankton
assemblages at the extremes of the Red Sea”. In: <em>FEMS Microbiology
Ecology</em> 92.3. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiw002">10.1093/femsec/fiw002</a>.</p>
<p><a name=bib-Pernice2016945></a><a href="#cite-Pernice2016945">Pernice, M., C. Giner, R. Logares, et
al.</a> (2016). “Large variability of bathypelagic
microbial eukaryotic communities across the world’s oceans”. In: <em>ISME
Journal</em> 10.4, pp. 945-958. DOI:
<a href="https://doi.org/10.1038%2Fismej.2015.170">10.1038/ismej.2015.170</a>.</p>
<p><a name=bib-Rimet2016></a><a href="#cite-Rimet2016">Rimet, F., P. Chaumeil, F. Keck, et
al.</a> (2016). “R-Syst::diatom: An open-access and
curated barcode database for diatoms and freshwater monitoring”. In:
<em>Database-the Magazine of Electronic Database Reviews</em> 2016. DOI:
<a href="https://doi.org/10.1093%2Fdatabase%2Fbaw016">10.1093/database/baw016</a>.</p>
<p><a name=bib-Rocke201696></a><a href="#cite-Rocke201696">Rocke, E., H. Jing, X. Xia, et
al.</a> (2016). “Effects of hypoxia on the phylogenetic
composition and species distribution of protists in a subtropical
harbor”. In: <em>Microbial Ecology</em> 72.1, pp. 96-105. DOI:
<a href="https://doi.org/10.1007%2Fs00248-016-0751-7">10.1007/s00248-016-0751-7</a>.</p>
<p><a name=bib-Schiaffino20165249></a><a href="#cite-Schiaffino20165249">Schiaffino, M., E. Lara, L.
Fernández, et al.</a> (2016). “Microbial
eukaryote communities exhibit robust biogeographical patterns along a
gradient of Patagonian and Antarctic lakes”. In: <em>Environmental
Microbiology</em> 18.12, pp. 5249-5264. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.13566">10.1111/1462-2920.13566</a>.</p>
<p><a name=bib-Scholz201659></a><a href="#cite-Scholz201659">Scholz, B., L. Guillou, A. Marano, et
al.</a> (2016). “Zoosporic parasites infecting marine
diatoms-A black box that needs to be opened”. In: <em>Fungal Ecology</em> 19,
pp. 59-76. DOI:
<a href="https://doi.org/10.1016%2Fj.funeco.2015.09.002">10.1016/j.funeco.2015.09.002</a>.</p>
<p><a name=bib-Shazib2016128></a><a href="#cite-Shazib2016128">Shazib, S., P. Vdacn'y, J. Kim, et
al.</a> (2016). “Molecular phylogeny and species
delimitation within the ciliate genus Spirostomum (Ciliophora,
Postciliodesmatophora, Heterotrichea), using the internal transcribed
spacer region”. In: <em>Molecular Phylogenetics and Evolution</em> 102, pp.
128-144. DOI:
<a href="https://doi.org/10.1016%2Fj.ympev.2016.05.041">10.1016/j.ympev.2016.05.041</a>.</p>
<p><a name=bib-Simon2016></a><a href="#cite-Simon2016">Simon, M., P. LÃ³pez-Garc'ia, P. Deschamps,
et al.</a> (2016). “Resilience of freshwater communities
of small microbial eukaryotes undergoing severe drought events”. In:
<em>Frontiers in Microbiology</em> 7.MAY. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2016.00812">10.3389/fmicb.2016.00812</a>.</p>
<p><a name=bib-Singer201642></a><a href="#cite-Singer201642">Singer, D., E. Lara, M. Steciow, et
al.</a> (2016). “High-throughput sequencing reveals
diverse oomycete communities in oligotrophic peat bog micro-habitat”.
In: <em>Fungal Ecology</em> 23, pp. 42-47. DOI:
<a href="https://doi.org/10.1016%2Fj.funeco.2016.05.009">10.1016/j.funeco.2016.05.009</a>.</p>
<p><a name=bib-Sun2016></a><a href="#cite-Sun2016">Sun, P., J. Clamp, D. Xu, et
al.</a> (2016). “An integrative approach to phylogeny
reveals patterns of environmental distribution and novel evolutionary
relationships in a major group of ciliates”. In: <em>Scientific Reports</em>
6. DOI: <a href="https://doi.org/10.1038%2Fsrep21695">10.1038/srep21695</a>.</p>
<p><a name=bib-Volant2016608></a><a href="#cite-Volant2016608">Volant, A., M. Héry, A. Desoeuvre, et
al.</a> (2016). “Spatial distribution of eukaryotic
communities using high-throughput sequencing along a pollution gradient
in the arsenic-rich creek sediments of carnoulès mine, france”. In:
<em>Microbial Ecology</em> 72.3, pp. 608-620. DOI:
<a href="https://doi.org/10.1007%2Fs00248-016-0826-5">10.1007/s00248-016-0826-5</a>.</p>
<p><a name=bib-Voudanta2016></a><a href="#cite-Voudanta2016">Voudanta, E., K. Kormas, S. Monchy, et
al.</a> (2016). “Mussel biofiltration effects on
attached bacteria and unicellular eukaryotes in fish-rearing seawater”.
In: <em>PeerJ</em> 2016.3. DOI:
<a href="https://doi.org/10.7717%2Fpeerj.1829">10.7717/peerj.1829</a>.</p>
<p><a name=bib-Zaiko2016671></a><a href="#cite-Zaiko2016671">Zaiko, A., K. Schimanski, X. Pochon, et
al.</a> (2016). “Metabarcoding improves detection of
eukaryotes from early biofouling communities: implications for pest
monitoring and pathway management”. In: <em>Biofouling</em> 32.6, pp. 671-684.
DOI:
<a href="https://doi.org/10.1080%2F08927014.2016.1186165">10.1080/08927014.2016.1186165</a>.</p>
<p><a name=bib-Zoccarato201616></a><a href="#cite-Zoccarato201616">Zoccarato, L., A. Pallavicini, F.
Cerino, et al.</a> (2016). “Water mass dynamics
shape Ross Sea protist communities in mesopelagic and bathypelagic
layers”. In: <em>Progress in Oceanography</em> 149, pp. 16-26. DOI:
<a href="https://doi.org/10.1016%2Fj.pocean.2016.10.003">10.1016/j.pocean.2016.10.003</a>.</p>
</div>
<div id="section-7" class="section level2">
<h2>2015</h2>
<p><a name=bib-Balzano2015263></a><a href="#cite-Balzano2015263">Balzano, S., E. Abs, and S.
Leterme</a> (2015). “Protist diversity along a
salinity gradient in a coastal lagoon”. In: <em>Aquatic Microbial Ecology</em>
74.3, pp. 263-277. DOI:
<a href="https://doi.org/10.3354%2Fame01740">10.3354/ame01740</a>.</p>
<p><a name=bib-Bass2015499></a><a href="#cite-Bass2015499">Bass, D., G. Stentiford, D. Littlewood, et
al.</a> (2015). “Diverse applications of environmental
DNA methods in parasitology”. In: <em>Trends in Parasitology</em> 31.10, pp.
499-513. DOI:
<a href="https://doi.org/10.1016%2Fj.pt.2015.06.013">10.1016/j.pt.2015.06.013</a>.</p>
<p><a name=bib-delCampo20153195></a><a href="#cite-delCampo20153195">Campo, J. del, D. Mallo, R. Massana,
et al.</a> (2015). “Diversity and distribution of
unicellular opisthokonts along the European coast analysed using
high-throughput sequencing”. In: <em>Environmental Microbiology</em> 17.9, pp.
3195-3207. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.12759">10.1111/1462-2920.12759</a>.</p>
<p><a name=bib-Christaki2015></a><a href="#cite-Christaki2015">Christaki, U., C. Georges, S.
Genitsaris, et al.</a> (2015). “Microzooplankton
community associated with phytoplankton blooms in the naturally
iron-fertilized Kerguelen area (Southern Ocean)”. In: <em>FEMS
Microbiology Ecology</em> 91.7. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiv068">10.1093/femsec/fiv068</a>.</p>
<p><a name=bib-DeVargas2015></a><a href="#cite-DeVargas2015">De Vargas, C., S. Audic, N. Henry, et
al.</a> (2015). “Eukaryotic plankton diversity in the
sunlit ocean”. In: <em>Science (New York, N.Y.)</em> 348.6237. DOI:
<a href="https://doi.org/10.1126%2Fscience.1261605">10.1126/science.1261605</a>.</p>
<p><a name=bib-Decelle20151435></a><a href="#cite-Decelle20151435">Decelle, J., S. Romac, R. Stern, et
al.</a> (2015). “PhytoREF: A reference database of
the plastidial 16S rRNA gene of photosynthetic eukaryotes with curated
taxonomy”. In: <em>Molecular Ecology Resources</em> 15.6, pp. 1435-1445. DOI:
<a href="https://doi.org/10.1111%2F1755-0998.12401">10.1111/1755-0998.12401</a>.</p>
<p><a name=bib-Egge2015121></a><a href="#cite-Egge2015121">Egge, E., W. Eikrem, and B.
Edvardsen</a> (2015). “Deep-branching novel lineages
and high diversity of haptophytes in the Skagerrak (Norway) uncovered
by 454 pyrosequencing”. In: <em>Journal of Eukaryotic Microbiology</em> 62.1,
pp. 121-140. DOI:
<a href="https://doi.org/10.1111%2Fjeu.12157">10.1111/jeu.12157</a>.</p>
<p><a name=bib-Esling20152513></a><a href="#cite-Esling20152513">Esling, P., F. Lejzerowicz, and J.
Pawlowski</a> (2015). “Accurate multiplexing and
filtering for high-throughput amplicon-sequencing”. In: <em>Nucleic Acids
Research</em> 43.5, pp. 2513-2524. DOI:
<a href="https://doi.org/10.1093%2Fnar%2Fgkv107">10.1093/nar/gkv107</a>.</p>
<p><a name=bib-Geisen20154556></a><a href="#cite-Geisen20154556">Geisen, S., I. Laros, A. Vizcaíno, et
al.</a> (2015). “Not all are free-living:
High-throughput DNA metabarcoding reveals a diverse community of
protists parasitizing soil metazoa”. In: <em>Molecular Ecology</em> 24.17, pp.
4556-4569. DOI:
<a href="https://doi.org/10.1111%2Fmec.13238">10.1111/mec.13238</a>.</p>
<p><a name=bib-Genitsaris2015></a><a href="#cite-Genitsaris2015">Genitsaris, S., S. Monchy, E.
Viscogliosi, et al.</a> (2015). “Seasonal variations
of marine protist community structure based on taxon-specific traits
using the eastern English Channel as a model coastal system”. In: <em>FEMS
Microbiology Ecology</em> 91.5. DOI:
<a href="https://doi.org/10.1093%2Ffemsec%2Ffiv034">10.1093/femsec/fiv034</a>.</p>
<p><a name=bib-Goltsman20151014></a><a href="#cite-Goltsman20151014">Goltsman, D., L. Comolli, B. Thomas,
et al.</a> (2015). “Community transcriptomics
reveals unexpected high microbial diversity in acidophilic biofilm
communities”. In: <em>ISME Journal</em> 9, pp. 1014-1023. DOI:
<a href="https://doi.org/10.1038%2Fismej.2014.200">10.1038/ismej.2014.200</a>.</p>
<p><a name=bib-Gong20153722></a><a href="#cite-Gong20153722">Gong, J., F. Shi, B. Ma, et
al.</a> (2015). “Depth shapes <span class="math inline">\(\alpha\)</span>- and
<span class="math inline">\(\beta\)</span>-diversities of microbial eukaryotes in surficial sediments of
coastal ecosystems”. In: <em>Environmental Microbiology</em> 17.10, pp.
3722-3737. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.12763">10.1111/1462-2920.12763</a>.</p>
<p><a name=bib-Guardiola2015></a><a href="#cite-Guardiola2015">Guardiola, M., M. Uriz, P. Taberlet, et
al.</a> (2015). “Deep-sea, deep-sequencing:
Metabarcoding extracellular DNA from sediments of marine canyons”. In:
<em>PLoS ONE</em> 10.10. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pone.0139633">10.1371/journal.pone.0139633</a>.</p>
<p><a name=bib-Jeffries2015></a><a href="#cite-Jeffries2015">Jeffries, T., M. Ostrowski, R. Williams,
et al.</a> (2015). “Spatially extensive microbial
biogeography of the Indian Ocean provides insights into the unique
community structure of a pristine coral atoll”. In: <em>Scientific
Reports</em> 5. DOI:
<a href="https://doi.org/10.1038%2Fsrep15383">10.1038/srep15383</a>.</p>
<p><a name=bib-Jing201513483></a><a href="#cite-Jing201513483">Jing, H., E. Rocke, L. Kong, et
al.</a> (2015). “Protist communities in a marine
oxygen minimum zone off Costa Rica by 454 pyrosequencing”. In:
<em>Biogeosciences Discussions</em> 12.16, pp. 13483-13509. DOI:
<a href="https://doi.org/10.5194%2Fbgd-12-13483-2015">10.5194/bgd-12-13483-2015</a>.</p>
<p><a name=bib-Lejzerowicz2015></a><a href="#cite-Lejzerowicz2015">Lejzerowicz, F., P. Esling, L. Pillet,
et al.</a> (2015). “High-throughput sequencing and
morphology perform equally well for benthic monitoring of marine
ecosystems”. In: <em>Scientific Reports</em> 5. DOI:
<a href="https://doi.org/10.1038%2Fsrep13932">10.1038/srep13932</a>.</p>
<p><a name=bib-Lima-Mendez2015></a><a href="#cite-Lima-Mendez2015">Lima-Mendez, G., K. Faust, N. Henry,
et al.</a> (2015). “Determinants of community
structure in the global plankton interactome”. In: <em>Science (New York,
N.Y.)</em> 348.6237. DOI:
<a href="https://doi.org/10.1126%2Fscience.1262073">10.1126/science.1262073</a>.</p>
<p><a name=bib-Massana20153></a><a href="#cite-Massana20153">Massana, R.</a> (2015).
<em>Protistan diversity in environmental molecular surveys</em> , pp. 3-21.
DOI:
<a href="https://doi.org/10.1007%2F978-4-431-55130-0_1">10.1007/978-4-431-55130-0_1</a>.</p>
<p><a name=bib-Massana20154035></a><a href="#cite-Massana20154035">Massana, R., A. Gobet, S. Audic, et
al.</a> (2015). “Marine protist diversity in
European coastal waters and sediments as revealed by high-throughput
sequencing”. In: <em>Environmental Microbiology</em> 17.10, pp. 4035-4049.
DOI:
<a href="https://doi.org/10.1111%2F1462-2920.12955">10.1111/1462-2920.12955</a>.</p>
<p><a name=bib-Morard20151472></a><a href="#cite-Morard20151472">Morard, R., K. Darling, F. Mahé, et
al.</a> (2015). “PFR2: A curated database of
planktonic foraminifera 18S ribosomal DNA as a resource for studies of
plankton ecology, biogeography and evolution”. In: <em>Molecular Ecology
Resources</em> 15.6, pp. 1472-1485. DOI:
<a href="https://doi.org/10.1111%2F1755-0998.12410">10.1111/1755-0998.12410</a>.</p>
<p><a name=bib-Pochon2015241></a><a href="#cite-Pochon2015241">Pochon, X., A. Zaiko, G. Hopkins, et
al.</a> (2015). “Early detection of eukaryotic
communities from marine biofilm using high-throughput sequencing: an
assessment of different sampling devices”. In: <em>Biofouling</em> 31.3, pp.
241-251. DOI:
<a href="https://doi.org/10.1080%2F08927014.2015.1028923">10.1080/08927014.2015.1028923</a>.</p>
<p><a name=bib-Pringault2015189></a><a href="#cite-Pringault2015189">Pringault, O., J. Aube, O. Bouchez,
et al.</a> (2015). “Contrasted effects of natural
complex mixtures of PAHs and metals on oxygen cycle in a microbial
mat”. In: <em>Chemosphere</em> 135, pp. 189-201. DOI:
<a href="https://doi.org/10.1016%2Fj.chemosphere.2015.04.037">10.1016/j.chemosphere.2015.04.037</a>.</p>
<p><a name=bib-Richards2015></a><a href="#cite-Richards2015">Richards, T., G. Leonard, F. Mahé, et
al.</a> (2015). “Molecular diversity and distribution
of marine fungi across 130 european environmental samples”. In:
<em>Proceedings of the Royal Society B: Biological Sciences</em> 282.1819.
DOI:
<a href="https://doi.org/10.1098%2Frspb.2015.2243">10.1098/rspb.2015.2243</a>.</p>
<p><a name=bib-Simon20151941></a><a href="#cite-Simon20151941">Simon, M., P. LÃ³pez-Garc'ia, P.
Deschamps, et al.</a> (2015). “Marked seasonality and
high spatial variability of protist communities in shallow freshwater
systems”. In: <em>ISME Journal</em> 9.9, pp. 1941-1953. DOI:
<a href="https://doi.org/10.1038%2Fismej.2015.6">10.1038/ismej.2015.6</a>.</p>
<p><a name=bib-Wu2015826></a><a href="#cite-Wu2015826">Wu, W., L. Wang, Y. Liao, et
al.</a> (2015). “Microbial eukaryotic diversity and
distribution in a river plume and cyclonic eddy-influenced ecosystem in
the South China Sea”. In: <em>MicrobiologyOpen</em> 4.5, pp. 826-840. DOI:
<a href="https://doi.org/10.1002%2Fmbo3.282">10.1002/mbo3.282</a>.</p>
</div>
<div id="section-8" class="section level2">
<h2>2014</h2>
<p><a name=bib-Chambouvet2014></a><a href="#cite-Chambouvet2014">Chambouvet, A., C. Berney, S. Romac, et
al.</a> (2014). “Diverse molecular signatures for
ribosomally ‘active’ Perkinsea in marine sediments”. In: <em>BMC
Microbiology</em> 14.1. DOI:
<a href="https://doi.org/10.1186%2F1471-2180-14-110">10.1186/1471-2180-14-110</a>.</p>
<p><a name=bib-Christaki201413></a><a href="#cite-Christaki201413">Christaki, U., K. Kormas, S.
Genitsaris, et al.</a> (2014). “Winter-summer
succession of unicellular eukaryotes in a meso-eutrophic coastal
system”. In: <em>Microbial Ecology</em> 67.1, pp. 13-23. DOI:
<a href="https://doi.org/10.1007%2Fs00248-013-0290-4">10.1007/s00248-013-0290-4</a>.</p>
<p><a name=bib-Decelle2014></a><a href="#cite-Decelle2014">Decelle, J., S. Romac, E. Sasaki, et
al.</a> (2014). “Intracellular diversity of the V4 and
V9 regions of the 18S rRNA in marine protists (radiolarians) assessed
by high-throughput sequencing”. In: <em>PLoS ONE</em> 9.8. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pone.0104297">10.1371/journal.pone.0104297</a>.</p>
<p><a name=bib-Georges20145847></a><a href="#cite-Georges20145847">Georges, C., S. Monchy, S. Genitsaris,
et al.</a> (2014). “Protist community composition
during early phytoplankton blooms in the naturally iron-fertilized
Kerguelen area (Southern Ocean)”. In: <em>Biogeosciences (Online)</em> 11.20,
pp. 5847-5863. DOI:
<a href="https://doi.org/10.5194%2Fbg-11-5847-2014">10.5194/bg-11-5847-2014</a>.</p>
<p><a name=bib-Heger2014214></a><a href="#cite-Heger2014214">Heger, T., V. Edgcomb, E. Kim, et
al.</a> (2014). “A resurgence in field research is
essential to better understand the diversity, ecology, and evolution of
microbial eukaryotes”. In: <em>Journal of Eukaryotic Microbiology</em> 61.2,
pp. 214-223. DOI:
<a href="https://doi.org/10.1111%2Fjeu.12095">10.1111/jeu.12095</a>.</p>
<p><a name=bib-Lara2014645></a><a href="#cite-Lara2014645">Lara, E., C. Seppey, G. Garraza, et
al.</a> (2014). “Planktonic eukaryote molecular
diversity: Discrimination of minerotrophic and ombrotrophic peatland
pools in Tierra del Fuego (Argentina)”. In: <em>Journal of Plankton
Research</em> 37.3, pp. 645-655. DOI:
<a href="https://doi.org/10.1093%2Fplankt%2Ffbv016">10.1093/plankt/fbv016</a>.</p>
<p><a name=bib-Lejzerowicz201417></a><a href="#cite-Lejzerowicz201417">Lejzerowicz, F., P. Esling, and J.
Pawlowski</a> (2014). “Patchiness of deep-sea
benthic Foraminifera across the Southern Ocean: Insights from
high-throughput DNA sequencing”. In: <em>Deep-Sea Research Part II:
Topical Studies in Oceanography</em> 108, pp. 17-26. DOI:
<a href="https://doi.org/10.1016%2Fj.dsr2.2014.07.018">10.1016/j.dsr2.2014.07.018</a>.</p>
<p><a name=bib-Lentendu20143341></a><a href="#cite-Lentendu20143341">Lentendu, G., T. Wubet, A.
Chatzinotas, et al.</a> (2014). “Effects of
long-term differential fertilization on eukaryotic microbial
communities in an arable soil: A multiple barcoding approach”. In:
<em>Molecular Ecology</em> 23.13, pp. 3341-3355. DOI:
<a href="https://doi.org/10.1111%2Fmec.12819">10.1111/mec.12819</a>.</p>
<p><a name=bib-Lie20144363></a><a href="#cite-Lie20144363">Lie, A., Z. Liu, S. Hu, et
al.</a> (2014). “Investigating microbial eukaryotic
diversity from a global census: Insights from a comparison of pyrotag
and full-length sequences of 18S rRNA genes”. In: <em>Applied and
Environmental Microbiology</em> 80.14, pp. 4363-4373. DOI:
<a href="https://doi.org/10.1128%2FAEM.00057-14">10.1128/AEM.00057-14</a>.</p>
<p><a name=bib-Logares2014813></a><a href="#cite-Logares2014813">Logares, R., S. Audic, D. Bass, et
al.</a> (2014). “Patterns of rare and abundant
marine microbial eukaryotes”. In: <em>Current Biology</em> 24.8, pp. 813-821.
DOI:
<a href="https://doi.org/10.1016%2Fj.cub.2014.02.050">10.1016/j.cub.2014.02.050</a>.</p>
<p><a name=bib-Massana2014854></a><a href="#cite-Massana2014854">Massana, R., J. Del Campo, M. Sieracki,
et al.</a> (2014). “Exploring the uncultured
microeukaryote majority in the oceans: Reevaluation of ribogroups
within stramenopiles”. In: <em>ISME Journal</em> 8.4, pp. 854-866. DOI:
<a href="https://doi.org/10.1038%2Fismej.2013.204">10.1038/ismej.2013.204</a>.</p>
<p><a name=bib-Nanjappa2014></a><a href="#cite-Nanjappa2014">Nanjappa, D., S. Audic, S. Romac, et
al.</a> (2014). “Assessment of species diversity and
distribution of an ancient diatom lineage using a DNA metabarcoding
approach”. In: <em>PLoS ONE</em> 9.8. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pone.0103810">10.1371/journal.pone.0103810</a>.</p>
<p><a name=bib-Parfrey2014></a><a href="#cite-Parfrey2014">Parfrey, L., W. Walters, C. Lauber, et
al.</a> (2014). “Communities of microbial eukaryotes in
the mammalian gut within the context of environmental eukaryotic
diversity”. In: <em>Frontiers in Microbiology</em> 5.JUN. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2014.00298">10.3389/fmicb.2014.00298</a>.</p>
<p><a name=bib-Parris2014></a><a href="#cite-Parris2014">Parris, D., S. Ganesh, V. Edgcomb, et
al.</a> (2014). “Microbial eukaryote diversity in the
marine oxygen minimum zone off northern Chile”. In: <em>Frontiers in
Microbiology</em> 5.OCT. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2014.00543">10.3389/fmicb.2014.00543</a>.</p>
<p><a name=bib-Patterson2014115></a><a href="#cite-Patterson2014115">Patterson,
D.</a> (2014). “Helping Protists to find their
place in a big data world”. In: <em>Acta Protozoologica</em> 53.1, pp.
115-128. DOI:
<a href="https://doi.org/10.4467%2F16890027AP.14.011.1448">10.4467/16890027AP.14.011.1448</a>.</p>
<p><a name=bib-Taylor2014960></a><a href="#cite-Taylor2014960">Taylor, J. and M.
Cunliffe</a> (2014). “High-throughput sequencing
reveals neustonic and planktonic microbial eukaryote diversity in
coastal waters”. In: <em>Journal of Phycology</em> 50.5, pp. 960-965. DOI:
<a href="https://doi.org/10.1111%2Fjpy.12228">10.1111/jpy.12228</a>.</p>
<p><a name=bib-Wu2014563></a><a href="#cite-Wu2014563">Wu, W., B. Huang, Y. Liao, et
al.</a> (2014). “Picoeukaryotic diversity and
distribution in the subtropical-tropical South China Sea”. In: <em>FEMS
Microbiology Ecology</em> 89.3, pp. 563-579. DOI:
<a href="https://doi.org/10.1111%2F1574-6941.12357">10.1111/1574-6941.12357</a>.</p>
<p><a name=bib-Xiuhai2014304></a><a href="#cite-Xiuhai2014304">Xiuhai, L.</a> (2014).
“An OpenFlow switch model using kepler GPU”. In: <em>Journal of Software
Engineering</em> 8.4, pp. 304-313. DOI:
<a href="https://doi.org/10.3923%2Fjse.2014.304.313">10.3923/jse.2014.304.313</a>.</p>
<p><a name=bib-Xu2014175></a><a href="#cite-Xu2014175">Xu, Y., T. Vick-Majors, R. Morgan-Kiss, et
al.</a> (2014). “Ciliate diversity, community structure,
and Novel Taxa in lakes of the McMurdo Dry Valleys, Antarctica”. In:
<em>Biological Bulletin</em> 227.2, pp. 175-190. DOI:
<a href="https://doi.org/10.1086%2FBBLv227n2p175">10.1086/BBLv227n2p175</a>.</p>
<p><a name=bib-Yilmaz2014></a><a href="#cite-Yilmaz2014">Yilmaz, P., L. Parfrey, P. Yarza, et
al.</a> (2014). “The SILVA and”all-species Living Tree
Project (LTP)” taxonomic frameworks”. In: <em>Nucleic Acids Research</em>
42.D1, pp. D643-D648. DOI:
<a href="https://doi.org/10.1093%2Fnar%2Fgkt1209">10.1093/nar/gkt1209</a>.</p>
</div>
<div id="section-9" class="section level2">
<h2>2013</h2>
<p><a name=bib-Adl2013328></a><a href="#cite-Adl2013328">Adl, S., A. Habura, and Y.
Eglit</a> (2013). “Amplification primers of SSU rDNA for
soil protists”. In: <em>Soil Biology and Biochemistry</em> 69, pp. 328-342.
DOI:
<a href="https://doi.org/10.1016%2Fj.soilbio.2013.10.024">10.1016/j.soilbio.2013.10.024</a>.</p>
<p><a name=bib-delCampo2013489></a><a href="#cite-delCampo2013489">Campo, J. del, V. Balagué, I. Forn, et
al.</a> (2013). “Culturing bias in marine
heterotrophic flagellates analyzed through seawater enrichment
incubations”. In: <em>Microbial Ecology</em> 66.3, pp. 489-499. DOI:
<a href="https://doi.org/10.1007%2Fs00248-013-0251-y">10.1007/s00248-013-0251-y</a>.</p>
<p><a name=bib-Cheung2013351></a><a href="#cite-Cheung2013351">Cheung, M., W. Nong, H. Kwan, et
al.</a> (2013). “Composition of heterotrophic
flagellates in coastal waters of different trophic status”. In:
<em>Current Microbiology</em> 67.3, pp. 351-355. DOI:
<a href="https://doi.org/10.1007%2Fs00284-013-0374-x">10.1007/s00284-013-0374-x</a>.</p>
<p><a name=bib-Decelle2013></a><a href="#cite-Decelle2013">Decelle, J., P. Martin, K. Paborstava, et
al.</a> (2013). “Diversity, ecology and biogeochemistry
of cyst-forming acantharia (radiolaria) in the oceans”. In: <em>PLoS ONE</em>
8.1. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pone.0053598">10.1371/journal.pone.0053598</a>.</p>
<p><a name=bib-DelCampo2013802></a><a href="#cite-DelCampo2013802">Del Campo, J. and I.
Ruiz-Trillo</a> (2013). “Environmental survey
meta-analysis reveals hidden diversity among unicellular opisthokonts”.
In: <em>Molecular Biology and Evolution</em> 30.4, pp. 802-805. DOI:
<a href="https://doi.org/10.1093%2Fmolbev%2Fmst006">10.1093/molbev/mst006</a>.</p>
<p><a name=bib-Egge2013></a><a href="#cite-Egge2013">Egge, E., L. Bittner, T. Andersen, et
al.</a> (2013). “454 pyrosequencing to describe microbial
eukaryotic community composition, diversity and relative abundance: A
test for marine haptophytes”. In: <em>PLoS ONE</em> 8.9. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pone.0074371">10.1371/journal.pone.0074371</a>.</p>
<p><a name=bib-Risse-Buhl2013467></a><a href="#cite-Risse-Buhl2013467">Risse-Buhl, U., M. Herrmann, P.
Lange, et al.</a> (2013). “Phagotrophic protist
diversity in the groundwater of a karstified aquifer - Morphological
and molecular analysis”. In: <em>Journal of Eukaryotic Microbiology</em> 60.5,
pp. 467-479. DOI:
<a href="https://doi.org/10.1111%2Fjeu.12054">10.1111/jeu.12054</a>.</p>
<p><a name=bib-Thaler2013347></a><a href="#cite-Thaler2013347">Thaler, M. and C.
Lovejoy</a> (2013). “Environmental selection of
marine stramenopile clades in the Arctic Ocean and coastal waters”. In:
<em>Polar Biology</em> 37.3, pp. 347-357. DOI:
<a href="https://doi.org/10.1007%2Fs00300-013-1435-0">10.1007/s00300-013-1435-0</a>.</p>
<p><a name=bib-Vinogradov2013391></a><a href="#cite-Vinogradov2013391">Vinogradov, S., X. Bailly, D. Smith,
et al.</a> (2013). “Microbial eukaryote globins”.
In: <em>Advances in Microbial Physiology</em> 63, pp. 391-446. DOI:
<a href="https://doi.org/10.1016%2FB978-0-12-407693-8.00009-1">10.1016/B978-0-12-407693-8.00009-1</a>.</p>
</div>
