---
title: "Papers citing EukRef"
author: vaulot
date: '2019-10-15'
categories: ["papers"]
tags: ["EukRef"]
---

<script src="/rmarkdown-libs/header-attrs/header-attrs.js"></script>


<p><img src="./../eukref_paper_2018.png" /></p>
<p>del Campo J., Kolisko M., Boscaro V., Santoferrara LF., Nenarokov S., Massana R., Guillou L., Simpson A., Berney C., de Vargas C., Brown MW., Keeling PJ., Wegener Parfrey L. 2018. <a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2005849">EukRef: Phylogenetic curation of ribosomal RNA to enhance understanding of eukaryotic diversity and distribution</a>. PLOS Biology 16:e2005849. DOI: 10.1371/journal.pbio.2005849.</p>
<p>Number of papers citing EukRef: 29</p>
<div id="section" class="section level2">
<h2>2021</h2>
<p><a name=bib-Dias2021></a><a href="#cite-Dias2021">Dias, R., P. de Souza, M. Rossi, et
al.</a> (2021). “Ciliates as bioindicators of water
quality: A case study in the neotropical region and evidence of
phylogenetic signals (18S-rDNA)”. In: <em>Environmental Pollution</em> 268.
cited By 0. DOI:
<a href="https://doi.org/10.1016%2Fj.envpol.2020.115760">10.1016/j.envpol.2020.115760</a>.</p>
<p><a name=bib-Dumack202177></a><a href="#cite-Dumack202177">Dumack, K. and M.
Bonkowski</a> (2021). “Protists in the plant
microbiome: an untapped field of research”. In: <em>Methods in Molecular
Biology</em> 2232. cited By 0, pp. 77-84. DOI:
<a href="https://doi.org/10.1007%2F978-1-0716-1040-4_8">10.1007/978-1-0716-1040-4_8</a>.</p>
<p><a name=bib-Lavrinienko202119></a><a href="#cite-Lavrinienko202119">Lavrinienko, A., T. Jernfors, J.
KoskimÃ¤ki, et al.</a> (2021). “Does
Intraspecific Variation in rDNA Copy Number Affect Analysis of
Microbial Communities?” In: <em>Trends in Microbiology</em> 29.1. cited By 3,
pp. 19-27. DOI:
<a href="https://doi.org/10.1016%2Fj.tim.2020.05.019">10.1016/j.tim.2020.05.019</a>.</p>
<p><a name=bib-Mann2021></a><a href="#cite-Mann2021">Mann, D., R. Trobajo, S. Sato, et
al.</a> (2021). “Ripe for reassessment: A synthesis of
available molecular data for the speciose diatom family
Bacillariaceae”. In: <em>Molecular Phylogenetics and Evolution</em> 158. cited
By 0. DOI:
<a href="https://doi.org/10.1016%2Fj.ympev.2020.106985">10.1016/j.ympev.2020.106985</a>.</p>
<p><a name=bib-Obertegger2021640></a><a href="#cite-Obertegger2021640">Obertegger, U., M. Pindo, and G.
Flaim</a> (2021). “Do inferences about freshwater
phytoplankton communities change when based on microscopy or
high-throughput sequencing data?” In: <em>Freshwater Biology</em> 66.4. cited
By 0, pp. 640-655. DOI:
<a href="https://doi.org/10.1111%2Ffwb.13667">10.1111/fwb.13667</a>.</p>
<p><a name=bib-NoAuthor2021></a><a href="#cite-NoAuthor2021">Rajter, I. Ewers, N. Graupner, et
al.</a> (2021). “Colpodean ciliate phylogeny and
reference alignments for phylogenetic placements”. In: <em>European
Journal of Protistology</em> 77. cited By 0. DOI:
<a href="https://doi.org/10.1016%2Fj.ejop.2020.125747">10.1016/j.ejop.2020.125747</a>.</p>
<p><a name=bib-Schiwitza2021></a><a href="#cite-Schiwitza2021">Schiwitza, S. and F.
Nitsche</a> (2021). “A Needle in the Haystack â€“
Mapping Sequences to Morphology Exemplified by the Loricate
Choanoflagellate Enibas thessalia sp. nov. (Acanthoecida,
Acanthoecidae)”. In: <em>Protist</em> 172.1. cited By 0. DOI:
<a href="https://doi.org/10.1016%2Fj.protis.2020.125782">10.1016/j.protis.2020.125782</a>.</p>
</div>
<div id="section-1" class="section level2">
<h2>2020</h2>
<p><a name=bib-Djemiel2020></a><a href="#cite-Djemiel2020">Djemiel, C., D. Plassard, S. Terrat, et
al.</a> (2020). “Âµgreen-db: A reference database for
the 23S rRNA gene of eukaryotic plastids and cyanobacteria”. In:
<em>Scientific Reports</em> 10.1. cited By 1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-020-62555-1">10.1038/s41598-020-62555-1</a>.</p>
<p><a name=bib-Fawley20202699></a><a href="#cite-Fawley20202699">Fawley, M. and K.
Fawley</a> (2020). “Identification of eukaryotic
microalgal strains”. In: <em>Journal of Applied Phycology</em> 32.5. cited By
2, pp. 2699-2709. DOI:
<a href="https://doi.org/10.1007%2Fs10811-020-02190-5">10.1007/s10811-020-02190-5</a>.</p>
<p><a name=bib-Fiore-Donno2020></a><a href="#cite-Fiore-Donno2020">Fiore-Donno, A., T. Richter-Heitmann,
and M. Bonkowski</a> (2020). “Contrasting Responses
of Protistan Plant Parasites and Phagotrophs to Ecosystems, Land
Management and Soil Properties”. In: <em>Frontiers in Microbiology</em> 11.
cited By 4. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2020.01823">10.3389/fmicb.2020.01823</a>.</p>
<p><a name=bib-Gottschling2020193></a><a href="#cite-Gottschling2020193">Gottschling, M., J. ChacÃ³n, A.
Å½erdoner ÄŒalasan, et al.</a> (2020).
“Phylogenetic placement of environmental sequences using taxonomically
reliable databases helps to rigorously assess dinophyte biodiversity in
Bavarian lakes (Germany)”. In: <em>Freshwater Biology</em> 65.2. cited By 8,
pp. 193-208. DOI:
<a href="https://doi.org/10.1111%2Ffwb.13413">10.1111/fwb.13413</a>.</p>
<p><a name=bib-Irion20202835></a><a href="#cite-Irion20202835">Irion, S., L. Jardillier, I.
Sassenhagen, et al.</a> (2020). “Marked
spatiotemporal variations in small phytoplankton structure in
contrasted waters of the Southern Ocean (Kerguelen area)”. In:
<em>Limnology and Oceanography</em> 65.11. cited By 1, pp. 2835-2852. DOI:
<a href="https://doi.org/10.1002%2Flno.11555">10.1002/lno.11555</a>.</p>
<p><a name=bib-Kolisko2020></a><a href="#cite-Kolisko2020">Kolisko, M., O. Flegontova, A. Karnkowska,
et al.</a> (2020). “EukRef-excavates: Seven curated SSU
ribosomal RNA gene databases”. In: <em>Database</em> 2020. cited By 0. DOI:
<a href="https://doi.org/10.1093%2Fdatabase%2Fbaaa080">10.1093/database/baaa080</a>.</p>
<p><a name=bib-Mann2020609></a><a href="#cite-Mann2020609">Mann, A., F. Mazel, M. Lemay, et
al.</a> (2020). “Biodiversity of protists and nematodes
in the wild nonhuman primate gut”. In: <em>ISME Journal</em> 14.2. cited By 1,
pp. 609-622. DOI:
<a href="https://doi.org/10.1038%2Fs41396-019-0551-4">10.1038/s41396-019-0551-4</a>.</p>
<p><a name=bib-Martini2020></a><a href="#cite-Martini2020">Martini, S., F. Larras, A. BoyÃ©, et
al.</a> (2020). “Functional trait-based approaches as a
common framework for aquatic ecologists”. In: <em>Limnology and
Oceanography</em>. cited By 0. DOI:
<a href="https://doi.org/10.1002%2Flno.11655">10.1002/lno.11655</a>.</p>
<p><a name=bib-Santoferrara2020612></a><a href="#cite-Santoferrara2020612">Santoferrara, L., F. Burki, S.
Filker, et al.</a> (2020). “Perspectives from
Ten Years of Protist Studies by High-Throughput Metabarcoding”. In:
<em>Journal of Eukaryotic Microbiology</em> 67.5. cited By 3, pp. 612-622.
DOI: <a href="https://doi.org/10.1111%2Fjeu.12813">10.1111/jeu.12813</a>.</p>
</div>
<div id="section-2" class="section level2">
<h2>2019</h2>
<p><a name=bib-Adl20194></a><a href="#cite-Adl20194">Adl, S., D. Bass, C. Lane, et
al.</a> (2019). “Revisions to the Classification,
Nomenclature, and Diversity of Eukaryotes”. In: <em>Journal of Eukaryotic
Microbiology</em> 66.1. cited By 242, pp. 4-119. DOI:
<a href="https://doi.org/10.1111%2Fjeu.12691">10.1111/jeu.12691</a>.</p>
<p><a name=bib-Caron2019197></a><a href="#cite-Caron2019197">Caron, D. and S. Hu</a>
(2019). “Are We Overestimating Protistan Diversity in Nature?” In:
<em>Trends in Microbiology</em> 27.3. cited By 20, pp. 197-205. DOI:
<a href="https://doi.org/10.1016%2Fj.tim.2018.10.009">10.1016/j.tim.2018.10.009</a>.</p>
<p><a name=bib-DelCampo2019></a><a href="#cite-DelCampo2019">Del Campo, J., T. Heger, R.
RodrÃ­guez-MartÃ­nez, et al.</a> (2019). “Assessing
the diversity and distribution of apicomplexans in host and free-living
environments using high-throughput amplicon data and a phylogenetically
informed reference framework”. In: <em>Frontiers in Microbiology</em> 10.OCT.
cited By 7. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2019.02373">10.3389/fmicb.2019.02373</a>.</p>
<p><a name=bib-Ku2019></a><a href="#cite-Ku2019">Ku, C. and A. SebÃ©-PedrÃ³s</a>
(2019). “Using single-cell transcriptomics to understand functional
states and interactions in microbial eukaryotes”. In: <em>Philosophical
Transactions of the Royal Society B: Biological Sciences</em> 374.1786.
cited By 5. DOI:
<a href="https://doi.org/10.1098%2Frstb.2019.0098">10.1098/rstb.2019.0098</a>.</p>
<p><a name=bib-Linard20193303></a><a href="#cite-Linard20193303">Linard, B., K. Swenson, and F.
Pardi</a> (2019). “Rapid alignment-free phylogenetic
identification of metagenomic sequences”. In: <em>Bioinformatics</em> 35.18.
cited By 3, pp. 3303-3312. DOI:
<a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtz068">10.1093/bioinformatics/btz068</a>.</p>
<p><a name=bib-Lutz2019115></a><a href="#cite-Lutz2019115">Lutz, S., L. ProchÃ¡zkovÃ¡, L. Benning, et
al.</a> (2019). “Evaluating amplicon highâ€“throughput
sequencing data of microalgae living in melting snow: Improvements and
limitations”. In: <em>Fottea</em> 19.2. cited By 5, pp. 115-131. DOI:
<a href="https://doi.org/10.5507%2Ffot.2019.003">10.5507/fot.2019.003</a>.</p>
<p><a name=bib-Metz20193885></a><a href="#cite-Metz20193885">Metz, S., D. Singer, I. Domaizon, et
al.</a> (2019). “Global distribution of
Trebouxiophyceae diversity explored by high-throughput sequencing and
phylogenetic approaches”. In: <em>Environmental Microbiology</em> 21.10. cited
By 3, pp. 3885-3895. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.14738">10.1111/1462-2920.14738</a>.</p>
<p><a name=bib-Morard2019></a><a href="#cite-Morard2019">Morard, R., N. Vollmar, M. Greco, et
al.</a> (2019). “Unassigned diversity of planktonic
foraminifera from environmental sequencing revealed as known but
neglected species”. In: <em>PLoS ONE</em> 14.3. cited By 7. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pone.0213936">10.1371/journal.pone.0213936</a>.</p>
<p><a name=bib-Richardson2019></a><a href="#cite-Richardson2019">Richardson, E. and J.
Dacks</a> (2019). “Microbial Eukaryotes in oil sands
environments: Heterotrophs in the spotlight”. In: <em>Microorganisms</em> 7.6.
cited By 0. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms7060178">10.3390/microorganisms7060178</a>.</p>
<p><a name=bib-Rimet2019></a><a href="#cite-Rimet2019">Rimet, F., E. Gusev, M. Kahlert, et
al.</a> (2019). “Diat.barcode, an open-access curated
barcode library for diatoms”. In: <em>Scientific Reports</em> 9.1. cited By
21. DOI:
<a href="https://doi.org/10.1038%2Fs41598-019-51500-6">10.1038/s41598-019-51500-6</a>.</p>
<p><a name=bib-Santoferrara2019571></a><a href="#cite-Santoferrara2019571">Santoferrara,
L.</a> (2019). “Current practice in plankton
metabarcoding: Optimization and error management”. In: <em>Journal of
Plankton Research</em> 41.5. cited By 7, pp. 571-582. DOI:
<a href="https://doi.org/10.1093%2Fplankt%2Ffbz041">10.1093/plankt/fbz041</a>.</p>
<p><a name=bib-Yu2019474></a><a href="#cite-Yu2019474">Yu, Z. and N. Chen</a> (2019).
“Emerging trends in red tide and major research progresses
[å›½å†…å¤–èµ¤æ½®çš„å‘å±•è¶‹åŠ¿ä¸Žç ”ç©¶çƒ­ç‚¹]”. In: <em>Oceanologia et
Limnologia Sinica</em> 50.3. cited By 3, pp. 474-486. DOI:
<a href="https://doi.org/10.11693%2Fhyhz20190200041">10.11693/hyhz20190200041</a>.</p>
</div>
<div id="section-3" class="section level2">
<h2>2018</h2>
<p><a name=bib-Duarte2018></a><a href="#cite-Duarte2018">Duarte, S., L. Nunes, P. Borges, et
al.</a> (2018). “A bridge too far? An integrative
framework linking classical protist taxonomy and metabarcoding in lower
termites”. In: <em>Frontiers in Microbiology</em> 9.NOV. cited By 3. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2018.02620">10.3389/fmicb.2018.02620</a>.</p>
</div>
