---
title: "Papers citing EukRef"
author: vaulot
date: '2019-10-15'
categories: ["papers"]
tags: ["EukRef"]
---

<script src="/rmarkdown-libs/header-attrs/header-attrs.js"></script>


<p><img src="eukref_paper_2018.png" /></p>
<p>del Campo J., Kolisko M., Boscaro V., Santoferrara LF., Nenarokov S., Massana R., Guillou L., Simpson A., Berney C., de Vargas C., Brown MW., Keeling PJ., Wegener Parfrey L. 2018. <a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2005849">EukRef: Phylogenetic curation of ribosomal RNA to enhance understanding of eukaryotic diversity and distribution</a>. PLOS Biology 16:e2005849. DOI: 10.1371/journal.pbio.2005849.</p>
<div id="number-of-papers-citing-eukref-57" class="section level3">
<h3>Number of papers citing EukRef: 57</h3>
</div>
<div id="section" class="section level2">
<h2>2022</h2>
<p><a name=bib-Cahoon2022538></a><a href="#cite-Cahoon2022538">Cahoon, A. and R.
VanGundy</a> (2022). “Alveolates (dinoflagellates,
ciliates and apicomplexans) and Rhizarians are the most common
microbial eukaryotes in temperate Appalachian karst caves”. In:
<em>Environmental Microbiology Reports</em> 14.4, pp. 538-548. DOI:
<a href="https://doi.org/10.1111%2F1758-2229.13060">10.1111/1758-2229.13060</a>.</p>
<p><a name=bib-Caracciolo20223761></a><a href="#cite-Caracciolo20223761">Caracciolo, M., F. Rigaut-Jalabert,
S. Romac, et al.</a> (2022). “Seasonal dynamics
of marine protist communities in tidally mixed coastal waters”. In:
<em>Molecular Ecology</em> 31.14, pp. 3761-3783. DOI:
<a href="https://doi.org/10.1111%2Fmec.16539">10.1111/mec.16539</a>.</p>
<p><a name=bib-Cho2022></a><a href="#cite-Cho2022">Cho, A., D. Tikhonenkov, E. Hehenberger, et
al.</a> (2022). “Monophyly of diverse Bigyromonadea and
their impact on phylogenomic relationships within stramenopiles”. In:
<em>Molecular Phylogenetics and Evolution</em> 171. DOI:
<a href="https://doi.org/10.1016%2Fj.ympev.2022.107468">10.1016/j.ympev.2022.107468</a>.</p>
<p><a name=bib-Duerschlag2022465></a><a href="#cite-Duerschlag2022465">Duerschlag, J., W. Mohr, T.
Ferdelman, et al.</a> (2022). “Niche partitioning
by photosynthetic plankton as a driver of CO2-fixation across the
oligotrophic South Pacific Subtropical Ocean”. In: <em>ISME Journal</em> 16.2,
pp. 465-476. DOI:
<a href="https://doi.org/10.1038%2Fs41396-021-01072-z">10.1038/s41396-021-01072-z</a>.</p>
<p><a name=bib-EshghiSahraei2022></a><a href="#cite-EshghiSahraei2022">Eshghi Sahraei, S., B. Furneaux, K.
Kluting, et al.</a> (2022). “Effects of
operational taxonomic unit inference methods on soil microeukaryote
community analysis using long-read metabarcoding”. In: <em>Ecology and
Evolution</em> 12.3. DOI:
<a href="https://doi.org/10.1002%2Fece3.8676">10.1002/ece3.8676</a>.</p>
<p><a name=bib-Frantal2022></a><a href="#cite-Frantal2022">Frantal, D., S. Agatha, D. Beisser, et
al.</a> (2022). “Molecular Data Reveal a Cryptic
Diversity in the Genus Urotricha (Alveolata, Ciliophora, Prostomatida),
a Key Player in Freshwater Lakes, With Remarks on Morphology, Food
Preferences, and Distribution”. In: <em>Frontiers in Microbiology</em> 12.
DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2021.787290">10.3389/fmicb.2021.787290</a>.</p>
<p><a name=bib-Igai20221317></a><a href="#cite-Igai20221317">Igai, K., O. Kitade, J. Fu, et
al.</a> (2022). “Fine-scale genetic diversity and
putative ecotypes of oxymonad protists coinhabiting the hindgut of
Reticulitermes speratus”. In: <em>Molecular Ecology</em> 31.4, pp. 1317-1331.
DOI: <a href="https://doi.org/10.1111%2Fmec.16309">10.1111/mec.16309</a>.</p>
<p><a name=bib-Metz20221731></a><a href="#cite-Metz20221731">Metz, S., P. Huber, V. Accattatis, et
al.</a> (2022). “Freshwater protists: unveiling the
unexplored in a large floodplain system”. In: <em>Environmental
Microbiology</em> 24.4, pp. 1731-1745. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.15838">10.1111/1462-2920.15838</a>.</p>
<p><a name=bib-Panova2022></a><a href="#cite-Panova2022">Panova, M., M. Varfolomeeva, E. Gafarova,
et al.</a> (2022). “First insights into the gut
microbiomes and the diet of the Littorina snail ecotypes, a recently
emerged marine evolutionary model”. In: <em>Evolutionary Applications</em>.
DOI: <a href="https://doi.org/10.1111%2Feva.13447">10.1111/eva.13447</a>.</p>
<p><a name=bib-Reid202289></a><a href="#cite-Reid202289">Reid, B., J. Servis, M. Timmers, et
al.</a> (2022). “18S rDNA amplicon sequence data (V1-V3)
of the Palmyra Atoll National Wildlife Refuge, Central Pacific”. In:
<em>Metabarcoding and Metagenomics</em> 6, pp. 89-99. DOI:
<a href="https://doi.org/10.3897%2Fmbmg.6.78762">10.3897/mbmg.6.78762</a>.</p>
<p><a name=bib-Sandin20222979></a><a href="#cite-Sandin20222979">Sandin, M., S. Romac, and F.
Not</a> (2022). “Intra-genomic rRNA gene variability
of Nassellaria and Spumellaria (Rhizaria, Radiolaria) assessed by
Sanger, MinION and Illumina sequencing”. In: <em>Environmental
Microbiology</em> 24.7, pp. 2979-2993. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.16081">10.1111/1462-2920.16081</a>.</p>
<p><a name=bib-Tikhonenkov2022></a><a href="#cite-Tikhonenkov2022">Tikhonenkov, D., M. Jamy, A. Borodina,
et al.</a> (2022). “On the origin of TSAR:
Morphology, diversity and phylogeny of Telonemia”. In: <em>Open Biology</em>
12.3. DOI:
<a href="https://doi.org/10.1098%2Frsob.210325">10.1098/rsob.210325</a>.</p>
<p><a name=bib-Zafeiropoulos2022></a><a href="#cite-Zafeiropoulos2022">Zafeiropoulos, H., S. Paragkamian,
S. Ninidakis, et al.</a> (2022). “PREGO: A
Literature and Data-Mining Resource to Associate Microorganisms,
Biological Processes, and Environment Types”. In: <em>Microorganisms</em>
10.2. DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms10020293">10.3390/microorganisms10020293</a>.</p>
</div>
<div id="section-1" class="section level2">
<h2>2021</h2>
<p><a name=bib-Burki2021R1267></a><a href="#cite-Burki2021R1267">Burki, F., M. Sandin, and M.
Jamy</a> (2021). “Diversity and ecology of protists
revealed by metabarcoding”. In: <em>Current Biology</em> 31.19, pp.
R1267-R1280. DOI:
<a href="https://doi.org/10.1016%2Fj.cub.2021.07.066">10.1016/j.cub.2021.07.066</a>.</p>
<p><a name=bib-Canino2021199></a><a href="#cite-Canino2021199">Canino, A., A. Bouchez, C.
Laplace-Treyture, et al.</a> (2021). “Phytool, a
ShinyApp to homogenise taxonomy of freshwater microalgae from DNA
barcodes and microscopic observations”. In: <em>Metabarcoding and
Metagenomics</em> 5, pp. 199-205. DOI:
<a href="https://doi.org/10.3897%2Fmbmg.5.74096">10.3897/mbmg.5.74096</a>.</p>
<p><a name=bib-Diaz-Torres2021></a><a href="#cite-Diaz-Torres2021">DÃ­az-Torres, O., J. de Anda, O.
Lugo-Melchor, et al.</a> (2021). “Rapid Changes in
the Phytoplankton Community of a Subtropical, Shallow, Hypereutrophic
Lake During the Rainy Season”. In: <em>Frontiers in Microbiology</em> 12. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2021.617151">10.3389/fmicb.2021.617151</a>.</p>
<p><a name=bib-Dias2021></a><a href="#cite-Dias2021">Dias, R., P. de Souza, M. Rossi, et
al.</a> (2021). “Ciliates as bioindicators of water
quality: A case study in the neotropical region and evidence of
phylogenetic signals (18S-rDNA)”. In: <em>Environmental Pollution</em> 268.
DOI:
<a href="https://doi.org/10.1016%2Fj.envpol.2020.115760">10.1016/j.envpol.2020.115760</a>.</p>
<p><a name=bib-Dumack202177></a><a href="#cite-Dumack202177">Dumack, K. and M.
Bonkowski</a> (2021). “Protists in the plant
microbiome: an untapped field of research”. In: <em>Methods in Molecular
Biology</em> 2232, pp. 77-84. DOI:
<a href="https://doi.org/10.1007%2F978-1-0716-1040-4_8">10.1007/978-1-0716-1040-4_8</a>.</p>
<p><a name=bib-Gao2021></a><a href="#cite-Gao2021">Gao, B., L. Chi, Y. Zhu, et
al.</a> (2021). “An introduction to next generation
sequencing bioinformatic analysis in gut microbiome studies”. In:
<em>Biomolecules</em> 11.4. DOI:
<a href="https://doi.org/10.3390%2Fbiom11040530">10.3390/biom11040530</a>.</p>
<p><a name=bib-Greco2021113></a><a href="#cite-Greco2021113">Greco, M., R. Morard, and M.
Kucera</a> (2021). “Single-cell metabarcoding reveals
biotic interactions of the Arctic calcifier Neogloboquadrina pachyderma
with the eukaryotic pelagic community”. In: <em>Journal of Plankton
Research</em> 43.2, pp. 113-125. DOI:
<a href="https://doi.org/10.1093%2Fplankt%2Ffbab015">10.1093/plankt/fbab015</a>.</p>
<p><a name=bib-Lavrinienko202119></a><a href="#cite-Lavrinienko202119">Lavrinienko, A., T. Jernfors, J.
KoskimÃ¤ki, et al.</a> (2021). “Does
Intraspecific Variation in rDNA Copy Number Affect Analysis of
Microbial Communities?” In: <em>Trends in Microbiology</em> 29.1, pp. 19-27.
DOI:
<a href="https://doi.org/10.1016%2Fj.tim.2020.05.019">10.1016/j.tim.2020.05.019</a>.</p>
<p><a name=bib-Mann2021></a><a href="#cite-Mann2021">Mann, D., R. Trobajo, S. Sato, et
al.</a> (2021). “Ripe for reassessment: A synthesis of
available molecular data for the speciose diatom family
Bacillariaceae”. In: <em>Molecular Phylogenetics and Evolution</em> 158. DOI:
<a href="https://doi.org/10.1016%2Fj.ympev.2020.106985">10.1016/j.ympev.2020.106985</a>.</p>
<p><a name=bib-Martini2021965></a><a href="#cite-Martini2021965">Martini, S., F. Larras, A. BoyÃ©, et
al.</a> (2021). “Functional trait-based approaches
as a common framework for aquatic ecologists”. In: <em>Limnology and
Oceanography</em> 66.3, pp. 965-994. DOI:
<a href="https://doi.org/10.1002%2Flno.11655">10.1002/lno.11655</a>.</p>
<p><a name=bib-Obertegger2021640></a><a href="#cite-Obertegger2021640">Obertegger, U., M. Pindo, and G.
Flaim</a> (2021). “Do inferences about freshwater
phytoplankton communities change when based on microscopy or
high-throughput sequencing data?” In: <em>Freshwater Biology</em> 66.4, pp.
640-655. DOI: <a href="https://doi.org/10.1111%2Ffwb.13667">10.1111/fwb.13667</a>.</p>
<p><a name=bib-Rajter2021></a><a href="#cite-Rajter2021">Rajter, I. Ewers, N. Graupner, et
al.</a> (2021). “Colpodean ciliate phylogeny and
reference alignments for phylogenetic placements”. In: <em>European
Journal of Protistology</em> 77. DOI:
<a href="https://doi.org/10.1016%2Fj.ejop.2020.125747">10.1016/j.ejop.2020.125747</a>.</p>
<p><a name=bib-Regaldo2021343></a><a href="#cite-Regaldo2021343">Regaldo, L., V. MÃ¡rquez, M. Latorre
Rapela, et al.</a> (2021). <em>Prokaryotic and
Eukaryotic Diversity in Wastewater Treatment: Phenotypic
Characterization, Molecular Techniques, and Bioprospecting</em> , pp.
343-365. DOI:
<a href="https://doi.org/10.1016%2FB978-0-12-821881-5.00017-9">10.1016/B978-0-12-821881-5.00017-9</a>.</p>
<p><a name=bib-Rimet202117></a><a href="#cite-Rimet202117">Rimet, F., E. Aylagas, A. Borja, et
al.</a> (2021). “Metadata standards and practical
guidelines for specimen and DNA curation when building barcode
reference libraries for aquatic life”. In: <em>Metabarcoding and
Metagenomics</em> 5, pp. 17-33. DOI:
<a href="https://doi.org/10.3897%2Fmbmg.5.58056">10.3897/mbmg.5.58056</a>.</p>
<p><a name=bib-Schiwitza2021></a><a href="#cite-Schiwitza2021">Schiwitza, S. and F.
Nitsche</a> (2021). “A Needle in the Haystack â€“
Mapping Sequences to Morphology Exemplified by the Loricate
Choanoflagellate Enibas thessalia sp. nov. (Acanthoecida,
Acanthoecidae)”. In: <em>Protist</em> 172.1. DOI:
<a href="https://doi.org/10.1016%2Fj.protis.2020.125782">10.1016/j.protis.2020.125782</a>.</p>
<p><a name=bib-Snyder20213356></a><a href="#cite-Snyder20213356">Snyder, R., J. Moss, L. Santoferrara,
et al.</a> (2021). “Ciliate microzooplankton from
the Northeastern Gulf of Mexico”. In: <em>ICES Journal of Marine Science</em>
78.9, pp. 3356-3371. DOI:
<a href="https://doi.org/10.1093%2Ficesjms%2Ffsab002">10.1093/icesjms/fsab002</a>.</p>
<p><a name=bib-Tedersoo20211></a><a href="#cite-Tedersoo20211">Tedersoo, L., M. Albertsen, S. Anslan,
et al.</a> (2021). “Perspectives and Benefits of
High-Throughput Long-Read Sequencing in Microbial Ecology”. In:
<em>Applied and Environmental Microbiology</em> 87.17, pp. 1-19. DOI:
<a href="https://doi.org/10.1128%2FAEM.00626-21">10.1128/AEM.00626-21</a>.</p>
<p><a name=bib-Tonjer20214926></a><a href="#cite-Tonjer20214926">Tonjer, L., E. Thoen, L. Morgado, et
al.</a> (2021). “Fungal community dynamics across a
forestâ€“alpine ecotone”. In: <em>Molecular Ecology</em> 30.19, pp. 4926-4938.
DOI: <a href="https://doi.org/10.1111%2Fmec.16095">10.1111/mec.16095</a>.</p>
<p><a name=bib-Walker2021195></a><a href="#cite-Walker2021195">Walker, L., T. Hoppe, and M.
Silliker</a> (2021). <em>Molecular techniques and
current research approaches</em> , pp. 195-229. DOI:
<a href="https://doi.org/10.1016%2FB978-0-12-824281-0.00006-3">10.1016/B978-0-12-824281-0.00006-3</a>.</p>
<p><a name=bib-Zhou20216331></a><a href="#cite-Zhou20216331">Zhou, Y., B. Sun, B. Xie, et
al.</a> (2021). “Warming reshaped the microbial
hierarchical interactions”. In: <em>Global Change Biology</em> 27.24, pp.
6331-6347. DOI:
<a href="https://doi.org/10.1111%2Fgcb.15891">10.1111/gcb.15891</a>.</p>
</div>
<div id="section-2" class="section level2">
<h2>2020</h2>
<p><a name=bib-Djemiel2020></a><a href="#cite-Djemiel2020">Djemiel, C., D. Plassard, S. Terrat, et
al.</a> (2020). “Âµgreen-db: A reference database for
the 23S rRNA gene of eukaryotic plastids and cyanobacteria”. In:
<em>Scientific Reports</em> 10.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-020-62555-1">10.1038/s41598-020-62555-1</a>.</p>
<p><a name=bib-Fawley20202699></a><a href="#cite-Fawley20202699">Fawley, M. and K.
Fawley</a> (2020). “Identification of eukaryotic
microalgal strains”. In: <em>Journal of Applied Phycology</em> 32.5, pp.
2699-2709. DOI:
<a href="https://doi.org/10.1007%2Fs10811-020-02190-5">10.1007/s10811-020-02190-5</a>.</p>
<p><a name=bib-Fiore-Donno2020></a><a href="#cite-Fiore-Donno2020">Fiore-Donno, A., T. Richter-Heitmann,
and M. Bonkowski</a> (2020). “Contrasting Responses
of Protistan Plant Parasites and Phagotrophs to Ecosystems, Land
Management and Soil Properties”. In: <em>Frontiers in Microbiology</em> 11.
DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2020.01823">10.3389/fmicb.2020.01823</a>.</p>
<p><a name=bib-Gottschling2020193></a><a href="#cite-Gottschling2020193">Gottschling, M., J. ChacÃ³n, A.
Å½erdoner ÄŒalasan, et al.</a> (2020).
“Phylogenetic placement of environmental sequences using taxonomically
reliable databases helps to rigorously assess dinophyte biodiversity in
Bavarian lakes (Germany)”. In: <em>Freshwater Biology</em> 65.2, pp. 193-208.
DOI: <a href="https://doi.org/10.1111%2Ffwb.13413">10.1111/fwb.13413</a>.</p>
<p><a name=bib-Irion20202835></a><a href="#cite-Irion20202835">Irion, S., L. Jardillier, I.
Sassenhagen, et al.</a> (2020). “Marked
spatiotemporal variations in small phytoplankton structure in
contrasted waters of the Southern Ocean (Kerguelen area)”. In:
<em>Limnology and Oceanography</em> 65.11, pp. 2835-2852. DOI:
<a href="https://doi.org/10.1002%2Flno.11555">10.1002/lno.11555</a>.</p>
<p><a name=bib-Kolisko2020></a><a href="#cite-Kolisko2020">Kolisko, M., O. Flegontova, A. Karnkowska,
et al.</a> (2020). “EukRef-excavates: Seven curated SSU
ribosomal RNA gene databases”. In: <em>Database</em> 2020. DOI:
<a href="https://doi.org/10.1093%2Fdatabase%2Fbaaa080">10.1093/database/baaa080</a>.</p>
<p><a name=bib-Lukesova202078></a><a href="#cite-Lukesova202078">LukeÅ¡ovÃ¡, S., M. Karlicki, L.
TomeÄkovÃ¡ HadariovÃ¡, et al.</a> (2020). “Analyses
of environmental sequences and two regions of chloroplast genomes
revealed the presence of new clades of photosynthetic euglenids in
marine environments”. In: <em>Environmental Microbiology Reports</em> 12.1,
pp. 78-91. DOI:
<a href="https://doi.org/10.1111%2F1758-2229.12817">10.1111/1758-2229.12817</a>.</p>
<p><a name=bib-Mann2020609></a><a href="#cite-Mann2020609">Mann, A., F. Mazel, M. Lemay, et
al.</a> (2020). “Biodiversity of protists and nematodes
in the wild nonhuman primate gut”. In: <em>ISME Journal</em> 14.2, pp.
609-622. DOI:
<a href="https://doi.org/10.1038%2Fs41396-019-0551-4">10.1038/s41396-019-0551-4</a>.</p>
<p><a name=bib-Santoferrara2020612></a><a href="#cite-Santoferrara2020612">Santoferrara, L., F. Burki, S.
Filker, et al.</a> (2020). “Perspectives from
Ten Years of Protist Studies by High-Throughput Metabarcoding”. In:
<em>Journal of Eukaryotic Microbiology</em> 67.5, pp. 612-622. DOI:
<a href="https://doi.org/10.1111%2Fjeu.12813">10.1111/jeu.12813</a>.</p>
<p><a name=bib-Tumova2020></a><a href="#cite-Tumova2020">TÅ¯movÃ¡, P., E. NohÃ½nkovÃ¡, A. Klingl, et
al.</a> (2020). “A rapid workflow for the
characterization of small numbers of unicellular eukaryotes by using
correlative light and electron microscopy”. In: <em>Journal of
Microbiological Methods</em> 172. DOI:
<a href="https://doi.org/10.1016%2Fj.mimet.2020.105888">10.1016/j.mimet.2020.105888</a>.</p>
</div>
<div id="section-3" class="section level2">
<h2>2019</h2>
<p><a name=bib-Adl20194></a><a href="#cite-Adl20194">Adl, S., D. Bass, C. Lane, et
al.</a> (2019). “Revisions to the Classification,
Nomenclature, and Diversity of Eukaryotes”. In: <em>Journal of Eukaryotic
Microbiology</em> 66.1, pp. 4-119. DOI:
<a href="https://doi.org/10.1111%2Fjeu.12691">10.1111/jeu.12691</a>.</p>
<p><a name=bib-Caron2019197></a><a href="#cite-Caron2019197">Caron, D. and S. Hu</a>
(2019). “Are We Overestimating Protistan Diversity in Nature?” In:
<em>Trends in Microbiology</em> 27.3, pp. 197-205. DOI:
<a href="https://doi.org/10.1016%2Fj.tim.2018.10.009">10.1016/j.tim.2018.10.009</a>.</p>
<p><a name=bib-DelCampo2019></a><a href="#cite-DelCampo2019">Del Campo, J., T. Heger, R.
RodrÃ­guez-MartÃ­nez, et al.</a> (2019). “Assessing
the diversity and distribution of apicomplexans in host and free-living
environments using high-throughput amplicon data and a phylogenetically
informed reference framework”. In: <em>Frontiers in Microbiology</em> 10.OCT.
DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2019.02373">10.3389/fmicb.2019.02373</a>.</p>
<p><a name=bib-Ku2019></a><a href="#cite-Ku2019">Ku, C. and A. SebÃ©-PedrÃ³s</a>
(2019). “Using single-cell transcriptomics to understand functional
states and interactions in microbial eukaryotes”. In: <em>Philosophical
Transactions of the Royal Society B: Biological Sciences</em> 374.1786.
DOI:
<a href="https://doi.org/10.1098%2Frstb.2019.0098">10.1098/rstb.2019.0098</a>.</p>
<p><a name=bib-Linard20193303></a><a href="#cite-Linard20193303">Linard, B., K. Swenson, and F.
Pardi</a> (2019). “Rapid alignment-free phylogenetic
identification of metagenomic sequences”. In: <em>Bioinformatics</em> 35.18,
pp. 3303-3312. DOI:
<a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtz068">10.1093/bioinformatics/btz068</a>.</p>
<p><a name=bib-Lutz2019115></a><a href="#cite-Lutz2019115">Lutz, S., L. ProchÃ¡zkovÃ¡, L. Benning, et
al.</a> (2019). “Evaluating amplicon highâ€“throughput
sequencing data of microalgae living in melting snow: Improvements and
limitations”. In: <em>Fottea</em> 19.2, pp. 115-131. DOI:
<a href="https://doi.org/10.5507%2Ffot.2019.003">10.5507/fot.2019.003</a>.</p>
<p><a name=bib-Metz20193885></a><a href="#cite-Metz20193885">Metz, S., D. Singer, I. Domaizon, et
al.</a> (2019). “Global distribution of
Trebouxiophyceae diversity explored by high-throughput sequencing and
phylogenetic approaches”. In: <em>Environmental Microbiology</em> 21.10, pp.
3885-3895. DOI:
<a href="https://doi.org/10.1111%2F1462-2920.14738">10.1111/1462-2920.14738</a>.</p>
<p><a name=bib-Morard2019></a><a href="#cite-Morard2019">Morard, R., N. Vollmar, M. Greco, et
al.</a> (2019). “Unassigned diversity of planktonic
foraminifera from environmental sequencing revealed as known but
neglected species”. In: <em>PLoS ONE</em> 14.3. DOI:
<a href="https://doi.org/10.1371%2Fjournal.pone.0213936">10.1371/journal.pone.0213936</a>.</p>
<p><a name=bib-Przybos2019527></a><a href="#cite-Przybos2019527">PrzyboÅš, E. and S.
Tarcz</a> (2019). “Global molecular variation of
Paramecium jenningsi complex (Ciliophora, Protista): a starting point
for further, detailed biogeography surveys”. In: <em>Systematics and
Biodiversity</em> 17.5, pp. 527-539. DOI:
<a href="https://doi.org/10.1080%2F14772000.2019.1643424">10.1080/14772000.2019.1643424</a>.</p>
<p><a name=bib-Richardson2019></a><a href="#cite-Richardson2019">Richardson, E. and J.
Dacks</a> (2019). “Microbial Eukaryotes in oil sands
environments: Heterotrophs in the spotlight”. In: <em>Microorganisms</em> 7.6.
DOI:
<a href="https://doi.org/10.3390%2Fmicroorganisms7060178">10.3390/microorganisms7060178</a>.</p>
<p><a name=bib-Rimet2019></a><a href="#cite-Rimet2019">Rimet, F., E. Gusev, M. Kahlert, et
al.</a> (2019). “Diat.barcode, an open-access curated
barcode library for diatoms”. In: <em>Scientific Reports</em> 9.1. DOI:
<a href="https://doi.org/10.1038%2Fs41598-019-51500-6">10.1038/s41598-019-51500-6</a>.</p>
<p><a name=bib-Santoferrara2019571></a><a href="#cite-Santoferrara2019571">Santoferrara,
L.</a> (2019). “Current practice in plankton
metabarcoding: Optimization and error management”. In: <em>Journal of
Plankton Research</em> 41.5, pp. 571-582. DOI:
<a href="https://doi.org/10.1093%2Fplankt%2Ffbz041">10.1093/plankt/fbz041</a>.</p>
<p><a name=bib-Yu2019474></a><a href="#cite-Yu2019474">Yu, Z. and N. Chen</a> (2019).
“Emerging trends in red tide and major research progresses
[å›½å†…å¤–èµ¤æ½®çš„å’å±•è¶‹åŠ¿ä¸Žç ”ç©¶çƒ­ç‚¹]”. In: <em>Oceanologia et
Limnologia Sinica</em> 50.3, pp. 474-486. DOI:
<a href="https://doi.org/10.11693%2Fhyhz20190200041">10.11693/hyhz20190200041</a>.</p>
</div>
<div id="section-4" class="section level2">
<h2>2018</h2>
<p><a name=bib-Duarte2018></a><a href="#cite-Duarte2018">Duarte, S., L. Nunes, P. Borges, et
al.</a> (2018). “A bridge too far? An integrative
framework linking classical protist taxonomy and metabarcoding in lower
termites”. In: <em>Frontiers in Microbiology</em> 9.NOV. DOI:
<a href="https://doi.org/10.3389%2Ffmicb.2018.02620">10.3389/fmicb.2018.02620</a>.</p>
</div>
