---
title: "Papers citing EukRef"
author: vaulot
date: '2019-10-15'
categories: ["papers"]
tags: []
bibliography: eukref.bib
csl: science.csl
nocite: '@*'
---



<p><img src="./../eukref_paper_2018.png" /></p>
<p>del Campo J., Kolisko M., Boscaro V., Santoferrara LF., Nenarokov S., Massana R., Guillou L., Simpson A., Berney C., de Vargas C., Brown MW., Keeling PJ., Wegener Parfrey L. 2018. <a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2005849">EukRef: Phylogenetic curation of ribosomal RNA to enhance understanding of eukaryotic diversity and distribution</a>. PLOS Biology 16:e2005849. DOI: 10.1371/journal.pbio.2005849.</p>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Rimet2019">
<p>1. F. Rimet, E. Gusev, M. Kahlert, M. G. Kelly, M. Kulikovskiy, Y. Maltsev, D. G. Mann, M. Pfannkuchen, R. Trobajo, V. Vasselon, J. Zimmermann, A. Bouchez, Diat.barcode, an open-access curated barcode library for diatoms. <em>Scientific Reports</em>. <strong>9</strong> (2019), doi:<a href="https://doi.org/10.1038/s41598-019-51500-6">10.1038/s41598-019-51500-6</a>.</p>
</div>
<div id="ref-Ku2019">
<p>2. C. Ku, A. Sebé-Pedrós, Using single-cell transcriptomics to understand functional states and interactions in microbial eukaryotes. <em>Philosophical Transactions of the Royal Society B: Biological Sciences</em>. <strong>374</strong> (2019), doi:<a href="https://doi.org/10.1098/rstb.2019.0098">10.1098/rstb.2019.0098</a>.</p>
</div>
<div id="ref-Metz20193885">
<p>3. S. Metz, D. Singer, I. Domaizon, F. Unrein, E. Lara, Global distribution of trebouxiophyceae diversity explored by high-throughput sequencing and phylogenetic approaches. <em>Environmental Microbiology</em>. <strong>21</strong>, 3885–3895 (2019).</p>
</div>
<div id="ref-Linard20193303">
<p>4. B. Linard, K. Swenson, F. Pardi, R. Schwartz, Rapid alignment-free phylogenetic identification of metagenomic sequences. <em>Bioinformatics</em>. <strong>35</strong>, 3303–3312 (2019).</p>
</div>
<div id="ref-Przybo2019527">
<p>5. E. Przybo, S. Tarcz, Global molecular variation of paramecium jenningsi complex (ciliophora, protista): A starting point for further, detailed biogeography surveys. <em>Systematics and Biodiversity</em>. <strong>17</strong>, 527–539 (2019).</p>
</div>
<div id="ref-Richardson2019">
<p>6. E. Richardson, J. B. Dacks, Microbial eukaryotes in oil sands environments: Heterotrophs in the spotlight. <em>Microorganisms</em>. <strong>7</strong> (2019), doi:<a href="https://doi.org/10.3390/microorganisms7060178">10.3390/microorganisms7060178</a>.</p>
</div>
<div id="ref-Yu2019474">
<p>7. Z.-M. Yu, N.-S. Chen, Emerging trends in red tide and major research progresses [国内外赤潮的发展趋势与研究热点]. <em>Oceanologia et Limnologia Sinica</em>. <strong>50</strong>, 474–486 (2019).</p>
</div>
<div id="ref-Morard2019">
<p>8. R. Morard, N. M. Vollmar, M. Greco, M. Kucera, Unassigned diversity of planktonic foraminifera from environmental sequencing revealed as known but neglected species. <em>PLoS ONE</em>. <strong>14</strong> (2019), doi:<a href="https://doi.org/10.1371/journal.pone.0213936">10.1371/journal.pone.0213936</a>.</p>
</div>
<div id="ref-Caron2019197">
<p>9. D. A. Caron, S. K. Hu, Are we overestimating protistan diversity in nature? <em>Trends in Microbiology</em>. <strong>27</strong>, 197–205 (2019).</p>
</div>
<div id="ref-Adl20194">
<p>10. S. M. Adl, D. Bass, C. E. Lane, J. Lukeš, C. L. Schoch, A. Smirnov, S. Agatha, C. Berney, M. W. Brown, F. Burki, P. Cárdenas, I. Čepička, L. Chistyakova, J. del Campo, M. Dunthorn, B. Edvardsen, Y. Eglit, L. Guillou, V. Hampl, A. A. Heiss, M. Hoppenrath, T. Y. James, A. Karnkowska, S. Karpov, E. Kim, M. Kolisko, A. Kudryavtsev, D. J. Lahr, E. Lara, L. Le Gall, D. H. Lynn, D. G. Mann, R. Massana, E. A. Mitchell, C. Morrow, J. S. Park, J. W. Pawlowski, M. J. Powell, D. J. Richter, S. Rueckert, L. Shadwick, S. Shimano, F. W. Spiegel, G. Torruella, N. Youssef, V. Zlatogursky, Q. Zhang, Revisions to the classification, nomenclature, and diversity of eukaryotes. <em>Journal of Eukaryotic Microbiology</em>. <strong>66</strong>, 4–119 (2019).</p>
</div>
<div id="ref-DelCampo2019">
<p>11. J. Del Campo, T. J. Heger, R. Rodríguez-Martínez, A. Z. Worden, T. A. Richards, R. Massana, P. J. Keeling, Assessing the diversity and distribution of apicomplexans in host and free-living environments using high-throughput amplicon data and a phylogenetically informed reference framework. <em>Frontiers in Microbiology</em>. <strong>10</strong> (2019), doi:<a href="https://doi.org/10.3389/fmicb.2019.02373">10.3389/fmicb.2019.02373</a>.</p>
</div>
<div id="ref-Lutz2019115">
<p>12. S. Lutz, L. Procházková, L. G. Benning, L. Nedbalová, D. Remias, Evaluating amplicon high–throughput sequencing data of microalgae living in melting snow: Improvements and limitations. <em>Fottea</em>. <strong>19</strong>, 115–131 (2019).</p>
</div>
<div id="ref-Gottschling2019">
<p>13. M. Gottschling, J. Chacón, A. Žerdoner Čalasan, S. Neuhaus, J. Kretschmann, H. Stibor, U. John, Phylogenetic placement of environmental sequences using taxonomically reliable databases helps to rigorously assess dinophyte biodiversity in bavarian lakes (germany). <em>Freshwater Biology</em> (2019), doi:<a href="https://doi.org/10.1111/fwb.13413">10.1111/fwb.13413</a>.</p>
</div>
<div id="ref-Duarte2018">
<p>14. S. Duarte, L. Nunes, P. A. Borges, T. Nobre, A bridge too far? An integrative framework linking classical protist taxonomy and metabarcoding in lower termites. <em>Frontiers in Microbiology</em>. <strong>9</strong> (2018), doi:<a href="https://doi.org/10.3389/fmicb.2018.02620">10.3389/fmicb.2018.02620</a>.</p>
</div>
</div>
</div>
