+++
widget = "blank"  # See https://sourcethemes.com/academic/docs/page-builder/
headless = true  # This file represents a page section.
active = true  # Activate this widget? true/false
weight = 15  # Order that this section will appear.

title = "Papers citing PR2"
subtitle = ""
date = "2019-08-01"

[design]
  # Choose how many columns the section has. Valid values: 1 or 2.
  columns = "1"

[design.background]
  # Apply a background color, gradient, or image.
  #   Uncomment (by removing `#`) an option to apply it.
  #   Choose a light or dark text color by setting `text_color_light`.
  #   Any HTML color name or Hex value is valid.

  # Background color.
  # color = "navy"
  
  # Background gradient.
  # gradient_start = "DeepSkyBlue"
  # gradient_end = "SkyBlue"
  
  # Background image.
  image = "headers/bubbles-wide.jpg"  # Name of image in `static/img/`.
  image_darken = 0.6  # Darken the image? Range 0-1 where 0 is transparent and 1 is opaque.
  image_size = "cover"  #  Options are `cover` (default), `contain`, or `actual` size.
  image_position = "center"  # Options include `left`, `center` (default), or `right`.
  image_parallax = true  # Use a fun parallax-like fixed background effect? true/false

  # Text color (true=light or false=dark).
  text_color_light = true

[design.spacing]
  # Customize the section spacing. Order is top, right, bottom, left.
  padding = ["20px", "0", "20px", "0"]

[advanced]
 # Custom CSS. 
 css_style = ""
 
 # CSS class.
 css_class = ""
+++

Total number of papers citing PR2: **324** (November 2019)

![](./../../img/pr2_citations_years.jpg)

## 2019
1. Kolody BC., McCrow JP., Allen LZ., Aylward FO., Fontanez KM., Moustafa A., Moniruzzaman M., Chavez FP., Scholin CA., Allen EE., Worden AZ., Delong EF., Allen AE. 2019. Diel transcriptional response of a California Current plankton microbiome to light, low iron, and enduring viral infection. The ISME Journal. DOI: 10.1038/s41396-019-0472-2.
1. Wu W., Huang B. 2019. Protist diversity and community assembly in surface sediments of the South China Sea. MicrobiologyOpen:e891. DOI: 10.1002/mbo3.891.
1. Cleary DFR., Gomes NCM. 2019. Baseline information on prokaryotic and microeukaryotic plankton communities inside and outside of Indonesian marine lakes. Journal of Sea Research 148–149:23–32. DOI: 10.1016/j.seares.2019.04.001.
1. Marquardt M., Skogseth R., Wiedmann I., Vader A., Reigstad M., Cottier F., Gabrielsen T. 2019. Vertical export of marine pelagic protists in an ice-free high-Arctic fjord (Adventfjorden, West Spitsbergen) throughout 2011-2012. Aquatic Microbial Ecology 83:65–82. DOI: 10.3354/ame01904.
1. Kataoka T., Kondo R. 2019. Protistan community composition in anoxic sediments from three salinity-disparate Japanese lakes. Estuarine, Coastal and Shelf Science 224:34–42. DOI: 10.1016/j.ecss.2019.04.046.
1. Gómez-Consarnau L., Needham DM., Weber PK., Fuhrman JA., Mayali X. 2019. Influence of Light on Particulate Organic Matter Utilization by Attached and Free-Living Marine Bacteria. Frontiers in Microbiology 10:1–11. DOI: 10.3389/fmicb.2019.01204.
1. Cleary DFR. 2019. A comparison of microeukaryote communities inhabiting sponges and seawater in a Taiwanese coral reef system. Annals of Microbiology. DOI: 10.1007/s13213-019-01476-5.
1. Boeuf D., Edwards BR., Eppley JM., Hu SK., Poff KE., Romano AE., Caron DA., Karl DM., DeLong EF. 2019. Biological composition and microbial dynamics of sinking particulate organic matter at abyssal depths in the oligotrophic open ocean. Proceedings of the National Academy of Sciences:201903080. DOI: 10.1073/pnas.1903080116.
1. Wilschut RA., Geisen S., Martens H., Kostenko O., Hollander M., Hooven FC., Weser C., Snoek LB., Bloem J., Caković D., Čelik T., Koorem K., Krigas N., Manrubia M., Ramirez KS., Tsiafouli MA., Vreš B., Putten WH. 2019. Latitudinal variation in soil nematode communities under climate warming‐related range‐expanding and native plants. Global Change Biology:gcb.14657. DOI: 10.1111/gcb.14657.
1. Ribeiro H., Martins A., Gonçalves M., Guedes M., Tomasino MP., Dias N., Dias A., Mucha AP., Carvalho MF., Almeida CMR., Ramos S., Almeida JM., Silva E., Magalhães C. 2019. Development of an autonomous biosampler to capture in situ aquatic microbiomes. PLOS ONE 14:e0216882. DOI: 10.1371/journal.pone.0216882.
1. Liu L., Chen H., Liu M., Yang JR., Xiao P., Wilkinson DM., Yang J. 2019. Response of the eukaryotic plankton community to the cyanobacterial biomass cycle over 6 years in two subtropical reservoirs. The ISME Journal. DOI: 10.1038/s41396-019-0417-9.
1. Giner CR., Balagué V., Krabberød AK., Ferrera I., Reñé A., Garcés E., Gasol JM., Logares R., Massana R. 2019. Quantifying long‐term recurrence in planktonic microbial eukaryotes. Molecular Ecology 28:923–935. DOI: 10.1111/mec.14929.
1. Lentendu G., Mahé F., Bass D., Rueckert S., Stoeck T., Dunthorn M. 2018. Consistent patterns of high alpha and low beta diversity in tropical parasitic and free-living protists. Molecular Ecology 27:2846–2857. DOI: 10.1111/mec.14731.
1. Enberg S., Majaneva M., Autio R., Blomster J., Rintala J. 2018. Phases of microalgal succession in sea ice and the water column in the Baltic Sea from autumn to spring. Marine Ecology Progress Series 599:19–34. DOI: 10.3354/meps12645.
1. Giner CR., Balagué V., Pernice MC., Duarte CM. 2019. Marked changes in diversity and relative activity of picoeukaryotes with depth in the global ocean. bioRxiv. DOI: 10.1101/552604.
1. Metz S., Lopes dos Santos A., Berman MC., Bigeard E. 2019. Diversity of photosynthetic picoeukaryotes in eutrophic shallow lakes as assessed by combining flow cytometry cell-sorting and high throughput sequencing. bioRxiv. DOI: 10.1101/551598.
1. Pasulka A., Hu SK., Countway PD., Coyne KJ., Cary SC., Heidelberg KB., Caron DA. 2019. <scp>SSU</scp> ‐ <scp>rRNA</scp> Gene Sequencing Survey of Benthic Microbial Eukaryotes from Guaymas Basin Hydrothermal Vent. Journal of Eukaryotic Microbiology:jeu.12711. DOI: 10.1111/jeu.12711.
1. Ramond P., Sourisseau M., Simon N., Romac S., Schmitt S., Rigaut-Jalabert F., Henry N., de Vargas C., Siano R. 2019. Coupling between taxonomic and functional diversity in protistan coastal communities. Environmental Microbiology. DOI: 10.1111/1462-2920.14537.
1. Venter PC., Nitsche F., Arndt H. 2018. The Hidden Diversity of Flagellated Protists in Soil. Protist 169:432–449. DOI: 10.1016/j.protis.2018.04.007.
1. Adl SM., Bass D., Lane CE., Lukeš J., Schoch CL., Smirnov A., Agatha S., Berney C., Brown MW., Burki F., Cárdenas P., Čepička I., Chistyakova L., del Campo J., Dunthorn M., Edvardsen B., Eglit Y., Guillou L., Hampl V., Heiss AA., Hoppenrath M., James TY., Karpov S., Kim E., Kolisko M., Kudryavtsev A., Lahr DJG., Lara E., Le Gall L., Lynn DH., Mann DG., Massana i Molera R., Mitchell EAD., Morrow C., Park JS., Pawlowski JW., Powell MJ., Richter DJ., Rueckert S., Shadwick L., Shimano S., Spiegel FW., Torruella i Cortes G., Youssef N., Zlatogursky V., Zhang Q. 2019. Revisions to the Classification, Nomenclature, and Diversity of Eukaryotes. Journal of Eukaryotic Microbiology 66:4–119. DOI: 10.1111/jeu.12691.
1. Moore LR., Huang T., Ostrowski M., Mazard S. 2019. Unicellular Cyanobacteria Are Important Components of Phytoplankton Communities in Australia ’ s Northern Oceanic Ecoregions. Frontiers in Microbiology 9:1–16. DOI: 10.3389/fmicb.2018.03356.
1. de Sousa AGG., Tomasino MP., Duarte P., Fernández-Méndez M., Assmy P., Ribeiro H., Surkont J., Leite RB., Pereira-Leal JB., Torgo L., Magalhães C. 2019. Diversity and Composition of Pelagic Prokaryotic and Protist Communities in a Thin Arctic Sea-Ice Regime. Microbial Ecology. DOI: 10.1007/s00248-018-01314-2.
1. Faure E., Not F., Benoiston A-S., Labadie K., Bittner L., Ayata S-D. 2019. Mixotrophic protists display contrasted biogeographies in the global ocean. The ISME Journal. DOI: 10.1038/s41396-018-0340-5.
1. Gaonkar CC., Piredda R., Minucci C., Mann DG., Montresor M., Sarno D., Kooistra WHCF. 2018. Annotated 18S and 28S rDNA reference sequences of taxa in the planktonic diatom family Chaetocerotaceae. PLOS ONE 13:e0208929. DOI: 10.1371/journal.pone.0208929.
1. Varkey D., Mazard S., Jeffries TC., Hughes DJ., Seymour J., Paulsen IT., Ostrowski M. 2018. Stormwater influences phytoplankton assemblages within the diverse, but impacted Sydney Harbour estuary. Plos One 13:e0209857. DOI: 10.1371/journal.pone.0209857.
1. Angelakis E., Bachar D., Yasir M., Musso D., Djossou F., Gaborit B., Brah S., Diallo A., Ndombe GM., Mediannikov O., Robert C., Azhar EI., Bibi F., Nsana NS., Parra H-J., Akiana J., Sokhna C., Davoust B., Dutour A., Raoult D. 2019. Treponema species enrich the gut microbiota of traditional rural populations but are absent from urban individuals. New Microbes and New Infections 27:14–21. DOI: 10.1016/j.nmni.2018.10.009.

## 2018
1. Wangensteen, O.S., Cebrian, E., Palacin, C. & Turon, X. 2018. Under the canopy: Community-wide effects of invasive algae in Marine Protected Areas revealed by metabarcoding. Mar. Pollut. Bull. 127:54–66.
1. von Ammon, U., Wood, S.A., Laroche, O., Zaiko, A., Tait, L., Lavery, S., Inglis, G. et al. 2018. The impact of artificial surfaces on marine bacterial and eukaryotic biofouling assemblages: A high-throughput sequencing analysis. Mar. Environ. Res. 133:57–66.
1. Tragin, M., Zingone, A. & Vaulot, D. 2018. Comparison of coastal phytoplankton composition estimated from the V4 and V9 regions of 18S rRNA gene with a focus on photosynthetic groups and especially Chlorophyta. Environ. Microbiol. 20:1–41.
1. Fiore-Donno, A.M., Rixen, C., Rippin, M., Glaser, K., Samolov, E., Karsten, U., Becker, B. et al. 2018. New barcoded primers for efficient retrieval of cercozoan sequences in high-throughput environmental diversity surveys, with emphasis on worldwide biological soil crusts. Mol. Ecol. Resour. 18:229–39.
1. Flegontova, O., Flegontov, P., Malviya, S., Poulain, J., de Vargas, C., Bowler, C., Lukeš, J. et al. 2018. Neobodonids are dominant kinetoplastids in the global ocean. Environ. Microbiol. 20:878–89.
1. Vader, A., Laughinghouse, H.D., Griffiths, C., Jakobsen, K.S. & Gabrielsen, T.M. 2018. Proton-pumping rhodopsins are abundantly expressed by microbial eukaryotes in a high-Arctic fjord. Environ. Microbiol. 20:890–902.
1. Szelecz, I.I., Lösch, S., Seppey, C.V.W., Lara, E., Singer, D., Sorge, F., Tschui, J. et al. 2018. Comparative analysis of bones, mites, soil chemistry, nematodes and soil micro-eukaryotes from a suspected homicide to estimate the post-mortem interval. Sci. Rep. 8:25.
1. Vader, A., Laughinghouse, H.D., Griffiths, C., Jakobsen, K.S. & Gabrielsen, T.M. 2018. Proton-pumping rhodopsins are abundantly expressed by microbial eukaryotes in a high-Arctic fjord. Environ. Microbiol. 20:890–902.
1. Gołębiewski, M., Tarasek, A., Sikora, M., Deja-Sikora, E., Tretyn, A. & Niklińska, M. 2018. Rapid Microbial Community Changes During Initial Stages of Pine Litter Decomposition. Microb. Ecol.
1. Kisand, V., Talas, L., Kisand, A., Stivrins, N., Reitalu, T., Alliksaar, T., Vassiljev, J. et al. 2018. From microbial eukaryotes to metazoan vertebrates: Wide spectrum paleo-diversity in sedimentary ancient DNA over the last ~14,500 years. Geobiology. 1–12.
1. Oliverio, A.M., Power, J.F., Washburne, A., Cary, S.C., Stott, M.B. & Fierer, N. 2018. The ecology and diversity of microbial eukaryotes in geothermal springs. ISME J. 1.
1. Bahram, M., Hildebrand, F., Forslund, S.K., Anderson, J.L., Soudzilovskaia, N.A., Bodegom, P.M., Bengtsson-Palme, J. et al. 2018. Structure and function of the global topsoil microbiome. Nature.
1. Fiore-Donno, A.M., Rixen, C., Rippin, M., Glaser, K., Samolov, E., Karsten, U., Becker, B. et al. 2018. New barcoded primers for efficient retrieval of cercozoan sequences in high-throughput environmental diversity surveys, with emphasis on worldwide biological soil crusts. Mol. Ecol. Resour. 18:229–39.
1. Needham, D.M., Fichot, E.B., Wang, E., Berdjeb, L., Cram, J.A., Fichot, C.G. & Fuhrman, J.A. 2018. Dynamics and interactions of highly resolved marine plankton via automated high frequency sampling. bioRxiv.
1. Hunsucker, K.Z., Vora, G.J., Hunsucker, J.T., Gardner, H., Leary, D.H., Kim, S., Lin, B. et al. 2018. Biofilm community structure and the associated drag penalties of a groomed fouling release ship hull coating. Biofouling. 34:162–72.
1. Stern, R., Kraberg, A., Bresnan, E., Kooistra, W.H.C.F., Lovejoy, C., Montresor, M., Morán, X.A.G. et al. 2018. Molecular analyses of protists in long-term observation programmes—current status and future perspectives. J. Plankton Res. in press.
1. Mukherjee, I., Hodoki, Y. & Nakano, S. 2018. Seasonal dynamics of heterotrophic and plastidic protists in the water column of Lake Biwa, Japan. Aquat. Microb. Ecol. 80:123–37.
1. Rachik, S., Christaki, U., Li, L.L., Genitsaris, S., Breton, E. & Monchy, S. 2018. Diversity and potential activity patterns of planktonic eukaryotic microbes in a mesoeutrophic coastal area (eastern English Channel). PLoS One. 13:e0196987.
1. Leff, J.W., Bardgett, R.D., Wilkinson, A., Jackson, B.G., Pritchard, W.J., De Long, J.R., Oakley, S. et al. 2018. Predicting the structure of soil communities from plant community taxonomy, phylogeny, and traits. ISME J.
1. Herve, V., Leroy, B., Pires, A.D.S., Lopez, P.J., Hervé, V., Leroy, B., Da Silva Pires, A. et al. 2018. Aquatic urban ecology at the scale of a capital: community structure and interactions in street gutters. ISME J. 12:253–66.
1. López-Escardó, D., Paps, J., de Vargas, C., Massana, R., Ruiz-Trillo, I. & del Campo, J. 2018. Metabarcoding analysis on European coastal samples reveals new molecular metazoan diversity. Sci. Rep. 8:9106.
1. Xie, Y., Floehr, T., Zhang, X., Xiao, H., Yang, J., Xia, P., Burton, G.A. et al. 2018. In situ microbiota distinguished primary anthropogenic stressor in freshwater sediments. Environ. Pollut. 239:189–97.
1. Wylezich, C., Herlemann, D.P.R. & Juergens, K. 2018. Improved 18S rDNA amplification protocol for assessing protist diversity in oxygen-deficient marine systems. Aquat. Microb. Ecol. 81:83–94.
1. Tan, S. & Liu, H. 2018. Unravel the hidden protistan diversity: application of blocking primers to suppress PCR amplification of metazoan DNA. Appl. Microbiol. Biotechnol. 102:389–401.
1. Cai, J., Bai, C., Tang, X., Dai, J., Gong, Y., Hu, Y., Shao, K. et al. 2018. Characterization of bacterial and microbial eukaryotic communities associated with an ephemeral hypoxia event in Taihu Lake, a shallow eutrophic Chinese lake. Environ. Sci. Pollut. Res. 25:31543–57.
1. del Campo, J., Kolisko, M., Boscaro, V., Santoferrara, L.F., Nenarokov, S., Massana, R., Guillou, L. et al. 2018. EukRef: Phylogenetic curation of ribosomal RNA to enhance understanding of eukaryotic diversity and distribution. PLOS Biol. 16:e2005849.
1. Zouari, A.B., Hassen, M.B., Balague, V., Sahli, E., Ben Kacem, M.Y., Akrout, F., Hamza, A. et al. 2018. Picoeukaryotic diversity in the Gulf of Gabes: variability patterns and relationships to nutrients and water masses. Aquat. Microb. Ecol. 81:37–53.
1. Forster, D., Filker, S., Kochems, R., Breiner, H.-W., Cordier, T., Pawlowski, J. & Stoeck, T. 2018. A Comparison of Different Ciliate Metabarcode Genes as Bioindicators for Environmental Impact Assessments of Salmon Aquaculture. J. Eukaryot. Microbiol. 1–15.
1. Lewitus, E., Bittner, L., Malviya, S., Bowler, C. & Morlon, H. 2018. Clade-specific diversification dynamics of marine diatoms since the Jurassic. Nat. Ecol. Evol. 2:1715–23.
1. Hernández-Ruiz, M., Barber-Lluch, E., Prieto, A., Álvarez-Salgado, X.A., Logares, R. & Teira, E. 2018. Seasonal succession of small planktonic eukaryotes inhabiting surface waters of a coastal upwelling system. Environ. Microbiol. in press.
1. Boscaro, V., Syberg-Olsen, M.J., Irwin, N.A.T., del Campo, J. & Keeling, P.J. 2018. What Can Environmental Sequences Tell Us About the Distribution of Low-Rank Taxa? The Case of Euplotes (Ciliophora, Spirotrichea), Including a Description of Euplotes enigma sp. nov. J. Eukaryot. Microbiol. 0–3.
1. Kamp, A., Petro, C., Røy, H., Nielsen, S., Carvalho, P., Stief, P. & Schramm, A. 2018. Intracellular nitrate in sediments of an oxygen-deficient marine basin is linked to pelagic diatoms. FEMS Microbiol. Ecol. 94:1–12.
1. Ser-Giacomi, E., Zinger, L., Malviya, S., De Vargas, C., Karsenti, E., Bowler, C. & De Monte, S. 2018. Ubiquitous abundance distribution of non-dominant plankton across the global ocean. Nat. Ecol. Evol.
1. Gebert, M.J., Delgado-Baquerizo, M., Oliverio, A.M., Webster, T.M., Nichols, L.M., Honda, J.R., Chan, E.D. et al. 2018. Ecological Analyses of Mycobacteria in Showerhead Biofilms and Their Relevance to Human Health. MBio. 9:366088.
1. Guo, S., Xiong, W., Xu, H., Hang, X., Liu, H., Xun, W., Li, R. et al. 2018. Continuous application of different fertilizers induces distinct bulk and rhizosphere soil protist communities. Eur. J. Soil Biol. 88:8–14.
1. Boast, A.P., Weyrich, L.S., Wood, J.R., Metcalf, J.L., Knight, R. & Cooper, A. 2018. Coprolites reveal ecological interactions lost with the extinction of New Zealand birds. Proc. Natl. Acad. Sci. U. S. A. 115:1546–51.
1. Clerissi, C., Brunet, S., Vidal-Dupiol, J., Adjeroud, M., Lepage, P., Guillou, L., Escoubas, J.-M. et al. 2018. Protists Within Corals: The Hidden Diversity. Front. Microbiol. 9:2043.
1. Arroyo, A.S., López-Escardó, D., Kim, E., Ruiz-Trillo, I. & Najle, S.R. 2018. Novel Diversity of Deeply Branching Holomycota and Unicellular Holozoans Revealed by Metabarcoding in Middle Paraná River, Argentina. Front. Ecol. Evol. 6:99.
1. Wu, W., Lu, H.-P., Sastri, A., Yeh, Y.-C., Gong, G.-C., Chou, W.-C. & Hsieh, C.-H. 2018. Contrasting the relative importance of species sorting and dispersal limitation in shaping marine bacterial versus protist communities. ISME J. 12:485–94.
1. Ammon, U. von, Wood, S.A., Laroche, O., Zaiko, A., Tait, L., Lavery, S., Inglis, G.J. et al. 2018. Combining morpho-taxonomy and metabarcoding enhances the detection of non-indigenous marine pests in biofouling communities. Sci. Rep. 8:16290.
1. Laroche, O., Wood, S.A., Tremblay, L.A., Ellis, J.I., Lear, G. & Pochon, X. 2018. A cross-taxa study using environmental DNA/RNA metabarcoding to measure biological impacts of offshore oil and gas drilling and production operations. Mar. Pollut. Bull. 127:97–107.
1. Boaventura, C.M., Coelho, F.J.R.C., Martins, P.T., Pires, A.C.C., Duarte, L.N., Uetanabaro, A.P.T., Cleary, D.F.R. et al. 2018. Micro-eukaryotic plankton diversity in an intensive aquaculture system for production of Scophthalmus maximus and Solea senegalensis. AQUACULTURE. 490:321–8.
1. Piwosz, K., Całkiewicz, J., Gołębiewski, M. & Creer, S. 2018. Diversity and community composition of pico- and nanoplanktonic protists in the Vistula River estuary (Gulf of Gdańsk, Baltic Sea). Estuar. Coast. Shelf Sci. 207:242–9.
1. Zhu, P., Wang, Y., Shi, T., Huang, G. & Gong, J. 2018. Genetic Diversity of Benthic Microbial Eukaryotes in Response to Spatial Heterogeneity of Sediment Geochemistry in a Mangrove Ecosystem. ESTUARIES AND COASTS. 41:751–64.
1. Leblanc, K., Queguiner, B., Diaz, F., Cornet, V., Michel-Rodriguez, M., de Madron, X.D., Bowler, C. et al. 2018. Nanoplanktonic diatoms are globally overlooked but play a role in spring blooms and carbon export. Nat. Commun. 9.
1. van der Reis, A.L. 2018. Preliminary analysis of New Zealand scampi ( Metanephrops challengeri ) diet using metabarcoding. PeerJ Prepr.
1. Morard, R., Garet-Delmas, M.-J., Mahe, F., Romac, S., Poulain, J., Kucera, M. & de Vargas, C. 2018. Surface ocean metabarcoding confirms limited diversity in planktonic foraminifera but reveals unknown hyper-abundant lineages. Sci. Rep. 8.
1. Berdjeb, L., Parada, A., Needham, D.M. & Fuhrman, J.A. 2018. Short-term dynamics and interactions of marine protist communities during the spring–summer transition. ISME J.

## 2017
1. Fletcher, L.M., Zaiko, A., Atalah, J., Richter, I., Dufour, C.M., Pochon, X., Wood, S.A. et al. 2017. Bilge water as a vector for the spread of marine pests: a morphological, metabarcoding and experimental assessment. Biol. Invasions. 19:2851–67.
1. Balzano, S. & Leterme, S.C. 2017. Persistent and top ranked microbial taxa can dominate marine microbial communities. Microb. Ecol.
1. Salvitti, L., Wood, S.A., Fairweather, R., Culliford, D., McNabb, P. & Cary, S.C. 2017. In situ accumulation of tetrodotoxin in non-toxic Pleurobranchaea maculata (Opisthobranchia). Aquat. Sci. 79:335–44.
1. Smith, K.F., Kohli, G.S., Murray, S.A. & Rhodes, L.L. 2017. Assessment of the metabarcoding approach for community analysis of benthic-epiphytic dinoflagellates using mock communities. NEW Zeal. J. Mar. Freshw. Res. 51:555–76.
1. Filker, S., Forster, D., Weinisch, L., Mora-Ruiz, M., Gonzalez, B., Eugenia Farias, M., Rossello-Mora, R. et al. 2017. Transition boundaries for protistan species turnover in hypersaline waters of different biogeographic regions. Environ. Microbiol. 19:3186–200.
1. Leray, M. & Knowlton, N. 2017. Random sampling causes the low reproducibility of rare eukaryotic OTUs in Illumina COI metabarcoding. PeerJ. 5.
1. Torruella, G., Moreira, D. & López-García, P. 2017. Phylogenetic and ecological diversity of apusomonads, a lineage of deep-branching eukaryotes. Environ. Microbiol. Rep. 00:1–7.
1. Machida, R.J., Leray, M., Ho, S.-L. & Knowlton, N. 2017. Data Descriptor: Metazoan mitochondrial gene sequence reference datasets for taxonomic assignment of environmental samples. Sci. DATA. 4.
1. dos Santos, A.L., Gourvil, P., Tragin, M., Noel, M.-H., Decelle, J., Romac, S., Vaulot, D. et al. 2017. Diversity and oceanic distribution of prasinophytes clade VII, the dominant group of green algae in oceanic waters. ISME J. 11:512–28.
1. Simon, N., Foulon, E., Grulois, D., Six, C., Desdevises, Y., Latimier, M., Le Gall, F. et al. 2017. Revision of the Genus Micromonas Manton et Parke (Chlorophyta, Mamiellophyceae), of the Type Species M. pusilla (Butcher) Manton &amp; Parke and of the Species M. commoda van Baren, Bachy and Worden and Description of Two New Species Based on the Genetic. Protist. 168:612–35.
1. Biard, T., Bigeard, E., Audic, S.S., Poulain, J., Gutierrez-Rodriguez, A., Pesant, S.S., Stemmann, L. et al. 2017. Biogeography and diversity of Collodaria (Radiolaria) in the global ocean. ISME J. 11:1331–44.
1. Piredda, R., Sarno, D., Lange, C.B., Tomasino, M.P., Zingone, A. & Montresor, M. 2017. Diatom resting stages in surface sediments: a pilot study comparing Next Generation Sequencing and Serial Dilution Cultures. Cryptogam. Algol. 38:31–46.
1. Wu, W., Logares, R., Huang, B. & Hsieh, C. 2017. Abundant and rare picoeukaryotic sub-communities present contrasting patterns in the epipelagic waters of marginal seas in the northwestern Pacific Ocean. Environ. Microbiol. 19:287–300.
1. Lee, K.H., Ab Samad, L.S., Lwin, P.M., Riedel, S.F., Magin, A., Bashir, M., Vaishampayan, P.A. et al. 2017. On the Rocks: Microbiological Quality and Microbial Diversity of Packaged Ice in Southern California. J. Food Prot. 80:1041–9.
1. Mahe, F., de Vargas, C., Bass, D., Czech, L., Stamatakis, A., Lara, E., Singer, D. et al. 2017. Parasites dominate hyperdiverse soil protist communities in Neotropical rainforests. Nat. Ecol. Evol. 1:0091.
1. Domaizon, I., Winegardner, A., Capo, E., Gauthier, J. & Gregory-Eaves, I. 2017. DNA-based methods in paleolimnology: new opportunities for investigating long-term dynamics of lacustrine biodiversity. J. Paleolimnol. 58:1–21.
1. Huang, T. 2017. Spatial heterogeneity defines microbial eukaryotic communities in a subtropical hotspot of production and nitrogen fixation. Environ. Microbiol.
1. Lohan, K.M.P., Fleischer, R.C., Carney, K.J., Holzer, K.K. & Ruiz, G.M. 2017. Molecular characterisation of protistan species and communities in ships’ ballast water across three US coasts. Divers. Distrib. 23:680–91.
1. Penna, A., Casabianca, S., Guerra, A.F., Vernesi, C. & Scardi, M. 2017. Analysis of phytoplankton assemblage structure in the Mediterranean Sea based on high-throughput sequencing of partial 18S rRNA sequences. Mar. Genomics. 36:49–55.
1. Torruella, G., Moreira, D. & Lopez-Garcia, P. 2017. Phylogenetic and ecological diversity of apusomonads, a lineage of deep-branching eukaryotes. Environ. Microbiol. Rep. 9:113–9.
1. Hugerth, L.W. & Andersson, A.F. 2017. Analysing Microbial Community Composition through Amplicon Sequencing: From Sampling to Hypothesis Testing. Front. Microbiol. 8.
1. Saghai, A., Gutierrez-Preciado, A., Deschamps, P., Moreira, D., Bertolino, P., Ragon, M. & Lopez-Garcia, P. 2017. Unveiling microbial interactions in stratified mat communities from a warm saline shallow pond. Environ. Microbiol. 19:2405–21.
1. Deiner, K., Bik, H.M., Machler, E., Seymour, M., Lacoursiere-Roussel, A., Altermatt, F., Creer, S. et al. 2017. Environmental DNA metabarcoding: Transforming how we survey animal and plant communities. Mol. Ecol. 26:5872–95.
1. Golebiewski, M., Calkiewicz, J., Creer, S. & Piwosz, K. 2017. Tideless estuaries in brackish seas as possible freshwater-marine transition zones for bacteria: the case study of the Vistula river estuary. Environ. Microbiol. Rep. 9:129–43.
1. Xie, Y., Wang, J., Yang, J., Giesy, J.P., Yu, H. & Zhang, X. 2017. Environmental DNA metabarcoding reveals primary chemical contaminants in freshwater sediments from different land-use types. Chemosphere. 172:201–9.
1. Weber, F., Mylnikov, A.P., Juergens, K. & Wylezich, C. 2017. Culturing Heterotrophic Protists from the Baltic Sea: Mostly the ``Usual Suspects{’’} but a Few Novelties as Well. J. Eukaryot. Microbiol. 64:153–63.
1. Kataoka, T., Yamaguchi, H., Sato, M., Watanabe, T., Taniuchi, Y., Kuwata, A. & Kawachi, M. 2017. Seasonal and geographical distribution of near-surface small photosynthetic eukaryotes in the western North Pacific determined by pyrosequencing of 18S rDNA. FEMS Microbiol. Ecol. 93:fiw229.
1. Dorrell, R.G., Gile, G., McCallum, G., Méheust, R., Bapteste, E.P., Klinger, C.M., Brillet-Guéguen, L. et al. 2017. Chimeric origins of ochrophytes and haptophytes revealed through an ancient plastid proteome. Elife. 6:1–45.
1. Marie, D., Le Gall, F., Edern, R., Gourvil, P. & Vaulot, D. 2017. Improvement of phytoplankton culture isolation using single cell sorting by flow cytometry. J. Phycol. 53:271–82.
1. Santos, S.S., Nunes, I., Nielsen, T.K., Jacquiod, S., Hansen, L.H. & Winding, A. 2017. Soil DNA Extraction Procedure Influences Protist 18S rRNA Gene Community Profiling Outcome. Protist. 168:283–93.
1. Sun, P., Huang, L., Xu, D., Huang, B., Chen, N. & Warren, A. 2017. Marked seasonality and high spatial variation in estuarine ciliates are driven by exchanges between the `abundant’ and `intermediate’ biospheres. Sci. Rep. 7.
1. Venter, P.C., Nitsche, F., Domonell, A., Heger, P. & Arndt, H. 2017. The Protistan Microbiome of Grassland Soil: Diversity in the Mesoscale. Protist. 168:546–64.
1. Xu, D., Jiao, N., Ren, R. & Warren, A. 2017. Distribution and Diversity of Microbial Eukaryotes in Bathypelagic Waters of the South China Sea. J. Eukaryot. Microbiol. 64:370–82.
1. Dunthorn, M., Kauserud, H., Bass, D., Mayor, J. & Mahe, F. 2017. Yeasts dominate soil fungal communities in three lowland Neotropical rainforests. Environ. Microbiol. Rep. 9:668–75.
1. Gran-Stadniczenko, S., Supraha, L., Egge, E.D. & Edvardsen, B. 2017. Haptophyte Diversity and Vertical Distribution Explored by 18S and 28S Ribosomal RNA Gene Metabarcoding and Scanning Electron Microscopy. J. Eukaryot. Microbiol. 64:514–32.
1. Berney, C.C., Ciuprina, A., Bender, S., Brodie, J., Edgcomb, V., Kim, E., Rajan, J. et al. 2017. UniEuk : Time to Speak a Common Language in Protistology! J. Eukaryot. Microbiol. 64:407–11.
1. Seppey, C.V.W., Singer, D., Dumack, K., Fournier, B., Belbahri, L., Mitchell, E.A.D. & Lara, E. 2017. Distribution patterns of soil microbial eukaryotes suggests widespread algivory by phagotrophic protists as an alternative pathway for nutrient cycling. SOIL Biol. Biochem. 112:68–76.
1. Khomich, M., Kauserud, H., Logares, R., Rasconi, S. & Andersen, T. 2017. Planktonic protistan communities in lakes along a large-scale environmental gradient. FEMS Microbiol. Ecol. 93.
1. Wang, Y., Guo, X., Zheng, P., Zou, S., Li, G. & Gong, J. 2017. Distinct seasonality of chytrid-dominated benthic fungal communities in the neritic oceans (Bohai Sea and North Yellow Sea). FUNGAL Ecol. 30:55–66.
1. Wampach, L., Heintz-Buschart, A., Hogan, A., Muller, E.E.L., Narayanasamy, S., Laczny, C.C., Hugerth, L.W. et al. 2017. Colonization and Succession within the Human Gut Microbiome by Archaea, Bacteria, and Microeukaryotes during the First Year of Life. Front. Microbiol. 8.
1. Morard, R., Lejzerowicz, F., Darling, K.F., Lecroq-Bennet, B., Pedersen, M.W., Orlando, L., Pawlowski, J. et al. 2017. Planktonic foraminifera-derived environmental DNA extracted from abyssal sediments preserves patterns of plankton macroecology. BIOGEOSCIENCES. 14:2741–54.
1. Heinz, K.G.H., Zanoni, P.R.S., Oliveira, R.R., Medina-Silva, R., Simo, T.L.L., Trindade, F.J., Pereira, L.M. et al. 2017. Recycled Paper Sludge Microbial Community as a Potential Source of Cellulase and Xylanase Enzymes. WASTE AND BIOMASS VALORIZATION. 8:1907–17.
1. Tirichine, L., Rastogi, A. & Bowler, C. 2017. Recent progress in diatom genomics and epigenomics. Curr. Opin. Plant Biol. 36:46–55.
1. Foster, Z.S.L., Sharpton, T.J. & Grünwald, N.J. 2017. Metacoder: An R package for visualization and manipulation of community taxonomic diversity data. PLOS Comput. Biol. 13:e1005404.
1. Lohan, K.M.P., Fleischer, R.C., Torchin, M.E. & Ruiz, G.M. 2017. Protistan Biogeography: A Snapshot Across a Major Shipping Corridor Spanning Two Oceans. Protist. 168:183–96.
1. Pearman, J.K., Ellis, J., Irigoien, X., Sarma, Y.V.B., Jones, B.H. & Carvalho, S. 2017. Microbial planktonic communities in the Red Sea: high levels of spatial and temporal variability shaped by nutrient availability and turbulence. Sci. Rep. 7:6611.
1. Simon, N., Foulon, E., Grulois, D., Six, C., Desdevises, Y., Latimier, M., Le Gall, F. et al. 2017. Revision of the Genus Micromonas Manton et Parke (Chlorophyta, Mamiellophyceae), of the Type Species M-pusilla (Butcher) Manton & Parke and of the Species M-commoda van Baren, Bachy and Worden and Description of Two New Species Based on the Genetic and Ph. Protist. 168:612–35.
1. Kaysen, A., Heintz-Buschart, A., Muller, E.E.L., Narayanasamy, S., Wampach, L., Laczny, C.C., Graf, N. et al. 2017. Integrated meta-omic analyses of the gastrointestinal tract microbiome in patients undergoing allogeneic hematopoietic stem cell transplantation. Transl. Res. 186:79–94.
1. Xie, Y., Hong, S., Kim, S., Zhang, X., Yang, J., Giesy, J.P., Wang, T. et al. 2017. Ecogenomic responses of benthic communities under multiple stressors along the marine and adjacent riverine areas of northern Bohai Sea, China. Chemosphere. 172:166–74.
1. Xu, D., Li, R., Hu, C., Sun, P., Jiao, N. & Warren, A. 2017. Microbial Eukaryote Diversity and Activity in the Water Column of the South China Sea Based on DNA and RNA High Throughput Sequencing. Front. Microbiol. 8:1–14.
1. Piredda, R., Tomasino, M.P., D’Erchia, A.M., Manzari, C., Pesole, G., Montresor, M., Kooistra, W.H.C.F. et al. 2017. Diversity and temporal patterns of planktonic protist assemblages at a Mediterranean Long Term Ecological Research site. FEMS Microbiol. Ecol. 93:fiw200.
1. Alves-de-Souza, C., Benevides, T.S., Santos, J.B.O., Von Dassow, P., Guillou, L. & Menezes, M. 2017. Does environmental heterogeneity explain temporal β diversity of small eukaryotic phytoplankton? Example from a tropical eutrophic coastal lagoon. J. Plankton Res. in press:1–17.
1. Li, S., Bronner, G., Lepère, C., Kong, F. & Shi, X. 2017. Temporal and spatial variations in the composition of freshwater photosynthetic picoeukaryotes revealed by MiSeq sequencing from flow cytometry sorted samples. Environ. Microbiol. 1146–7.
1. Zouari, A.B. 2017. Picoeukaryotic diversity in the Gulf of Gabès: variability patterns and relationships to nutrients and water masses. Aquat. Microb. Ecol. submitted:1–42.
1. Reich, M. & Labes, A. 2017. How to boost marine fungal research: A first step towards a multidisciplinary approach by combining molecular fungal ecology and natural products chemistry. Mar. Genomics. 36:57–75.
1. Meshram, A.R., Vader, A., Kristiansen, S. & Gabrielsen, T.M. 2017. Microbial Eukaryotes in an Arctic Under-Ice Spring Bloom North of Svalbard. Front. Microbiol. 8:3–6.
1. Pochon, X., Zaiko, A., Fletcher, L.M., Laroche, O. & Wood, S.A. 2017. Wanted dead or alive? Using metabarcoding of environmental DNA and RNA to distinguish living assemblages for biosecurity applications. PLoS One. 12:e0187636.
1. Schnittler, M., Shchepin, O.N., Dagamac, N.H.A., Dahl, M.B. & Novozhilov, Y.K. 2017. Barcoding myxomycetes with molecular markers: challenges and opportunities. Nova Hedwigia. 104:323–41.
1. Laroche, O., Wood, S.A., Tremblay, L.A., Lear, G., Ellis, J.I. & Pochon, X. 2017. Metabarcoding monitoring analysis: the pros and cons of using co-extracted environmental DNA and RNA data to assess offshore oil production impacts on benthic communities. PeerJ. 5.
1. Gérikas Ribeiro, C., Lopes dos Santos, A., Marie, D., Brandini, P. & Vaulot, D. 2017. Relationships between photosynthetic eukaryotes and nitrogen-fixing cyanobacteria off Brazil. ISME J. submitted.
1. Jacob, F., Melachio, T.T., Njitchouang, G.R., Gimonneau, G., Njiokou, F., Abate, L., Christen, R. et al. 2017. Intestinal Bacterial Communities of Trypanosome-Infected and Uninfected Glossina palpalis palpalis from Three Human African Trypanomiasis Foci in Cameroon. Front. Microbiol. 8.

## 2016
1. Ploch, S., Rose, L.E., Bass, D. & Bonkowski, M. 2016. High Diversity Revealed in Leaf-Associated Protists (Rhizaria: Cercozoa) of Brassicaceae. J. Eukaryot. Microbiol. 63:635–41.
1. Nitsche, F. 2016. A phylogenetic and morphological re-investigation of Diaphanoeca spiralifurca, Didymoeca elongata and Polyoeca dichotoma (Acanthoecida/Choanomonadida) from the Caribbean Sea. Eur. J. Protistol. 52:58–64.
1. Schiaffino, M.R., Lara, E., Fernández, L.D., Balagué, V., Singer, D., Seppey, C.C.W., Massana, R. et al. 2016. Microbial eukaryote communities exhibit robust biogeographical patterns along a gradient of Patagonian and Antarctic lakes. Environ. Microbiol. 18:5249–64.
1. Dubourg, G., Lagier, J.-C., Hue, S., Surenaud, M., Bachar, D., Robert, C., Michelle, C. et al. 2016. Gut microbiota associated with HIV infection is significantly enriched in bacteria tolerant to oxygen. BMJ OPEN Gastroenterol. 3.
1. Gimmler, A., Korn, R., de Vargas, C., Audic, S. & Stoeck, T. 2016. The Tara Oceans voyage reveals global diversity and distribution patterns of marine planktonic ciliates. Sci. Rep. 6:33555.
1. Giner, C.R., Forn, I., Romac, S., Logares, R., de Vargas, C. & Massana, R. 2016. Environmental sequencing provides reasonable estimates of the relative abundance of specific picoeukaryotes. Appl. Environ. Microbiol. AEM.00560-16.
1. Scholz, B., Guillou, L., Marano, A. V, Neuhauser, S., Sullivan, B.K., Karsten, U., Küpper, F.C. et al. 2016. Zoosporic parasites infecting marine diatoms – A black box that needs to be opened. Fungal Ecol. 19:59–76.
1. Groendahl, S. 2016. The best of two worlds : A combined approach for analyzing microalgal diversity via metabarcoding and morphology-based methods. PLoS One.
1. Genitsaris, S., Monchy, S., Denonfoux, J., Ferreira, S., Kormas, K.A., Sime-Ngando, T., Viscogliosi, E. et al. 2016. Marine microbial community structure assessed from combined metagenomic analysis and ribosomal amplicon deep-sequencing. Mar. Biol. Res. 12:30–42.
1. Li, Y., He, Y., Wei, X., Liu, X. & Bi, Y. 2016. Unveiling picoeukaryotic diversity and distribution in the bay of large reservoir system. Aquat. Microb. Ecol.
1. Pagenkopp Lohan, K.M., Fleischer, R.C., Carney, K.J., Holzer, K.K. & Ruiz, G.M. 2016. Amplicon-Based Pyrosequencing Reveals High Diversity of Protistan Parasites in Ships’ Ballast Water: Implications for Biogeography and Infectious Diseases. Microb. Ecol. 71:530–42.
1. Aguilar, M., Richardson, E., Tan, B., Walker, G., Dunfield, P.F., Bass, D., Nesbo, C. et al. 2016. Next-generation Sequencing Assessment of Eukaryotic Diversity in Oil Sands Tailings Ponds Sediments and Surface Water. J. Eukaryot. Microbiol. 63:732–43.
1. Shazib, S.U.A., Vd’acny, P., Kim, J.H., Jang, S.W. & Shin, M.K. 2016. Molecular phylogeny and species delimitation within the ciliate genus Spirostomum (Ciliophora, Postciliodesmatophora, Heterotrichea), using the internal transcribed spacer region. Mol. Phylogenet. Evol. 102:128–44.
1. Angelakis, E., Yasir, M., Bachar, D., Azhar, E.I., Lagier, J.-C., Bibi, F., Jiman-Fatani, A.A. et al. 2016. Gut microbiome and dietary patterns in different Saudi populations and monkeys. Sci. Rep. 6.
1. Singer, D., Lara, E., Steciow, M.M., Seppey, C.V.W., Paredes, N., Pillonel, A., Oszako, T. et al. 2016. High-throughput sequencing reveals diverse oomycete communities in oligotrophic peat bog micro-habitat. FUNGAL Ecol. 23:42–7.
1. Coelho, F.J.R.C., Louvado, A., Domingues, P.M., Cleary, D.F.R., Ferreira, M., Almeida, A., Cunha, M.R. et al. 2016. Integrated analysis of bacterial and microeukaryotic communities from differentially active mud volcanoes in the Gulf of Cadiz. Sci. Rep. 6.
1. Pedros-Alio, C. 2016. The contribution of high throughput sequencing to the study of marine microbial diversity. In Microbial Ecology of the Oceans. 3rd ed.
1. Pearman, J.K., Anlauf, H., Irigoien, X. & Carvalho, S. 2016. Please mind the gap - Visual census and cryptic biodiversity assessment at central Red Sea coral reefs. Mar. Environ. Res. 118:20–30.
1. Pan, J., del Campo, J. & Keeling, P.J. 2016. Reference Tree and Environmental Sequence Diversity of Labyrinthulomycetes. J. Eukaryot. Microbiol.
1. Boyer, F., Mercier, C., Bonin, A., Le Bras, Y., Taberlet, P. & Coissac, E. 2016. OBITOOLS: a UNIX-inspired software package for DNA metabarcoding. Mol. Ecol. Resour. 16:176–82.
1. Santoferrara, L.F., Bachy, C., Alder, V.A., Gong, J., Kim, Y.-O., Sacca, A., da Silva Neto, I.D. et al. 2016. Updating Biodiversity Studies in Loricate Protists: The Case of the Tintinnids (Alveolata, Ciliophora, Spirotrichea). J. Eukaryot. Microbiol. 63:651–6.
1. Ramirez-Sanchez, O., Perez-Rodriguez, P., Delaye, L. & Tiessen, A. 2016. Plant Proteins Are Smaller Because They Are Encoded by Fewer Exons than Animal Proteins. GENOMICS PROTEOMICS Bioinforma. 14:357–70.
1. Ichinomiya, M., dos Santos, A.L., Gourvil, P., Yoshikawa, S., Kamiya, M., Ohki, K., Audic, S. et al. 2016. Diversity and oceanic distribution of the Parmales (Bolidophyceae), a picoplanktonic group closely related to diatoms. ISME J. 10:2419–34.
1. Cornejo-Castillo, F.M., Cabello, A.M., Salazar, G., Sanchez-Baracaldo, P., Lima-mendez, G., Hingamp, P., Alberti, A. et al. 2016. Cyanobacterial symbionts diverged in the late Cretaceous towards lineage-specific nitrogen fixation factories in single-celled phytoplankton. Nat. Commun. 7:1–9.
1. Pawlowski, J., Lejzerowicz, F., Apotheloz-Perret-Gentil, L., Visco, J. & Esling, P. 2016. Protist metabarcoding and environmental biomonitoring: Time for change. Eur. J. Protistol. 55:12–25.
1. Rimet, F.F., Chaumeil, P., Keck, F.F., Kermarrec, L.L., Vasselon, V., Kahlert, M., Franc, A. et al. 2016. R-Syst::diatom: an open-access and curated barcode database for diatoms and freshwater monitoring. Database. 2016:baw016.
1. Needham, D.M. & Fuhrman, J.A. 2016. Pronounced daily succession of phytoplankton , archaea and bacteria following a spring bloom. Nat. Microbiol. 1:1–7.
1. Dupont, A.O.C., Griffiths, R.I., Bell, T. & Bass, D. 2016. Differences in soil micro-eukaryotic communities over soil pH gradients are strongly driven by parasites and saprotrophs. Environ. Microbiol. 18:2010–24.
1. Campo, J. del, Guillou, L., Hehenberger, E., Logares, R., López-García, P., Massana, R., del Campo, J. et al. 2016. Ecological and evolutionary significance of novel protist lineages. Eur. J. Protistol. 55:4–11.
1. Weiner, A.K.M., Morard, R., Weinkauf, M.F., Darling, K.F., André, A., Quillévéré, F., Ujiie, Y. et al. 2016. Methodology for single-cell genetic analysis of planktonic foraminifera for studies of protist diversity and evolution. Fontiers Mar. Sci. 3:1–15.
1. Cooper, M.K., Phalen, D.N., Donahoe, S.L., Rose, K. & Slapeta, J. 2016. The utility of diversity profiling using Illumina 18S rRNA gene amplicon deep sequencing to detect and discriminate Toxoplasma gondii among the cyst-forming coccidia. Vet. Parasitol. 216:38–45.
1. Yoon, T.-H. 2016. Development of a cost-effective metabarcoding strategy for analysis of the marine phytoplankton community. PeerJ.
1. Pearman, J.K., Kürten, S., Sarma, Y.V.B., Jones, B.H., Carvalho, S., Kuerten, S., Sarma, Y.V.B. et al. 2016. Biodiversity patterns of plankton assemblages at the extremes of the Red Sea. FEMS Microbiol. Ecol. 92.
1. Pernice, M.C., Giner, C.R., Logares, R., Perera-Bel, J., Acinas, S.G., Duarte, C.M., Gasol, J.M. et al. 2016. Large variability of bathypelagic microbial eukaryotic communities across the world’s oceans. ISME J. 10:945–58.
1. Zaiko, A., Schimanski, K., Pochon, X., Hopkins, G.A., Goldstien, S., Floerl, O. & Wood, S.A. 2016. Metabarcoding improves detection of eukaryotes from early biofouling communities: implications for pest monitoring and pathway management. Biofouling. 32:671–84.
1. Moustaka-Gouni, M., Kormas, K.A., Scotti, M., Vardaka, E. & Sommer, U. 2016. Warming and Acidification Effects on Planktonic Heterotrophic Pico- and Nanoflagellates in a Mesocosm Experiment. Protist. 167:389–410.
1. Forster, D., Dunthorn, M., Stoeck, T. & Mahe, F. 2016. Comparison of three clustering approaches for detecting novel environmental microbial diversity. PeerJ. 4.
1. Leray, M. & Knowlton, N. 2016. Censusing marine eukaryotic diversity in the twenty-first century. Philos. Trans. R. Soc. B Biol. Sci. 371:20150331.
1. Malviya, S., Scalco, E., Audic, S., Vincent, F., Veluchamy, A., Bittner, L., Poulain, J. et al. 2016. Insights into global diatom distribution and diversity in the world’s ocean. Proc. Natl. Acad. Sci. 348:1–10.
1. Tragin, M., Lopes dos Santos, A., Christen, R. & Vaulot, D. 2016. Diversity and ecology of green microalgae in marine systems: an overview based on 18S rRNA gene sequences. Perspect. Phycol. 3:141–54.
1. Simon, M., Lopez-Garcia, P., Deschamps, P., Restoux, G., Bertolino, P., Moreira, D. & Jardillier, L. 2016. Resilience of Freshwater Communities of Small Microbial Eukaryotes Undergoing Severe Drought Events. Front. Microbiol. 7.
1. Le Bescot, N., Mahe, F., Audic, S.S., Dimier, C.C., Garet, M.-J.M.-J., Poulain, J., Wincker, P. et al. 2016. Global patterns of pelagic dinoflagellate diversity across protist size classes unveiled by metabarcoding. Environ. Microbiol. 18:609–26.
1. Geisen, S. 2016. Thorough high-throughput sequencing analyses unravels huge diversities of soil parasitic protists. Environ. Microbiol. 18:1669–72.
1. Genitsaris, S., Monchy, S., Breton, E., Lecuyer, E. & Christaki, U. 2016. Small-scale variability of protistan planktonic communities relative to environmental pressures and biotic interactions at two adjacent coastal stations. Mar. Ecol. Prog. Ser. 548:61–75.
1. Zoccarato, L., Pallavicini, A., Cerino, F., Umani, S.F. & Celussi, M. 2016. Water mass dynamics shape Ross Sea protist communities in mesopelagic and bathypelagic layers. Prog. Oceanogr. 149:16–26.
1. Guardiola, M., Wangensteen, O.S., Taberlet, P., Coissac, E., Uriz, M.J., Turon, X., Jesus Uriz, M. et al. 2016. Spatio-temporal monitoring of deep-sea communities using metabarcoding of sediment DNA and RNA. PeerJ. 4:e2807.
1. Bass, D., Silberman, J.D., Brown, M.W., Pearce, R.A., Tice, A.K., Jousset, A., Geisen, S. et al. 2016. Coprophilic amoebae and flagellates, including Guttulinopsis, Rosculus and Helkesimastix, characterise a divergent and diverse rhizarian radiation and contribute to a large diversity of faecal-associated protists. Environ. Microbiol. 18:1604–19.
1. Forster, D., Dunthorn, M., Mahé, F., Dolan, J.R., Audic, S., Bass, D., Bittner, L. et al. 2016. Benthic protists: the under-charted majority. FEMS Microbiol. Ecol. 92:fiw120.
1. Frank, N., Abuldhauge, T.H., Daniel, R.J., Nitsche, F., Thomsen, H.A. & Richter, D.J. 2016. Bridging the gap between morphological species and molecular barcodes − exemplified by loricate choanoflagellates. Eur. J. Protistol. 57:26–37.
1. Kozlov, A.M., Zhang, J., Yilmaz, P., Gloeckner, F.O. & Stamatakis, A. 2016. Phylogeny-aware identification and correction of taxonomically mislabeled sequences. Nucleic Acids Res. 44:5022–33.
1. Voudanta, E., Kormas, K.A., Monchy, S., Delegrange, A., Vincent, D., Genitsaris, S. & Christaki, U. 2016. Mussel biofiltration effects on attached bacteria and unicellular eukaryotes in fish-rearing seawater. PeerJ. 4.
1. Hu, Y.O.O., Karlson, B., Charvet, S. & Andersson, A.F. 2016. Diversity of Pico- to Mesoplankton along the 2000 km Salinity Gradient of the Baltic Sea. Front. Microbiol. 7.
1. Amato, K.R., Metcalf, J.L., Song, S.J., Hale, V.L., Clayton, J., Ackermann, G., Humphrey, G. et al. 2016. Using the gut microbiota as a novel tool for examining colobine primate GI health. Glob. Ecol. Conserv. 7:225–37.
1. Hartikainen, H., Bass, D., Briscoe, A.G., Knipe, H., Green, A.J. & Okamura, B. 2016. Assessing myxozoan presence and diversity using environmental DNA. Int. J. Parasitol. 46:781–92.
1. Marquardt, M., Vader, A., Stübner, E.I., Reigstad, M. & Gabrielsen, T.M. 2016. Strong Seasonality of Marine Microbial Eukaryotes in a High-Arctic Fjord (Isfjorden, in West Spitsbergen, Norway). Appl. Environ. Microbiol. 82:1868–80.
1. Obbels, D., Verleyen, E., Mano, M.-J., Namsaraev, Z., Sweetlove, M., Tytgat, B., Fernandez-Carazo, R. et al. 2016. Bacterial and eukaryotic biodiversity patterns in terrestrial and aquatic habitats in the Sor Rondane Mountains, Dronning Maud Land, East Antarctica. FEMS Microbiol. Ecol. 92.
1. More, R.P. & Purohit, H.J. 2016. The Identification of Discriminating Patterns from 16S rRNA Gene to Generate Signature for Bacillus Genus. J. Comput. Biol. 23:651–61.
1. Volant, A., Hery, M., Desoeuvre, A., Casiot, C., Morin, G., Bertin, P.N. & Bruneel, O. 2016. Spatial Distribution of Eukaryotic Communities Using High-Throughput Sequencing Along a Pollution Gradient in the Arsenic-Rich Creek Sediments of CarnoulSs Mine, France. Microb. Ecol. 72:608–20.
1. Kretzschmar, M., Kuhnt, A., Bonkowski, M. & Fiore-Donno, A.M. 2016. Phylogeny of the Highly Divergent Echinosteliales (Amoebozoa). J. Eukaryot. Microbiol. 63:453–9.
1. Pasulka, A.L., Levin, L.A., Steele, J.A., Case, D.H., Landry, M.R. & Orphan, V.J. 2016. Microbial eukaryotic distributions and diversity patterns in a deep-sea methane seep ecosystem. Environ. Microbiol. 18:3022–43.

## 2015
1. Lara, E., Seppey, C.V.W., Gonzalez Garraza, G., Singer, D., Victoria Quiroga, M., Mataloni, G., Garraza, G.G. et al. 2015. Planktonic eukaryote molecular diversity: discrimination of minerotrophic and ombrotrophic peatland pools in Tierra del Fuego (Argentina). J. Plankton Res. 37:645–55.
1. Jeffries, T.C., Ostrowski, M., Williams, R.B., Xie, C., Jensen, R.M., Grzymski, J.J., Senstius, S.J. et al. 2015. Spatially extensive microbial biogeography of the Indian Ocean provides insights into the unique community structure of a pristine coral atoll. Sci. Rep. 5.
1. Jing, H., Rocke, E., Kong, L., Xia, X., Liu, H. & Landry, M.R. 2015. Protist communities in a marine oxygen minimum zone off Costa Rica by 454 pyrosequencing. Biogeosciences Discuss. 12:13483–509.
1. Gong, J., Shi, F., Ma, B., Dong, J., Pachiadaki, M., Zhang, X. & Edgcomb, V.P. 2015. Depth shapes alpha- and beta-diversities of microbial eukaryotes in surficial sediments of coastal ecosystems. Environ. Microbiol. 17:3722–37.
1. Simon, M., Lopez-Garcia, P., Deschamps, P., Moreira, D., Restoux, G., Bertolino, P. & Jardillier, L. 2015. Marked seasonality and high spatial variability of protist communities in shallow freshwater systems. ISME J. 9:1941–53.
1. Decelle, J., Romac, S., Stern, R.F., Bendif, E.M., Zingone, A., Audic, S., Guiry, M.D. et al. 2015. PhytoREF: a reference database of the plastidial 16S rRNA gene of photosynthetic eukaryotes with curated taxonomy. Mol. Ecol. Resour. 15:1435–1445.
1. Pringault, O., Aube, J., Bouchez, O., Klopp, C., Mariette, J., Escudie, F., Senin, P. et al. 2015. Contrasted effects of natural complex mixtures of PAHs and metals on oxygen cycle in a microbial mat. Chemosphere. 135:189–201.
1. Sørensen, N., Richardson, K., Lund-Hansen, L., Sorrell, B. & Daugbjerg, N. 2015. Molecular diversity of picoeukaryotes in sea ice at the North Pole: implications for dispersal and function. J. Plankton Res.
1. Pochon, X., Zaiko, A., Hopkins, G.A., Banks, J.C. & Wood, S.A. 2015. Early detection of eukaryotic communities from marine biofilm using high-throughput sequencing: an assessment of different sampling devices. Biofouling. 31:241–51.
1. Goltsman, D.S.A., Comolli, L.R., Thomas, B.C. & Banfield, J.F. 2015. Community transcriptomics reveals unexpected high microbial diversity in acidophilic biofilm communities. ISME J. 9:1014–23.
1. Massana, R., Gobet, A., Audic, S., Bass, D., Bittner, L., Boutte, C., Chambouvet, A. et al. 2015. Marine protist diversity in European coastal waters and sediments as revealed by high-throughput sequencing. Environ. Microbiol. 17:4035–49.
1. Richards, T.A., Leonard, G., Mahe, F., del Campov, J., Romac, S., Jones, M.D.M., Maguire, F. et al. 2015. Molecular diversity and distribution of marine fungi across 130 European environmental samples. Proc. R. Soc. B-BIOLOGICAL Sci. 282.
1. Esling, P., Lejzerowicz, F. & Pawlowski, J. 2015. Accurate multiplexing and filtering for high-throughput amplicon-sequencing. Nucleic Acids Res. 43:2513–24.
1. del Campo, J., Mallo, D., Massana, R., de Vargas, C., Richards, T. a. & Ruiz-Trillo, I.I. 2015. Diversity and distribution of unicellular opisthokonts along the European coast analysed using high-throughput sequencing. Environ. Microbiol. 17:3195–207.
1. Rodríguez-Marconi, S., De la Iglesia, R., Díez, B., Fonseca, C.A.C.A., Hajdu, E., Trefault, N., Rodriguez-Marconi, S. et al. 2015. Characterization of Bacterial, Archaeal and Eukaryote Symbionts from Antarctic Sponges Reveals a High Diversity at a Three-Domain Level and a Particular Signature for This Ecosystem. PLoS One. 10:e0138837.
1. Wu, W., Wang, L., Liao, Y. & Huang, B. 2015. Microbial eukaryotic diversity and distribution in a river plume and cyclonic eddy-influenced ecosystem in the South China Sea. Microbiologyopen. 4:826–40.
1. Saghai, A., Zivanovic, Y., Zeyen, N., Moreira, D., Benzerara, K., Deschamps, P., Bertolino, P. et al. 2015. Metagenome-based diversity analyses suggest a significant contribution of non-cyanobacterial lineages to carbonate precipitation in modern microbialites. Front. Microbiol. 6.
1. Esling, P., Lejzerowicz, F. & Pawlowski, J. 2015. Accurate multiplexing and filtering for high-throughput amplicon-sequencing. Nucleic Acids Res. 1–12.
1. Kinebanyan, N. 2015. Plankton diversity and ecology in the California Current Ecosystem revealed by bioinformatics approaches. Paris Descartes.
1. Torstensson, A., Dinasquet, J., Chierici, M., Fransson, A., Riemann, L. & Wulff, A. 2015. Physicochemical control of bacterial and protist community composition and diversity in Antarctic sea ice. Environ. Microbiol. 17:3869–81.
1. Mahe, F., Mayor, J., Bunge, J., Chi, J., Siemensmeyer, T., Stoeck, T., Wahl, B. et al. 2015. Comparing High-throughput Platforms for Sequencing the V4 Region of SSU-rDNA in Environmental Microbial Eukaryotic Diversity Surveys. J. Eukaryot. Microbiol. 62:338–45.
1. Balzano, S., Abs, E. & Leterme, S.S.C. 2015. Protist diversity along a salinity gradient in a coastal lagoon. Aquat. Microb. Ecol. 74:263–77.
1. Guardiola, M., Uriz, M.J., Taberlet, P., Coissac, E., Wangensteen, O.S. & Turon, X. 2015. Deep-Sea, Deep-Sequencing: Metabarcoding Extracellular DNA from Sediments of Marine Canyons. PLoS One. 10.
1. Mahé​, F., Rognes​, T., Quince, C., de Vargas, C. & Dunthorn, M. 2015. Swarm v2: highly-scalable and high-resolution amplicon clustering. PeerJ. 3:e1420.
1. Lima-Mendez, G., Faust, K., Henry, N., Decelle, J., Colin, S., Carcillo, F., Chaffron, S. et al. 2015. Determinants of community structure in the global plankton interactome. Science. 348:1262073–1262073.
1. Mahe, F., Rognes, T., Quince, C., de Vargas, C. & Dunthorn, M. 2015. Swarm v2: highly-scalable and high-resolution amplicon clustering. PeerJ. 3.
1. Morard, R.R., Darling, K.F., Mahé, F., Audic, S.S., Ujiié, Y., Weiner, A.K.M., André, A. et al. 2015. PFR 2 : a curated database of planktonic foraminifera 18S ribosomal DNA as a resource for studies of plankton ecology, biogeography and evolution. Mol. Ecol. Resour. 15:1472–85.
1. Panek, T., Taborsky, P., Pachiadaki, M.G., Hroudova, M., Vlcek, C., Edgcomb, V.P. & Cepicka, I. 2015. Combined Culture-Based and Culture-Independent Approaches Provide Insights into Diversity of Jakobids, an Extremely Plesiomorphic Eukaryotic Lineage. Front. Microbiol. 6.
1. Geisen, S., Laros, I., Vizcaino, A., Bonkowski, M. & De Groot, G.A. 2015. Not all are free-living: high-throughput DNA metabarcoding reveals a diverse community of protists parasitizing soil metazoa. Mol. Ecol. 24:4556–69.
1. de Vargas, C., Audic, S., Henry, N., Decelle, J., Mahe, F., Logares, R., Lara, E. et al. 2015. Eukaryotic plankton diversity in the sunlit ocean. Science. 348:1261605.
1. Egge, E.S., Eikrem, W. & Edvardsen, B. 2015. Deep-branching Novel Lineages and High Diversity of Haptophytes in the Skagerrak (Norway) Uncovered by 454 pyrosequencing. J. Eukaryot. Microbiol. 62:121–40.
1. Lejzerowicz, F., Esling, P., Pillet, L., Wilding, T.A., Black, K.D. & Pawlowski, J. 2015. High-throughput sequencing and morphology perform equally well for benthic monitoring of marine ecosystems. Sci. Rep. 5.

## 2014
1. Nanjappa, D., Audic, S., Romac, S., Kooistra, W.H.C.F. & Zingone, A. 2014. Assessment of Species Diversity and Distribution of an Ancient Diatom Lineage Using a DNA Metabarcoding Approach. PLoS One. 9:e103810.
1. Lie, A.A.Y., Liu, Z., Hu, S.K., Jones, A.C., Kim, D.Y., Countway, P.D., Amaral-Zettler, L.A. et al. 2014. Investigating microbial eukaryotic diversity from a global census: Insights from a comparison of pyrotag and full-length sequences of 18S rRNA genes. Appl. Environ. Microbiol. 80:4363–73.
1. Xu, Y., Vick-Majors, T., Morgan-Kiss, R., Priscu, J.C. & Amaral-Zettler, L. 2014. Ciliate Diversity, Community Structure, and Novel Taxa in Lakes of the McMurdo Dry Valleys, Antarctica. Biol. Bull. 227:175–90.
1. Decelle, J., Romac, S., Sasaki, E., Not, F., Mahe, F. & Mahé, F. 2014. Intracellular Diversity of the V4 and V9 Regions of the 18S rRNA in Marine Protists (Radiolarians) Assessed by High-Throughput Sequencing. PLoS One. 9:e104297.
1. Wu, W., Huang, B., Liao, Y. & Sun, P. 2014. Picoeukaryotic diversity and distribution in the subtropicaltropical South China Sea. FEMS Microbiol. Ecol. 89:563–79.
1. Patterson, D.J. 2014. Helping Protists to Find Their Place in a Big Data World. ACTA Protozool. 53:115–28.
1. Chambouvet, A., Berney, C., Romac, S., Audic, S., Maguire, F., De Vargas, C. & Richards, T.A. 2014. Diverse molecular signatures for ribosomally `active’ Perkinsea in marine sediments. BMC Microbiol. 14.
1. Massana, R., del Campo, J., Sieracki, M.E., Audic, S. & Logares, R. 2014. Exploring the uncultured microeukaryote majority in the oceans: reevaluation of ribogroups within stramenopiles. ISME J. 8:854–66.
1. Logares, R., Audic, S., Bass, D., Bittner, L., Boutte, C., Christen, R., Claverie, J.-M. et al. 2014. Patterns of Rare and Abundant Marine Microbial Eukaryotes. Curr. Biol. 24:813–21.
1. Taylor, J.D. & Cunliffe, M. 2014. High-throughput sequencing reveals neustonic and planktonic microbial eukaryote diversity in coastal waters. J. Phycol. 50:960–5.
1. Parfrey, L.W., Walters, W.A., Lauber, C.L., Clemente, J.C., Berg-Lyons, D., Teiling, C., Kodira, C. et al. 2014. Communities of microbial eukaryotes in the mammalian gut within the context of environmental eukaryotic diversity. Front. Microbiol. 5.
1. Yilmaz, P., Parfrey, L.W., Yarza, P., Gerken, J., Pruesse, E., Quast, C., Schweer, T. et al. 2014. The SILVA and ``All-species Living Tree Project (LTP){’’} taxonomic frameworks. Nucleic Acids Res. 42:D643–8.
1. Chambouvet, A., Berney, C., Romac, S., Audic, S., Maguire, F., De Vargas, C. & Richards, T. 2014. Diverse molecular signatures for ribosomally “active” Perkinsea in marine sediments. BMC Microbiol. 14:110.
1. Thaler, M. & Lovejoy, C. 2014. Environmental selection of marine stramenopile clades in the Arctic Ocean and coastal waters. POLAR Biol. 37:347–57.

## 2013
1. Decelle, J., Martin, P., Paborstava, K., Pond, D.W., Tarling, G., Mahe, F., de Vargas, C. et al. 2013. Diversity, Ecology and Biogeochemistry of Cyst-Forming Acantharia (Radiolaria) in the Oceans. PLoS One. 8.
1. Guillou, L., Bachar, D., Audic, S., Bass, D., Berney, C., Bittner, L., Boutte, C. et al. 2013. The Protist Ribosomal Reference database (PR2): a catalog of unicellular eukaryote Small Sub-Unit rRNA sequences with curated taxonomy. Nucleic Acids Res. 41:D597–604.
1. Cheung, M.K., Nong, W., Kwan, H.S. & Wong, C.K. 2013. Composition of Heterotrophic Flagellates in Coastal Waters of Different Trophic Status. Curr. Microbiol. 67:351–5.
1. Risse-Buhl, U., Herrmann, M., Lange, P., Akob, D.M., Pizani, N., Schoenborn, W., Totsche, K.U. et al. 2013. Phagotrophic Protist Diversity in the Groundwater of a Karstified Aquifer - Morphological and Molecular Analysis. J. Eukaryot. Microbiol. 60:467–79.
1. Seenivasan, R., Sausen, N., Medlin, L.K. & Melkonian, M. 2013. Picomonas judraskeda Gen. Et Sp. Nov.: The First Identified Member of the Picozoa Phylum Nov., a Widespread Group of Picoeukaryotes, Formerly Known as ‘Picobiliphytes.’ PLoS One. 8:e59565.
1. Egge, E., Bittner, L., Andersen, T., Audic, S., de Vargas, C. & Edvardsen, B. 2013. 454 Pyrosequencing to Describe Microbial Eukaryotic Community Composition, Diversity and Relative Abundance: A Test for Marine Haptophytes. PLoS One. 8:e74371.
1. Fernandez-Suarez, X.M. & Galperin, M.Y. 2013. The 2013 Nucleic Acids Research Database Issue and the online Molecular Biology Database Collection. Nucleic Acids Res. 41:D1–7.
