---
authors:
- vaulot
categories:
- documentation
date: "2018-07-01"
draft: false
featured: false
image:
  caption: 'Image credit: [**Unsplash**](https://unsplash.com/photos/CpkOjOcXdUY)'
  focal_point: ""
  placement: 2
  preview_only: false
lastmod: "2021-01-18"
projects: []
subtitle: ''
summary: List of all taxonomic revisions made to date
tags:
- Documentation
- Taxonomy
- History
title: 'Taxonomic groups annotated'
---

## Groups with updated annotations

Division | Class | Group | Who | Date |  Version
--- | --- | --- | --- | --- | ---
Alveolata | Apicomplexa | | J. del Campo | 2019 |  4.12, 4.14  
Alveolata | Ciliates | | W. Ting, C. Bachy | 2017 | 4.7
Alveolata | Ciliates | | V. Boscaro,  L. Santoferrara, E. Gentekaki and Q. Zhang | 2018 | 4.11
Alveolata | Dinoflagellates | | S. Mordret, D. Sarno | 2018  | 4.9  
Alveolata | Dinoflagellates | Suessiales | J. del Campo | 2021  | 4.13
Chlorophyta |  |  | M.Tragin, A. Lopes dos Santos | 2015 | 3.0 
Excavata |  |  | J. del Campo | 2021  | 4.14
Haptophyta |  |  |  B. Edvardsen | 2015 | 4.0
Prasinodermophyta |  |  | D. Vaulot | 2021 | 4.13  
Rhizaria | Collodaria | | T. Biard | 2015 | 2.0
Rhizaria | Foraminifera | | R. Morard | 2021
Rhizaria | Radiolaria | | M. Mendez Sandin. | 2021
Stramenopiles |  |  |  R. Massana | 2019 | 4.12
Stramenopiles |  | *Cafeteria* |  Alex Schoenle | 2021 | 4.14
Stramenopiles | Labyrinthulomycetes |  |  J. del Campo | 2021 | 4.14
Stramenopiles | Diatoms  |  _Chaetoceros_ | C. Gaonkar | 2019 | 4.12
Stramenopiles | Diatoms  | Thalassiosirales | L. Arsenieff | 2021 | 4.13
Stramenopiles | Bolidophyceae |  |  D. Vaulot | 2017 | 4.6
Stramenopiles | Pelagophyceae | | A.M. Cabello | 2021 | 4.13
Stramenopiles |   Chrysophyceae | | D. Vaulot | 2021 | 4.13

## Groups with planned annotations

Division | Class | Who | Date planned
--- | --- | --- | ---
Eukaryota | mitochondria | D. Vaulot | 2021


## List of references used to annotate database

### Eukaryotic taxonomy
* Adl SM., Bass D., Lane CE., Lukeš J., Schoch CL., Smirnov A., Agatha S., Berney C., Brown MW., Burki F., Cárdenas P., Čepička I., Chistyakova L., del Campo J., Dunthorn M., Edvardsen B., Eglit Y., Guillou L., Hampl V., Heiss AA., Hoppenrath M., James TY., Karpov S., Kim E., Kolisko M., Kudryavtsev A., Lahr DJG., Lara E., Le Gall L., Lynn DH., Mann DG., Massana i Molera R., Mitchell EAD., Morrow C., Park JS., Pawlowski JW., Powell MJ., Richter DJ., Rueckert S., Shadwick L., Shimano S., Spiegel FW., Torruella i Cortes G., Youssef N., Zlatogursky V., Zhang Q. 2019. Revisions to the Classification, Nomenclature, and Diversity of Eukaryotes. Journal of Eukaryotic Microbiology 66:4–119. DOI: 10.1111/jeu.12691.
* Brown, M.W., Heiss, A.A., Kamikawa, R., Inagaki, Y., Yabuki, A., Tice, A.K., Shiratori, T., Ishida, K.I., Hashimoto, T., Simpson, A.G. and Roger, A.J., 2018. Phylogenomics places orphan protistan lineages in a novel eukaryotic super-group. *Genome biology and evolution*, *10*(2), pp.427-433.
* Strassert, J.F., Jamy, M., Mylnikov, A.P., Tikhonenkov, D.V. and Burki, F., 2019. New phylogenomic analysis of the enigmatic phylum Telonemia further resolves the eukaryote tree of life. *Molecular biology and evolution*, *36*(4), pp.757-765.
* Gawryluk, R.M., Tikhonenkov, D.V., Hehenberger, E., Husnik, F., Mylnikov, A.P. and Keeling, P.J., 2019. Non-photosynthetic predators are sister to red algae. *Nature*, *572*(7768), pp.240-243.



### Chlorophyta

* Tragin, M., Lopes dos Santos, A., Christen, R., and Vaulot, D. (2016). Diversity and ecology of green microalgae in marine systems: an overview based on 18S rRNA gene sequences. Perspect. Phycol. 3, 141–154. doi:10.1127/pip/2016/0059.
* Lopes dos Santos A., Pollina T., Gourvil P., Corre E., Marie D., Garrido JL., Rodríguez F., Noël M-H., Vaulot D., Eikrem W. 2017. Chloropicophyceae, a new class of picophytoplanktonic prasinophytes. Scientific Reports 7:14019. DOI: 10.1038/s41598-017-12412-5.
* Daugbjerg N., Fassel NMD., Moestrup Ø. 2019. Microscopy and phylogeny of _Pyramimonas tatianae_ sp. nov. (Pyramimonadales, Chlorophyta), a scaly quadriflagellate from Golden Horn Bay (eastern Russia) and formal description of Pyramimonadophyceae classis nova . European Journal of Phycology 0:1–15. DOI: 10.1080/09670262.2019.1638524
* 

### Ochrophyta/Stramenopiles
* Adl, S.M., Bass, D., Lane, C.E., Lukeš, J., Schoch, C.L., Smirnov, A., Agatha, S. et al. 2019. Revisions to the Classification, Nomenclature, and Diversity of Eukaryotes. J. Eukaryot. Microbiol. 66:4–119.
* Ichinomiya, M., dos Santos, A. L., Gourvil, P., Yoshikawa, S., Kamiya, M., Ohki, K., et al. (2016). Diversity and oceanic distribution of the Parmales (Bolidophyceae), a picoplanktonic group closely related to diatoms. ISME J. 10, 2419–2434. doi:10.1038/ismej.2016.38.
* Derelle, R., López-García, P., Timpano, H. & Moreira, D. 2016. A Phylogenomic Framework to Study the Diversity and Evolution of Stramenopiles (= Heterokonts). Mol. Biol. Evol. 33:2890–8.
* Gaonkar, C.C., Piredda, R., Minucci, C., Mann, D.G., Montresor, M., Sarno, D. & Kooistra, W.H.C.F. 2018. Annotated 18S and 28S rDNA reference sequences of taxa in the planktonic diatom family Chaetocerotaceae. PLoS One. 13:e0208929.
* Massana, R., del Campo, J., Sieracki, M.E., Audic, S. & Logares, R. 2014. Exploring the uncultured microeukaryote majority in the oceans: reevaluation of ribogroups within stramenopiles. ISME J. 8:854–66.
* Arsenieff L., Le Gall F., Rigaut-Jalabert F., Mahé F., Sarno D., Gouhier L., Baudoux A-C., Simon N. 2020. Diversity and dynamics of relevant nanoplanktonic diatoms in the Western English Channel. The ISME Journal. DOI: 10.1038/s41396-020-0659-6.
* Han KY., Graf L., Reyes CP., Melkonian B., Andersen RA., Yoon HS., Melkonian M. 2018. A Re-investigation of _Sarcinochrysis marina_ (Sarcinochrysidales, Pelagophyceae) from its Type Locality and the Descriptions of _Arachnochrysis_, _Pelagospilus_, _Sargassococcus_ and _Sungminbooa_ genera nov. Protist 169:79–106. DOI: 10.1016/j.protis.2017.12.004.
* Andersen RA., Graf L., Malakhov Y., Yoon HS. 2017. Rediscovery of the Ochromonas type species _Ochromonas triangulata_ (Chrysophyceae) from its type locality (Lake Veysove, Donetsk region, Ukraine). Phycologia 56:591–604. DOI: 10.2216/17-15.1.
* Schoenle, A., Hohlfeld, M., Rosse, M., Filz, P., Wylezich, C., Nitsche, F., & Arndt, H. (2020). Global comparison of bicosoecid Cafeteria-like flagellates from the deep ocean and surface waters, with reorganization of the family Cafeteriaceae. European Journal of Protistology, 73, 125665. https://doi.org/10.1016/j.ejop.2019.125665.
* Jirsová, D., Füssy, Z., Richtová, J., Gruber, A., & Oborník, M. (2019). Morphology, ultrastructure, and mitochondrial genome of the marine non-photosynthetic bicosoecid Cafileria marina gen. Et sp. Nov. Microorganisms, 7(8), 240. https://doi.org/10.3390/microorganisms7080240
* Pan, J., del Campo, J., & Keeling, P. J. (2017). Reference Tree and Environmental Sequence Diversity of Labyrinthulomycetes. Journal of Eukaryotic Microbiology, 64(1), 88–96. https://doi.org/10.1111/jeu.12342

### Haptophyta
* Edvardsen, B., Egge, E. S., and Vaulot, D. (2016). Diversity and distribution of haptophytes revealed by environmental sequencing and metabarcoding – a review. Perspect. Phycol. 3, 77–91. [doi:10.1127/pip/2016/0052](https://dx.doi.org/10.1127/pip/2016/0052).
* Kawachi, M., Nakayama, T., Kayama, M., Nomura, M., Miyashita, H., Bojo, O., Rhodes, L., Sym, S., Pienaar, R. N., Probert, I., Inouye, I., & Kamikawa, R. (2021). Rappemonads are haptophyte phytoplankton. Current Biology. https://doi.org/10.1016/j.cub.2021.03.012

### Ciliates
* Agatha, S. and Strüder-Kypke, M.C. (2014) What Morphology and Molecules Tell Us about the Evolution of Oligotrichea (Alveolata, Ciliophora). Acta Protozool. 2014: 7790.
* Bachy, C., Gómez, F., López-García, P., Dolan, J.R., and Moreira, D. (2012) Molecular Phylogeny of Tintinnid Ciliates (Tintinnida, Ciliophora). Protist 163: 873–887.
* Boscaro V., Santoferrara L., Zhang Q., Gentekaki E., Syberg-Olsen M., del Campo J., Keeling P., 2018. EukRef–Ciliophora: A manually curated, phylogeny‐based database of small subunit rRNA gene sequences of ciliates. Environmental Microbiology, 20: 2218-2230.
* Santoferrara, L.F., Alder, V.V., and McManus, G.B. (2017) Phylogeny, classification and diversity of Choreotrichia and Oligotrichia (Ciliophora, Spirotrichea). Mol. Phylogenet. Evol. 112: 12–22.
* Zhang, Q., Agatha, S., Zhang, W., Dong, J., Yu, Y., Jiao, N., and Gong, J. (2017) Three rDNA Loci-Based Phylogenies of Tintinnid Ciliates (Ciliophora, Spirotrichea, Choreotrichida). J. Eukaryot. Microbiol. 64: 226–241.

### Dinoflagellates
* Mordret, S., Piredda, R., Vaulot, D., Montresor, M., Kooistra, W.H.C.F. & Sarno, D. 2018. <scp>dinoref</scp> : A curated dinoflagellate (Dinophyceae) reference database for the 18S rRNA gene. Mol. Ecol. Resour. in press.
* Janouškovec, Jan, Gregory S. Gavelis, Fabien Burki, Donna Dinh, Tsvetan R. Bachvaroff, Sebastian G. Gornik, Kelley J. Bright, et al. 2017. Major Transitions in Dinoflagellate Evolution Unveiled by Phylotranscriptomics. Proceedings of the National Academy of Sciences 114 (2): E171–80. https://doi.org/10.1073/pnas.1614842114.
* LaJeunesse, Todd C., John Everett Parkinson, Paul W. Gabrielson, Hae Jin Jeong, James Davis Reimer, Christian R. Voolstra, and Scott R. Santos. 2018. Systematic Revision of Symbiodiniaceae Highlights the Antiquity and Diversity of Coral Endosymbionts. Current Biology 28 (16): 2570-2580.e6. https://doi.org/10.1016/j.cub.2018.07.008.

### Apicomplexa
* del Campo J., Heger TJ., Rodríguez-Martínez R., Worden AZ., Richards TA., Massana R., Keeling PJ. 2019. Assessing the Diversity and Distribution of Apicomplexans in Host and Free-Living Environments Using High-Throughput Amplicon Data and a Phylogenetically Informed Reference Framework. Frontiers in Microbiology 10:1–15. DOI: 10.3389/fmicb.2019.02373.

### Plastids
* Decelle, J., Romac, S., Stern, R.F., Bendif, E.M., Zingone, A., Audic, S., Guiry, M.D. et al. 2015. PhytoREF: a reference database of the plastidial 16S rRNA gene of photosynthetic eukaryotes with curated taxonomy. Mol. Ecol. Resour. 15:1435–144

### Radiolaria
* Biard, T., Bigeard, E., Audic, S., Poulain, J., Stemmann, L., Not, F., 2017. Biogeography and diversity of Collodaria (Radiolaria) in the global ocean. Nat. Publ. Gr. 1–42. [doi:10.1038/ismej.2017.12](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5437347/pdf/ismej201712a.pdf)  
* Cavalier-Smith, T., Chao, E.E., Lewis, R., 2018. Multigene phylogeny and cell evolution of chromist infrakingdom Rhizaria: contrasting cell organisation of sister phyla Cercozoa and Retaria. Protoplasma 255, 1517–1574. [doi:10.1007/s00709-018-1241-1](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6133090/)  
* Capella-Gutiérrez, S., Silla-Martínez, J.M., Gabaldón, T., 2009. trimAl: A tool for automated alignment trimming in large-scale phylogenetic analyses. Bioinformatics 25, 1972–1973. [doi:10.1093/bioinformatics/btp348](https://academic.oup.com/bioinformatics/article/25/15/1972/213148)  
* Decelle, J., Suzuki, N., Mahé, F., Vargas, C. De, Not, F., 2012b. Molecular Phylogeny and Morphological Evolution of the Acantharea (Radiolaria). Protist 163, 435–450. [doi:10.1016/j.protis.2011.10.002](https://www.sciencedirect.com/science/article/abs/pii/S1434461011000988?via%3Dihub)  
* Gouy, M., Guindon, S., Gascuel, O., 2010. SeaView Version 4: A Multiplatform Graphical User Interface for Sequence Alignment and Phylogenetic Tree Building. Mol. Biol. Evol. 27, 221–224. [doi:10.1093/molbev/msp259](https://academic.oup.com/mbe/article/27/2/221/970247)  
* Katoh, K., Standley, D.M., 2013. MAFFT multiple sequence alignment software version 7: Improvements in performance and usability. Mol. Biol. Evol. 30, 772–780. [doi:10.1093/molbev/mst010](https://academic.oup.com/mbe/article/30/4/772/1073398)  
* Larsson, A. (2014). AliView: a fast and lightweight alignment viewer and editor for large data sets. Bioinformatics30(22): 3276-3278. [http://dx.doi.org/10.1093/bioinformatics/btu531](https://academic.oup.com/bioinformatics/article/30/22/3276/2391211)  
* Nakamura, Y., Sandin, M.M., Suzuki N., Somiya R., Tuji A., Not, F, 2020. Phylogenetic revision of the order Entactinaria - Paleozoic relict Radiolaria (Rhizaria, SAR). Protist 171. [doi:10.1016/j.protis.2019.125712](https://reader.elsevier.com/reader/sd/pii/S1434461019300707?token=5DD9A1B3581FD91E73FDF99BF683DCF68658895F7A186C03566C0C1146EA7F8FB4FDB2E76FFA7D188CA812DFEFB57B9D)  
* Rambaut A (2016) FigTree version 1.4.3. [http://tree.bio.ed.ac.uk/software/figtree/](http://tree.bio.ed.ac.uk/software/figtree/)  
Rognes, T., Flouri, T., Nichols, B., Quince, C., Mahé, F., 2016. VSEARCH: a versatile open source tool for metagenomics. PeerJ 4, e2584.[doi:10.7717/peerj.2584](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5075697/pdf/peerj-04-2584.pdf)  
* Sandin M.M., Biard T., Romac S., O’Dogherty L., Suzuki N., Not F., 2020. Morpho-molecular diversity and evolutionary analyses suggest hidden life styles in Spumellaria (Radiolaria) bioRxiv 2020.06.29.176917; doi: [https://doi.org/10.1101/2020.06.29.176917](https://www.biorxiv.org/content/10.1101/2020.06.29.176917v1)   
* Sandin, M.M., Pillet, L., Biard, T., Poirier, C., Bigeard, E., Romac, S., Suzuki, N., Not, F., 2019. Time Calibrated Morpho-molecular Classification of Nassellaria (Radiolaria). Protist 170, 187–208. [doi:10.1016/j.protis.2019.02.002](https://reader.elsevier.com/reader/sd/pii/S143446101830110X?token=5A45B3EAB7E3260253DBBDAC41398181BF8FEB3E40B4CD9F7BEF12B3DF4312F96D955633C3DB076EA70E8D77242FE6F3)  
* Schloss, P.D., Westcott, S.L., Ryabin, T., Hall, J.R., Hartmann, M., Hollister, E.B., Lesniewski, R.A., Oakley, B.B., Parks, D.H., Robinson, C.J., Sahl, J.W., Stres, B., Thallinger, G.G., Van Horn, D.J., Weber, C.F., 2009. Introducing mothur: Open-source, platform-independent, community-supported software for describing and comparing microbial communities. Appl. Environ. Microbiol. 75, 7537–7541.[doi:10.1128/AEM.01541-09](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2786419/pdf/1541-09.pdf) 
* Stamatakis, A., 2014. RAxML version 8: A tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics 30, 1312–1313. [doi:10.1093/bioinformatics/btu033](https://academic.oup.com/bioinformatics/article/30/9/1312/238053) 
