---
title: "Installation and use R package pr2database"
author: vaulot
date: '2018-08-15'
categories: ["documentation"]
tags: []
---



<p>The PR2 database is now provided as a R package called <strong>pr2database</strong>. This page provides instruction to install and use the package. Other examples on how to use the pr2database package are provided <a href="https://vaulot.github.io/pr2/PR2_analysis.html">here</a>.</p>
<div id="installation" class="section level1">
<h1>Installation</h1>
<p>Install from the GitHub web site using the devtools package</p>
<pre class="r"><code>install.packages(devtools)
devtools::install_github(&quot;pr2database/pr2database&quot;)</code></pre>
<pre><code>* installing *source* package &#39;pr2database&#39; ...
** R
** data
*** moving datasets to lazyload DB
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
  converting help for package &#39;pr2database&#39;
    finding HTML links ... fini
    pr2                                     html  
** building package indices
** testing if installed package can be loaded
*** arch - i386
*** arch - x64
* DONE (pr2database)
In R CMD INSTALL</code></pre>
</div>
<div id="loading-the-database" class="section level1">
<h1>Loading the database</h1>
<p>The PR2 database is provided as a data frame (or a tibble). This is a join between the following tables:
* pr2_main
* pr2_taxonomy
* pr2_sequence
* pr2_metadata</p>
<pre class="r"><code>library(&quot;pr2database&quot;)

data(&quot;pr2&quot;)

# List of the different columns available - see the help of the package for information on each field

colnames(pr2)
#&gt;  [1] &quot;pr2_main_id&quot;                &quot;pr2_accession&quot;             
#&gt;  [3] &quot;genbank_accession&quot;          &quot;start&quot;                     
#&gt;  [5] &quot;end&quot;                        &quot;label&quot;                     
#&gt;  [7] &quot;gene&quot;                       &quot;organelle&quot;                 
#&gt;  [9] &quot;species&quot;                    &quot;chimera&quot;                   
#&gt; [11] &quot;chimera_remark&quot;             &quot;reference_sequence&quot;        
#&gt; [13] &quot;added_version&quot;              &quot;removed_version&quot;           
#&gt; [15] &quot;edited_version&quot;             &quot;edited_by&quot;                 
#&gt; [17] &quot;edited_remark&quot;              &quot;remark&quot;                    
#&gt; [19] &quot;taxo_id&quot;                    &quot;kingdom&quot;                   
#&gt; [21] &quot;supergroup&quot;                 &quot;division&quot;                  
#&gt; [23] &quot;class&quot;                      &quot;order&quot;                     
#&gt; [25] &quot;family&quot;                     &quot;genus&quot;                     
#&gt; [27] &quot;taxon_trophic_mode&quot;         &quot;taxo_edited_version&quot;       
#&gt; [29] &quot;taxo_edited_by&quot;             &quot;taxo_removed_version&quot;      
#&gt; [31] &quot;taxo_remark&quot;                &quot;reference&quot;                 
#&gt; [33] &quot;seq_id&quot;                     &quot;sequence&quot;                  
#&gt; [35] &quot;sequence_length&quot;            &quot;ambiguities&quot;               
#&gt; [37] &quot;sequence_hash&quot;              &quot;pr2_metadata_id&quot;           
#&gt; [39] &quot;gb_date&quot;                    &quot;gb_locus&quot;                  
#&gt; [41] &quot;gb_definition&quot;              &quot;gb_organism&quot;               
#&gt; [43] &quot;gb_organelle&quot;               &quot;gb_taxonomy&quot;               
#&gt; [45] &quot;gb_strain&quot;                  &quot;gb_culture_collection&quot;     
#&gt; [47] &quot;gb_clone&quot;                   &quot;gb_isolate&quot;                
#&gt; [49] &quot;gb_isolation_source&quot;        &quot;gb_specimen_voucher&quot;       
#&gt; [51] &quot;gb_host&quot;                    &quot;gb_collection_date&quot;        
#&gt; [53] &quot;gb_environmental_sample&quot;    &quot;gb_country&quot;                
#&gt; [55] &quot;gb_lat_lon&quot;                 &quot;gb_collected_by&quot;           
#&gt; [57] &quot;gb_note&quot;                    &quot;gb_references&quot;             
#&gt; [59] &quot;gb_publication&quot;             &quot;gb_authors&quot;                
#&gt; [61] &quot;gb_journal&quot;                 &quot;pubmed_id&quot;                 
#&gt; [63] &quot;eukref_name&quot;                &quot;eukref_source&quot;             
#&gt; [65] &quot;eukref_env_material&quot;        &quot;eukref_env_biome&quot;          
#&gt; [67] &quot;eukref_biotic_relationship&quot; &quot;eukref_specific_host&quot;      
#&gt; [69] &quot;eukref_geo_loc_name&quot;        &quot;eukref_notes&quot;              
#&gt; [71] &quot;pr2_sample_type&quot;            &quot;pr2_sample_method&quot;         
#&gt; [73] &quot;pr2_latitude&quot;               &quot;pr2_longitude&quot;             
#&gt; [75] &quot;pr2_ocean&quot;                  &quot;pr2_sea&quot;                   
#&gt; [77] &quot;pr2_sea_lat&quot;                &quot;pr2_sea_lon&quot;               
#&gt; [79] &quot;pr2_continent&quot;              &quot;pr2_country&quot;               
#&gt; [81] &quot;pr2_location&quot;               &quot;pr2_location_geoname&quot;      
#&gt; [83] &quot;pr2_location_geotype&quot;       &quot;pr2_location_lat&quot;          
#&gt; [85] &quot;pr2_location_lon&quot;           &quot;pr2_country_geocode&quot;       
#&gt; [87] &quot;pr2_country_lat&quot;            &quot;pr2_country_lon&quot;           
#&gt; [89] &quot;pr2_sequence_origin&quot;        &quot;pr2_size_fraction&quot;         
#&gt; [91] &quot;pr2_size_fraction_min&quot;      &quot;pr2_size_fraction_max&quot;     
#&gt; [93] &quot;metadata_remark&quot;</code></pre>
</div>
<div id="working-with-the-database" class="section level1">
<h1>Working with the database</h1>
<div id="install-and-load-the-libraries" class="section level2">
<h2>Install and load the libraries</h2>
<p>The following examples makes use of the specifc R libraries</p>
<p>Install the libraries</p>
<pre class="r"><code>install.packages(&quot;dplyr&quot;)      # For filtering the data
install.package(&quot;ggplot2&quot;)     # To plot data
install.package(&quot;maps&quot;)        # To plot maps

source(&quot;https://bioconductor.org/biocLite.R&quot;)  # This package is on Bioconductor
biocLite(&quot;Biostrings&quot;)         # To save fasta files</code></pre>
<p>Load the libraries</p>
<pre class="r"><code>  library(dplyr)
  library(ggplot2)    # For plots
  library(Biostrings) # To save fasta files</code></pre>
</div>
<div id="selecting-sequences-from-a-specific-taxon" class="section level2">
<h2>Selecting sequences from a specific taxon</h2>
<p>Let us select all the available sequences for the Mamiellophyceae <em>Ostreococcus</em></p>
<pre class="r"><code>
  # Filter only the sequences for which the column genus contains Ostreococcus
  pr2_ostreo &lt;- pr2 %&gt;% dplyr::filter(genus == &quot;Ostreococcus&quot;)

  # Select only the columns of interest
  pr2_ostreo &lt;- pr2_ostreo %&gt;% dplyr::select( genbank_accession, species, 
                                              pr2_sample_type, gb_strain, gb_clone, 
                                              pr2_latitude, pr2_longitude, 
                                              sequence_length, sequence  )
  
  pr2_ostreo
#&gt; # A tibble: 325 x 9
#&gt;    genbank_accessi~ species pr2_sample_type gb_strain gb_clone pr2_latitude
#&gt;    &lt;chr&gt;            &lt;chr&gt;   &lt;chr&gt;           &lt;chr&gt;     &lt;chr&gt;           &lt;dbl&gt;
#&gt;  1 AF525872         Ostreo~ environmental   &lt;NA&gt;      UEPACIp5         NA  
#&gt;  2 EU562149         Ostreo~ environmental   &lt;NA&gt;      IND2.6           NA  
#&gt;  3 AY425309         Ostreo~ environmental   &lt;NA&gt;      RA01041~         NA  
#&gt;  4 GQ426346         Ostreo~ culture         CB6       &lt;NA&gt;             NA  
#&gt;  5 KC583118         Ostreo~ environmental   &lt;NA&gt;      RS.12f.~         NA  
#&gt;  6 JN862906         Ostreo~ culture         BCC48000  &lt;NA&gt;             NA  
#&gt;  7 JQ692065         Ostreo~ environmental   &lt;NA&gt;      PUPF_60         -43.3
#&gt;  8 FR874749         Ostreo~ environmental   &lt;NA&gt;      1815F12          60.3
#&gt;  9 FJ431431         Ostreo~ environmental   &lt;NA&gt;      RA07100~         NA  
#&gt; 10 EU561670         Ostreo~ environmental   &lt;NA&gt;      IND1.11         -35.0
#&gt; # ... with 315 more rows, and 3 more variables: pr2_longitude &lt;dbl&gt;,
#&gt; #   sequence_length &lt;int&gt;, sequence &lt;chr&gt;</code></pre>
</div>
<div id="exporting-the-sequences-to-fasta" class="section level2">
<h2>Exporting the sequences to fasta</h2>
<p>We will save the <em>Ostreococcus</em> sequences to a FASTA file. This is easy done with the bioconductor package BioStrings.</p>
<pre class="r"><code>
  # Importing the sequence in a Biostring set 

  seq_ostreo &lt;- Biostrings::DNAStringSet(pr2_ostreo$sequence)

  # Constructing the name of each sequecne (the first line of the fasta file)
  # using the genbank accession, species name, strain name and clone name

  names(seq_ostreo) &lt;- paste(pr2_ostreo$genbank_accession, pr2_ostreo$species,
                             &quot;strain&quot;,pr2_ostreo$gb_strain,
                             &quot;clone&quot;,pr2_ostreo$gb_clone, 
                              sep=&quot;|&quot;)

  # Displaying the Biostring set
  seq_ostreo
#&gt;   A DNAStringSet instance of length 325
#&gt;       width seq                                        names               
#&gt;   [1]  1766 ACCTGGTTGATCCTGCCAGT...TGAACCTGCAGAAGGATCA AF525872|Ostreoco...
#&gt;   [2]   836 AAAGCTCGTAGTCGGATTTT...GGGCCGCACGCGCGCTACA EU562149|Ostreoco...
#&gt;   [3]  1728 GCCAGTAGTCATATGCTTGT...AAGTCGTAACAAGGTTTCC AY425309|Ostreoco...
#&gt;   [4]  1652 AGCCATGCATGTCTAAGTAT...ATTACCGTGGGAAATTCGT GQ426346|Ostreoco...
#&gt;   [5]  1764 CCTGGTTGATCCTGCCAGTA...GTGAACCTGCAGAAGGATC KC583118|Ostreoco...
#&gt;   ...   ... ...
#&gt; [321]  1570 CAATTTGAATGAGATTCAAA...AAAAAGACCAAGCCGGAAG KF285522|Ostreoco...
#&gt; [322]  1570 CAATTTGAATGAGATTCAAA...AAAAAGACCAAGCCGGAAG KF285529|Ostreoco...
#&gt; [323]  1570 CAATTTGAATGAGATTCAAA...AAAAAGACCAAGCCGGAAG KF285531|Ostreoco...
#&gt; [324]  1570 CAATTTGAATGAGATTCAAA...AAAAAGACCAAGCCGGAAG NC_008289|Ostreoc...
#&gt; [325]  1570 CAATTTGAATGAGATTCAAA...AAAAAGACCAAGCCGGAAG NC_008289|Ostreoc...
    
  # Saving the sequences as a fasta file
  Biostrings::writeXStringSet(seq_ostreo, &quot;examples/pr2_ostreo.fasta&quot;, width = 80)</code></pre>
<p>The fasta file will look as follows</p>
<pre><code>&gt;AF525872|Ostreococcus_lucimarinus|strain|NA|clone|UEPACIp5
ACCTGGTTGATCCTGCCAGTAGTCATATGCTTGTCTCAAAGATTAAGCCATGCATGTCTAAGTATAAGCGTTATACTGTG
AAACTGCGAATGGCTCATTAAATCAGCAATAGTTTCTTTGGTGGTGTTTACTACTCGGATAACCGTAGTAATTCTAGAGC
TAATACGTGCGTAAATCCCGACTTCGGAAGGGACGTATTTATTAGATAAAGACCG...
&gt;EU562149|Ostreococcus_lucimarinus|strain|NA|clone|IND2.6
AAAGCTCGTAGTCGGATTTTGGCTGAGAACGGTCGGTCCGCCGTTAGGTGTGCACTGACTGGTCTCAGCTTCCTGGTGAG
GAGGTGTGCTTCATCGCCACTTAGTCACCGTGGTTACTTTGAAAAAATTAGAGTGTTCAAAGCGGGCTTACGCTTGAATA
TATTAGCATGGAATAACACCATAGGACTCCTGTCCTATTTCGTTGGTCTCGGGACGGGAGTAATGATTAAGATGAACAGT
TGGGGGCATTCGTATTTCATTGTCAGAGGTGAAATTCTTGGATTT...
&gt;AY425309|Ostreococcus_lucimarinus|strain|NA|clone|RA010412.39
GCCAGTAGTCATATGCTTGTCTCAAAGATTAAGCCATGCATGTCTAAGTATAAGCGTTATACTGTGAAACTGCGAATGGC
TCATTAAATCAGCAATAGTTTCTTTGGTGGTGTTTACTACTCGGATAACCGT...</code></pre>
</div>
<div id="doing-an-histogram-of-the-sequence-length" class="section level2">
<h2>Doing an histogram of the sequence length</h2>
<pre class="r"><code>  ggplot(pr2_ostreo) + 
    geom_histogram(aes(sequence_length), binwidth = 50, fill=&quot;blue&quot;) + 
    xlim(0,2000) + xlab(&quot;Sequence length&quot;) + ylab(&quot;Number of sequences&quot;) + 
    ggtitle(&quot;Ostreococcus lucimarinus&quot;)</code></pre>
<p><img src="/post/documentation/11-PR2-R-database-package_files/figure-html/sequence_histogram-1.png" width="672" /></p>
</div>
<div id="drawing-a-map-of-sequence-locations" class="section level2">
<h2>Drawing a map of sequence locations</h2>
<pre class="r"><code>  library(maps)
  world &lt;- map_data(&quot;world&quot;)

  ggplot() + 
    geom_polygon(data = world, aes(x=long, y = lat, group = group), fill=&quot;grey&quot;) + 
    coord_fixed(1.3) +
    geom_point(data=pr2_ostreo, aes(x=pr2_longitude, y=pr2_latitude), fill=&quot;blue&quot;, size=2, shape=21) + 
    ggtitle(&quot;Ostreococcus lucimarinus&quot;)</code></pre>
<p><img src="/post/documentation/11-PR2-R-database-package_files/figure-html/sequence_map-1.png" width="672" /></p>
</div>
<div id="drawing-a-map-of-sequence-locations-obtained-by-fuzzy-matching" class="section level2">
<h2>Drawing a map of sequence locations obtained by fuzzy matching</h2>
<p>A very good tutorial by <a href="https://twitter.com/MargaretBrisbin">Margaret Mars Brisbin</a> on how to combine PR2 metadata with Python to locate sequences using all the metadata information (lat, long, country and fuzzy localization): <a href="https://maggimars.github.io/eukGeoBlast/eGB.html" class="uri">https://maggimars.github.io/eukGeoBlast/eGB.html</a> and <a href="https://github.com/maggimars/eukGeoBlast" class="uri">https://github.com/maggimars/eukGeoBlast</a>. This code has been used to incorporate more geo-localisation information into PR2 version 4.12.0</p>
</div>
<div id="number-of-sequences-per-country" class="section level2">
<h2>Number of sequences per country</h2>
<p>Version 4.12.0 incorporate better geo-localisation using the approach pioneered by<a href="https://twitter.com/MargaretBrisbin">Margaret Mars Brisbin</a></p>
<div id="number-of-sequences-per-country-of-origin" class="section level3">
<h3>Number of sequences per country of origin</h3>
<pre class="r"><code>  
  countries &lt;- pr2 %&gt;% 
    count(pr2_country) %&gt;% 
    arrange(-n) %&gt;% 
    filter(!is.na(pr2_country) &amp; n &gt; 500)
  
  ggplot(countries, aes(x = reorder(pr2_country, n), y = n)) + 
    geom_col() +
    coord_flip() +
    xlab(&quot;&quot;)  + ylab(&quot;Number of PR2 sequences&quot;)</code></pre>
<p><img src="/post/documentation/11-PR2-R-database-package_files/figure-html/sequence_country-1.png" width="672" /></p>
</div>
<div id="number-of-sequences-per-ocean-of-origin" class="section level3">
<h3>Number of sequences per ocean of origin</h3>
<pre class="r"><code>  
  oceans &lt;- pr2 %&gt;% 
    count(pr2_ocean) %&gt;% 
    arrange(-n) %&gt;% 
    filter(!is.na(pr2_ocean))
  
  ggplot(oceans, aes(x = reorder(pr2_ocean, n), y = n)) + 
    geom_col() +
    coord_flip() +
    xlab(&quot;&quot;)  + ylab(&quot;Number of PR2 sequences&quot;)</code></pre>
<p><img src="/post/documentation/11-PR2-R-database-package_files/figure-html/sequence_oceans-1.png" width="672" /></p>
</div>
</div>
</div>
